<html>
    <head>
        <meta charset="utf-8">
        
            <script src="lib/bindings/utils.js"></script>
            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/vis-network/9.1.2/dist/dist/vis-network.min.css" integrity="sha512-WgxfT5LWjfszlPHXRmBWHkV2eceiWTOBvrKCNbdgDYTHrT2AeLCGbF4sZlZw3UMN3WtL0tGUoIAKsu8mllg/XA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
            <script src="https://cdnjs.cloudflare.com/ajax/libs/vis-network/9.1.2/dist/vis-network.min.js" integrity="sha512-LnvoEWDFrqGHlHmDD2101OrLcbsfkrzoSpvtSQtxK3RMnRV0eOkhhBN2dXHKRrUU8p2DGRTk35n4O8nWSVe1mQ==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
            
        
<center>
<h1></h1>
</center>

<!-- <link rel="stylesheet" href="../node_modules/vis/dist/vis.min.css" type="text/css" />
<script type="text/javascript" src="../node_modules/vis/dist/vis.js"> </script>-->
        <link
          href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css"
          rel="stylesheet"
          integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6"
          crossorigin="anonymous"
        />
        <script
          src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/js/bootstrap.bundle.min.js"
          integrity="sha384-JEW9xMcG8R+pH31jmWH6WWP0WintQrMb4s7ZOdauHnUtxwoG2vI5DkLtS3qm9Ekf"
          crossorigin="anonymous"
        ></script>


        <center>
          <h1></h1>
        </center>
        <style type="text/css">

             #mynetwork {
                 width: 100%;
                 height: 2160px;
                 background-color: #222222;
                 border: 1px solid lightgray;
                 position: relative;
                 float: left;
             }

             
             #loadingBar {
                 position:absolute;
                 top:0px;
                 left:0px;
                 width: 100%;
                 height: 2160px;
                 background-color:rgba(200,200,200,0.8);
                 -webkit-transition: all 0.5s ease;
                 -moz-transition: all 0.5s ease;
                 -ms-transition: all 0.5s ease;
                 -o-transition: all 0.5s ease;
                 transition: all 0.5s ease;
                 opacity:1;
             }

             #bar {
                 position:absolute;
                 top:0px;
                 left:0px;
                 width:20px;
                 height:20px;
                 margin:auto auto auto auto;
                 border-radius:11px;
                 border:2px solid rgba(30,30,30,0.05);
                 background: rgb(0, 173, 246); /* Old browsers */
                 box-shadow: 2px 0px 4px rgba(0,0,0,0.4);
             }

             #border {
                 position:absolute;
                 top:10px;
                 left:10px;
                 width:500px;
                 height:23px;
                 margin:auto auto auto auto;
                 box-shadow: 0px 0px 4px rgba(0,0,0,0.2);
                 border-radius:10px;
             }

             #text {
                 position:absolute;
                 top:8px;
                 left:530px;
                 width:30px;
                 height:50px;
                 margin:auto auto auto auto;
                 font-size:22px;
                 color: #000000;
             }

             div.outerBorder {
                 position:relative;
                 top:400px;
                 width:600px;
                 height:44px;
                 margin:auto auto auto auto;
                 border:8px solid rgba(0,0,0,0.1);
                 background: rgb(252,252,252); /* Old browsers */
                 background: -moz-linear-gradient(top,  rgba(252,252,252,1) 0%, rgba(237,237,237,1) 100%); /* FF3.6+ */
                 background: -webkit-gradient(linear, left top, left bottom, color-stop(0%,rgba(252,252,252,1)), color-stop(100%,rgba(237,237,237,1))); /* Chrome,Safari4+ */
                 background: -webkit-linear-gradient(top,  rgba(252,252,252,1) 0%,rgba(237,237,237,1) 100%); /* Chrome10+,Safari5.1+ */
                 background: -o-linear-gradient(top,  rgba(252,252,252,1) 0%,rgba(237,237,237,1) 100%); /* Opera 11.10+ */
                 background: -ms-linear-gradient(top,  rgba(252,252,252,1) 0%,rgba(237,237,237,1) 100%); /* IE10+ */
                 background: linear-gradient(to bottom,  rgba(252,252,252,1) 0%,rgba(237,237,237,1) 100%); /* W3C */
                 filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#fcfcfc', endColorstr='#ededed',GradientType=0 ); /* IE6-9 */
                 border-radius:72px;
                 box-shadow: 0px 0px 10px rgba(0,0,0,0.2);
             }
             

             

             
        </style>
    </head>


    <body>
        <div class="card" style="width: 100%">
            
            
            <div id="mynetwork" class="card-body"></div>
        </div>

        
            <div id="loadingBar">
              <div class="outerBorder">
                <div id="text">0%</div>
                <div id="border">
                  <div id="bar"></div>
                </div>
              </div>
            </div>
        
        

        <script type="text/javascript">

              // initialize global variables.
              var edges;
              var nodes;
              var allNodes;
              var allEdges;
              var nodeColors;
              var originalNodes;
              var network;
              var container;
              var options, data;
              var filter = {
                  item : '',
                  property : '',
                  value : []
              };

              

              

              // This method is responsible for drawing the graph, returns the drawn network
              function drawGraph() {
                  var container = document.getElementById('mynetwork');

                  

                  // parsing and collecting nodes and edges from the python
                  nodes = new vis.DataSet([{"color": "#97c2fc", "font": {"color": "white"}, "id": "C-type cytochromes", "label": "C-type cytochromes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "aerobic respiration", "label": "aerobic respiration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "anaerobic respiration", "label": "anaerobic respiration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Heterologous production", "label": "Heterologous production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli", "label": "Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Heterologous production limitations", "label": "Heterologous production limitations", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "ccmABCDEFGH genes expression", "label": "ccmABCDEFGH genes expression", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Anaerobic conditions", "label": "Anaerobic conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "pEC86 vector", "label": "pEC86 vector", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "ccm genes co-expression", "label": "ccm genes co-expression", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Co-expression methods", "label": "Co-expression methods", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Expression host restriction", "label": "Expression host restriction", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vector choice restriction", "label": "Vector choice restriction", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens Vmax X2", "label": "Vibrio natriegens Vmax X2", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Recombinant production", "label": "Recombinant production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Acidithiobacillus ferrooxidans CCM4253", "label": "Acidithiobacillus ferrooxidans CCM4253", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Co-expression of ccm genes", "label": "Co-expression of ccm genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Holo-c-type cytochromes production necessity", "label": "Holo-c-type cytochromes production necessity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli T7 Express", "label": "Escherichia coli T7 Express", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Holo-c-type cytochromes co-expression production", "label": "Holo-c-type cytochromes co-expression production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Inner membrane cytochrome CycA production inability", "label": "Inner membrane cytochrome CycA production inability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens Vmax X2 cell extracts", "label": "Vibrio natriegens Vmax X2 cell extracts", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Recombinant holo-proteins quantity", "label": "Recombinant holo-proteins quantity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "T7 Express cell extracts", "label": "T7 Express cell extracts", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Redox proteins", "label": "Redox proteins", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Intended cell compartment translocation", "label": "Intended cell compartment translocation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens", "label": "Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "soluble proteins", "label": "soluble proteins", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vmax X2", "label": "Vmax X2", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bacterial and fungal proteins in milligram scale", "label": "bacterial and fungal proteins in milligram scale", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cholera toxin", "label": "cholera toxin", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "N-acetyl glucosamine-binding protein A", "label": "N-acetyl glucosamine-binding protein A", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "heat-labile enterotoxin", "label": "heat-labile enterotoxin", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fungal nematotoxin CCTX2", "label": "fungal nematotoxin CCTX2", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Type II secretion system", "label": "Type II secretion system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "secretion of cholera toxin", "label": "secretion of cholera toxin", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "secretion of N-acetyl glucosamine-binding protein A", "label": "secretion of N-acetyl glucosamine-binding protein A", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "secretion of heat-labile enterotoxin", "label": "secretion of heat-labile enterotoxin", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "recovery from growth media", "label": "recovery from growth media", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "higher Protein yields compared to Escherichia coli", "label": "higher Protein yields compared to Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "protein perdeuteration in amounts suitable for structural biology studies", "label": "protein perdeuteration in amounts suitable for structural biology studies", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "deuterated minimal media with deuterium oxide", "label": "deuterated minimal media with deuterium oxide", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "3-fold increase in yield compared to Escherichia coli", "label": "3-fold increase in yield compared to Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "natural transformation", "label": "natural transformation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "master competence regulator tfoX from Vibrio cholerae", "label": "master competence regulator tfoX from Vibrio cholerae", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "naturally competent cells", "label": "naturally competent cells", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "new minimal competence media", "label": "new minimal competence media", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "uses acetate as an energy source", "label": "uses acetate as an energy source", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "transformed", "label": "transformed", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "recovered", "label": "recovered", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "competent to plasmids", "label": "competent to plasmids", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "recombination with linear DNA", "label": "recombination with linear DNA", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cotransformation of plasmids and linear DNA", "label": "cotransformation of plasmids and linear DNA", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "zero capital protocol", "label": "zero capital protocol", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "transformation process at room temperature range", "label": "transformation process at room temperature range", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high-efficiency protocol", "label": "high-efficiency protocol", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "transformation process with incubator", "label": "transformation process with incubator", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "retain naturally competent state when frozen", "label": "retain naturally competent state when frozen", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "transformable immediately upon thawing", "label": "transformable immediately upon thawing", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "optimized transformation protocol", "label": "optimized transformation protocol", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "requires 50 minutes of hands-on time", "label": "requires 50 minutes of hands-on time", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "grows quickly even on plates", "label": "grows quickly even on plates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "natural competence strain and media", "label": "natural competence strain and media", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "ideal for automated directed evolution applications", "label": "ideal for automated directed evolution applications", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "naturally competent Vibrio natriegens", "label": "naturally competent Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "could compete with Escherichia coli", "label": "could compete with Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bioproduction processes", "label": "bioproduction processes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "biomass production", "label": "biomass production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "similar extracellular electron transfer pathway to electroactive organisms to Iron (III", "label": "similar extracellular electron transfer pathway to electroactive organisms to Iron (III", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "porin-cytochrome circuit", "label": "porin-cytochrome circuit", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "electroactive microbes", "label": "electroactive microbes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reduce toxic metals", "label": "reduce toxic metals", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "carcinogenic Chromium (VI)", "label": "carcinogenic Chromium (VI)", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "converted to Chromium (III", "label": "converted to Chromium (III", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bioremediation of Chromium (VI", "label": "bioremediation of Chromium (VI", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens density", "label": "Vibrio natriegens density", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Chromium (VI", "label": "Chromium (VI", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "slower Chromium (VI", "label": "slower Chromium (VI", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens metabolism", "label": "Vibrio natriegens metabolism", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "outer-membrane c-type cytochromes", "label": "outer-membrane c-type cytochromes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Chromium (III", "label": "Chromium (III", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "adsorption of metallic ions", "label": "adsorption of metallic ions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Chromosome rearrangement by LoxP-mediated evolution", "label": "Chromosome rearrangement by LoxP-mediated evolution", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Chromosome architecture", "label": "Chromosome architecture", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Phenotypes", "label": "Phenotypes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "CRISPR-associated transposases", "label": "CRISPR-associated transposases", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Insertion of LoxP sites", "label": "Insertion of LoxP sites", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "LoxP sites", "label": "LoxP sites", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Genome of wild-type strains", "label": "Genome of wild-type strains", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Multiplex genome integration tool derived from CRISPR-associated transposases", "label": "Multiplex genome integration tool derived from CRISPR-associated transposases", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Insertion of cargo genes", "label": "Insertion of cargo genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Genomic loci", "label": "Genomic loci", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "30 LoxP sites", "label": "30 LoxP sites", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Chromosome 2 of Vibrio natrigens", "label": "Chromosome 2 of Vibrio natrigens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Rigorously induced Cre recombinase", "label": "Rigorously induced Cre recombinase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Chromosome Rearrangement and Modification by LoxP-mediated Evolution", "label": "Chromosome Rearrangement and Modification by LoxP-mediated Evolution", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Growth characterization", "label": "Growth characterization", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "~358 kilobase fragment on Chromosome 2", "label": "~358 kilobase fragment on Chromosome 2", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Rapid growth of Vibrio natrigens", "label": "Rapid growth of Vibrio natrigens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Enabling tools", "label": "Enabling tools", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Identification of genomic regions influencing rapid growth", "label": "Identification of genomic regions influencing rapid growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Genomic regions influencing rapid growth", "label": "Genomic regions influencing rapid growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Demonstration", "label": "Demonstration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Extension to other organisms", "label": "Extension to other organisms", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Other organisms", "label": "Other organisms", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Pseudomonas putida", "label": "Pseudomonas putida", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Bacillus subtilis", "label": "Bacillus subtilis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Xenobiotic agrochemical leaching", "label": "Xenobiotic agrochemical leaching", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Threaten microbial diversity", "label": "Threaten microbial diversity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Shotgun sequencing", "label": "Shotgun sequencing", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Pichavaram mangrove metagenomic dataset", "label": "Pichavaram mangrove metagenomic dataset", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "SqueezeMeta", "label": "SqueezeMeta", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "DIAMOND database", "label": "DIAMOND database", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Taxonomical classification of microbes", "label": "Taxonomical classification of microbes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "KEGG database", "label": "KEGG database", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Confirmed xenobiotic degradation pathways", "label": "Confirmed xenobiotic degradation pathways", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Microbial consortia", "label": "Microbial consortia", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Degradation of fluorobenzoate", "label": "Degradation of fluorobenzoate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Degradation of furfural", "label": "Degradation of furfural", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Degradation of steroids", "label": "Degradation of steroids", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Marinobacter", "label": "Marinobacter", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Methyloceanibacter", "label": "Methyloceanibacter", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Gramella sp.", "label": "Gramella sp.", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Afipia", "label": "Afipia", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Nitrosopumilus sp.", "label": "Nitrosopumilus sp.", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Phototrophicus methaneseepsis", "label": "Phototrophicus methaneseepsis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Proteobacteria", "label": "Proteobacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "in vivo in vivo random mutagenesis", "label": "in vivo in vivo random mutagenesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased capacity for biosorbing dysprosium", "label": "increased capacity for biosorbing dysprosium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased selectivity between lanthanum and lutetium", "label": "increased selectivity between lanthanum and lutetium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mutations in ABC transporters", "label": "mutations in ABC transporters", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "biosorption changes", "label": "biosorption changes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mutations in nonessential protein in the BAM outer membrane \u03b2-barrel protein insertion complex", "label": "mutations in nonessential protein in the BAM outer membrane \u03b2-barrel protein insertion complex", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "thermal strategy", "label": "thermal strategy", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high performance production of N-acetyl-d-neuraminic acid", "label": "high performance production of N-acetyl-d-neuraminic acid", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fast-growing Vibrio natriegens", "label": "fast-growing Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "thermophilic enzymes", "label": "thermophilic enzymes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high-temperature range whole-cell catalysis", "label": "high-temperature range whole-cell catalysis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "highest Neu5Ac titer", "label": "highest Neu5Ac titer", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "productivity of 71.6 grams per liter hour", "label": "productivity of 71.6 grams per liter hour", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "temperature range elevation", "label": "temperature range elevation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "elimination of acetic acid", "label": "elimination of acetic acid", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "exceptional formic acid tolerance", "label": "exceptional formic acid tolerance", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "metabolic capacity", "label": "metabolic capacity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "rewiring the serine cycle", "label": "rewiring the serine cycle", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "non-native closed loop", "label": "non-native closed loop", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "rewiring the TCA cycle", "label": "rewiring the TCA cycle", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "improved formic acid-utilizing ability", "label": "improved formic acid-utilizing ability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "laboratory evolution", "label": "laboratory evolution", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "introduction of foreign indigoidine-forming pathway", "label": "introduction of foreign indigoidine-forming pathway", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "production of 29.0 grams per liter indigoidine", "label": "production of 29.0 grams per liter indigoidine", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "consumption of 165.3 grams per liter formate", "label": "consumption of 165.3 grams per liter formate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "consumption rate of 2.3 grams per liter per hour", "label": "consumption rate of 2.3 grams per liter per hour", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "horizontal gene transfer", "label": "horizontal gene transfer", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "agarose-degrading ability of Vibrio natriegens WPAGA4", "label": "agarose-degrading ability of Vibrio natriegens WPAGA4", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "agarose degradation pathway of Vibrio natriegens WPAGA4", "label": "agarose degradation pathway of Vibrio natriegens WPAGA4", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Aga3418", "label": "Aga3418", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "agarose degradation product neoagarobiose", "label": "agarose degradation product neoagarobiose", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Aga3420", "label": "Aga3420", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Aga3472", "label": "Aga3472", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Aga3419", "label": "Aga3419", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "agarose degradation product neoagarotetraose", "label": "agarose degradation product neoagarotetraose", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "expression level ratio of Aga3418", "label": "expression level ratio of Aga3418", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Aga3419, Aga3420, and Aga3472, optimal agarose degradation by Vibrio natriegens WPAGA4", "label": "Aga3419, Aga3420, and Aga3472, optimal agarose degradation by Vibrio natriegens WPAGA4", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genomic information", "label": "genomic information", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "occurrence of horizontal gene transfer", "label": "occurrence of horizontal gene transfer", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "expression level ratio of agarase genes", "label": "expression level ratio of agarase genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "crucial factor for agarose degradation by Vibrio natriegens WPAGA4", "label": "crucial factor for agarose degradation by Vibrio natriegens WPAGA4", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "promotion of deep-sea carbon cycle", "label": "promotion of deep-sea carbon cycle", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "production of soluble proteins in high yields", "label": "production of soluble proteins in high yields", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vmax\u2122 X2", "label": "Vmax\u2122 X2", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "production of soluble bacterial and fungal proteins in milligram scale", "label": "production of soluble bacterial and fungal proteins in milligram scale", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "alternative host platform for production of proteins", "label": "alternative host platform for production of proteins", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "secreted by Type II secretion system", "label": "secreted by Type II secretion system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "N-acetyl glucosamine binding protein A", "label": "N-acetyl glucosamine binding protein A", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "recovered from growth media", "label": "recovered from growth media", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "6- to 26-fold increase in Protein yields compared to Escherichia coli", "label": "6- to 26-fold increase in Protein yields compared to Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "3-fold increase in yield using deuterated minimal media compared to Escherichia coli", "label": "3-fold increase in yield using deuterated minimal media compared to Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "simplified downstream purification procedure", "label": "simplified downstream purification procedure", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "struggled to produce soluble proteins in milligram scale", "label": "struggled to produce soluble proteins in milligram scale", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "promising host for production of challenging expression targets", "label": "promising host for production of challenging expression targets", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "promising host for protein deuteration in amounts suitable for structural biology studies", "label": "promising host for protein deuteration in amounts suitable for structural biology studies", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Polyhydroxybutyrate production", "label": "Polyhydroxybutyrate production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inhibitory substances presence", "label": "inhibitory substances presence", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "organic wastes poor utilization", "label": "organic wastes poor utilization", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "slow microbial growth", "label": "slow microbial growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high energy input for feedstock sterilization", "label": "high energy input for feedstock sterilization", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "base editing", "label": "base editing", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "key genes identification", "label": "key genes identification", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "key genes overexpression", "label": "key genes overexpression", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "glycerol", "label": "glycerol", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sugars", "label": "sugars", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens metabolic engineering", "label": "Vibrio natriegens metabolic engineering", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Polyhydroxybutyrate productivity increase", "label": "Polyhydroxybutyrate productivity increase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "tolerance to toxic substances", "label": "tolerance to toxic substances", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "activity maintenance under nonsterilized fermentation conditions", "label": "activity maintenance under nonsterilized fermentation conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Engineered Escherichia coli MG1655", "label": "Engineered Escherichia coli MG1655", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "lower Polyhydroxybutyrate productivity than engineered Vibrio natriegens", "label": "lower Polyhydroxybutyrate productivity than engineered Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "lower tolerance to toxic substances than engineered Vibrio natriegens", "label": "lower tolerance to toxic substances than engineered Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "tyrosinase enzyme Tyr1", "label": "tyrosinase enzyme Tyr1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "melanin production biosynthesis", "label": "melanin production biosynthesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "tyrosine", "label": "tyrosine", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "copper responsive promoter", "label": "copper responsive promoter", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "heterologous protein expression regulation", "label": "heterologous protein expression regulation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "recombinant microbial system", "label": "recombinant microbial system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "optimized minimally defined medium", "label": "optimized minimally defined medium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "melanin production yield", "label": "melanin production yield", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "volumetric productivity of melanin production", "label": "volumetric productivity of melanin production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "conversion rates of tyrosine to melanin production", "label": "conversion rates of tyrosine to melanin production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high sodium sodium chloride concentrations", "label": "high sodium sodium chloride concentrations", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "corrosion processes on metal surfaces", "label": "corrosion processes on metal surfaces", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "low-sodium chloride chemically defined medium", "label": "low-sodium chloride chemically defined medium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high-cell-density fed-batch cultivations", "label": "high-cell-density fed-batch cultivations", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sodium sodium chloride", "label": "sodium sodium chloride", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "precipitation of ammonium magnesium phosphate", "label": "precipitation of ammonium magnesium phosphate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sodium citrate", "label": "sodium citrate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "prevention of precipitates", "label": "prevention of precipitates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "exponential feeding", "label": "exponential feeding", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "biomass concentrations of more than 60 grams per liter", "label": "biomass concentrations of more than 60 grams per liter", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "disodium hydrogen phosphate", "label": "disodium hydrogen phosphate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "disodium sulfate", "label": "disodium sulfate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens ", "label": "Vibrio natriegens ", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "pyruvate production", "label": "pyruvate production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "deletion of two prophage gene clusters ", "label": "deletion of two prophage gene clusters ", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "deletion of several essential genes for the biosynthesis of byproducts ", "label": "deletion of several essential genes for the biosynthesis of byproducts ", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "down-regulation of aceE ", "label": "down-regulation of aceE ", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reduced carbon flux into the tricarboxylic acid cycle", "label": "reduced carbon flux into the tricarboxylic acid cycle", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fine-tuning of ppc gene expression ", "label": "fine-tuning of ppc gene expression ", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "balance between cell growth and pyruvate synthesis", "label": "balance between cell growth and pyruvate synthesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "PYR32 strain ", "label": "PYR32 strain ", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "54.22 grams per liter pyruvate production", "label": "54.22 grams per liter pyruvate production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "1.17 mol/mol yield of pyruvate", "label": "1.17 mol/mol yield of pyruvate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "3.39 grams per liter per hour average productivity of pyruvate", "label": "3.39 grams per liter per hour average productivity of pyruvate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "efficient conversion of sucrose into pyruvate", "label": "efficient conversion of sucrose into pyruvate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "efficient conversion of gluconate into pyruvate", "label": "efficient conversion of gluconate into pyruvate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "secretory expression of large molecular weight proteins", "label": "secretory expression of large molecular weight proteins", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "protein synthesis efficiency", "label": "protein synthesis efficiency", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Pullulanase PulA", "label": "Pullulanase PulA", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "secretory expression in Vibrio natriegens VnDX", "label": "secretory expression in Vibrio natriegens VnDX", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Pullulanase PulN2", "label": "Pullulanase PulN2", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "signal peptide", "label": "signal peptide", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "secretory expression", "label": "secretory expression", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fermentation temperature range", "label": "fermentation temperature range", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inducer concentration", "label": "inducer concentration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "glycine concentration", "label": "glycine concentration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fermentation time", "label": "fermentation time", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "extracellular enzyme activities", "label": "extracellular enzyme activities", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "comparison between Vibrio natriegens VnDX and Escherichia coli BL21(DE3", "label": "comparison between Vibrio natriegens VnDX and Escherichia coli BL21(DE3", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Pullulanase PulA extracellular enzyme activity in Vibrio natriegens VnDX", "label": "Pullulanase PulA extracellular enzyme activity in Vibrio natriegens VnDX", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "61.6 U/mL", "label": "61.6 U/mL", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Pullulanase PulN2 extracellular enzyme activity in Vibrio natriegens VnDX", "label": "Pullulanase PulN2 extracellular enzyme activity in Vibrio natriegens VnDX", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "64.3 U/mL", "label": "64.3 U/mL", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "110% of Escherichia coli BL21(DE3", "label": "110% of Escherichia coli BL21(DE3", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "62% of Escherichia coli BL21(DE3", "label": "62% of Escherichia coli BL21(DE3", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fast-growing", "label": "fast-growing", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "non-pathogenic", "label": "non-pathogenic", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high-level recombinant protein production", "label": "high-level recombinant protein production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fast DNA propagation", "label": "fast DNA propagation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "short generation time", "label": "short generation time", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "proteosynthetic activity", "label": "proteosynthetic activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "versatile metabolism", "label": "versatile metabolism", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "future industrial platform", "label": "future industrial platform", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fermentation processes operating with high productivity", "label": "fermentation processes operating with high productivity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "enhanced membrane permeability", "label": "enhanced membrane permeability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "facilitating transport of target proteins into extracellular environment", "label": "facilitating transport of target proteins into extracellular environment", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "IMAC chromatography", "label": "IMAC chromatography", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "purification of AfKatG from growth medium", "label": "purification of AfKatG from growth medium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "purification of MDBP from growth medium", "label": "purification of MDBP from growth medium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "purification of purification of Taq polymerase from growth medium from growth medium", "label": "purification of purification of Taq polymerase from growth medium from growth medium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "AfKatG", "label": "AfKatG", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "total yield from growth medium", "label": "total yield from growth medium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "MDBP", "label": "MDBP", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "purification of Taq polymerase from growth medium", "label": "purification of Taq polymerase from growth medium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens based system", "label": "Vibrio natriegens based system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "extracellular tag-less recombinant protein producer", "label": "extracellular tag-less recombinant protein producer", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "amino acid balancing", "label": "amino acid balancing", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "performance of Cell-free protein synthesis systems", "label": "performance of Cell-free protein synthesis systems", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli system", "label": "Escherichia coli system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high efficiency in the utilization of amino acids", "label": "high efficiency in the utilization of amino acids", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens based systems", "label": "Vibrio natriegens based systems", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fractional yields of over 70%", "label": "fractional yields of over 70%", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Leishmania tarentolae based systems", "label": "Leishmania tarentolae based systems", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Cell-free protein synthesis systems", "label": "Cell-free protein synthesis systems", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "protein molar concentration", "label": "protein molar concentration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Ostreid herpesvirus-1", "label": "Ostreid herpesvirus-1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mass mortalities of hatchery larvae of Pacific oysters", "label": "mass mortalities of hatchery larvae of Pacific oysters", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mass mortalities of Pacific oysters", "label": "mass mortalities of Pacific oysters", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio alginolyticus", "label": "Vibrio alginolyticus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Pseudoalteromonas spp.", "label": "Pseudoalteromonas spp.", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mass mortalities of Pacific oysters during the cold season", "label": "mass mortalities of Pacific oysters during the cold season", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens growth temperature range", "label": "Vibrio natriegens growth temperature range", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "faster growth at 25 degrees Celsius", "label": "faster growth at 25 degrees Celsius", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens haemolytic activity temperature range", "label": "Vibrio natriegens haemolytic activity temperature range", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "more remarkable haemolytic activity at 25 degrees Celsius", "label": "more remarkable haemolytic activity at 25 degrees Celsius", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens siderophore production capacity temperature range", "label": "Vibrio natriegens siderophore production capacity temperature range", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "more remarkable siderophore production capacity at 25 degrees Celsius", "label": "more remarkable siderophore production capacity at 25 degrees Celsius", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio alginolyticus growth temperature range", "label": "Vibrio alginolyticus growth temperature range", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio alginolyticus haemolytic activity temperature range", "label": "Vibrio alginolyticus haemolytic activity temperature range", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio alginolyticus siderophore production capacity temperature range", "label": "Vibrio alginolyticus siderophore production capacity temperature range", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens mortality rate temperature range", "label": "Vibrio natriegens mortality rate temperature range", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "higher accumulative mortalities at 25 degrees Celsius", "label": "higher accumulative mortalities at 25 degrees Celsius", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio alginolyticus mortality rate temperature range", "label": "Vibrio alginolyticus mortality rate temperature range", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Gluconobacter oxydans levansucrases", "label": "Gluconobacter oxydans levansucrases", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "lactosucrose production", "label": "lactosucrose production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fructooligosaccharides production", "label": "fructooligosaccharides production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "naturally competent Vibrio natriegens levansucrases", "label": "naturally competent Vibrio natriegens levansucrases", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Novosphingobium aromaticivorans levansucrases", "label": "Novosphingobium aromaticivorans levansucrases", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Burkholderia graminis levansucrases", "label": "Burkholderia graminis levansucrases", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "whey permeate", "label": "whey permeate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "milk permeate", "label": "milk permeate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "transfructosylation activity", "label": "transfructosylation activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "acceptor specificity of levansucrases", "label": "acceptor specificity of levansucrases", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "end-product profile", "label": "end-product profile", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "thermodynamic equilibrium", "label": "thermodynamic equilibrium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "coral bleaching event of 2020", "label": "coral bleaching event of 2020", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bloom of vibrios", "label": "bloom of vibrios", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "spatial distance", "label": "spatial distance", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio community", "label": "Vibrio community", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "coral species", "label": "coral species", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio harveyi", "label": "Vibrio harveyi", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "relative abundance", "label": "relative abundance", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "niche breadth", "label": "niche breadth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Acroporidae", "label": "Acroporidae", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "TfoX", "label": "TfoX", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "competency", "label": "competency", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "QstR", "label": "QstR", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "iron metabolism iModulon", "label": "iron metabolism iModulon", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "translation iModulon", "label": "translation iModulon", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "novel iModulon of unknown function", "label": "novel iModulon of unknown function", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "motility behavior iModulon", "label": "motility behavior iModulon", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "polycations iModulon", "label": "polycations iModulon", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reactive oxygen species responses iModulon", "label": "reactive oxygen species responses iModulon", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "gene deletion", "label": "gene deletion", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "aerobic glucose-limited fed-batch fermentation", "label": "aerobic glucose-limited fed-batch fermentation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "biomass concentration", "label": "biomass concentration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "viscosity of culture broth", "label": "viscosity of culture broth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "formation of exopolysaccharides", "label": "formation of exopolysaccharides", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "deletion of transcriptional regulator cpsR", "label": "deletion of transcriptional regulator cpsR", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "carbohydrate content of exopolysaccharides", "label": "carbohydrate content of exopolysaccharides", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "deletion of cps cluster", "label": "deletion of cps cluster", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "amount of exopolysaccharides secreted", "label": "amount of exopolysaccharides secreted", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "exopolysaccharides composition", "label": "exopolysaccharides composition", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "anaerobic conditions on glucose", "label": "anaerobic conditions on glucose", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "deletion of cpsR in Vibrio natriegens Succ1", "label": "deletion of cpsR in Vibrio natriegens Succ1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "deletion of cps cluster in Vibrio natriegens Succ1", "label": "deletion of cps cluster in Vibrio natriegens Succ1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens Succ1 deletion of cpsR", "label": "Vibrio natriegens Succ1 deletion of cpsR", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "final succinate concentration", "label": "final succinate concentration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "volumetric productivity of succinate", "label": "volumetric productivity of succinate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "product yield of succinate", "label": "product yield of succinate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens Succ1 deletion of cps cluster", "label": "Vibrio natriegens Succ1 deletion of cps cluster", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high growth and substrate consumption rates", "label": "high growth and substrate consumption rates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens as an emerging host for biotechnology", "label": "Vibrio natriegens as an emerging host for biotechnology", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "feed-batch processes", "label": "feed-batch processes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high space-time yields in industrial processes", "label": "high space-time yields in industrial processes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "anaerobic cultivation of Vibrio natriegens Succ1", "label": "anaerobic cultivation of Vibrio natriegens Succ1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased viscosity at the end of cultivation", "label": "increased viscosity at the end of cultivation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "deletion of transcriptional regulator cpsR and cps cluster in Vibrio natriegens Succ1", "label": "deletion of transcriptional regulator cpsR and cps cluster in Vibrio natriegens Succ1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "viscosity of culture broth reduction", "label": "viscosity of culture broth reduction", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "product yield of succinate in Vibrio natriegens Succ1 deletion of cpsR and cps cluster", "label": "product yield of succinate in Vibrio natriegens Succ1 deletion of cpsR and cps cluster", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "theoretical maximum succinate yield", "label": "theoretical maximum succinate yield", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "malonate semialdehyde ", "label": "malonate semialdehyde ", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "beta-alanine", "label": "beta-alanine", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "beta-alanine ", "label": "beta-alanine ", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "malonate semialdehyde", "label": "malonate semialdehyde", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens Coenzyme A-acylating methyl malonate semialdehyde dehydrogenases", "label": "Vibrio natriegens Coenzyme A-acylating methyl malonate semialdehyde dehydrogenases", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Pseudomonas aeruginosa beta-alanine/pyruvate transaminase", "label": "Pseudomonas aeruginosa beta-alanine/pyruvate transaminase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "acetyl-Coenzyme A", "label": "acetyl-Coenzyme A", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli MG1655", "label": "Escherichia coli MG1655", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli BA02", "label": "Escherichia coli BA02", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "optical density at 600 nanometer", "label": "optical density at 600 nanometer", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli BA07\u2206", "label": "Escherichia coli BA07\u2206", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "native 3-hydroxy acid dehydrogenase", "label": "native 3-hydroxy acid dehydrogenase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "3-hydroxypropionate production", "label": "3-hydroxypropionate production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "replication-induced gene copy numbers", "label": "replication-induced gene copy numbers", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cellular functions", "label": "cellular functions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "central pathways", "label": "central pathways", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "spatio-temporal expression pattern", "label": "spatio-temporal expression pattern", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "growth phase dependent gene expression", "label": "growth phase dependent gene expression", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "evolutionary gene migration", "label": "evolutionary gene migration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "chromosome evolution", "label": "chromosome evolution", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high growth rates related transient copy numbers", "label": "high growth rates related transient copy numbers", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "transient chromosomal copy numbers", "label": "transient chromosomal copy numbers", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "new standard biotechnological organism", "label": "new standard biotechnological organism", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fast growth", "label": "fast growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens productivity", "label": "Vibrio natriegens productivity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "surpasses Escherichia coli", "label": "surpasses Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genomic tools availability", "label": "genomic tools availability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "similar extracellular electron transfer pathway to electroactive organisms", "label": "similar extracellular electron transfer pathway to electroactive organisms", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "donates electrons from metabolism to electrode", "label": "donates electrons from metabolism to electrode", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "electrons donation via direct contact", "label": "electrons donation via direct contact", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "electrons donation via artificial redox mediator", "label": "electrons donation via artificial redox mediator", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "artificial mediator use", "label": "artificial mediator use", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "current densities up to 196 microamperes per square centimeter", "label": "current densities up to 196 microamperes per square centimeter", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "direct electron transfer", "label": "direct electron transfer", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "6.6 microamperes per square centimeter achieved within first 24 hours of cultivation", "label": "6.6 microamperes per square centimeter achieved within first 24 hours of cultivation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mediated system", "label": "mediated system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "production of formate from glucose", "label": "production of formate from glucose", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "production of acetate from glucose", "label": "production of acetate from glucose", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "production of succinate from glucose", "label": "production of succinate from glucose", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "superior electron-transfer capabilities", "label": "superior electron-transfer capabilities", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "outstanding metabolism", "label": "outstanding metabolism", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Mycobacterial aminoacylase gene", "label": "Mycobacterial aminoacylase gene", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "isopropyl \u03b2-d-1-thiogalactopyranoside", "label": "isopropyl \u03b2-d-1-thiogalactopyranoside", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "soluble aminoacylase in Vibrio natriegens", "label": "soluble aminoacylase in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "lactose", "label": "lactose", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "soluble recombinant protein in Escherichia coli", "label": "soluble recombinant protein in Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "GroEL/S chaperones overexpression", "label": "GroEL/S chaperones overexpression", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased aminoacylase activity in Vibrio natriegens", "label": "increased aminoacylase activity in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased aminoacylase activity in Escherichia coli", "label": "increased aminoacylase activity in Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Cpn60/10 chaperonins co-expression at 12 Celsius", "label": "Cpn60/10 chaperonins co-expression at 12 Celsius", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased aminoacylase activity in Escherichia coli ArcticExpress", "label": "increased aminoacylase activity in Escherichia coli ArcticExpress", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "pH 7.0", "label": "pH 7.0", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "stable aminoacylase activity", "label": "stable aminoacylase activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "70 Celsius", "label": "70 Celsius", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "maximum aminoacylase activity", "label": "maximum aminoacylase activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "40 Celsius and pH 7.0 for 5 days", "label": "40 Celsius and pH 7.0 for 5 days", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "aminoacylase stability", "label": "aminoacylase stability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "short-chain acyl-amino acids preference", "label": "short-chain acyl-amino acids preference", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "aminoacylase activity", "label": "aminoacylase activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "N-lauroyl-L-methionine", "label": "N-lauroyl-L-methionine", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "aminoacylase hydrolysis acceptance", "label": "aminoacylase hydrolysis acceptance", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "lauric acid and methionine in aqueous system", "label": "lauric acid and methionine in aqueous system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "lauroyl-methionine synthesis", "label": "lauroyl-methionine synthesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "low-temperature range preservation", "label": "low-temperature range preservation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "DNA damage", "label": "DNA damage", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "suppression of respiratory electron transport chain", "label": "suppression of respiratory electron transport chain", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in reactive oxygen species", "label": "increase in reactive oxygen species", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cold-induced loss of culturability", "label": "cold-induced loss of culturability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "volume of liquid culture medium", "label": "volume of liquid culture medium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "relationship between cold-induced loss of culturability and oxygen availability", "label": "relationship between cold-induced loss of culturability and oxygen availability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "deletion of Sodium-translocating NADH-ubiquinone oxidoreductase", "label": "deletion of Sodium-translocating NADH-ubiquinone oxidoreductase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "accelerated cold-induced loss of culturability", "label": "accelerated cold-induced loss of culturability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in intracellular reactive oxygen species levels", "label": "increase in intracellular reactive oxygen species levels", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "marine bacteria", "label": "marine bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "synthetic biology", "label": "synthetic biology", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "metabolic engineering", "label": "metabolic engineering", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "emerging marine chassis organisms", "label": "emerging marine chassis organisms", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Halomonas bluephagenesis", "label": "Halomonas bluephagenesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genetically enhanced microbes", "label": "genetically enhanced microbes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "valorization of algal polysaccharides", "label": "valorization of algal polysaccharides", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "tackling climate change", "label": "tackling climate change", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "tackling marine pollution", "label": "tackling marine pollution", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Gram-negative bacterium", "label": "Gram-negative bacterium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "gram-negative bacterium", "label": "gram-negative bacterium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "exceptional high growth rates", "label": "exceptional high growth rates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "standard biotechnological host", "label": "standard biotechnological host", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genome-scale metabolic model", "label": "genome-scale metabolic model", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "iLC858", "label": "iLC858", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "automated draft assembly", "label": "automated draft assembly", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "extensive manual curation", "label": "extensive manual curation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "predicted yields", "label": "predicted yields", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "central metabolic fluxes", "label": "central metabolic fluxes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "viable carbon substrates", "label": "viable carbon substrates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "essential genes", "label": "essential genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mass spectrometry-based proteomics data", "label": "mass spectrometry-based proteomics data", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "enzyme-encoding genes expression", "label": "enzyme-encoding genes expression", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "metabolic comparison between Escherichia coli and Vibrio natriegens", "label": "metabolic comparison between Escherichia coli and Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "analysis of model architecture of respiratory and ATP-generating system", "label": "analysis of model architecture of respiratory and ATP-generating system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sodium-dependent oxaloacetate decarboxylase pump discovery", "label": "sodium-dependent oxaloacetate decarboxylase pump discovery", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "investigation of halophilic adaptations", "label": "investigation of halophilic adaptations", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Resource Balance Analysis model creation", "label": "Resource Balance Analysis model creation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "study of allocation of carbon resources", "label": "study of allocation of carbon resources", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "microbial fuel cells", "label": "microbial fuel cells", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "electroactive industrial microorganisms", "label": "electroactive industrial microorganisms", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bioproduct and electricity production", "label": "bioproduct and electricity production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "next-generation industrial microorganism chassis", "label": "next-generation industrial microorganism chassis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "insufficient secretion of electron mediators", "label": "insufficient secretion of electron mediators", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "electron mediators", "label": "electron mediators", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "electronic transfer efficiency of Vibrio natriegens microbial fuel cells", "label": "electronic transfer efficiency of Vibrio natriegens microbial fuel cells", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "media and carbon sources", "label": "media and carbon sources", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bioelectricity generation performance of Vibrio natriegens microbial fuel cells", "label": "bioelectricity generation performance of Vibrio natriegens microbial fuel cells", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "1% glycerol", "label": "1% glycerol", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "highest power density in Vibrio natriegens microbial fuel cells", "label": "highest power density in Vibrio natriegens microbial fuel cells", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "phenazine-1-carboxamide synthesis", "label": "phenazine-1-carboxamide synthesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "phenazine-1-carboxylic acid synthesis", "label": "phenazine-1-carboxylic acid synthesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "1-hydroxyphenazine synthesis", "label": "1-hydroxyphenazine synthesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "shikimate pathway", "label": "shikimate pathway", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "phenazine synthesis and modification pathways", "label": "phenazine synthesis and modification pathways", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high salt-tolerant electroactive microorganism", "label": "high salt-tolerant electroactive microorganism", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "feedstock flexibility electroactive microorganism", "label": "feedstock flexibility electroactive microorganism", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "microbial electrofermentation application potential", "label": "microbial electrofermentation application potential", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "in silico exploration", "label": "in silico exploration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "levanase family biodiversity", "label": "levanase family biodiversity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sequence-driven approach", "label": "sequence-driven approach", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "levanases", "label": "levanases", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "hydrolyze low-molecular-weight levan", "label": "hydrolyze low-molecular-weight levan", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "hydrolyze high-molecular-weight levan", "label": "hydrolyze high-molecular-weight levan", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "thermal stability", "label": "thermal stability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "active levanases", "label": "active levanases", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "preferential hydrolysis of levan over inulin", "label": "preferential hydrolysis of levan over inulin", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "levanase hydrolysis", "label": "levanase hydrolysis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "levanbiose production", "label": "levanbiose production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "levantriose production", "label": "levantriose production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "levantetraose production", "label": "levantetraose production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "LEV9 from Belliella Baltica", "label": "LEV9 from Belliella Baltica", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "catalytic properties characterized", "label": "catalytic properties characterized", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "LEV36 from Dyadobacter fermentans", "label": "LEV36 from Dyadobacter fermentans", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "LEV37 from Capnocytophaga ochracea", "label": "LEV37 from Capnocytophaga ochracea", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "LEV79 from Vibrio natriegens", "label": "LEV79 from Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "LEV91 from Paenarthrobacter aurescens", "label": "LEV91 from Paenarthrobacter aurescens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "novel levanases identification", "label": "novel levanases identification", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "production of levan-type FOSs", "label": "production of levan-type FOSs", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "microbial colony growth", "label": "microbial colony growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "activity of cells on edges", "label": "activity of cells on edges", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "speckle imaging technology", "label": "speckle imaging technology", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "visualization of zones of increased microbial activity", "label": "visualization of zones of increased microbial activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "laser speckle imaging technique", "label": "laser speckle imaging technique", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "record of microbial colony growth", "label": "record of microbial colony growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "visualization of microbial growth activity", "label": "visualization of microbial growth activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "speckle imaging technique", "label": "speckle imaging technique", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "laser speckle imaging technique effectiveness", "label": "laser speckle imaging technique effectiveness", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Staphylococcus aureus", "label": "Staphylococcus aureus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "speckle analysis system", "label": "speckle analysis system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Synechococcus elongatus", "label": "Synechococcus elongatus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "carbon dioxide sequestration module", "label": "carbon dioxide sequestration module", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sucrose utilization module", "label": "sucrose utilization module", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "carbon dioxide", "label": "carbon dioxide", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "various valuable chemicals", "label": "various valuable chemicals", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "adaptation of the consensus -10 and -35 sequence boxes", "label": "adaptation of the consensus -10 and -35 sequence boxes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "improvement in strength of lac and tet promoters", "label": "improvement in strength of lac and tet promoters", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "improvement in control of lac and tet promoters", "label": "improvement in control of lac and tet promoters", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "improvement in portability of lac and tet promoters", "label": "improvement in portability of lac and tet promoters", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "independent control of the transcriptional regulators", "label": "independent control of the transcriptional regulators", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "incorporation of a strong ribosome binding site", "label": "incorporation of a strong ribosome binding site", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "use of the far-red fluorescent protein mCardinal", "label": "use of the far-red fluorescent protein mCardinal", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "improvement in the signal-to-background ratio of promoter measurements", "label": "improvement in the signal-to-background ratio of promoter measurements", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "production of cocaine esterase enzyme", "label": "production of cocaine esterase enzyme", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "validation of improvement in OFF state control and inducibility", "label": "validation of improvement in OFF state control and inducibility", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "promoters portability", "label": "promoters portability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "demonstration in Gram-negative species Pseudomonas putida and Vibrio natriegens", "label": "demonstration in Gram-negative species Pseudomonas putida and Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fastest-growing bacteria", "label": "fastest-growing bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "doubling time less than 10 min", "label": "doubling time less than 10 min", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "T7 expression system", "label": "T7 expression system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "introduced into Vibrio natriegens", "label": "introduced into Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "heterologous protein expression system", "label": "heterologous protein expression system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "commercial strain Vmax1", "label": "commercial strain Vmax1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "variant VnDX", "label": "variant VnDX", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli BL21(DE3)", "label": "Escherichia coli BL21(DE3)", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "backup expression chassis", "label": "backup expression chassis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "T7 RNA polymerase strength", "label": "T7 RNA polymerase strength", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "not optimal for every recombinant protein", "label": "not optimal for every recombinant protein", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "promoter sequence change of T7 RNA polymerase", "label": "promoter sequence change of T7 RNA polymerase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "different expression strengths", "label": "different expression strengths", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "ribosome binding site sequence change of T7 RNA polymerase", "label": "ribosome binding site sequence change of T7 RNA polymerase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "VnDX variant library", "label": "VnDX variant library", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fine-tuning T7 RNA polymerase", "label": "fine-tuning T7 RNA polymerase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "industrially used enzyme glucose dehydrogenase", "label": "industrially used enzyme glucose dehydrogenase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reporter protein", "label": "reporter protein", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "variant VnDX-tet", "label": "variant VnDX-tet", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased glucose dehydrogenase activity by 109%", "label": "increased glucose dehydrogenase activity by 109%", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "promoter of T7 RNA polymerase changed from PlacUV5 to Ptet", "label": "promoter of T7 RNA polymerase changed from PlacUV5 to Ptet", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased glucose dehydrogenase activity", "label": "increased glucose dehydrogenase activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "VnDX-RBS12/pGDH", "label": "VnDX-RBS12/pGDH", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "highest glucose dehydrogenase activity increased by 12.6%", "label": "highest glucose dehydrogenase activity increased by 12.6%", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "ribosome binding site sequence change", "label": "ribosome binding site sequence change", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "different T7 RNA polymerase translation levels", "label": "different T7 RNA polymerase translation levels", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "choice for expressing various recombinant proteins", "label": "choice for expressing various recombinant proteins", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "expanded application for expressing various recombinant proteins", "label": "expanded application for expressing various recombinant proteins", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens strain PWH3a", "label": "Vibrio natriegens strain PWH3a", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "complete genome sequence", "label": "complete genome sequence", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "two circular chromosomes", "label": "two circular chromosomes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "4,650 coding sequences", "label": "4,650 coding sequences", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "34 rRNA", "label": "34 rRNA", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "4 noncoding RNA", "label": "4 noncoding RNA", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "131 tRNA genes", "label": "131 tRNA genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "one Mu-like prophage sequence", "label": "one Mu-like prophage sequence", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "functional promoters P1", "label": "functional promoters P1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens strain Vmax", "label": "Vibrio natriegens strain Vmax", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "functional promoters P2-1", "label": "functional promoters P2-1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "functional promoters P2-2", "label": "functional promoters P2-2", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high salt concentrations", "label": "high salt concentrations", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "loss of microbial activities", "label": "loss of microbial activities", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "polyethylene terephthalate degradation", "label": "polyethylene terephthalate degradation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "engineered strains", "label": "engineered strains", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "chlorpyrifos degradation", "label": "chlorpyrifos degradation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "hexabromocyclododecanes degradation", "label": "hexabromocyclododecanes degradation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "degradation rate", "label": "degradation rate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "polyethylene terephthalate", "label": "polyethylene terephthalate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "chlorpyrifos", "label": "chlorpyrifos", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "hexabromocyclododecanes", "label": "hexabromocyclododecanes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "expressing the chitin-binding protein GbpA", "label": "expressing the chitin-binding protein GbpA", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "immobilization strategy", "label": "immobilization strategy", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "marine bacteria V. natriegens Vmax", "label": "marine bacteria V. natriegens Vmax", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "degradation of marine pollution", "label": "degradation of marine pollution", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "multidrug-resistant Vibrio natriegens strain AbY-1805", "label": "multidrug-resistant Vibrio natriegens strain AbY-1805", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "death of juvenile Pacific abalones", "label": "death of juvenile Pacific abalones", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "lytic phage vB_VnaS-L3", "label": "lytic phage vB_VnaS-L3", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "infect Vibrio natriegens AbY-1805", "label": "infect Vibrio natriegens AbY-1805", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "short latent period", "label": "short latent period", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "large burst size", "label": "large burst size", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "treatment with vB_VnaS-L3", "label": "treatment with vB_VnaS-L3", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reduced mortality of juvenile abalones", "label": "reduced mortality of juvenile abalones", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "maintained abalone feeding capacity", "label": "maintained abalone feeding capacity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "novel marine Siphoviridae-family phage", "label": "novel marine Siphoviridae-family phage", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "narrow host range", "label": "narrow host range", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "viability at wide range of pH", "label": "viability at wide range of pH", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "viability at wide range of temperature", "label": "viability at wide range of temperature", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "viability at wide range of salinity", "label": "viability at wide range of salinity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "stepwise metabolic engineering", "label": "stepwise metabolic engineering", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cyanobacteria CO2 sequestration module", "label": "cyanobacteria CO2 sequestration module", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "super-coupled module", "label": "super-coupled module", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "enhanced photosynthetic electron transport", "label": "enhanced photosynthetic electron transport", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "intercellular communication", "label": "intercellular communication", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "extracellular vesicles", "label": "extracellular vesicles", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "HCPC", "label": "HCPC", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "channeling CO2 into valuable chemicals", "label": "channeling CO2 into valuable chemicals", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "overall release of -22.27 to -606.59 kgCO2 e kg-1 in the end products", "label": "overall release of -22.27 to -606.59 kgCO2 e kg-1 in the end products", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens WPAGA4", "label": "Vibrio natriegens WPAGA4", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "16S rRNA gene sequence", "label": "16S rRNA gene sequence", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "average nucleotide identity", "label": "average nucleotide identity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "digital DNA-DNA hybridization", "label": "digital DNA-DNA hybridization", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "4561 coding genes", "label": "4561 coding genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "37 rRNA", "label": "37 rRNA", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "131 tRNA", "label": "131 tRNA", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "59 other non-coding RNA genes", "label": "59 other non-coding RNA genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "GH50 agarase gene", "label": "GH50 agarase gene", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "expression in Escherichia coli cells", "label": "expression in Escherichia coli cells", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "recombinant Aga3420", "label": "recombinant Aga3420", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "optimum temperature range of 40 Celsius", "label": "optimum temperature range of 40 Celsius", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "optimum pH of 7.0", "label": "optimum pH of 7.0", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "product Neoagarobiose during agarose degradation", "label": "product Neoagarobiose during agarose degradation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "stability at 10-30 Celsius for 50 min", "label": "stability at 10-30 Celsius for 50 min", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Km value of 2.8 mg/mL", "label": "Km value of 2.8 mg/mL", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vmax value of 78.1 U/mg", "label": "Vmax value of 78.1 U/mg", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "kcat value of 376.9 s^-1", "label": "kcat value of 376.9 s^-1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "relative activity of 59.7% at 10 Celsius", "label": "relative activity of 59.7% at 10 Celsius", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "relative activity of 41.2% at 0 Celsius", "label": "relative activity of 41.2% at 0 Celsius", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "capability to degrade and metabolize agarose in cold deep-sea environments", "label": "capability to degrade and metabolize agarose in cold deep-sea environments", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "suitability for industrial enzyme in NA2 production", "label": "suitability for industrial enzyme in NA2 production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "industrial potential in medical and cosmetic fields", "label": "industrial potential in medical and cosmetic fields", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genome-scale metabolic models", "label": "genome-scale metabolic models", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genome-editing protocols", "label": "genome-editing protocols", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "metabolic engineering of Vibrio natriegens", "label": "metabolic engineering of Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "production of 1,3-propanediol", "label": "production of 1,3-propanediol", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nick/gap repair and negative supercoiling reaction", "label": "nick/gap repair and negative supercoiling reaction", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "negative supercoiling of the whole genomes extracted from Escherichia coli", "label": "negative supercoiling of the whole genomes extracted from Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "negative supercoiling of the whole genomes extracted from Vibrio natriegens", "label": "negative supercoiling of the whole genomes extracted from Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli replication-cycle reaction", "label": "Escherichia coli replication-cycle reaction", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "enzymatic replication of 1-2 megabase chromosomes", "label": "enzymatic replication of 1-2 megabase chromosomes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "modified nick/gap repair and negative supercoiling reaction performed at a low salt concentration", "label": "modified nick/gap repair and negative supercoiling reaction performed at a low salt concentration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "electroporation-ready 1 megabase chromosomes", "label": "electroporation-ready 1 megabase chromosomes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "portable chromosome format", "label": "portable chromosome format", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genetically stable 1 megabase chromosomes", "label": "genetically stable 1 megabase chromosomes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "low nutritional requirements", "label": "low nutritional requirements", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high salt tolerance", "label": "high salt tolerance", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "rapid growth rate", "label": "rapid growth rate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "deleting two byproduct encoding genes", "label": "deleting two byproduct encoding genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "2,3-butanediol production", "label": "2,3-butanediol production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "non-sterilized fermentation", "label": "non-sterilized fermentation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "seawater-based minimal medium", "label": "seawater-based minimal medium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "optimized fermentative conditions", "label": "optimized fermentative conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "2,3-butanediol production of 41.27 grams per liter", "label": "2,3-butanediol production of 41.27 grams per liter", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "productivity of 3.44 grams per liter per hour", "label": "productivity of 3.44 grams per liter per hour", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "yield of 0.39 grams per gram glucose", "label": "yield of 0.39 grams per gram glucose", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "recombinant strain Vibrio natriegens Delta frdA Delta ldhA-pETRABC", "label": "recombinant strain Vibrio natriegens Delta frdA Delta ldhA-pETRABC", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Clustered regularly interspaced short palindromic repeat/CRISPR-associated gene systems", "label": "Clustered regularly interspaced short palindromic repeat/CRISPR-associated gene systems", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bacteriophages", "label": "bacteriophages", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "study bacteriophage functional genomics", "label": "study bacteriophage functional genomics", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "evolution of prokaryote CRISPR arrays", "label": "evolution of prokaryote CRISPR arrays", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "phage Genome edit by natural transformation platform", "label": "phage Genome edit by natural transformation platform", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "phage gene diversity", "label": "phage gene diversity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "development of phage synthetic biology", "label": "development of phage synthetic biology", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "development of nanotechnology", "label": "development of nanotechnology", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "discovery of new molecular biology tools", "label": "discovery of new molecular biology tools", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "efficiency and accuracy of phage TT4P2 gene editing", "label": "efficiency and accuracy of phage TT4P2 gene editing", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "target and cleave invaders", "label": "target and cleave invaders", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "CRISPR SWAPnDROP", "label": "CRISPR SWAPnDROP", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "large-scale in-vivo DNA transfer between bacterial species", "label": "large-scale in-vivo DNA transfer between bacterial species", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Genome edit by natural transformation in various species", "label": "Genome edit by natural transformation in various species", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "excision of large chromosomal regions", "label": "excision of large chromosomal regions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "transfer of large chromosomal regions", "label": "transfer of large chromosomal regions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "integration of large chromosomal regions", "label": "integration of large chromosomal regions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "scarless Genome edit by natural transformation", "label": "scarless Genome edit by natural transformation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "marker-free Genome edit by natural transformation", "label": "marker-free Genome edit by natural transformation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "iterative Genome edit by natural transformation", "label": "iterative Genome edit by natural transformation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "parallel insertions and deletions in Genome edit by natural transformation", "label": "parallel insertions and deletions in Genome edit by natural transformation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "DNA library applications", "label": "DNA library applications", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "recycling of standardized parts", "label": "recycling of standardized parts", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "CRISPR SWAPnDROP multi-color scarless co-selection system", "label": "CRISPR SWAPnDROP multi-color scarless co-selection system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "editing efficiency improvement", "label": "editing efficiency improvement", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "visual quality controls", "label": "visual quality controls", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fastest growing organism", "label": "fastest growing organism", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "minimum doubling time of 9.4 minutes", "label": "minimum doubling time of 9.4 minutes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "utilize over 60 different carbon sources", "label": "utilize over 60 different carbon sources", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "non-pathogenic properties", "label": "non-pathogenic properties", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "alternative to E. coli", "label": "alternative to E. coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "engineered V. natriegens strain Vmax Express", "label": "engineered V. natriegens strain Vmax Express", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "ability to incorporate non-canonical amino acid", "label": "ability to incorporate non-canonical amino acid", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "non-canonical amino acid p-azido-L-phenylalanine", "label": "non-canonical amino acid p-azido-L-phenylalanine", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "incorporation into recombinant proteins", "label": "incorporation into recombinant proteins", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "enhanced yellow fluorescent protein", "label": "enhanced yellow fluorescent protein", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "incorporation of p-azido-L-phenylalanine", "label": "incorporation of p-azido-L-phenylalanine", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "MlaC", "label": "MlaC", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "p-azido-L-phenylalanine incorporation into enhanced yellow fluorescent protein", "label": "p-azido-L-phenylalanine incorporation into enhanced yellow fluorescent protein", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "improved suppression efficiency", "label": "improved suppression efficiency", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "protein titer", "label": "protein titer", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "expression levels of MlaC-p-azido-L-phenylalanine", "label": "expression levels of MlaC-p-azido-L-phenylalanine", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "exceeded those of E. coli BL21 cells", "label": "exceeded those of E. coli BL21 cells", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "enhanced yellow fluorescent protein-p-azido-L-phenylalanine", "label": "enhanced yellow fluorescent protein-p-azido-L-phenylalanine", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "expression and isotopic labeling in minimal medium", "label": "expression and isotopic labeling in minimal medium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "azido-group", "label": "azido-group", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "coupling site for NMR sensitive 19F-tags", "label": "coupling site for NMR sensitive 19F-tags", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vmax", "label": "Vmax", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "flexible expression host", "label": "flexible expression host", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "suitable for incorporation of non-canonical amino acids in recombinant proteins", "label": "suitable for incorporation of non-canonical amino acids in recombinant proteins", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "potential to surpass Protein yields of E. coli", "label": "potential to surpass Protein yields of E. coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "expression of isotopically labeled proteins containing non-canonical amino acids", "label": "expression of isotopically labeled proteins containing non-canonical amino acids", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "isotopically labeled proteins containing non-canonical amino acids", "label": "isotopically labeled proteins containing non-canonical amino acids", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "chemically modified for application in protein-observed 19F-NMR", "label": "chemically modified for application in protein-observed 19F-NMR", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Genome editing", "label": "Genome editing", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Natural transformation", "label": "Natural transformation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Incorporation of DNA into genome", "label": "Incorporation of DNA into genome", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Uptake of multiple DNA fragments", "label": "Uptake of multiple DNA fragments", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Multiple DNA fragments", "label": "Multiple DNA fragments", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Multiple simultaneous edits", "label": "Multiple simultaneous edits", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "NT-CRISPR", "label": "NT-CRISPR", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Combination of natural transformation with CRISPR-Cas9 counterselection", "label": "Combination of natural transformation with CRISPR-Cas9 counterselection", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Genome edit by natural transformation", "label": "Genome edit by natural transformation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Induction of Induction of CRISPR-Cas9 targeting wild type sequence", "label": "Induction of Induction of CRISPR-Cas9 targeting wild type sequence", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Induction of CRISPR-Cas9 targeting wild type sequence", "label": "Induction of CRISPR-Cas9 targeting wild type sequence", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Death of non-edited cells", "label": "Death of non-edited cells", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "High cell killing efficiencies", "label": "High cell killing efficiencies", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Dispensability of antibiotic resistance markers", "label": "Dispensability of antibiotic resistance markers", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Scarless edits", "label": "Scarless edits", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Markerless edits", "label": "Markerless edits", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Single-base precision", "label": "Single-base precision", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Deletions", "label": "Deletions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Integrations", "label": "Integrations", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Single-base modifications", "label": "Single-base modifications", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Editing efficiencies", "label": "Editing efficiencies", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Up to 100%", "label": "Up to 100%", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Simultaneous deletion of multiple chromosomal regions", "label": "Simultaneous deletion of multiple chromosomal regions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "SpG Cas9", "label": "SpG Cas9", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Compatibility with NT-CRISPR", "label": "Compatibility with NT-CRISPR", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Broadens target spectrum", "label": "Broadens target spectrum", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "synthetic biology applications", "label": "synthetic biology applications", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "identification of artificial promoters portability and 5\u0027 untranslated regions", "label": "identification of artificial promoters portability and 5\u0027 untranslated regions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "production of fluorescent proteins", "label": "production of fluorescent proteins", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "DNA sequence and motif analysis", "label": "DNA sequence and motif analysis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "artificial regulatory sequence", "label": "artificial regulatory sequence", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "algorithms", "label": "algorithms", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "rapid growth", "label": "rapid growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high soluble protein expression system capacity", "label": "high soluble protein expression system capacity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "maximum soluble protein expression system at 37 degrees Celsius in 4 hours", "label": "maximum soluble protein expression system at 37 degrees Celsius in 4 hours", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "maximum soluble protein expression system at 16 degrees Celsius in 16 hours", "label": "maximum soluble protein expression system at 16 degrees Celsius in 16 hours", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "l-DOPA production", "label": "l-DOPA production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "maximum productivity over 10.0 grams per liter per hour", "label": "maximum productivity over 10.0 grams per liter per hour", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nearly two-fold higher than previously reported", "label": "nearly two-fold higher than previously reported", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "54.0 grams per liter l-DOPA", "label": "54.0 grams per liter l-DOPA", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "catechol conversion rate greater than 95%", "label": "catechol conversion rate greater than 95%", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "remarkable potential as a whole-cell catalysis chassis for the production of valuable chemicals", "label": "remarkable potential as a whole-cell catalysis chassis for the production of valuable chemicals", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high succinate titre", "label": "high succinate titre", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high succinate yield", "label": "high succinate yield", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high succinate productivity", "label": "high succinate productivity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "optimization of process conditions", "label": "optimization of process conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high biomass-specific succinate production rate", "label": "high biomass-specific succinate production rate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "succinate production", "label": "succinate production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reductive tricarboxylic acid cycle", "label": "reductive tricarboxylic acid cycle", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "oxidative tricarboxylic acid / glyoxylate pathway", "label": "oxidative tricarboxylic acid / glyoxylate pathway", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "minimal medium devoid of complex additives and expensive supplements", "label": "minimal medium devoid of complex additives and expensive supplements", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "final succinate titre of 60.4 grams per liter", "label": "final succinate titre of 60.4 grams per liter", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "maximum succinate productivity of 20.8 grams per liter per hour", "label": "maximum succinate productivity of 20.8 grams per liter per hour", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "overall volumetric succinate productivity of 8.6 grams per liter per hour", "label": "overall volumetric succinate productivity of 8.6 grams per liter per hour", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "promising new host for molecular biology and next generation bioprocesses", "label": "promising new host for molecular biology and next generation bioprocesses", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Gram-negative", "label": "Gram-negative", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "slight-halophilic bacterium", "label": "slight-halophilic bacterium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high nutrient versatility", "label": "high nutrient versatility", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reported doubling time of under 10 minutes", "label": "reported doubling time of under 10 minutes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "LysG", "label": "LysG", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "expression in Vibrio natriegens", "label": "expression in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "LysG-based sensors", "label": "LysG-based sensors", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "detection of positively charged amino acids", "label": "detection of positively charged amino acids", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "random mutagenesis", "label": "random mutagenesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "LysG-based sensor used to screen for amino acid producer strains", "label": "LysG-based sensor used to screen for amino acid producer strains", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fluorescence-activated cell sorting", "label": "fluorescence-activated cell sorting", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sorting of sorting of highly fluorescent cells", "label": "sorting of sorting of highly fluorescent cells", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sorting of highly fluorescent cells", "label": "sorting of highly fluorescent cells", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "potential amino acid producer cells", "label": "potential amino acid producer cells", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "amino acid producer strains", "label": "amino acid producer strains", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "producing L-lysine", "label": "producing L-lysine", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "producing L-arginine", "label": "producing L-arginine", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "producing L-histidine", "label": "producing L-histidine", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "producer strains", "label": "producer strains", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "extracellularly producing up to 1 millimolar of effector amino acid", "label": "extracellularly producing up to 1 millimolar of effector amino acid", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genome sequencing of producer strains", "label": "genome sequencing of producer strains", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "insight into the amino acid production metabolism of Vibrio natriegens", "label": "insight into the amino acid production metabolism of Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "expression and application of transcription factor-based biosensors in Vibrio natriegens", "label": "expression and application of transcription factor-based biosensors in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "insight into the underlying physiology", "label": "insight into the underlying physiology", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fast growth rate", "label": "fast growth rate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "doubling time of less than 10 min", "label": "doubling time of less than 10 min", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high substrate uptake rates", "label": "high substrate uptake rates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "metabolic prowess", "label": "metabolic prowess", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "next-generation workhorse for rapid molecular biology", "label": "next-generation workhorse for rapid molecular biology", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "next-generation workhorse for protein expression system", "label": "next-generation workhorse for protein expression system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "next-generation workhorse for metabolic engineering", "label": "next-generation workhorse for metabolic engineering", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "unnatural amino acid mutagenesis", "label": "unnatural amino acid mutagenesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "soluble Protein yields", "label": "soluble Protein yields", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "superfolder green fluorescent Protein yields", "label": "superfolder green fluorescent Protein yields", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "noncanonical \u03b1-amino acids", "label": "noncanonical \u03b1-amino acids", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Top10", "label": "Top10", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "BL21", "label": "BL21", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli C321.\u0394A.opt", "label": "Escherichia coli C321.\u0394A.opt", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli C321.\u0394A.exp", "label": "Escherichia coli C321.\u0394A.exp", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "misincorporated natural \u03b1-amino acids levels", "label": "misincorporated natural \u03b1-amino acids levels", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "orthogonal aminoacyl tRNA synthetase", "label": "orthogonal aminoacyl tRNA synthetase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "type VI secretion system", "label": "type VI secretion system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "externally induced on/off switch", "label": "externally induced on/off switch", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "effective antibacterial weapon", "label": "effective antibacterial weapon", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "delivered effector repertoire", "label": "delivered effector repertoire", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "toxicity range", "label": "toxicity range", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Genetic engineering", "label": "Genetic engineering", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "microbial metabolism", "label": "microbial metabolism", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bioconversions", "label": "bioconversions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Synthetic biology tools", "label": "Synthetic biology tools", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nonmodel organisms application", "label": "nonmodel organisms application", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bio-based industry hosts", "label": "bio-based industry hosts", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Prokaryotic newcomers", "label": "Prokaryotic newcomers", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "metabolic capabilities", "label": "metabolic capabilities", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cellular capabilities", "label": "cellular capabilities", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "industrial application potential", "label": "industrial application potential", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "industrial application drawbacks", "label": "industrial application drawbacks", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "chassis for synthetic and molecular biology", "label": "chassis for synthetic and molecular biology", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Marburg Collection", "label": "Marburg Collection", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Golden Gate cloning toolbox", "label": "Golden Gate cloning toolbox", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "de novo assembly of plasmids", "label": "de novo assembly of plasmids", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "plasmid replication origin", "label": "plasmid replication origin", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "resistance part", "label": "resistance part", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "modular circuit assembly", "label": "modular circuit assembly", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inversion of individual transcription units", "label": "inversion of individual transcription units", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reduce transcriptional crosstalk", "label": "reduce transcriptional crosstalk", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "microplate reader measurement workflow", "label": "microplate reader measurement workflow", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "quantitative characterization of genetic parts", "label": "quantitative characterization of genetic parts", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Cell-free protein synthesis system", "label": "Cell-free protein synthesis system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Corynebacterium glutamate", "label": "Corynebacterium glutamate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Codon usage", "label": "Codon usage", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Protein yield", "label": "Protein yield", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Plasmid synthesis competent cell selection", "label": "Plasmid synthesis competent cell selection", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Plasmid concentration", "label": "Plasmid concentration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Ribosomal binding site", "label": "Ribosomal binding site", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Cell-free protein synthesis system reagent components", "label": "Cell-free protein synthesis system reagent components", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Therapeutic protein production", "label": "Therapeutic protein production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Receptor-binding domain protein of SARS-CoV-2 production", "label": "Receptor-binding domain protein of SARS-CoV-2 production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "protein expression system", "label": "protein expression system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Ochrobactrum anthropi-derived \u03c9-transaminase", "label": "Ochrobactrum anthropi-derived \u03c9-transaminase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "strongly expressed in Escherichia coli", "label": "strongly expressed in Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "weakly in Vibrio natriegens", "label": "weakly in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Ochrobactrum anthropi-derived \u03c9-transaminase C-terminal", "label": "Ochrobactrum anthropi-derived \u03c9-transaminase C-terminal", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "green fluorescent protein fusion", "label": "green fluorescent protein fusion", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens mutants", "label": "Vibrio natriegens mutants", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in protein yield of Ochrobactrum anthropi-derived \u03c9-transaminase", "label": "increase in protein yield of Ochrobactrum anthropi-derived \u03c9-transaminase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in enzyme activity of Ochrobactrum anthropi-derived \u03c9-transaminase", "label": "increase in enzyme activity of Ochrobactrum anthropi-derived \u03c9-transaminase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "ultraviolet mutagenesis", "label": "ultraviolet mutagenesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in protein yield of genes of interest", "label": "increase in protein yield of genes of interest", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "OATA colorimetric assay", "label": "OATA colorimetric assay", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Y457 variants in rpoC", "label": "Y457 variants in rpoC", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in recombinant protein yield", "label": "increase in recombinant protein yield", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in pET plasmids copy number", "label": "increase in pET plasmids copy number", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in soluble protein yield of genes of interest", "label": "increase in soluble protein yield of genes of interest", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "GOI-GFP fusion", "label": "GOI-GFP fusion", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "powerful tool of forward genetics", "label": "powerful tool of forward genetics", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Ter binding sites", "label": "Ter binding sites", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "protection of protection of linear expression templates", "label": "protection of protection of linear expression templates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "DNA-binding protein Tus", "label": "DNA-binding protein Tus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "protection of linear expression templates", "label": "protection of linear expression templates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mCherry encoding", "label": "mCherry encoding", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "deGFP encoding", "label": "deGFP encoding", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli extracts", "label": "Escherichia coli extracts", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "our method comparison", "label": "our method comparison", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "GamS-mediated protection scheme", "label": "GamS-mediated protection scheme", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Tus-Ter system", "label": "Tus-Ter system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens extracts", "label": "Vibrio natriegens extracts", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "linear templates", "label": "linear templates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "plasmid-level expression", "label": "plasmid-level expression", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Optimal conditions", "label": "Optimal conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Generation times of less than 10 minutes", "label": "Generation times of less than 10 minutes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Fastest growing non-pathogenic bacterium", "label": "Fastest growing non-pathogenic bacterium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Full genome availability", "label": "Full genome availability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Genetic engineering tools availability", "label": "Genetic engineering tools availability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Wide range of carbon sources utilization", "label": "Wide range of carbon sources utilization", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Attractive host for biotechnological production", "label": "Attractive host for biotechnological production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Fed-batch cultivations", "label": "Fed-batch cultivations", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "High-cell-density cultivations", "label": "High-cell-density cultivations", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Deep-well plates", "label": "Deep-well plates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Lab-scale bioreactor cultivations", "label": "Lab-scale bioreactor cultivations", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Different temperature ranges in mineral salt medium", "label": "Different temperature ranges in mineral salt medium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Mineral salt medium", "label": "Mineral salt medium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Exponential glucose feeding", "label": "Exponential glucose feeding", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Cell death", "label": "Cell death", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Constant glucose-feeding", "label": "Constant glucose-feeding", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Initial NaCl concentrations of 15-18 grams per Liter", "label": "Initial NaCl concentrations of 15-18 grams per Liter", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Positive effect on cell growth", "label": "Positive effect on cell growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Temperature reduction from 37 to 30 degrees Celsius", "label": "Temperature reduction from 37 to 30 degrees Celsius", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Maximal growth rate", "label": "Maximal growth rate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "1.36 per hour", "label": "1.36 per hour", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Specific oxygen uptake rate", "label": "Specific oxygen uptake rate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "22 millimoles per gram CDW per hour", "label": "22 millimoles per gram CDW per hour", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "High biomass yields", "label": "High biomass yields", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Up to 55 grams per Liter", "label": "Up to 55 grams per Liter", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Fed-batch strategies", "label": "Fed-batch strategies", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Potential of Vibrio natriegens as a strong producer in industrial biotechnology", "label": "Potential of Vibrio natriegens as a strong producer in industrial biotechnology", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "motility behavior", "label": "motility behavior", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bacteria", "label": "bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "environmental applications", "label": "environmental applications", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "food industry applications", "label": "food industry applications", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "biomedical applications", "label": "biomedical applications", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "species", "label": "species", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Magnetococcus marinus", "label": "Magnetococcus marinus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio fischeri", "label": "Vibrio fischeri", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "microfluidic environments", "label": "microfluidic environments", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "various levels of confinement", "label": "various levels of confinement", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "geometrical complexity", "label": "geometrical complexity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "moderate confinement", "label": "moderate confinement", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "moved parallel to the wall", "label": "moved parallel to the wall", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "stable movement parallel to the wall", "label": "stable movement parallel to the wall", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "incidental wall escape events", "label": "incidental wall escape events", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "flipping between wall accumulator and wall escaper regimes", "label": "flipping between wall accumulator and wall escaper regimes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "tighter confining environments", "label": "tighter confining environments", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "motility governed by steric interactions", "label": "motility governed by steric interactions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "steric interactions", "label": "steric interactions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "motility", "label": "motility", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mesoscale regions", "label": "mesoscale regions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "impacts of hydrodynamics and steric interactions overlap", "label": "impacts of hydrodynamics and steric interactions overlap", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "hydrodynamics and steric interactions overlap", "label": "hydrodynamics and steric interactions overlap", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "push bacteria in the same directions in linear channels", "label": "push bacteria in the same directions in linear channels", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "smooth bacterial movement", "label": "smooth bacterial movement", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "oppositional in meandered channels", "label": "oppositional in meandered channels", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "chaotic movement", "label": "chaotic movement", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bacterial trapping", "label": "bacterial trapping", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "microfluidic devices", "label": "microfluidic devices", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "single-cell genomic screening", "label": "single-cell genomic screening", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bacterial entrapment for diagnostics", "label": "bacterial entrapment for diagnostics", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "biocomputation", "label": "biocomputation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Darobactin A", "label": "Darobactin A", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "outer membrane protein BamA", "label": "outer membrane protein BamA", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Gram-negative bacteria", "label": "Gram-negative bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Darobactin A production", "label": "Darobactin A production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "expression constructs optimization", "label": "expression constructs optimization", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Darobactin A production titers increase", "label": "Darobactin A production titers increase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fermentation time decrease", "label": "fermentation time decrease", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "minimal Darobactin A biosynthetic gene cluster", "label": "minimal Darobactin A biosynthetic gene cluster", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Euglena gracilis co-cultivated with naturally competent Vibrio natriegens", "label": "Euglena gracilis co-cultivated with naturally competent Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased total chlorophyll", "label": "increased total chlorophyll", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased microalgal growth", "label": "increased microalgal growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased dry weight", "label": "increased dry weight", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased storage sugar paramylon content", "label": "increased storage sugar paramylon content", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in methyl carbamate", "label": "increase in methyl carbamate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in ectoine", "label": "increase in ectoine", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in choline", "label": "increase in choline", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in methyl N-methylanthranilate", "label": "increase in methyl N-methylanthranilate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in gentiatibetine", "label": "increase in gentiatibetine", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in 4R-aminopentanoic acid", "label": "increase in 4R-aminopentanoic acid", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in glu-val", "label": "increase in glu-val", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high growth rates", "label": "high growth rates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high substrate consumption rates", "label": "high substrate consumption rates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "ability to utilize a variety of substrates", "label": "ability to utilize a variety of substrates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "easy to handle in lab", "label": "easy to handle in lab", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "development of wide range of genetic tools", "label": "development of wide range of genetic tools", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genetic engineering of production strains", "label": "genetic engineering of production strains", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "comprehensive understanding of metabolism and its regulation", "label": "comprehensive understanding of metabolism and its regulation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "exploit full potential of Vibrio natriegens", "label": "exploit full potential of Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "available genetic engineering tools", "label": "available genetic engineering tools", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "obstacles", "label": "obstacles", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "establish Vibrio natriegens as industrial production host", "label": "establish Vibrio natriegens as industrial production host", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bacterial colony-forming units identification", "label": "bacterial colony-forming units identification", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "laser speckle imaging method", "label": "laser speckle imaging method", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "microbial activity assessment", "label": "microbial activity assessment", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "visual inspection under white light illumination", "label": "visual inspection under white light illumination", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "growth of Vibrio natriegens", "label": "growth of Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bacterial colonies on the solid medium observation", "label": "bacterial colonies on the solid medium observation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "visual examination under white light illumination", "label": "visual examination under white light illumination", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "laser speckle correlation analysis", "label": "laser speckle correlation analysis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Gompertz model", "label": "Gompertz model", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "colony radius growth curves description", "label": "colony radius growth curves description", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "laser speckle contrast imaging", "label": "laser speckle contrast imaging", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "colony growth detection", "label": "colony growth detection", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "signal processing", "label": "signal processing", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "standard CFU counting method under white light illumination", "label": "standard CFU counting method under white light illumination", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "1,3-propanediol production", "label": "1,3-propanediol production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "heterologous pathway", "label": "heterologous pathway", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "engineering cellular transcriptional regulators", "label": "engineering cellular transcriptional regulators", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "global transcriptomic analysis", "label": "global transcriptomic analysis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cofactor engineering", "label": "cofactor engineering", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased intracellular reducing power", "label": "increased intracellular reducing power", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "pathway engineering", "label": "pathway engineering", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reduced toxic intermediate accumulation", "label": "reduced toxic intermediate accumulation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "glycerol oxidation pathway engineering", "label": "glycerol oxidation pathway engineering", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "elimination of byproduct formation", "label": "elimination of byproduct formation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "final engineered strain", "label": "final engineered strain", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "1,3-propanediol titer of 56.2 g/L", "label": "1,3-propanediol titer of 56.2 g/L", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "1,3-propanediol yield of 0.61 mol/mol", "label": "1,3-propanediol yield of 0.61 mol/mol", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "1,3-propanediol productivity of 2.36 g/L/h", "label": "1,3-propanediol productivity of 2.36 g/L/h", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "model bacteria in studies", "label": "model bacteria in studies", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fast-growing gram-negative bacterium", "label": "fast-growing gram-negative bacterium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genetic code expansion methodologies", "label": "genetic code expansion methodologies", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "adaptee of genetic code expansion methodology", "label": "adaptee of genetic code expansion methodology", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genetic code expansion", "label": "genetic code expansion", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "incorporation of unnatural amino acids into proteins", "label": "incorporation of unnatural amino acids into proteins", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "demonstrated in Vibrio natriegens", "label": "demonstrated in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "recombinant protein expression system systems", "label": "recombinant protein expression system systems", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reduction in production cost of enzymes", "label": "reduction in production cost of enzymes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "protein expression system host", "label": "protein expression system host", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "doubling time", "label": "doubling time", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "pET plasmids", "label": "pET plasmids", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens strain VnDX", "label": "Vibrio natriegens strain VnDX", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genes of interest", "label": "genes of interest", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "soluble expression in Vibrio natriegens", "label": "soluble expression in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "soluble expression in Escherichia coli", "label": "soluble expression in Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "better expression in Vibrio natriegens", "label": "better expression in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Terrific Broth medium", "label": "Terrific Broth medium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "consensus \"what to try first\" protocol for Vibrio natriegens", "label": "consensus \"what to try first\" protocol for Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased catalytic efficiency under optimized expression conditions", "label": "increased catalytic efficiency under optimized expression conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "pET-compatible expression host", "label": "pET-compatible expression host", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "highly expressed in Vibrio natriegens", "label": "highly expressed in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "not expressed well in Escherichia coli", "label": "not expressed well in Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Ciona intestinalis molecule against microbes-A24", "label": "Ciona intestinalis molecule against microbes-A24", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "antimicrobial peptide", "label": "antimicrobial peptide", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "plasmid expressing recombinant Ciona intestinalis molecule against microbes-A24", "label": "plasmid expressing recombinant Ciona intestinalis molecule against microbes-A24", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "recombinant Ciona intestinalis molecule against microbes-A24-expressing Bacillus subtilis", "label": "recombinant Ciona intestinalis molecule against microbes-A24-expressing Bacillus subtilis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bactericidal activity", "label": "bactericidal activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "lactoferricin-expressing strains", "label": "lactoferricin-expressing strains", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "pepsin-digested extracts from recombinant Ciona intestinalis molecule against microbes-A24-expressing strains", "label": "pepsin-digested extracts from recombinant Ciona intestinalis molecule against microbes-A24-expressing strains", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bactericidal activity against halophilic bacteria", "label": "bactericidal activity against halophilic bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bactericidal activity of recombinant Ciona intestinalis molecule against microbes-A24-expressing Bacillus subtilis strains", "label": "bactericidal activity of recombinant Ciona intestinalis molecule against microbes-A24-expressing Bacillus subtilis strains", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "efficacy of Tetracycline against Vibrio natriegens", "label": "efficacy of Tetracycline against Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "efficacy of Tetracycline against clearance of Vibrio parahaemolyticus in vivo", "label": "efficacy of Tetracycline against clearance of Vibrio parahaemolyticus in vivo", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "efficacy of Tetracycline against Vibrio alginolyticus", "label": "efficacy of Tetracycline against Vibrio alginolyticus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "recombinant Ciona intestinalis molecule against microbes-A24-expressing Bacillus subtilis strains", "label": "recombinant Ciona intestinalis molecule against microbes-A24-expressing Bacillus subtilis strains", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "protection against halophilic bacterial infection in marine fish and shellfish", "label": "protection against halophilic bacterial infection in marine fish and shellfish", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Alginate lyases", "label": "Alginate lyases", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Alginate oligosaccharides", "label": "Alginate oligosaccharides", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Prebiotic effects", "label": "Prebiotic effects", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Antidiabetic effects", "label": "Antidiabetic effects", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Extracellular production of recombinant proteins in Escherichia coli", "label": "Extracellular production of recombinant proteins in Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Simplified downstream processing", "label": "Simplified downstream processing", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "High productivity", "label": "High productivity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Aly01", "label": "Aly01", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Release of trisaccharide from alginate", "label": "Release of trisaccharide from alginate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Strictly Sodium Chloride activated", "label": "Strictly Sodium Chloride activated", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Green fluorescent protein", "label": "Green fluorescent protein", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Successfully secreted in Escherichia coli with Aly01 signal peptide", "label": "Successfully secreted in Escherichia coli with Aly01 signal peptide", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Terrific Broth medium supplemented with glycine and calcium", "label": "Terrific Broth medium supplemented with glycine and calcium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Reduction of lag phase of protein secretion to 3 hours from 12 hours", "label": "Reduction of lag phase of protein secretion to 3 hours from 12 hours", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Calcium", "label": "Calcium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Remedied glycine-related cell growth impairment", "label": "Remedied glycine-related cell growth impairment", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Enzyme productivity", "label": "Enzyme productivity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Reached maximum of 4.55 Units per milliliter", "label": "Reached maximum of 4.55 Units per milliliter", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Aly01 signal peptide", "label": "Aly01 signal peptide", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genome editing", "label": "genome editing", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genetic modification", "label": "genetic modification", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genome editing tools", "label": "genome editing tools", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high efficiency", "label": "high efficiency", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fidelity", "label": "fidelity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens strains", "label": "Vibrio natriegens strains", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "physiological characteristics", "label": "physiological characteristics", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "CICC 10908 strain", "label": "CICC 10908 strain", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "stable growth", "label": "stable growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genome editing study", "label": "genome editing study", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "natural transformation system", "label": "natural transformation system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "optimized natural transformation", "label": "optimized natural transformation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "antibiotic resistance marker cat-sacB integration", "label": "antibiotic resistance marker cat-sacB integration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Kan(R", "label": "Kan(R", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "double-selection cassette", "label": "double-selection cassette", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "seamless genome editing", "label": "seamless genome editing", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "positive rates", "label": "positive rates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "gene complementation", "label": "gene complementation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "gene insertion", "label": "gene insertion", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "gene substitution", "label": "gene substitution", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "transformation of recombinant plasmid", "label": "transformation of recombinant plasmid", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "seamless genome editing system", "label": "seamless genome editing system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "biotechnology applications", "label": "biotechnology applications", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrionaceae bacteria genome replication", "label": "Vibrionaceae bacteria genome replication", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased expression of genes close to origin of replication of Chromosome 1 during fast growth", "label": "increased expression of genes close to origin of replication of Chromosome 1 during fast growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "origin of replication of Chromosome 1-proximate growth-related genes", "label": "origin of replication of Chromosome 1-proximate growth-related genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased gene expression during fast growth", "label": "increased gene expression during fast growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "core genes", "label": "core genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bias towards origin of replication of Chromosome 1", "label": "bias towards origin of replication of Chromosome 1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "softcore genes", "label": "softcore genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "shell genes", "label": "shell genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "overrepresentation close to terminus of Chromosome 1", "label": "overrepresentation close to terminus of Chromosome 1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "core and softcore genes", "label": "core and softcore genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "overrepresentation around origin of replication of Chromosome 1", "label": "overrepresentation around origin of replication of Chromosome 1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "shell and cloud genes", "label": "shell and cloud genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "global gene expression patterns", "label": "global gene expression patterns", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "correlation with chromosomal distance to origin of replication of Chromosome 1", "label": "correlation with chromosomal distance to origin of replication of Chromosome 1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "pangene categories", "label": "pangene categories", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "contribution to skewed expression pattern at fast-growing conditions", "label": "contribution to skewed expression pattern at fast-growing conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "softcore", "label": "softcore", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "shell, and cloud genes, responsible for elevated expression at slow-growing conditions", "label": "shell, and cloud genes, responsible for elevated expression at slow-growing conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "core/softcore genes", "label": "core/softcore genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "enriched in spatially separated intracellular regions", "label": "enriched in spatially separated intracellular regions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "shell/cloud genes", "label": "shell/cloud genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genomic location of genes", "label": "genomic location of genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "link to cellular placement", "label": "link to cellular placement", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Aquatic Killer 99", "label": "Aquatic Killer 99", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "simulated microgravity", "label": "simulated microgravity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "growth rate of bacteria", "label": "growth rate of bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "shear force", "label": "shear force", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "generation time", "label": "generation time", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "growth rate of Vibrio natriegens", "label": "growth rate of Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "long-term culturing under simulated microgravity", "label": "long-term culturing under simulated microgravity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mutant with slower growth rate", "label": "mutant with slower growth rate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "transcriptome data", "label": "transcriptome data", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mechanical signal transduction genes up-regulated", "label": "mechanical signal transduction genes up-regulated", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "flagellin biogenesis genes up-regulated", "label": "flagellin biogenesis genes up-regulated", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "adaptive response genes down-regulated", "label": "adaptive response genes down-regulated", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "anaerobic metabolism genes down-regulated", "label": "anaerobic metabolism genes down-regulated", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nitrogen metabolism genes down-regulated", "label": "nitrogen metabolism genes down-regulated", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "chromosome segregation genes down-regulated", "label": "chromosome segregation genes down-regulated", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cell vitality genes down-regulated", "label": "cell vitality genes down-regulated", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genome-wide chromosome conformation capture", "label": "genome-wide chromosome conformation capture", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "changes of genome 3D organization", "label": "changes of genome 3D organization", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "differentially expressed genes", "label": "differentially expressed genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "whole genome re-sequencing", "label": "whole genome re-sequencing", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "structure variations", "label": "structure variations", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "down-regulated genes", "label": "down-regulated genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "decreased interaction frequency", "label": "decreased interaction frequency", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "well-orchestrated chromosomes replication", "label": "well-orchestrated chromosomes replication", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "local gene expression regulation", "label": "local gene expression regulation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "physiological characteristics alteration", "label": "physiological characteristics alteration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "growth rate", "label": "growth rate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cobalamin class-III dependent enzymes", "label": "cobalamin class-III dependent enzymes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reductive dehalogenases", "label": "reductive dehalogenases", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "organohalide respiration", "label": "organohalide respiration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "carbon-halogen bonds cleaving", "label": "carbon-halogen bonds cleaving", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bioremediation applications", "label": "bioremediation applications", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "heterologous expression systems", "label": "heterologous expression systems", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "active holo-enzyme production", "label": "active holo-enzyme production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "active holo-enzyme", "label": "active holo-enzyme", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "iron-sulphur cluster incorporation", "label": "iron-sulphur cluster incorporation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cobalamin incorporation", "label": "cobalamin incorporation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Bacillus megaterium", "label": "Bacillus megaterium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cobalamin class-III dependent enzymes expression", "label": "cobalamin class-III dependent enzymes expression", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli HMS174(DE3)", "label": "Escherichia coli HMS174(DE3)", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Shimwellia blattae", "label": "Shimwellia blattae", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natrigenes", "label": "Vibrio natrigenes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Nitratireductor pacificus pht-3B reductive dehalogenase", "label": "Nitratireductor pacificus pht-3B reductive dehalogenase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "model enzyme", "label": "model enzyme", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "epoxyqueuosine reductase from Streptococcus thermophilus", "label": "epoxyqueuosine reductase from Streptococcus thermophilus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "co-expression of BtuB", "label": "co-expression of BtuB", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "highest levels of cofactor incorporation", "label": "highest levels of cofactor incorporation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "co-expression of BtuB in E. coli", "label": "co-expression of BtuB in E. coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "appropriate balance between cofactor incorporation and protein yield", "label": "appropriate balance between cofactor incorporation and protein yield", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "rRNA transcription", "label": "rRNA transcription", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "ribosome synthesis", "label": "ribosome synthesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "transcriptional initiation rate of rRNA operon", "label": "transcriptional initiation rate of rRNA operon", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "rRNA synthesis", "label": "rRNA synthesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "rRNA chain elongation rate", "label": "rRNA chain elongation rate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bacterial ribosome synthesis capacity", "label": "bacterial ribosome synthesis capacity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "rRNA chain elongation rate of Escherichia coli", "label": "rRNA chain elongation rate of Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "constant under nutrient limitation", "label": "constant under nutrient limitation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "constant under chloramphenicol inhibition", "label": "constant under chloramphenicol inhibition", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "decreases under low temperature ranges", "label": "decreases under low temperature ranges", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "rRNA chain elongation rate of Vibrio natriegens", "label": "rRNA chain elongation rate of Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "double stranded DNA-binding protein", "label": "double stranded DNA-binding protein", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "single-chain Cro protein", "label": "single-chain Cro protein", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "CroP-LET method", "label": "CroP-LET method", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in superfolder green fluorescent protein expression levels", "label": "increase in superfolder green fluorescent protein expression levels", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli cell-free protein synthesis reactions", "label": "Escherichia coli cell-free protein synthesis reactions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "enhancement in yields in cell-free protein synthesis platforms from nonmodel organisms", "label": "enhancement in yields in cell-free protein synthesis platforms from nonmodel organisms", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens cell-free protein synthesis platform", "label": "Vibrio natriegens cell-free protein synthesis platform", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in superfolder green fluorescent protein yields", "label": "increase in superfolder green fluorescent protein yields", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "gene expression", "label": "gene expression", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "promoter region", "label": "promoter region", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "ribosome binding site element", "label": "ribosome binding site element", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "synthetic regulatory parts", "label": "synthetic regulatory parts", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fluorescence intensities", "label": "fluorescence intensities", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "stability", "label": "stability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nutrient-rich media", "label": "nutrient-rich media", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "synthetic regulatory parts stability", "label": "synthetic regulatory parts stability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "minimal media", "label": "minimal media", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "galactose permease", "label": "galactose permease", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "glucose transport", "label": "glucose transport", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "glucokinase", "label": "glucokinase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "galactose permease expression modulation", "label": "galactose permease expression modulation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Genome mining", "label": "Genome mining", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Levansucrase enzymes with increased transfructosylating activity and wider acceptor promiscuity", "label": "Levansucrase enzymes with increased transfructosylating activity and wider acceptor promiscuity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Levansucrase", "label": "Levansucrase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Fructooligosaccharides synthesis", "label": "Fructooligosaccharides synthesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "\u03b2-(2\u21926", "label": "\u03b2-(2\u21926", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Levansucrase from Gluconobacter oxydans", "label": "Levansucrase from Gluconobacter oxydans", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Fructooligosaccharides with up to 13 units in length", "label": "Fructooligosaccharides with up to 13 units in length", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Levansucrase from Novosphingobium aromaticivorans", "label": "Levansucrase from Novosphingobium aromaticivorans", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Alignment of amino acid sequences and substrate docking with homology models", "label": "Alignment of amino acid sequences and substrate docking with homology models", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Structural elements causing differences in product spectra", "label": "Structural elements causing differences in product spectra", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Raffinose", "label": "Raffinose", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Preferred donor molecule for Levansucrase from Vibrio natriegens", "label": "Preferred donor molecule for Levansucrase from Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Preferred donor molecule for Levansucrase from Novosphingobium aromaticivorans", "label": "Preferred donor molecule for Levansucrase from Novosphingobium aromaticivorans", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Preferred donor molecule for Levansucrase from Paraburkolderia graminis", "label": "Preferred donor molecule for Levansucrase from Paraburkolderia graminis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Levansucrases", "label": "Levansucrases", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Wide acceptor promiscuity", "label": "Wide acceptor promiscuity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Utilization of monosaccharides", "label": "Utilization of monosaccharides", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Utilization of two alcohols", "label": "Utilization of two alcohols", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "extracellular electron transfer", "label": "extracellular electron transfer", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fermentative conditions with gluconate", "label": "fermentative conditions with gluconate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fermentative conditions with glucosamine", "label": "fermentative conditions with glucosamine", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fermentative conditions with pyruvate", "label": "fermentative conditions with pyruvate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "requires CymA", "label": "requires CymA", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "requires PdsA", "label": "requires PdsA", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "requires MtrCAB", "label": "requires MtrCAB", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Fe(III", "label": "Fe(III", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "CymA", "label": "CymA", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "diverged from Shewanella", "label": "diverged from Shewanella", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "from Vibrionaceae", "label": "from Vibrionaceae", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "NapC", "label": "NapC", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "gammaproteobacteria", "label": "gammaproteobacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "anaerobic survival of Vibrio natriegens", "label": "anaerobic survival of Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "primary physiological function in Vibrionaceae", "label": "primary physiological function in Vibrionaceae", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens genes", "label": "Vibrio natriegens genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "functionally complemented Shewanella oneidensis mutants", "label": "functionally complemented Shewanella oneidensis mutants", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "insoluble oxide minerals utilization", "label": "insoluble oxide minerals utilization", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "enhanced survival of Vibrio natriegens under fermentative conditions", "label": "enhanced survival of Vibrio natriegens under fermentative conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fast growth rates", "label": "fast growth rates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "short doubling time", "label": "short doubling time", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high level of proteosynthetic activity", "label": "high level of proteosynthetic activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "non-pathogenicity", "label": "non-pathogenicity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "easy scale-up process", "label": "easy scale-up process", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "potential for using alternative carbon sources", "label": "potential for using alternative carbon sources", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "potential for further genetic and metabolic engineering", "label": "potential for further genetic and metabolic engineering", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "growth media", "label": "growth media", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "wide range of genetic tools", "label": "wide range of genetic tools", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "production of recombinant human growth hormone", "label": "production of recombinant human growth hormone", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "production of yeast alcohol dehydrogenase", "label": "production of yeast alcohol dehydrogenase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "production of production of archaeal catalase-peroxidase", "label": "production of production of archaeal catalase-peroxidase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sucrose", "label": "sucrose", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "higher biomass productions in Vibrio natriegens compared to Escherichia coli", "label": "higher biomass productions in Vibrio natriegens compared to Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "human growth hormone", "label": "human growth hormone", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "lower production in Vibrio natriegens compared to Escherichia coli", "label": "lower production in Vibrio natriegens compared to Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "yeast alcohol dehydrogenase", "label": "yeast alcohol dehydrogenase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "lower production and decreased solubility in Vibrio natriegens compared to Escherichia coli", "label": "lower production and decreased solubility in Vibrio natriegens compared to Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "production of archaeal catalase-peroxidase", "label": "production of archaeal catalase-peroxidase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "higher production in Vibrio natriegens compared to Escherichia coli", "label": "higher production in Vibrio natriegens compared to Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "susceptibility to condition changes", "label": "susceptibility to condition changes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inconsistent results", "label": "inconsistent results", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "non-pathogenic bacterium", "label": "non-pathogenic bacterium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "alternative chassis organism", "label": "alternative chassis organism", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "synthetic biology efforts", "label": "synthetic biology efforts", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "establishment of molecular biology tools", "label": "establishment of molecular biology tools", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "biotechnological production cycles", "label": "biotechnological production cycles", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "natural ecological niche", "label": "natural ecological niche", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "interactions with other organisms", "label": "interactions with other organisms", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genomic organization", "label": "genomic organization", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "diverse metabolism", "label": "diverse metabolism", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "synthetic biology chassis", "label": "synthetic biology chassis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "methods", "label": "methods", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "research community growth", "label": "research community growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "natural clay nanotubes loaded with biocide", "label": "natural clay nanotubes loaded with biocide", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "extended protection against marine microorganisms", "label": "extended protection against marine microorganisms", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "natural clay nanotubes loaded with silver", "label": "natural clay nanotubes loaded with silver", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "halloysite formulations with biocide-loaded nanoclay", "label": "halloysite formulations with biocide-loaded nanoclay", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "extended DCOIT release time to 12 months", "label": "extended DCOIT release time to 12 months", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "epoxy-halloysite nanocomposites", "label": "epoxy-halloysite nanocomposites", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "less adhesion and proliferation of marine bacteria Vibrio natriegens", "label": "less adhesion and proliferation of marine bacteria Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "DCOIT-loaded natural environmentally friendly halloysite clay", "label": "DCOIT-loaded natural environmentally friendly halloysite clay", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "improved antifouling properties of epoxy coating", "label": "improved antifouling properties of epoxy coating", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "silver particles synthesized on halloysite nanotubes", "label": "silver particles synthesized on halloysite nanotubes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "enhanced antifouling property of epoxy resin", "label": "enhanced antifouling property of epoxy resin", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "MnO particles and halloysite natural mixture", "label": "MnO particles and halloysite natural mixture", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "used as nonbiocide additive to marine coating", "label": "used as nonbiocide additive to marine coating", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Ag-halloysite composite", "label": "Ag-halloysite composite", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "no visible fouling in short-term White Sea water test", "label": "no visible fouling in short-term White Sea water test", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "melanin production", "label": "melanin production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "heterologous tyrosinase gene", "label": "heterologous tyrosinase gene", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "melanin", "label": "melanin", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "derived from dihydroxyindole-2-carboxylic acid", "label": "derived from dihydroxyindole-2-carboxylic acid", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "polymer", "label": "polymer", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nanoparticle structure", "label": "nanoparticle structure", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "protection of cells from UV irradiation", "label": "protection of cells from UV irradiation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "redox activity", "label": "redox activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "adsorption of trinitrotoluene", "label": "adsorption of trinitrotoluene", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "melanin nanoparticles", "label": "melanin nanoparticles", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "photoprotective properties", "label": "photoprotective properties", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "expressing heterologous tyrosinase gene", "label": "expressing heterologous tyrosinase gene", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "expressing heterologous tyrosinase gene with inducible promoters", "label": "expressing heterologous tyrosinase gene with inducible promoters", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "physical and chemical properties of melanin", "label": "physical and chemical properties of melanin", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genetic tractability", "label": "genetic tractability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "feedstock flexibility", "label": "feedstock flexibility", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "beta-carotene production", "label": "beta-carotene production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "violacein production", "label": "violacein production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio campbellii", "label": "Vibrio campbellii", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mevalonate biosynthetic pathway", "label": "mevalonate biosynthetic pathway", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "precursor abundance", "label": "precursor abundance", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Lactobacillus acidophilus", "label": "Lactobacillus acidophilus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "biosynthetic gene cluster", "label": "biosynthetic gene cluster", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Chromobacterium violaceum", "label": "Chromobacterium violaceum", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "small molecule drug production", "label": "small molecule drug production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "natural product production with industrially-relevant feedstocks", "label": "natural product production with industrially-relevant feedstocks", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "aerobic metabolism", "label": "aerobic metabolism", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "oxidative stress", "label": "oxidative stress", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "peroxide-sensing transcription factor", "label": "peroxide-sensing transcription factor", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mutations", "label": "mutations", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "oxidized conformation of OxyR", "label": "oxidized conformation of OxyR", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "constitutive expression of genes", "label": "constitutive expression of genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "constitutive activity of OxyR", "label": "constitutive activity of OxyR", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reduces DNA damage from reactive oxygen species", "label": "reduces DNA damage from reactive oxygen species", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reactive oxygen species", "label": "reactive oxygen species", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "activity of the SOS response regulator LexA", "label": "activity of the SOS response regulator LexA", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "lower growth", "label": "lower growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "calculations of proteome allocation", "label": "calculations of proteome allocation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genome-scale models of metabolism and macromolecular expression", "label": "genome-scale models of metabolism and macromolecular expression", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genetic changes in oxyR", "label": "genetic changes in oxyR", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "ribosome binding sites", "label": "ribosome binding sites", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "protein synthesis", "label": "protein synthesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "N-terminal affinity tags", "label": "N-terminal affinity tags", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mRNA secondary structures", "label": "mRNA secondary structures", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "expression optimization", "label": "expression optimization", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "periplasmic translocation sequences", "label": "periplasmic translocation sequences", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "identification of high producers", "label": "identification of high producers", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "thrombin cleavage site", "label": "thrombin cleavage site", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "removal of fusion tags", "label": "removal of fusion tags", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "expression in Escherichia coli", "label": "expression in Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "antifungal peptide", "label": "antifungal peptide", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "urate oxidase", "label": "urate oxidase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli ssYahJ secretion signal", "label": "Escherichia coli ssYahJ secretion signal", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "translocation to periplasm in Vibrio natriegens", "label": "translocation to periplasm in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "combinatorial library approach", "label": "combinatorial library approach", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "selection of high-producer clones", "label": "selection of high-producer clones", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "podophage Phriendly", "label": "podophage Phriendly", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Silver nanoparticles particle size", "label": "Silver nanoparticles particle size", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Minimum inhibitory concentration value", "label": "Minimum inhibitory concentration value", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Minimum bactericidal concentrations value", "label": "Minimum bactericidal concentrations value", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Antibacterial activity", "label": "Antibacterial activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Reactive oxygen species generation by bacteria", "label": "Reactive oxygen species generation by bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Bacterial membrane damage", "label": "Bacterial membrane damage", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Reactive oxygen species", "label": "Reactive oxygen species", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "spaceflight conditions", "label": "spaceflight conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "physiological microbial alterations", "label": "physiological microbial alterations", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "microgravity", "label": "microgravity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased growth yields", "label": "increased growth yields", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "decreased antibiotic susceptibility", "label": "decreased antibiotic susceptibility", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fast generation time", "label": "fast generation time", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens biomass increase", "label": "Vibrio natriegens biomass increase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "normal terrestrial gravity", "label": "normal terrestrial gravity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "final cell population", "label": "final cell population", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "no change in susceptibility to rifampicin", "label": "no change in susceptibility to rifampicin", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "model organism for microbial microgravity research", "label": "model organism for microbial microgravity research", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "resistance markers", "label": "resistance markers", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "ribosomal binding sites", "label": "ribosomal binding sites", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reporters", "label": "reporters", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "terminators", "label": "terminators", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "degradation tags", "label": "degradation tags", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "origin of replication sequences", "label": "origin of replication sequences", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "plasmid backbones", "label": "plasmid backbones", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genetic parts", "label": "genetic parts", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "plasmid stability", "label": "plasmid stability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "plasmid copy numbers", "label": "plasmid copy numbers", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "production load on cells", "label": "production load on cells", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "constructs", "label": "constructs", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "chemical induction", "label": "chemical induction", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "optogenetic induction", "label": "optogenetic induction", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "engineered circuit behavior", "label": "engineered circuit behavior", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "naturally competent Vibrio natriegens MK3", "label": "naturally competent Vibrio natriegens MK3", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "glycolipid biosurfactant", "label": "glycolipid biosurfactant", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "decrease in surface tension", "label": "decrease in surface tension", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reduction in biofilm formation", "label": "reduction in biofilm formation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reduction in bioluminescence", "label": "reduction in bioluminescence", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reduction in extracellular polysaccharide synthesis", "label": "reduction in extracellular polysaccharide synthesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reduction in quorum sensing", "label": "reduction in quorum sensing", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "antipathogenic effect against Vibrio harveyi", "label": "antipathogenic effect against Vibrio harveyi", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cocultivation with Vibrio natriegens", "label": "cocultivation with Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in biomass productivity of Euglena gracilis", "label": "increase in biomass productivity of Euglena gracilis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in paramylon content of Euglena gracilis", "label": "increase in paramylon content of Euglena gracilis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bacterial inoculum density", "label": "bacterial inoculum density", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "growth of Euglena gracilis", "label": "growth of Euglena gracilis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "timing of inoculation", "label": "timing of inoculation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "optimal deployment of Vibrio natriegens", "label": "optimal deployment of Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in biomass productivity of Euglena gracilis by more than 20%", "label": "increase in biomass productivity of Euglena gracilis by more than 20%", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in paramylon content of Euglena gracilis by more than 35%", "label": "increase in paramylon content of Euglena gracilis by more than 35%", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "indole-3-acetic acid biosynthesized by Vibrio natriegens", "label": "indole-3-acetic acid biosynthesized by Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cocultivation", "label": "cocultivation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "significantly larger paramylon granules in Euglena gracilis", "label": "significantly larger paramylon granules in Euglena gracilis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cocultivation with indole-3-acetic acid-producing Vibrio natriegens", "label": "cocultivation with indole-3-acetic acid-producing Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "enhanced production of paramylon in Euglena gracilis", "label": "enhanced production of paramylon in Euglena gracilis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "DNA-damaging agent mitomycin C", "label": "DNA-damaging agent mitomycin C", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "prophage regions induction", "label": "prophage regions induction", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "standard cultivation conditions", "label": "standard cultivation conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "spontaneous activation of prophage regions", "label": "spontaneous activation of prophage regions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "spontaneous activation of VNP1", "label": "spontaneous activation of VNP1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "spontaneous activation of VNP2", "label": "spontaneous activation of VNP2", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mCherry fusion to the major capsid proteins", "label": "mCherry fusion to the major capsid proteins", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "confirmation of activation of VNP1", "label": "confirmation of activation of VNP1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Transmission electron microscopy", "label": "Transmission electron microscopy", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "release of phage particles belonging to the Siphoviridae family", "label": "release of phage particles belonging to the Siphoviridae family", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "removal of proviral load", "label": "removal of proviral load", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "improved robustness toward DNA-damaging conditions", "label": "improved robustness toward DNA-damaging conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reduced cell lysis under hypo-osmotic conditions", "label": "reduced cell lysis under hypo-osmotic conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased pyruvate production", "label": "increased pyruvate production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "prophage-free variant", "label": "prophage-free variant", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "outcompeted the wild type in a competitive growth experiment", "label": "outcompeted the wild type in a competitive growth experiment", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "prophage-free strain", "label": "prophage-free strain", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "higher tolerance toward DNA damage", "label": "higher tolerance toward DNA damage", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "higher tolerance toward hypo-osmotic stress", "label": "higher tolerance toward hypo-osmotic stress", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "prophage elements", "label": "prophage elements", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "detrimental effects on growth and fitness", "label": "detrimental effects on growth and fitness", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "promising platform for biotechnological applications", "label": "promising platform for biotechnological applications", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "aggregates", "label": "aggregates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Nitrogen fixation", "label": "Nitrogen fixation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "heterotrophic diazotrophs", "label": "heterotrophic diazotrophs", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "polysaccharides", "label": "polysaccharides", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nitrogenase-immunolabeling", "label": "nitrogenase-immunolabeling", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "active heterotrophic diazotrophs", "label": "active heterotrophic diazotrophs", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Alcian blue", "label": "Alcian blue", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "concanavalin-A", "label": "concanavalin-A", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nucleic-acid staining", "label": "nucleic-acid staining", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "total bacteria", "label": "total bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "artificial aggregates", "label": "artificial aggregates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "natural aggregates", "label": "natural aggregates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli-based expression systems", "label": "Escherichia coli-based expression systems", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "recombinant proteins", "label": "recombinant proteins", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "M9 minimal medium", "label": "M9 minimal medium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "early time point of induction", "label": "early time point of induction", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cost-effectiveness", "label": "cost-effectiveness", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "convenience", "label": "convenience", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "time efficiency", "label": "time efficiency", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "FKBP", "label": "FKBP", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "EYFP", "label": "EYFP", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "correct folding", "label": "correct folding", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "corrosion of steel materials", "label": "corrosion of steel materials", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "formation of a biomineralized film", "label": "formation of a biomineralized film", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "biomineralized film", "label": "biomineralized film", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inhibition of Vibrio natriegens growth", "label": "inhibition of Vibrio natriegens growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mitigation of corrosion of steel in seawater", "label": "mitigation of corrosion of steel in seawater", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "stable anti-corrosion activity", "label": "stable anti-corrosion activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "extracellular polymeric substances and calcite composition", "label": "extracellular polymeric substances and calcite composition", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "formation of the biomineralized film", "label": "formation of the biomineralized film", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "clustered regularly interspaced short palindromic repeats interference screen", "label": "clustered regularly interspaced short palindromic repeats interference screen", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "minimal set of genes required for rapid wild-type growth", "label": "minimal set of genes required for rapid wild-type growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "predicted protein-coding genes", "label": "predicted protein-coding genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "core genes comprising putative essential and growth-supporting genes", "label": "core genes comprising putative essential and growth-supporting genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "enriched for respiratory pathways", "label": "enriched for respiratory pathways", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "located on chromosome 1", "label": "located on chromosome 1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "growth-neutral duplicates of core genes", "label": "growth-neutral duplicates of core genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "metabolic pathway annotations", "label": "metabolic pathway annotations", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "functional genomics", "label": "functional genomics", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "acceleration of biological studies and engineering of Vibrio natriegens", "label": "acceleration of biological studies and engineering of Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "emerging emerging microbial host for biotechnology", "label": "emerging emerging microbial host for biotechnology", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "preparation protocol", "label": "preparation protocol", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens crude cell extracts", "label": "Vibrio natriegens crude cell extracts", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reduced cost", "label": "reduced cost", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "user accessibility", "label": "user accessibility", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Cell-free protein synthesis", "label": "Cell-free protein synthesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "super folder Green Fluorescent Protein", "label": "super folder Green Fluorescent Protein", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "\u003e 260 micrograms per milliliter", "label": "\u003e 260 micrograms per milliliter", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "3 hours", "label": "3 hours", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "protocol", "label": "protocol", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "integration into protein synthesis pipelines", "label": "integration into protein synthesis pipelines", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "advances in bio-production", "label": "advances in bio-production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "advances in synthetic biology applications", "label": "advances in synthetic biology applications", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "lack of specific accessory genes for membrane insertion or cofactor assembly", "label": "lack of specific accessory genes for membrane insertion or cofactor assembly", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "suitable expression host for membrane-bound proteins from inhibition of growth of Vibrio cholerae", "label": "suitable expression host for membrane-bound proteins from inhibition of growth of Vibrio cholerae", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "NADH:quinone oxidoreductase", "label": "NADH:quinone oxidoreductase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased overall NADH:quinone oxidoreduction activity of membranes from transformed Vibrio natriegens", "label": "increased overall NADH:quinone oxidoreduction activity of membranes from transformed Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sensitivity toward Silver ion", "label": "sensitivity toward Silver ion", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inhibition of growth of Vibrio cholerae NADH:quinone oxidoreductase", "label": "inhibition of growth of Vibrio cholerae NADH:quinone oxidoreductase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "complete assembly in Vibrio natriegens", "label": "complete assembly in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "secondary transport system Mrp from inhibition of growth of Vibrio cholerae", "label": "secondary transport system Mrp from inhibition of growth of Vibrio cholerae", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "functional state in Vibrio natriegens demonstrated by in vivo Lithium ion transport", "label": "functional state in Vibrio natriegens demonstrated by in vivo Lithium ion transport", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "promising expression host for the production of membrane protein complexes from Gram-negative pathogens", "label": "promising expression host for the production of membrane protein complexes from Gram-negative pathogens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fast growing bacterium", "label": "fast growing bacterium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "microbial host for biotechnology", "label": "microbial host for biotechnology", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens cell-free expression system", "label": "Vibrio natriegens cell-free expression system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "superfolder GFP production", "label": "superfolder GFP production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "simplified crude extract preparation protocol", "label": "simplified crude extract preparation protocol", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "small-scale batch reaction", "label": "small-scale batch reaction", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "culturing conditions", "label": "culturing conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "translation kinetics", "label": "translation kinetics", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "protein yield", "label": "protein yield", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cell density", "label": "cell density", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "incubation temperature ranges", "label": "incubation temperature ranges", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "maximal protein yield", "label": "maximal protein yield", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "ions", "label": "ions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "ribosomal stability", "label": "ribosomal stability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "improved yield", "label": "improved yield", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "metabolic engineering applications", "label": "metabolic engineering applications", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "proteins", "label": "proteins", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genetic regulatory element libraries", "label": "genetic regulatory element libraries", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "optimization of biosynthetic pathways", "label": "optimization of biosynthetic pathways", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sensing biomarkers", "label": "sensing biomarkers", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cell-free protein synthesis systems", "label": "cell-free protein synthesis systems", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sonication", "label": "sonication", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "highly active extracts", "label": "highly active extracts", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "synthesis of superfolder green fluorescent protein", "label": "synthesis of superfolder green fluorescent protein", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cell-free reaction conditions", "label": "cell-free reaction conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens-derived cell-free protein synthesis platform", "label": "Vibrio natriegens-derived cell-free protein synthesis platform", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "production of antimicrobial peptides", "label": "production of antimicrobial peptides", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "lyophilization capability", "label": "lyophilization capability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "coupling with Vmax Express strain", "label": "coupling with Vmax Express strain", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "knockout library", "label": "knockout library", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in system productivity", "label": "increase in system productivity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sponge-bacteria interactions", "label": "sponge-bacteria interactions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "microbial diversity", "label": "microbial diversity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sponge metabolite profiles", "label": "sponge metabolite profiles", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bioactivity", "label": "bioactivity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "controlled aquarium system", "label": "controlled aquarium system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Aplysilla rosea", "label": "Aplysilla rosea", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens interaction", "label": "Vibrio natriegens interaction", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens exposure", "label": "Vibrio natriegens exposure", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "shifts in microbial communities", "label": "shifts in microbial communities", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased bacterial diversity", "label": "increased bacterial diversity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cell-free protein synthesis system", "label": "cell-free protein synthesis system", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cell-free extract", "label": "cell-free extract", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "eGFP synthesis", "label": "eGFP synthesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fast-growing Vibrio natriegens cultures", "label": "fast-growing Vibrio natriegens cultures", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "rRNA stability", "label": "rRNA stability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "substrate consumption", "label": "substrate consumption", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "magnesium homeostasis induction", "label": "magnesium homeostasis induction", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "translation capacity", "label": "translation capacity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "protein expression", "label": "protein expression", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "in vitro screening", "label": "in vitro screening", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "promoter strength testing", "label": "promoter strength testing", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "gene expression efficiency", "label": "gene expression efficiency", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cell-free protein synthesis toolbox", "label": "cell-free protein synthesis toolbox", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens platform establishment", "label": "Vibrio natriegens platform establishment", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "aerobic growth", "label": "aerobic growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "survival in 100 mM selenite", "label": "survival in 100 mM selenite", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "selenite", "label": "selenite", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens growth", "label": "Vibrio natriegens growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "selenium nanoparticles production", "label": "selenium nanoparticles production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bioremediation of selenite", "label": "bioremediation of selenite", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "selenium nanoparticles production by Vibrio natriegens", "label": "selenium nanoparticles production by Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "faster selenium nanoparticles production compared to other microorganisms", "label": "faster selenium nanoparticles production compared to other microorganisms", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "selenite-containing LB medium at 30 degrees Celsius", "label": "selenite-containing LB medium at 30 degrees Celsius", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "exponential growth phase of Vibrio natriegens", "label": "exponential growth phase of Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "selenium nanoparticles release after cell lysis", "label": "selenium nanoparticles release after cell lysis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "CcKPI1", "label": "CcKPI1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inhibitory activities against elastase", "label": "inhibitory activities against elastase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inhibitory activities against subtilisin A", "label": "inhibitory activities against subtilisin A", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inhibitory activities against proteinase K", "label": "inhibitory activities against proteinase K", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "rCcKPI1", "label": "rCcKPI1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "competitive inhibitory effects", "label": "competitive inhibitory effects", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "antimicrobial activity", "label": "antimicrobial activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "binding to Staphylococcus aureus", "label": "binding to Staphylococcus aureus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "binding to Bacillus subtilis", "label": "binding to Bacillus subtilis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "binding to Escherichia coli", "label": "binding to Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "binding to Vibrio vulnificus", "label": "binding to Vibrio vulnificus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "binding to Vibrio cholerae", "label": "binding to Vibrio cholerae", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "binding to Vibrio natriegens", "label": "binding to Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "binding to Vibrio mimicus", "label": "binding to Vibrio mimicus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "binding to Vibrio alginolyticus", "label": "binding to Vibrio alginolyticus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "binding to Vibrio parahaemolyticus", "label": "binding to Vibrio parahaemolyticus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "binding to Candida albicans", "label": "binding to Candida albicans", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "binding to Candida parapsilokis", "label": "binding to Candida parapsilokis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "binding to Candida glabrata", "label": "binding to Candida glabrata", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inhibition of growth of Staphylococcus aureus", "label": "inhibition of growth of Staphylococcus aureus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inhibition of growth of Bacillus subtilis", "label": "inhibition of growth of Bacillus subtilis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inhibition of growth of Escherichia coli", "label": "inhibition of growth of Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inhibition of growth of Vibrio vulnificus", "label": "inhibition of growth of Vibrio vulnificus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inhibition of growth of Vibrio cholerae", "label": "inhibition of growth of Vibrio cholerae", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inhibition of growth of Vibrio natriegens", "label": "inhibition of growth of Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inhibition of growth of Vibrio mimicus", "label": "inhibition of growth of Vibrio mimicus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inhibition of growth of Vibrio alginolyticus", "label": "inhibition of growth of Vibrio alginolyticus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inhibition of growth of Vibrio parahaemolyticus", "label": "inhibition of growth of Vibrio parahaemolyticus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inhibition of growth of Candida albicans", "label": "inhibition of growth of Candida albicans", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inhibition of growth of Candida parapsilokis", "label": "inhibition of growth of Candida parapsilokis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "antibacterial activities", "label": "antibacterial activities", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "antifungal activities", "label": "antifungal activities", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "role in immune defence of Cyanea capillata", "label": "role in immune defence of Cyanea capillata", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "core metabolism model", "label": "core metabolism model", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "biomass composition", "label": "biomass composition", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high-resolution carbon-13 metabolic flux analysis", "label": "high-resolution carbon-13 metabolic flux analysis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "intracellular fluxes estimation", "label": "intracellular fluxes estimation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "parallel labeling experiments optimal tracers [1,2-carbon-13]glucose", "label": "parallel labeling experiments optimal tracers [1,2-carbon-13]glucose", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "growth rate of 1.70 1/h", "label": "growth rate of 1.70 1/h", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "doubling time of 24 minutes", "label": "doubling time of 24 minutes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "glucose uptake rate of 3.90g/g/h", "label": "glucose uptake rate of 3.90g/g/h", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens core metabolism", "label": "Vibrio natriegens core metabolism", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "similarity to E. coli core metabolism", "label": "similarity to E. coli core metabolism", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "lower normalized flux through oxidative pentose phosphate pathway by 33%", "label": "lower normalized flux through oxidative pentose phosphate pathway by 33%", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "energy metabolism insights", "label": "energy metabolism insights", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "redox metabolism insights", "label": "redox metabolism insights", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "biomass-specific substrate consumption rate", "label": "biomass-specific substrate consumption rate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "rapid growth in rapid growth in BHIN complex medium", "label": "rapid growth in rapid growth in BHIN complex medium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "rapid growth in BHIN complex medium", "label": "rapid growth in BHIN complex medium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high biomass-specific substrate consumption rate under aerobic conditions", "label": "high biomass-specific substrate consumption rate under aerobic conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high biomass-specific substrate consumption rate under anaerobic conditions", "label": "high biomass-specific substrate consumption rate under anaerobic conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genetically engineered Vibrio natriegens", "label": "genetically engineered Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "overall volumetric productivity of alanine", "label": "overall volumetric productivity of alanine", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "promising microbial platform for industrial fermentation processes", "label": "promising microbial platform for industrial fermentation processes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "versatility in metabolism", "label": "versatility in metabolism", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "rapid conversion rates", "label": "rapid conversion rates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Photobacterium sp. TKY1 ", "label": "Photobacterium sp. TKY1 ", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "tributyltin (TBT", "label": "tributyltin (TBT", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "number of surviving cells", "label": "number of surviving cells", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "tributyltin (TBT) chloride ", "label": "tributyltin (TBT) chloride ", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Photobacterium ganghwense JCM 12487 sensitivity", "label": "Photobacterium ganghwense JCM 12487 sensitivity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Photobacterium halotolerans LMG 22194 sensitivity", "label": "Photobacterium halotolerans LMG 22194 sensitivity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens ATCC 14048 ", "label": "Vibrio natriegens ATCC 14048 ", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reduction in surviving cells in presence of Photobacterium sp. TKY1 and tributyltin (TBT", "label": "reduction in surviving cells in presence of Photobacterium sp. TKY1 and tributyltin (TBT", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genetic tools", "label": "genetic tools", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "development of Vibrio natriegens", "label": "development of Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "natural competence induction", "label": "natural competence induction", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "MuGENT", "label": "MuGENT", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "integration of multiple genome edits", "label": "integration of multiple genome edits", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high-efficiency genome editing", "label": "high-efficiency genome editing", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "generation of highly complex mutant populations", "label": "generation of highly complex mutant populations", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "metabolic engineering efforts", "label": "metabolic engineering efforts", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "PHB biosynthesis genes expression targeting via MuGENT", "label": "PHB biosynthesis genes expression targeting via MuGENT", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "enhanced PHB production in Vibrio natriegens", "label": "enhanced PHB production in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "edited strains", "label": "edited strains", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "\u223c100 times more PHB production than parent isolate", "label": "\u223c100 times more PHB production than parent isolate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "\u223c3.3 times more PHB production than rationally designed strain", "label": "\u223c3.3 times more PHB production than rationally designed strain", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bacterial host", "label": "bacterial host", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "fastest growth rate", "label": "fastest growth rate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "engineer Vibrio natriegens", "label": "engineer Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "advantages in biotech processes", "label": "advantages in biotech processes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli recA::luxCDABE bioreporter", "label": "Escherichia coli recA::luxCDABE bioreporter", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genotoxicity assessments", "label": "genotoxicity assessments", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "recA promoter", "label": "recA promoter", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli recA::luxCDABE bioreporter construction", "label": "Escherichia coli recA::luxCDABE bioreporter construction", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "recA promoter source", "label": "recA promoter source", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mitomycin C induction", "label": "mitomycin C induction", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "dose-response relationship in Escherichia coli recA::luxCDABE bioreporter", "label": "dose-response relationship in Escherichia coli recA::luxCDABE bioreporter", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "pollutants induction", "label": "pollutants induction", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "calculation as mitomycin C equivalents by Escherichia coli recA::luxCDABE bioreporter", "label": "calculation as mitomycin C equivalents by Escherichia coli recA::luxCDABE bioreporter", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "environmental samples induction", "label": "environmental samples induction", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "temperature range", "label": "temperature range", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genotoxicity evaluation by Escherichia coli recA::luxCDABE bioreporter", "label": "genotoxicity evaluation by Escherichia coli recA::luxCDABE bioreporter", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "pH range", "label": "pH range", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "salinity range", "label": "salinity range", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "environmental media", "label": "environmental media", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genotoxicity assessment by Escherichia coli recA::luxCDABE bioreporter", "label": "genotoxicity assessment by Escherichia coli recA::luxCDABE bioreporter", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens recA expression in Escherichia coli", "label": "Vibrio natriegens recA expression in Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "indication of a LexA-like regulator in Vibrio natriegens", "label": "indication of a LexA-like regulator in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens recA putative SOS box", "label": "Vibrio natriegens recA putative SOS box", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "similarity to Escherichia coli SOS box", "label": "similarity to Escherichia coli SOS box", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "groundwater samples from a petrochemical-contaminated site", "label": "groundwater samples from a petrochemical-contaminated site", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "chemical analyses of contamination levels", "label": "chemical analyses of contamination levels", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Porphyra haitanensis", "label": "Porphyra haitanensis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Seawater", "label": "Seawater", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Medium containing 1 per thousand agar", "label": "Medium containing 1 per thousand agar", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Dilution-plate method", "label": "Dilution-plate method", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Liquid culture", "label": "Liquid culture", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Crude enzymes", "label": "Crude enzymes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "DNS method", "label": "DNS method", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Neoagaro oligosaccharides", "label": "Neoagaro oligosaccharides", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "16S rDNA sequence analysis", "label": "16S rDNA sequence analysis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Morphological characteristics", "label": "Morphological characteristics", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Physiological biochemical characteristics", "label": "Physiological biochemical characteristics", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Basic Local Alignment Search Tool", "label": "Basic Local Alignment Search Tool", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "HPLC", "label": "HPLC", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Dasycladus vermicularis", "label": "Dasycladus vermicularis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sulfated hydroxylated coumarin derivative", "label": "sulfated hydroxylated coumarin derivative", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "protein cross linker", "label": "protein cross linker", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "comprehensive monitoring technique", "label": "comprehensive monitoring technique", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "identification of 4-(sulfooxy", "label": "identification of 4-(sulfooxy", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "4-(sulfooxy)phenylacetic acid", "label": "4-(sulfooxy)phenylacetic acid", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "proof of mass spectrometry-based structure", "label": "proof of mass spectrometry-based structure", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "4-(sulfooxy)benzoic acid", "label": "4-(sulfooxy)benzoic acid", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "desulfated phenols", "label": "desulfated phenols", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sulfatases of bacteria", "label": "sulfatases of bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "desulfated forms", "label": "desulfated forms", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased activity in biofouling experiments", "label": "increased activity in biofouling experiments", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sulfatation", "label": "sulfatation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "detoxification measure", "label": "detoxification measure", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Zosteric acid", "label": "Zosteric acid", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "para-coumaric acid", "label": "para-coumaric acid", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "biofilm inhibiting activity", "label": "biofilm inhibiting activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inactive", "label": "inactive", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sulfatase activity", "label": "sulfatase activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli inhibition", "label": "Escherichia coli inhibition", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens inhibition", "label": "Vibrio natriegens inhibition", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mixed electron donors", "label": "mixed electron donors", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "carbon dioxide fixing efficiency", "label": "carbon dioxide fixing efficiency", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "non-photosynthetic microbial communities", "label": "non-photosynthetic microbial communities", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "energy utilization efficiency", "label": "energy utilization efficiency", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nitrite", "label": "nitrite", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "thiosulfate", "label": "thiosulfate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "microbial inoculation concentration", "label": "microbial inoculation concentration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "net amount of carbon dioxide fixing", "label": "net amount of carbon dioxide fixing", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "autotrophic bacteria", "label": "autotrophic bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "energy/carbon source recycling", "label": "energy/carbon source recycling", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "heterotrophic bacteria", "label": "heterotrophic bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "symbiotic bacteria", "label": "symbiotic bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "C-type lectin", "label": "C-type lectin", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "pattern-recognition proteins of the non-self-innate immune system in invertebrates", "label": "pattern-recognition proteins of the non-self-innate immune system in invertebrates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "EsCTL1", "label": "EsCTL1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "169 amino acids", "label": "169 amino acids", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "EsCTL2", "label": "EsCTL2", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "164 amino acids", "label": "164 amino acids", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "carbohydrate-recognition domain", "label": "carbohydrate-recognition domain", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "not clustered with other reported lectins from crabs", "label": "not clustered with other reported lectins from crabs", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "expressed only in the hepatopancreas", "label": "expressed only in the hepatopancreas", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "challenge with lipopolysaccharide", "label": "challenge with lipopolysaccharide", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "regulation of expression levels of EsCTL1 and EsCTL2", "label": "regulation of expression levels of EsCTL1 and EsCTL2", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "challenge with peptidoglycan", "label": "challenge with peptidoglycan", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "challenge with Staphylococcus aureus", "label": "challenge with Staphylococcus aureus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "challenge with Aeromonas hydrophila", "label": "challenge with Aeromonas hydrophila", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "recombinant EsCTL1", "label": "recombinant EsCTL1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "agglutination of both Gram-positive bacteria", "label": "agglutination of both Gram-positive bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "recombinant EsCTL2", "label": "recombinant EsCTL2", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "agglutination of both Gram-negative bacteria", "label": "agglutination of both Gram-negative bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "direct binding to lipopolysaccharide", "label": "direct binding to lipopolysaccharide", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "direct binding to peptidoglycan", "label": "direct binding to peptidoglycan", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "direct binding to microorganisms", "label": "direct binding to microorganisms", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "rEsCTL1", "label": "rEsCTL1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "facilitation of the clearance of Vibrio parahaemolyticus in vivo", "label": "facilitation of the clearance of Vibrio parahaemolyticus in vivo", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "roles in the anti-bacterial immunity of Chinese mitten crab", "label": "roles in the anti-bacterial immunity of Chinese mitten crab", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "molecular biological analysis", "label": "molecular biological analysis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "detection of Vibrio parahaemolyticus", "label": "detection of Vibrio parahaemolyticus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "virulence factor genes", "label": "virulence factor genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "determinations of Vibrio parahaemolyticus population numbers", "label": "determinations of Vibrio parahaemolyticus population numbers", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "prevalence of pathogenic Vibrio parahaemolyticus strains", "label": "prevalence of pathogenic Vibrio parahaemolyticus strains", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Photobacterium damselae", "label": "Photobacterium damselae", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "isolated from pristine salt marsh estuary", "label": "isolated from pristine salt marsh estuary", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio diabolicus", "label": "Vibrio diabolicus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "strains similar to Vibrio tubiashii", "label": "strains similar to Vibrio tubiashii", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio parahaemolyticus hemolysin genes tdh", "label": "Vibrio parahaemolyticus hemolysin genes tdh", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "examined using established PCR primers and protocols", "label": "examined using established PCR primers and protocols", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio parahaemolyticus hemolysin genes trh", "label": "Vibrio parahaemolyticus hemolysin genes trh", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio parahaemolyticus hemolysin gene tlh", "label": "Vibrio parahaemolyticus hemolysin gene tlh", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio parahaemolyticus type III secretion system 2 alpha gene vscC2", "label": "Vibrio parahaemolyticus type III secretion system 2 alpha gene vscC2", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "virulence-related genes", "label": "virulence-related genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "occurrence at high frequencies in non-Vibrio parahaemolyticus Vibrionaceae species", "label": "occurrence at high frequencies in non-Vibrio parahaemolyticus Vibrionaceae species", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "contained virulence-related genes", "label": "contained virulence-related genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "does not ensure correct identification of virulent Vibrio parahaemolyticus", "label": "does not ensure correct identification of virulent Vibrio parahaemolyticus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio parahaemolyticus-like virulence factors", "label": "Vibrio parahaemolyticus-like virulence factors", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "potentially complicates tracking of Vibrio parahaemolyticus infections outbreaks", "label": "potentially complicates tracking of Vibrio parahaemolyticus infections outbreaks", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "EsCTLDcp", "label": "EsCTLDcp", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Chinese mitten crab Eriocheir sinensis", "label": "Chinese mitten crab Eriocheir sinensis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "pathogens", "label": "pathogens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "gram-positive bacteria Staphylococcus aureus", "label": "gram-positive bacteria Staphylococcus aureus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "gram-negative bacteria Aeromonas hydrophila", "label": "gram-negative bacteria Aeromonas hydrophila", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bacteria Bacillus thuringiensis", "label": "bacteria Bacillus thuringiensis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bacteria Bacillus subtilis", "label": "bacteria Bacillus subtilis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bacteria Escherichia coli", "label": "bacteria Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bacteria Vibrio natriegens", "label": "bacteria Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "envelope protein of white spot syndrome virus VP24", "label": "envelope protein of white spot syndrome virus VP24", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "innate immunity of Eriocheir sinensis", "label": "innate immunity of Eriocheir sinensis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "hepatopancreas expression regulation", "label": "hepatopancreas expression regulation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "agglutination of bacteria", "label": "agglutination of bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "binding to various bacteria", "label": "binding to various bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "clearance of Vibrio parahaemolyticus in vivo", "label": "clearance of Vibrio parahaemolyticus in vivo", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio cholerae", "label": "Vibrio cholerae", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "antibiotic resistance", "label": "antibiotic resistance", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "SXT/R391-like integrative and conjugative elements", "label": "SXT/R391-like integrative and conjugative elements", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio strains", "label": "Vibrio strains", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "novel genes", "label": "novel genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "ICEs-chromosome junction sequences", "label": "ICEs-chromosome junction sequences", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "core-genes for ICE transfer", "label": "core-genes for ICE transfer", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "novel cluster", "label": "novel cluster", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "annotated draft genome sequence", "label": "annotated draft genome sequence", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "generation time of \u003c10 min", "label": "generation time of \u003c10 min", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sequenced genome", "label": "sequenced genome", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens ATCC 14048", "label": "Vibrio natriegens ATCC 14048", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Monoclonal antibodies raised against Vibrio parahaemolyticus cell extracts", "label": "Monoclonal antibodies raised against Vibrio parahaemolyticus cell extracts", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "MAb-VP34", "label": "MAb-VP34", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Reacted in enzyme-linked immunosorbent assays", "label": "Reacted in enzyme-linked immunosorbent assays", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Highly specific for Vibrio parahaemolyticus", "label": "Highly specific for Vibrio parahaemolyticus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Recognized Vibrio parahaemolyticus F(0", "label": "Recognized Vibrio parahaemolyticus F(0", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "1) ATP synthase\u0027s delta subunit)\n(MAb-VP34", "label": "1) ATP synthase\u0027s delta subunit)\n(MAb-VP34", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Used to develop immunodot blotting assay (VP-Dot", "label": "Used to develop immunodot blotting assay (VP-Dot", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Immunodot blotting assay (VP-Dot)", "label": "Immunodot blotting assay (VP-Dot)", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Rapid identification of Vibrio parahaemolyticus colonies", "label": "Rapid identification of Vibrio parahaemolyticus colonies", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Reduced labor, time, Rapid identification of Vibrio parahaemolyticus colonies", "label": "Reduced labor, time, Rapid identification of Vibrio parahaemolyticus colonies", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "packhorse lobster", "label": "packhorse lobster", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio jasicida sp. nov.", "label": "Vibrio jasicida sp. nov.", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "abalone", "label": "abalone", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Atlantic salmon", "label": "Atlantic salmon", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "0.5-8.0% NaCl", "label": "0.5-8.0% NaCl", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "optimum 3.0-6.0% NaCl", "label": "optimum 3.0-6.0% NaCl", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "10-37 \u00b0C", "label": "10-37 \u00b0C", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "optimum 25-30 \u00b0C", "label": "optimum 25-30 \u00b0C", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "45.9 mol% DNA G+C content", "label": "45.9 mol% DNA G+C content", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio harveyi JMH612", "label": "Vibrio harveyi JMH612", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "N-acyl-homoserine lactone", "label": "N-acyl-homoserine lactone", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Agrobacterium tumefaciens KYC55", "label": "Agrobacterium tumefaciens KYC55", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "thin-layer chromatography", "label": "thin-layer chromatography", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio mediterranei VIB296", "label": "Vibrio mediterranei VIB296", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "N-acyl-homoserine lactone diversity", "label": "N-acyl-homoserine lactone diversity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Aliivibrio logei VIB414", "label": "Aliivibrio logei VIB414", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "luxS gene", "label": "luxS gene", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "autoinducer-2 activity", "label": "autoinducer-2 activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cholerae autoinducer-1-like activity", "label": "cholerae autoinducer-1-like activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio campbellii VIB285", "label": "Vibrio campbellii VIB285", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio furnissii VIB293", "label": "Vibrio furnissii VIB293", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio pomeroyi LMG20537", "label": "Vibrio pomeroyi LMG20537", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio harveyi VIB571", "label": "Vibrio harveyi VIB571", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio harveyi VIB645", "label": "Vibrio harveyi VIB645", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "N-acyl-homoserine lactone activity", "label": "N-acyl-homoserine lactone activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrionaceae strains", "label": "Vibrionaceae strains", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "quorum sensing signal molecule diversity", "label": "quorum sensing signal molecule diversity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "N-acyl-homoserine lactone production", "label": "N-acyl-homoserine lactone production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "autoinducer-2 production", "label": "autoinducer-2 production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "environmental conditions", "label": "environmental conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "plant growth-promoting bacteria strains", "label": "plant growth-promoting bacteria strains", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "phosphate-solubilization capacity", "label": "phosphate-solubilization capacity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nitrogen-fixation capacity", "label": "nitrogen-fixation capacity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "plant growth", "label": "plant growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "strain HN011", "label": "strain HN011", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "SZ7-1", "label": "SZ7-1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Klebsiella oxytoca", "label": "Klebsiella oxytoca", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "SZ002", "label": "SZ002", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Paenibacillus sp.", "label": "Paenibacillus sp.", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "SZ002 nifH gene", "label": "SZ002 nifH gene", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Klebsiella genus", "label": "Klebsiella genus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased plant dry weight", "label": "increased plant dry weight", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased total Nitrogen in plants", "label": "increased total Nitrogen in plants", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased total Phosphorus in plants", "label": "increased total Phosphorus in plants", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Nucleotide excision repair", "label": "Nucleotide excision repair", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "DNA lesions", "label": "DNA lesions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "uvrA", "label": "uvrA", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "DNA repair capabilities", "label": "DNA repair capabilities", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "uvrB", "label": "uvrB", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli uvrA mutants", "label": "Escherichia coli uvrA mutants", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli uvrB mutants", "label": "Escherichia coli uvrB mutants", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "luciferase gene", "label": "luciferase gene", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "transcriptional fusions", "label": "transcriptional fusions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "uvrA regulatory regions", "label": "uvrA regulatory regions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "luminescence induction", "label": "luminescence induction", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "UV irradiation", "label": "UV irradiation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "dichloro chalcone containing marine paint", "label": "dichloro chalcone containing marine paint", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reduction in maximum biofilm thickness on polymethylmethacrylate", "label": "reduction in maximum biofilm thickness on polymethylmethacrylate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in surface roughness", "label": "increase in surface roughness", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "decrease in surface hydrophobicity", "label": "decrease in surface hydrophobicity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "dichloro chalcone", "label": "dichloro chalcone", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "large number of dead cells on coated surfaces", "label": "large number of dead cells on coated surfaces", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Bacillus flexus", "label": "Bacillus flexus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "marine biofouling", "label": "marine biofouling", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Pseudomonas fluorescens", "label": "Pseudomonas fluorescens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Chalcone derivatives", "label": "Chalcone derivatives", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Compounds C-5", "label": "Compounds C-5", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Compounds C-16", "label": "Compounds C-16", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Compounds C-24", "label": "Compounds C-24", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Compounds C-33", "label": "Compounds C-33", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Compounds C-37", "label": "Compounds C-37", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Hydroxyl substitutions", "label": "Hydroxyl substitutions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "antibiofouling activity", "label": "antibiofouling activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Jurs descriptors", "label": "Jurs descriptors", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "hexadecanoic acid", "label": "hexadecanoic acid", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "evaporative fluxes", "label": "evaporative fluxes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "concentration of naturally competent Vibrio natriegens cells", "label": "concentration of naturally competent Vibrio natriegens cells", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "removal of hexadecanoic acid", "label": "removal of hexadecanoic acid", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "quantity of hexadecanoic acid at the interface", "label": "quantity of hexadecanoic acid at the interface", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Ultraviolet radiation", "label": "Ultraviolet radiation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Microbacterium maritypicum resistance", "label": "Microbacterium maritypicum resistance", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens resistance", "label": "Vibrio natriegens resistance", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "NaCl range 0-1000 mM", "label": "NaCl range 0-1000 mM", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Microbacterium maritypicum growth", "label": "Microbacterium maritypicum growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "NaCl range 250-1000 mM", "label": "NaCl range 250-1000 mM", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "NaCl optimum growth 0-400 mM", "label": "NaCl optimum growth 0-400 mM", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Microbacterium maritypicum optimum growth", "label": "Microbacterium maritypicum optimum growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "NaCl optimum growth 250-600 mM", "label": "NaCl optimum growth 250-600 mM", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens optimum growth", "label": "Vibrio natriegens optimum growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Ultraviolet A and B radiation", "label": "Ultraviolet A and B radiation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Ultraviolet C radiation", "label": "Ultraviolet C radiation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Photoreactivation in Microbacterium maritypicum", "label": "Photoreactivation in Microbacterium maritypicum", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Photoreactivation in Vibrio natriegens", "label": "Photoreactivation in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "NaCl 0.0 M", "label": "NaCl 0.0 M", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Microbacterium maritypicum UV-C resistance", "label": "Microbacterium maritypicum UV-C resistance", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "NaCl 0.6 M", "label": "NaCl 0.6 M", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio spp. strain M11", "label": "Vibrio spp. strain M11", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "poly-beta-hydroxybutyrate", "label": "poly-beta-hydroxybutyrate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio spp. strain M14", "label": "Vibrio spp. strain M14", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio spp. strain M20", "label": "Vibrio spp. strain M20", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio spp. strain M31", "label": "Vibrio spp. strain M31", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "growth on acetate", "label": "growth on acetate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "growth on glycerol", "label": "growth on glycerol", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "growth on succinate", "label": "growth on succinate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "growth on glucose", "label": "growth on glucose", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "growth on sucrose", "label": "growth on sucrose", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sodium chloride in medium", "label": "sodium chloride in medium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high poly-beta-hydroxybutyrate concentration in Vibrio spp. strain M11", "label": "high poly-beta-hydroxybutyrate concentration in Vibrio spp. strain M11", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens characteristic", "label": "Vibrio natriegens characteristic", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "exploitable trait for poly-beta-hydroxybutyrate production", "label": "exploitable trait for poly-beta-hydroxybutyrate production", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "DNA rearrangements", "label": "DNA rearrangements", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mutation resilience", "label": "mutation resilience", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "T4-like phages adaptation", "label": "T4-like phages adaptation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genome size", "label": "genome size", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "T4-like phages divergence", "label": "T4-like phages divergence", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "host range", "label": "host range", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "content and genetic organization of T4-like genes for DNA metabolism", "label": "content and genetic organization of T4-like genes for DNA metabolism", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mutational drift of the predicted T4-like gene products and their regulatory sites", "label": "mutational drift of the predicted T4-like gene products and their regulatory sites", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "content of open-reading frames that have no counterparts in T4 or other known organisms", "label": "content of open-reading frames that have no counterparts in T4 or other known organisms", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "diversity in sequence of DNA replication proteins", "label": "diversity in sequence of DNA replication proteins", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "diversity in molecular form of DNA replication proteins", "label": "diversity in molecular form of DNA replication proteins", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sequence divergence in protein-protein and protein-nucleic acid interaction sites", "label": "sequence divergence in protein-protein and protein-nucleic acid interaction sites", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "phage-encoded DNA polymerase gp43 split forms", "label": "phage-encoded DNA polymerase gp43 split forms", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "T4-like phages genetic variability", "label": "T4-like phages genetic variability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "poly(3-hydroxybutyrate", "label": "poly(3-hydroxybutyrate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "glucose", "label": "glucose", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "gluconate", "label": "gluconate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "frutose", "label": "frutose", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "molasses", "label": "molasses", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "poly(3-hydroxybutyrate) accumulation", "label": "poly(3-hydroxybutyrate) accumulation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cell growth", "label": "cell growth", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "PHA polymerase activity", "label": "PHA polymerase activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "beta-ketothiolase activity", "label": "beta-ketothiolase activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "acetoacetyl-Coenzyme A reductase activity", "label": "acetoacetyl-Coenzyme A reductase activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "de novo fatty acid synthesis", "label": "de novo fatty acid synthesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cerulenin", "label": "cerulenin", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "solar radiation", "label": "solar radiation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "pyrimidine dimers formation in calf-thymus DNA", "label": "pyrimidine dimers formation in calf-thymus DNA", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "pyrimidine dimers", "label": "pyrimidine dimers", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "decay rates of viral infectivity", "label": "decay rates of viral infectivity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "depth of water column", "label": "depth of water column", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "decay of solar radiation", "label": "decay of solar radiation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Solar ultraviolet radiation", "label": "Solar ultraviolet radiation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cell damage", "label": "cell damage", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "DNA repair", "label": "DNA repair", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Reduction in the earth\u0027s stratospheric ozone layer", "label": "Reduction in the earth\u0027s stratospheric ozone layer", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increased solar ultraviolet radiation effects", "label": "increased solar ultraviolet radiation effects", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "RecA protein", "label": "RecA protein", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "DNA damage repair indication", "label": "DNA damage repair indication", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "regulation of Dark Repair Systems", "label": "regulation of Dark Repair Systems", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Dark Repair Systems", "label": "Dark Repair Systems", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "excision repair", "label": "excision repair", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "postreplication recombinational repair", "label": "postreplication recombinational repair", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mutagenic or SOS repair", "label": "mutagenic or SOS repair", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Solar UVB radiation", "label": "Solar UVB radiation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reduced Vibrio natriegens viability", "label": "reduced Vibrio natriegens viability", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Solar UVA radiation", "label": "Solar UVA radiation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Unfiltered solar radiation exposure", "label": "Unfiltered solar radiation exposure", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reduced Vibrio natriegens survival", "label": "reduced Vibrio natriegens survival", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "UVB blocked radiation exposure", "label": "UVB blocked radiation exposure", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Visible light exposure", "label": "Visible light exposure", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "no effect on Vibrio natriegens survival", "label": "no effect on Vibrio natriegens survival", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Solar radiation", "label": "Solar radiation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "induced RecA protein expression", "label": "induced RecA protein expression", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "UVB-induced cyclobutyl pyrimidine dimers", "label": "UVB-induced cyclobutyl pyrimidine dimers", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "correlation with RecA expression", "label": "correlation with RecA expression", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "UVA radiation", "label": "UVA radiation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "RecA-regulated repair", "label": "RecA-regulated repair", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "survival after solar ultraviolet radiation exposure", "label": "survival after solar ultraviolet radiation exposure", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "RecA expression", "label": "RecA expression", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "monitor of bacterial repair after solar UVR exposure", "label": "monitor of bacterial repair after solar UVR exposure", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high rate of protein synthesis", "label": "high rate of protein synthesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in ribosome numbers", "label": "increase in ribosome numbers", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increasing growth rate", "label": "increasing growth rate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "large number of rRNA operons", "label": "large number of rRNA operons", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens rRNA promoters", "label": "Vibrio natriegens rRNA promoters", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "extremely strong", "label": "extremely strong", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens rRNA core promoters", "label": "Vibrio natriegens rRNA core promoters", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "active in vitro", "label": "active in vitro", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens RNAP", "label": "Vibrio natriegens RNAP", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "activation by UP elements", "label": "activation by UP elements", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Escherichia coli transcription factor Fis", "label": "Escherichia coli transcription factor Fis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "activated Vibrio natriegens rrn P1 promoters in vitro", "label": "activated Vibrio natriegens rrn P1 promoters in vitro", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high capacity for rRNA transcription", "label": "high capacity for rRNA transcription", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high capacity for ribosome synthesis in Vibrio natriegens", "label": "high capacity for ribosome synthesis in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high capacity for rRNA transcription in Vibrio natriegens", "label": "high capacity for rRNA transcription in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high gene dose", "label": "high gene dose", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio parahaemolyticus", "label": "Vibrio parahaemolyticus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "RNase P RNA structure", "label": "RNase P RNA structure", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio carchariae", "label": "Vibrio carchariae", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio proteolyticus", "label": "Vibrio proteolyticus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio pelagius", "label": "Vibrio pelagius", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "P3 helices", "label": "P3 helices", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "P12 helices", "label": "P12 helices", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "tandem repeats", "label": "tandem repeats", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "palindromic sequence", "label": "palindromic sequence", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "stem structure", "label": "stem structure", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "repetition number", "label": "repetition number", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "four repetitions of P3 helices", "label": "four repetitions of P3 helices", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "RNase P RNA genes", "label": "RNase P RNA genes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "location between two open reading frames", "label": "location between two open reading frames", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "similarity to hypothetical proteins in Escherichia coli", "label": "similarity to hypothetical proteins in Escherichia coli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio anguillarum", "label": "Vibrio anguillarum", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Little or no growth of trophozoites", "label": "Little or no growth of trophozoites", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Aeromonas salmonicida", "label": "Aeromonas salmonicida", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Streptococcus sp.", "label": "Streptococcus sp.", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Pasteurella piscicida", "label": "Pasteurella piscicida", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio vulnificus", "label": "Vibrio vulnificus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio campbelli", "label": "Vibrio campbelli", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio fluvialis", "label": "Vibrio fluvialis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Pseudomonas dondorofii", "label": "Pseudomonas dondorofii", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Aeromonas hydrophila", "label": "Aeromonas hydrophila", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Growth of trophozoites", "label": "Growth of trophozoites", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Paraphysomonas imperforata", "label": "Paraphysomonas imperforata", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Pteridomonas danica", "label": "Pteridomonas danica", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Cafeteria roenbergensis", "label": "Cafeteria roenbergensis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "incubation temperature", "label": "incubation temperature", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "timing of the maximum of flagellate numbers", "label": "timing of the maximum of flagellate numbers", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "concentrations at the time of maximum flagellate numbers", "label": "concentrations at the time of maximum flagellate numbers", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "gross growth efficiency", "label": "gross growth efficiency", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "effectiveness as grazer", "label": "effectiveness as grazer", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "threshold bacterial concentrations", "label": "threshold bacterial concentrations", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "percentage of Vibrio natriegens consumed", "label": "percentage of Vibrio natriegens consumed", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "peak concentrations of flagellates", "label": "peak concentrations of flagellates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "numbers of flagellates produced", "label": "numbers of flagellates produced", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "numbers of bacteria eaten", "label": "numbers of bacteria eaten", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bacterial concentration", "label": "bacterial concentration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "time to reach peak flagellate numbers", "label": "time to reach peak flagellate numbers", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "flagellate species", "label": "flagellate species", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "niche specialization", "label": "niche specialization", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "diversity", "label": "diversity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "16S rRNA sequence analyses", "label": "16S rRNA sequence analyses", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio pelagius strains CECT 4202T", "label": "Vibrio pelagius strains CECT 4202T", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio pelagius strains ATCC 25916T", "label": "Vibrio pelagius strains ATCC 25916T", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "genomic DNA-DNA similarity studies", "label": "genomic DNA-DNA similarity studies", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "physiological investigations", "label": "physiological investigations", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "originally described type species", "label": "originally described type species", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio pelagius strains NCIMB 1900T", "label": "Vibrio pelagius strains NCIMB 1900T", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "liquid flow velocity", "label": "liquid flow velocity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mass transfer in the diffusion boundary layer", "label": "mass transfer in the diffusion boundary layer", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "biomass detachment rate from the biofilm", "label": "biomass detachment rate from the biofilm", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "maximum maximum biofilm thickness", "label": "maximum maximum biofilm thickness", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "overall mass transfer coefficient", "label": "overall mass transfer coefficient", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "biofilm structure", "label": "biofilm structure", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "maximum biofilm thickness", "label": "maximum biofilm thickness", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reaction and diffusion", "label": "reaction and diffusion", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "effective diffusivity", "label": "effective diffusivity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "yield of biomass on acetate", "label": "yield of biomass on acetate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "endogenous respiration", "label": "endogenous respiration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "yield coefficients", "label": "yield coefficients", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "acetate uptake rates", "label": "acetate uptake rates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mathematical model", "label": "mathematical model", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "oxygen uptake rates", "label": "oxygen uptake rates", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "UV-B radiation", "label": "UV-B radiation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "survival of marine bacteria", "label": "survival of marine bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cyclobutane pyrimidine dimers", "label": "cyclobutane pyrimidine dimers", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Sphingomonas sp. strain RB2256", "label": "Sphingomonas sp. strain RB2256", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "little DNA damage", "label": "little DNA damage", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high level of UV-B resistance", "label": "high level of UV-B resistance", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "growth phase", "label": "growth phase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "UV-B sensitivity of marine bacteria", "label": "UV-B sensitivity of marine bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "starvation", "label": "starvation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "UV-B exposure", "label": "UV-B exposure", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "rates of photoreactivation restoration of infectivity", "label": "rates of photoreactivation restoration of infectivity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "UV-A exposure", "label": "UV-A exposure", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cool white light exposure", "label": "cool white light exposure", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "microbial community structure in surface water", "label": "microbial community structure in surface water", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "survival and CPD data", "label": "survival and CPD data", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "photoreactivation restoration of infectivity plus dark repair", "label": "photoreactivation restoration of infectivity plus dark repair", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "photoadaptation", "label": "photoadaptation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "X-ray microanalysis", "label": "X-ray microanalysis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "magnesium concentration", "label": "magnesium concentration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sodium concentration", "label": "sodium concentration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "potassium to sodium ratio", "label": "potassium to sodium ratio", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "magnesium", "label": "magnesium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "cell turgor", "label": "cell turgor", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "chloride", "label": "chloride", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "high internal chloride concentration", "label": "high internal chloride concentration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "dominating cation in marine bacterioplankton", "label": "dominating cation in marine bacterioplankton", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "adaptation in actively growing marine bacteria", "label": "adaptation in actively growing marine bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "sunlight-damaged natural viral communities", "label": "sunlight-damaged natural viral communities", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "photoreactivation restoration of infectivity", "label": "photoreactivation restoration of infectivity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "restoration of infectivity to natural viral communities", "label": "restoration of infectivity to natural viral communities", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "\u003e 312-nm-wavelength sunlight", "label": "\u003e 312-nm-wavelength sunlight", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "restoration of infectivity to sunlight-damaged viruses in oceanic waters", "label": "restoration of infectivity to sunlight-damaged viruses in oceanic waters", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "370 to 550 nm wavelengths", "label": "370 to 550 nm wavelengths", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "restoration of infectivity to damaged viruses", "label": "restoration of infectivity to damaged viruses", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "photoreactivation", "label": "photoreactivation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "compensation for sunlight-induced DNA damage in natural viral communities", "label": "compensation for sunlight-induced DNA damage in natural viral communities", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "aerobic N-replete conditions", "label": "aerobic N-replete conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "anaerobic N-deplete conditions", "label": "anaerobic N-deplete conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "transcriptional regulation of N2 fixation", "label": "transcriptional regulation of N2 fixation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "post-translational regulation of N2 fixation", "label": "post-translational regulation of N2 fixation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nifHDK mRNA", "label": "nifHDK mRNA", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nitrogenase enzyme", "label": "nitrogenase enzyme", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nifHDK mRNA levels highest at 140 min postshift", "label": "nifHDK mRNA levels highest at 140 min postshift", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nitrogenase enzyme levels highest between 6 and 15 h postshift", "label": "nitrogenase enzyme levels highest between 6 and 15 h postshift", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nifHDK mRNA undetectable between 6 and 9 h", "label": "nifHDK mRNA undetectable between 6 and 9 h", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nitrogenase activity peak between 6 and 9 h postshift", "label": "nitrogenase activity peak between 6 and 9 h postshift", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nifHDK mRNA absent", "label": "nifHDK mRNA absent", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nitrogenase enzyme presence in Vibrio natriegens", "label": "nitrogenase enzyme presence in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "anaerobic conditions", "label": "anaerobic conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nitrogenase stability in Vibrio natriegens", "label": "nitrogenase stability in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "N2-fixing conditions to non-N2-fixing conditions", "label": "N2-fixing conditions to non-N2-fixing conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "disappearance of nifHDK mRNA", "label": "disappearance of nifHDK mRNA", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "disappearance of Fe protein", "label": "disappearance of Fe protein", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "non-N2-fixing conditions to N2-fixing conditions", "label": "non-N2-fixing conditions to N2-fixing conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reappearance of nifHDK mRNA", "label": "reappearance of nifHDK mRNA", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "immunological detection", "label": "immunological detection", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "positive correlation with nitrogenase activity", "label": "positive correlation with nitrogenase activity", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens anaerobic induction", "label": "Vibrio natriegens anaerobic induction", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "continued N2 fixation", "label": "continued N2 fixation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Fluorescently stained viruses", "label": "Fluorescently stained viruses", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Specific strains of bacteria identification", "label": "Specific strains of bacteria identification", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Cyanobacteria identification", "label": "Cyanobacteria identification", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "YOYO-1", "label": "YOYO-1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Fluorescent staining", "label": "Fluorescent staining", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "POPO-1", "label": "POPO-1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Host cells", "label": "Host cells", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Quantification efficiency with fluorescently labeled virus probes", "label": "Quantification efficiency with fluorescently labeled virus probes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Strain PWH3a addition", "label": "Strain PWH3a addition", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Increase in virus concentration", "label": "Increase in virus concentration", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Collapse in populations of strain PWH3a", "label": "Collapse in populations of strain PWH3a", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Nutrient-enrichment", "label": "Nutrient-enrichment", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Increase in virus concentration infecting strain PWH3a", "label": "Increase in virus concentration infecting strain PWH3a", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Bacteria quantification in microbial communities", "label": "Bacteria quantification in microbial communities", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Low abundances of viruses", "label": "Low abundances of viruses", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Control over microbial community structure", "label": "Control over microbial community structure", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Viral abundance", "label": "Viral abundance", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Viral direct counts", "label": "Viral direct counts", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens plaque titers", "label": "Vibrio natriegens plaque titers", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Bacterial direct counts", "label": "Bacterial direct counts", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Chlorophyll a measurements", "label": "Chlorophyll a measurements", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Sediment viral parameters", "label": "Sediment viral parameters", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Water column microbial data", "label": "Water column microbial data", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Water column viral data", "label": "Water column viral data", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Water column salinity data", "label": "Water column salinity data", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Coliphage presence", "label": "Coliphage presence", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Fecal pollution indicators", "label": "Fecal pollution indicators", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "comparative analysis of the 16S rRNAs", "label": "comparative analysis of the 16S rRNAs", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "inter- and intrageneric relationships of the genus Vibrio", "label": "inter- and intrageneric relationships of the genus Vibrio", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "neighboring taxon of the family Enterobacteriaceae", "label": "neighboring taxon of the family Enterobacteriaceae", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "core of the genus Vibrio", "label": "core of the genus Vibrio", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio (Listonella) anguillarum", "label": "Vibrio (Listonella) anguillarum", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "outskirts of the genus Vibrio", "label": "outskirts of the genus Vibrio", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio diazotrophicus", "label": "Vibrio diazotrophicus", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio hollisae", "label": "Vibrio hollisae", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "variable regions around positions 80", "label": "variable regions around positions 80", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "180, and 450, target sites for genus- and species-specific oligonucleotide probes", "label": "180, and 450, target sites for genus- and species-specific oligonucleotide probes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "180, and 450, target sites for polymerase chain reaction primers", "label": "180, and 450, target sites for polymerase chain reaction primers", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Fourier transform-infrared spectroscopy", "label": "Fourier transform-infrared spectroscopy", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bacteria-polymer mixtures", "label": "bacteria-polymer mixtures", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "digester samples", "label": "digester samples", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "microbial biofilms", "label": "microbial biofilms", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Diffuse reflectance Fourier transform-infrared analysis", "label": "Diffuse reflectance Fourier transform-infrared analysis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "structural information", "label": "structural information", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "dominant amide I band", "label": "dominant amide I band", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "strong carbonyl stretch at approximately 1740 centimeters-1", "label": "strong carbonyl stretch at approximately 1740 centimeters-1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "differences in microbial community structure", "label": "differences in microbial community structure", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "specific band ratios", "label": "specific band ratios", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "composition of bacteria", "label": "composition of bacteria", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "composition of extracellular or intracellular storage product polymer mixtures", "label": "composition of extracellular or intracellular storage product polymer mixtures", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bacteria-gum arabic", "label": "bacteria-gum arabic", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "amide I/carbohydrate C-O approximately 1150 centimeters-1", "label": "amide I/carbohydrate C-O approximately 1150 centimeters-1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bacteria-poly-beta-hydroxybutyrate", "label": "bacteria-poly-beta-hydroxybutyrate", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "amide I/carbonyl approximately 1740 centimeters-1", "label": "amide I/carbonyl approximately 1740 centimeters-1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "key band ratios", "label": "key band ratios", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "compositions of the material", "label": "compositions of the material", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "bacteria grown under differing nutrient conditions", "label": "bacteria grown under differing nutrient conditions", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Vibrio natriegens biofilm", "label": "Vibrio natriegens biofilm", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in bands at approximately 1440 and 1090 centimeters-1", "label": "increase in bands at approximately 1440 and 1090 centimeters-1", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "flow-through attenuated total reflectance", "label": "flow-through attenuated total reflectance", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "real-time changes in biofilm formation", "label": "real-time changes in biofilm formation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "interactions within adherent microbial consortia", "label": "interactions within adherent microbial consortia", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Alteromonas haloplanktis 214", "label": "Alteromonas haloplanktis 214", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Na+", "label": "Na+", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Li+", "label": "Li+", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Toluene-treated cells", "label": "Toluene-treated cells", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "K+ sparing requirement requirement", "label": "K+ sparing requirement requirement", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Alteromonas haloplanktis", "label": "Alteromonas haloplanktis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Na+ requirement for alpha-aminoisobutyric acid uptake", "label": "Na+ requirement for alpha-aminoisobutyric acid uptake", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Photobacterium phosphoreum", "label": "Photobacterium phosphoreum", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "K+", "label": "K+", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Na+ activation", "label": "Na+ activation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "K+ sparing requirement", "label": "K+ sparing requirement", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "temperature", "label": "temperature", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "longer latent periods", "label": "longer latent periods", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reduced burst sizes", "label": "reduced burst sizes", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "reduced burst size for phage nt-6", "label": "reduced burst size for phage nt-6", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Na(+) presence ", "label": "Na(+) presence ", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "significant quantity of (14", "label": "significant quantity of (14", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "K(+) presence ", "label": "K(+) presence ", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mannitol minimal culture medium proliferation", "label": "mannitol minimal culture medium proliferation", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "protein synthesis increase", "label": "protein synthesis increase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "deoxyribonucleic acid quantity increase", "label": "deoxyribonucleic acid quantity increase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "ribonucleic acid quantity increase", "label": "ribonucleic acid quantity increase", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "significant amount of mannitol-binding protein synthesis", "label": "significant amount of mannitol-binding protein synthesis", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Na(+) absence ", "label": "Na(+) absence ", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "0.3 m K(+) presence ", "label": "0.3 m K(+) presence ", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mannitol presence ", "label": "mannitol presence ", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "nicotinamide adenine dinucleotide-linked mannitol-1-phosphate dehydrogenase presence", "label": "nicotinamide adenine dinucleotide-linked mannitol-1-phosphate dehydrogenase presence", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "broth-grown cells ", "label": "broth-grown cells ", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mannitol dehydrogenase absence", "label": "mannitol dehydrogenase absence", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mannitol-grown cells ", "label": "mannitol-grown cells ", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "ability to take up 14C-labeled mannitol", "label": "ability to take up 14C-labeled mannitol", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Na(+)", "label": "Na(+)", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "ability to take up 14C-labeled mannitol in Vibrio natriegens", "label": "ability to take up 14C-labeled mannitol in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "K(+)", "label": "K(+)", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "0.3 m K(+)", "label": "0.3 m K(+)", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "significant quantity of label uptake in Vibrio natriegens in absence of Na(+", "label": "significant quantity of label uptake in Vibrio natriegens in absence of Na(+", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "proliferation of Vibrio natriegens in mannitol minimal culture medium", "label": "proliferation of Vibrio natriegens in mannitol minimal culture medium", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in protein synthesis in Vibrio natriegens", "label": "increase in protein synthesis in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in deoxyribonucleic acid synthesis in Vibrio natriegens", "label": "increase in deoxyribonucleic acid synthesis in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "increase in ribonucleic acid synthesis in Vibrio natriegens", "label": "increase in ribonucleic acid synthesis in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "synthesis of mannitolbinding protein in Vibrio natriegens", "label": "synthesis of mannitolbinding protein in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "small quantity of mannitolbinding protein produced in Vibrio natriegens in absence of Na(+", "label": "small quantity of mannitolbinding protein produced in Vibrio natriegens in absence of Na(+", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mannitol and Na(+) and K(+)", "label": "mannitol and Na(+) and K(+)", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mannitol and 0.3 m K(+) but no Na(+)", "label": "mannitol and 0.3 m K(+) but no Na(+)", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "small quantity of mannitolbinding protein produced in Vibrio natriegens", "label": "small quantity of mannitolbinding protein produced in Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "mannitol", "label": "mannitol", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "presence of nicotinamide adenine dinucleotide-linked mannitol-1-phosphate dehydrogenase in mannitol-grown Vibrio natriegens", "label": "presence of nicotinamide adenine dinucleotide-linked mannitol-1-phosphate dehydrogenase in mannitol-grown Vibrio natriegens", "shape": "dot"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "absence of mannitol dehydrogenase in mannitol-grown Vibrio natriegens", "label": "absence of mannitol dehydrogenase in mannitol-grown Vibrio natriegens", "shape": "dot"}]);
                  edges = new vis.DataSet([{"from": "C-type cytochromes", "title": "Vibrio natriegens as a superior host for the production of c-type cytochromes and difficult-to-express redox proteins.", "to": "aerobic respiration"}, {"from": "C-type cytochromes", "title": "Vibrio natriegens as a superior host for the production of c-type cytochromes and difficult-to-express redox proteins.", "to": "anaerobic respiration"}, {"from": "Heterologous production", "title": "Vibrio natriegens as a superior host for the production of c-type cytochromes and difficult-to-express redox proteins.", "to": "C-type cytochromes"}, {"from": "Escherichia coli", "title": "Vibrio natriegens as a superior host for the production of c-type cytochromes and difficult-to-express redox proteins.", "to": "Heterologous production limitations"}, {"from": "ccmABCDEFGH genes expression", "title": "Vibrio natriegens as a superior host for the production of c-type cytochromes and difficult-to-express redox proteins.", "to": "Anaerobic conditions"}, {"from": "pEC86 vector", "title": "Vibrio natriegens as a superior host for the production of c-type cytochromes and difficult-to-express redox proteins.", "to": "ccm genes co-expression"}, {"from": "Co-expression methods", "title": "Vibrio natriegens as a superior host for the production of c-type cytochromes and difficult-to-express redox proteins.", "to": "Expression host restriction"}, {"from": "Co-expression methods", "title": "Vibrio natriegens as a superior host for the production of c-type cytochromes and difficult-to-express redox proteins.", "to": "Vector choice restriction"}, {"from": "Vibrio natriegens Vmax X2", "title": "Vibrio natriegens as a superior host for the production of c-type cytochromes and difficult-to-express redox proteins.", "to": "Recombinant production"}, {"from": "Acidithiobacillus ferrooxidans CCM4253", "title": "Vibrio natriegens as a superior host for the production of c-type cytochromes and difficult-to-express redox proteins.", "to": "Recombinant production"}, {"from": "Co-expression of ccm genes", "title": "Vibrio natriegens as a superior host for the production of c-type cytochromes and difficult-to-express redox proteins.", "to": "Holo-c-type cytochromes production necessity"}, {"from": "Escherichia coli T7 Express", "title": "Vibrio natriegens as a superior host for the production of c-type cytochromes and difficult-to-express redox proteins.", "to": "Holo-c-type cytochromes co-expression production"}, {"from": "Escherichia coli T7 Express", "title": "Vibrio natriegens as a superior host for the production of c-type cytochromes and difficult-to-express redox proteins.", "to": "Inner membrane cytochrome CycA production inability"}, {"from": "Vibrio natriegens Vmax X2 cell extracts", "title": "Vibrio natriegens as a superior host for the production of c-type cytochromes and difficult-to-express redox proteins.", "to": "Recombinant holo-proteins quantity"}, {"from": "T7 Express cell extracts", "title": "Vibrio natriegens as a superior host for the production of c-type cytochromes and difficult-to-express redox proteins.", "to": "Recombinant holo-proteins quantity"}, {"from": "Redox proteins", "title": "Vibrio natriegens as a superior host for the production of c-type cytochromes and difficult-to-express redox proteins.", "to": "Intended cell compartment translocation"}, {"from": "Vibrio natriegens", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for High-Yield Production of Challenging Expression Targets and for Protein Perdeuteration.", "to": "soluble proteins"}, {"from": "Vmax X2", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for High-Yield Production of Challenging Expression Targets and for Protein Perdeuteration.", "to": "bacterial and fungal proteins in milligram scale"}, {"from": "bacterial and fungal proteins in milligram scale", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for High-Yield Production of Challenging Expression Targets and for Protein Perdeuteration.", "to": "cholera toxin"}, {"from": "bacterial and fungal proteins in milligram scale", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for High-Yield Production of Challenging Expression Targets and for Protein Perdeuteration.", "to": "N-acetyl glucosamine-binding protein A"}, {"from": "bacterial and fungal proteins in milligram scale", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for High-Yield Production of Challenging Expression Targets and for Protein Perdeuteration.", "to": "heat-labile enterotoxin"}, {"from": "bacterial and fungal proteins in milligram scale", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for High-Yield Production of Challenging Expression Targets and for Protein Perdeuteration.", "to": "fungal nematotoxin CCTX2"}, {"from": "Type II secretion system", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for High-Yield Production of Challenging Expression Targets and for Protein Perdeuteration.", "to": "secretion of cholera toxin"}, {"from": "Type II secretion system", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for High-Yield Production of Challenging Expression Targets and for Protein Perdeuteration.", "to": "secretion of N-acetyl glucosamine-binding protein A"}, {"from": "Type II secretion system", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for High-Yield Production of Challenging Expression Targets and for Protein Perdeuteration.", "to": "secretion of heat-labile enterotoxin"}, {"from": "Vmax X2", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for High-Yield Production of Challenging Expression Targets and for Protein Perdeuteration.", "to": "secretion of cholera toxin"}, {"from": "Vmax X2", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for High-Yield Production of Challenging Expression Targets and for Protein Perdeuteration.", "to": "secretion of N-acetyl glucosamine-binding protein A"}, {"from": "Vmax X2", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for High-Yield Production of Challenging Expression Targets and for Protein Perdeuteration.", "to": "secretion of heat-labile enterotoxin"}, {"from": "secretion of cholera toxin", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for High-Yield Production of Challenging Expression Targets and for Protein Perdeuteration.", "to": "recovery from growth media"}, {"from": "secretion of N-acetyl glucosamine-binding protein A", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for High-Yield Production of Challenging Expression Targets and for Protein Perdeuteration.", "to": "recovery from growth media"}, {"from": "secretion of heat-labile enterotoxin", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for High-Yield Production of Challenging Expression Targets and for Protein Perdeuteration.", "to": "recovery from growth media"}, {"from": "Vmax X2", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for High-Yield Production of Challenging Expression Targets and for Protein Perdeuteration.", "to": "higher Protein yields compared to Escherichia coli"}, {"from": "Vmax X2", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for High-Yield Production of Challenging Expression Targets and for Protein Perdeuteration.", "to": "protein perdeuteration in amounts suitable for structural biology studies"}, {"from": "deuterated minimal media with deuterium oxide", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for High-Yield Production of Challenging Expression Targets and for Protein Perdeuteration.", "to": "3-fold increase in yield compared to Escherichia coli"}, {"from": "Vibrio natriegens", "title": "Efficient natural plasmid transformation of \u003ci\u003eVibrio natriegens\u003c/i\u003e enables zero-capital molecular biology.", "to": "natural transformation"}, {"from": "master competence regulator tfoX from Vibrio cholerae", "title": "Efficient natural plasmid transformation of \u003ci\u003eVibrio natriegens\u003c/i\u003e enables zero-capital molecular biology.", "to": "naturally competent cells"}, {"from": "new minimal competence media", "title": "Efficient natural plasmid transformation of \u003ci\u003eVibrio natriegens\u003c/i\u003e enables zero-capital molecular biology.", "to": "naturally competent cells"}, {"from": "new minimal competence media", "title": "Efficient natural plasmid transformation of \u003ci\u003eVibrio natriegens\u003c/i\u003e enables zero-capital molecular biology.", "to": "uses acetate as an energy source"}, {"from": "naturally competent cells", "title": "Efficient natural plasmid transformation of \u003ci\u003eVibrio natriegens\u003c/i\u003e enables zero-capital molecular biology.", "to": "transformed"}, {"from": "naturally competent cells", "title": "Efficient natural plasmid transformation of \u003ci\u003eVibrio natriegens\u003c/i\u003e enables zero-capital molecular biology.", "to": "recovered"}, {"from": "naturally competent cells", "title": "Efficient natural plasmid transformation of \u003ci\u003eVibrio natriegens\u003c/i\u003e enables zero-capital molecular biology.", "to": "competent to plasmids"}, {"from": "naturally competent cells", "title": "Efficient natural plasmid transformation of \u003ci\u003eVibrio natriegens\u003c/i\u003e enables zero-capital molecular biology.", "to": "recombination with linear DNA"}, {"from": "naturally competent cells", "title": "Efficient natural plasmid transformation of \u003ci\u003eVibrio natriegens\u003c/i\u003e enables zero-capital molecular biology.", "to": "cotransformation of plasmids and linear DNA"}, {"from": "zero capital protocol", "title": "Efficient natural plasmid transformation of \u003ci\u003eVibrio natriegens\u003c/i\u003e enables zero-capital molecular biology.", "to": "transformation process at room temperature range"}, {"from": "high-efficiency protocol", "title": "Efficient natural plasmid transformation of \u003ci\u003eVibrio natriegens\u003c/i\u003e enables zero-capital molecular biology.", "to": "transformation process with incubator"}, {"from": "naturally competent cells", "title": "Efficient natural plasmid transformation of \u003ci\u003eVibrio natriegens\u003c/i\u003e enables zero-capital molecular biology.", "to": "retain naturally competent state when frozen"}, {"from": "naturally competent cells", "title": "Efficient natural plasmid transformation of \u003ci\u003eVibrio natriegens\u003c/i\u003e enables zero-capital molecular biology.", "to": "transformable immediately upon thawing"}, {"from": "optimized transformation protocol", "title": "Efficient natural plasmid transformation of \u003ci\u003eVibrio natriegens\u003c/i\u003e enables zero-capital molecular biology.", "to": "requires 50 minutes of hands-on time"}, {"from": "Vibrio natriegens", "title": "Efficient natural plasmid transformation of \u003ci\u003eVibrio natriegens\u003c/i\u003e enables zero-capital molecular biology.", "to": "grows quickly even on plates"}, {"from": "natural competence strain and media", "title": "Efficient natural plasmid transformation of \u003ci\u003eVibrio natriegens\u003c/i\u003e enables zero-capital molecular biology.", "to": "ideal for automated directed evolution applications"}, {"from": "naturally competent Vibrio natriegens", "title": "Efficient natural plasmid transformation of \u003ci\u003eVibrio natriegens\u003c/i\u003e enables zero-capital molecular biology.", "to": "could compete with Escherichia coli"}, {"from": "Vibrio natriegens", "title": "Ultrafast removal of toxic Cr(VI) by the marine bacterium Vibrio natriegens.", "to": "bioproduction processes"}, {"from": "Vibrio natriegens", "title": "Ultrafast removal of toxic Cr(VI) by the marine bacterium Vibrio natriegens.", "to": "biomass production"}, {"from": "Vibrio natriegens", "title": "Ultrafast removal of toxic Cr(VI) by the marine bacterium Vibrio natriegens.", "to": "similar extracellular electron transfer pathway to electroactive organisms to Iron (III"}, {"from": "porin-cytochrome circuit", "title": "Ultrafast removal of toxic Cr(VI) by the marine bacterium Vibrio natriegens.", "to": "similar extracellular electron transfer pathway to electroactive organisms to Iron (III"}, {"from": "electroactive microbes", "title": "Ultrafast removal of toxic Cr(VI) by the marine bacterium Vibrio natriegens.", "to": "reduce toxic metals"}, {"from": "carcinogenic Chromium (VI)", "title": "Ultrafast removal of toxic Cr(VI) by the marine bacterium Vibrio natriegens.", "to": "converted to Chromium (III"}, {"from": "Vibrio natriegens", "title": "Ultrafast removal of toxic Cr(VI) by the marine bacterium Vibrio natriegens.", "to": "bioremediation of Chromium (VI"}, {"from": "Vibrio natriegens density", "title": "Ultrafast removal of toxic Cr(VI) by the marine bacterium Vibrio natriegens.", "to": "Chromium (VI"}, {"from": "Escherichia coli", "title": "Ultrafast removal of toxic Cr(VI) by the marine bacterium Vibrio natriegens.", "to": "slower Chromium (VI"}, {"from": "Vibrio natriegens metabolism", "title": "Ultrafast removal of toxic Cr(VI) by the marine bacterium Vibrio natriegens.", "to": "Chromium (VI"}, {"from": "outer-membrane c-type cytochromes", "title": "Ultrafast removal of toxic Cr(VI) by the marine bacterium Vibrio natriegens.", "to": "Chromium (VI"}, {"from": "Vibrio natriegens", "title": "Ultrafast removal of toxic Cr(VI) by the marine bacterium Vibrio natriegens.", "to": "Chromium (III"}, {"from": "Vibrio natriegens", "title": "Ultrafast removal of toxic Cr(VI) by the marine bacterium Vibrio natriegens.", "to": "adsorption of metallic ions"}, {"from": "Chromosome rearrangement by LoxP-mediated evolution", "title": "Chromosome recombination and modification by LoxP-mediated evolution in Vibrio natriegens using CRISPR-associated transposases.", "to": "Chromosome architecture"}, {"from": "Chromosome architecture", "title": "Chromosome recombination and modification by LoxP-mediated evolution in Vibrio natriegens using CRISPR-associated transposases.", "to": "Phenotypes"}, {"from": "CRISPR-associated transposases", "title": "Chromosome recombination and modification by LoxP-mediated evolution in Vibrio natriegens using CRISPR-associated transposases.", "to": "Insertion of LoxP sites"}, {"from": "LoxP sites", "title": "Chromosome recombination and modification by LoxP-mediated evolution in Vibrio natriegens using CRISPR-associated transposases.", "to": "Genome of wild-type strains"}, {"from": "Multiplex genome integration tool derived from CRISPR-associated transposases", "title": "Chromosome recombination and modification by LoxP-mediated evolution in Vibrio natriegens using CRISPR-associated transposases.", "to": "Insertion of cargo genes"}, {"from": "Insertion of cargo genes", "title": "Chromosome recombination and modification by LoxP-mediated evolution in Vibrio natriegens using CRISPR-associated transposases.", "to": "Genomic loci"}, {"from": "30 LoxP sites", "title": "Chromosome recombination and modification by LoxP-mediated evolution in Vibrio natriegens using CRISPR-associated transposases.", "to": "Chromosome 2 of Vibrio natrigens"}, {"from": "Rigorously induced Cre recombinase", "title": "Chromosome recombination and modification by LoxP-mediated evolution in Vibrio natriegens using CRISPR-associated transposases.", "to": "Chromosome Rearrangement and Modification by LoxP-mediated Evolution"}, {"from": "Chromosome Rearrangement and Modification by LoxP-mediated Evolution", "title": "Chromosome recombination and modification by LoxP-mediated evolution in Vibrio natriegens using CRISPR-associated transposases.", "to": "Growth characterization"}, {"from": "Growth characterization", "title": "Chromosome recombination and modification by LoxP-mediated evolution in Vibrio natriegens using CRISPR-associated transposases.", "to": "~358 kilobase fragment on Chromosome 2"}, {"from": "~358 kilobase fragment on Chromosome 2", "title": "Chromosome recombination and modification by LoxP-mediated evolution in Vibrio natriegens using CRISPR-associated transposases.", "to": "Rapid growth of Vibrio natrigens"}, {"from": "Enabling tools", "title": "Chromosome recombination and modification by LoxP-mediated evolution in Vibrio natriegens using CRISPR-associated transposases.", "to": "Identification of genomic regions influencing rapid growth"}, {"from": "Genomic regions influencing rapid growth", "title": "Chromosome recombination and modification by LoxP-mediated evolution in Vibrio natriegens using CRISPR-associated transposases.", "to": "Rapid growth of Vibrio natrigens"}, {"from": "Demonstration", "title": "Chromosome recombination and modification by LoxP-mediated evolution in Vibrio natriegens using CRISPR-associated transposases.", "to": "Extension to other organisms"}, {"from": "Other organisms", "title": "Chromosome recombination and modification by LoxP-mediated evolution in Vibrio natriegens using CRISPR-associated transposases.", "to": "Escherichia coli"}, {"from": "Other organisms", "title": "Chromosome recombination and modification by LoxP-mediated evolution in Vibrio natriegens using CRISPR-associated transposases.", "to": "Pseudomonas putida"}, {"from": "Other organisms", "title": "Chromosome recombination and modification by LoxP-mediated evolution in Vibrio natriegens using CRISPR-associated transposases.", "to": "Bacillus subtilis"}, {"from": "Xenobiotic agrochemical leaching", "title": "Unveiling bacterial consortium for xenobiotic biodegradation from Pichavaram mangrove forest soil: a metagenomic approach.", "to": "Threaten microbial diversity"}, {"from": "Shotgun sequencing", "title": "Unveiling bacterial consortium for xenobiotic biodegradation from Pichavaram mangrove forest soil: a metagenomic approach.", "to": "Pichavaram mangrove metagenomic dataset"}, {"from": "SqueezeMeta", "title": "Unveiling bacterial consortium for xenobiotic biodegradation from Pichavaram mangrove forest soil: a metagenomic approach.", "to": "Pichavaram mangrove metagenomic dataset"}, {"from": "DIAMOND database", "title": "Unveiling bacterial consortium for xenobiotic biodegradation from Pichavaram mangrove forest soil: a metagenomic approach.", "to": "Taxonomical classification of microbes"}, {"from": "KEGG database", "title": "Unveiling bacterial consortium for xenobiotic biodegradation from Pichavaram mangrove forest soil: a metagenomic approach.", "to": "Confirmed xenobiotic degradation pathways"}, {"from": "Microbial consortia", "title": "Unveiling bacterial consortium for xenobiotic biodegradation from Pichavaram mangrove forest soil: a metagenomic approach.", "to": "Degradation of fluorobenzoate"}, {"from": "Microbial consortia", "title": "Unveiling bacterial consortium for xenobiotic biodegradation from Pichavaram mangrove forest soil: a metagenomic approach.", "to": "Degradation of furfural"}, {"from": "Microbial consortia", "title": "Unveiling bacterial consortium for xenobiotic biodegradation from Pichavaram mangrove forest soil: a metagenomic approach.", "to": "Degradation of steroids"}, {"from": "Marinobacter", "title": "Unveiling bacterial consortium for xenobiotic biodegradation from Pichavaram mangrove forest soil: a metagenomic approach.", "to": "Degradation of fluorobenzoate"}, {"from": "Methyloceanibacter", "title": "Unveiling bacterial consortium for xenobiotic biodegradation from Pichavaram mangrove forest soil: a metagenomic approach.", "to": "Degradation of fluorobenzoate"}, {"from": "Vibrio natriegens", "title": "Unveiling bacterial consortium for xenobiotic biodegradation from Pichavaram mangrove forest soil: a metagenomic approach.", "to": "Degradation of fluorobenzoate"}, {"from": "Gramella sp.", "title": "Unveiling bacterial consortium for xenobiotic biodegradation from Pichavaram mangrove forest soil: a metagenomic approach.", "to": "Degradation of fluorobenzoate"}, {"from": "Afipia", "title": "Unveiling bacterial consortium for xenobiotic biodegradation from Pichavaram mangrove forest soil: a metagenomic approach.", "to": "Degradation of furfural"}, {"from": "Nitrosopumilus sp.", "title": "Unveiling bacterial consortium for xenobiotic biodegradation from Pichavaram mangrove forest soil: a metagenomic approach.", "to": "Degradation of furfural"}, {"from": "Phototrophicus methaneseepsis", "title": "Unveiling bacterial consortium for xenobiotic biodegradation from Pichavaram mangrove forest soil: a metagenomic approach.", "to": "Degradation of furfural"}, {"from": "Proteobacteria", "title": "Unveiling bacterial consortium for xenobiotic biodegradation from Pichavaram mangrove forest soil: a metagenomic approach.", "to": "Degradation of steroids"}, {"from": "in vivo in vivo random mutagenesis", "title": "Multiple Rounds of \u003ci\u003eIn Vivo\u003c/i\u003e Random Mutagenesis and Selection in \u003ci\u003eVibrio natriegens\u003c/i\u003e Result in Substantial Increases in REE Binding Capacity.", "to": "increased capacity for biosorbing dysprosium"}, {"from": "in vivo in vivo random mutagenesis", "title": "Multiple Rounds of \u003ci\u003eIn Vivo\u003c/i\u003e Random Mutagenesis and Selection in \u003ci\u003eVibrio natriegens\u003c/i\u003e Result in Substantial Increases in REE Binding Capacity.", "to": "increased selectivity between lanthanum and lutetium"}, {"from": "mutations in ABC transporters", "title": "Multiple Rounds of \u003ci\u003eIn Vivo\u003c/i\u003e Random Mutagenesis and Selection in \u003ci\u003eVibrio natriegens\u003c/i\u003e Result in Substantial Increases in REE Binding Capacity.", "to": "biosorption changes"}, {"from": "mutations in nonessential protein in the BAM outer membrane \u03b2-barrel protein insertion complex", "title": "Multiple Rounds of \u003ci\u003eIn Vivo\u003c/i\u003e Random Mutagenesis and Selection in \u003ci\u003eVibrio natriegens\u003c/i\u003e Result in Substantial Increases in REE Binding Capacity.", "to": "biosorption changes"}, {"from": "thermal strategy", "title": "High-Performance Production of \u003ci\u003eN\u003c/i\u003e-Acetyl-d-Neuraminic Acid with Whole Cells of Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e via a Thermal Strategy.", "to": "high performance production of N-acetyl-d-neuraminic acid"}, {"from": "fast-growing Vibrio natriegens", "title": "High-Performance Production of \u003ci\u003eN\u003c/i\u003e-Acetyl-d-Neuraminic Acid with Whole Cells of Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e via a Thermal Strategy.", "to": "high performance production of N-acetyl-d-neuraminic acid"}, {"from": "thermophilic enzymes", "title": "High-Performance Production of \u003ci\u003eN\u003c/i\u003e-Acetyl-d-Neuraminic Acid with Whole Cells of Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e via a Thermal Strategy.", "to": "high performance production of N-acetyl-d-neuraminic acid"}, {"from": "high-temperature range whole-cell catalysis", "title": "High-Performance Production of \u003ci\u003eN\u003c/i\u003e-Acetyl-d-Neuraminic Acid with Whole Cells of Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e via a Thermal Strategy.", "to": "high performance production of N-acetyl-d-neuraminic acid"}, {"from": "high performance production of N-acetyl-d-neuraminic acid", "title": "High-Performance Production of \u003ci\u003eN\u003c/i\u003e-Acetyl-d-Neuraminic Acid with Whole Cells of Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e via a Thermal Strategy.", "to": "highest Neu5Ac titer"}, {"from": "high performance production of N-acetyl-d-neuraminic acid", "title": "High-Performance Production of \u003ci\u003eN\u003c/i\u003e-Acetyl-d-Neuraminic Acid with Whole Cells of Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e via a Thermal Strategy.", "to": "productivity of 71.6 grams per liter hour"}, {"from": "temperature range elevation", "title": "High-Performance Production of \u003ci\u003eN\u003c/i\u003e-Acetyl-d-Neuraminic Acid with Whole Cells of Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e via a Thermal Strategy.", "to": "elimination of acetic acid"}, {"from": "Vibrio natriegens", "title": "Discovery and remodeling of Vibrio natriegens as a microbial platform for efficient formic acid biorefinery.", "to": "exceptional formic acid tolerance"}, {"from": "Vibrio natriegens", "title": "Discovery and remodeling of Vibrio natriegens as a microbial platform for efficient formic acid biorefinery.", "to": "metabolic capacity"}, {"from": "rewiring the serine cycle", "title": "Discovery and remodeling of Vibrio natriegens as a microbial platform for efficient formic acid biorefinery.", "to": "non-native closed loop"}, {"from": "rewiring the TCA cycle", "title": "Discovery and remodeling of Vibrio natriegens as a microbial platform for efficient formic acid biorefinery.", "to": "non-native closed loop"}, {"from": "non-native closed loop", "title": "Discovery and remodeling of Vibrio natriegens as a microbial platform for efficient formic acid biorefinery.", "to": "improved formic acid-utilizing ability"}, {"from": "laboratory evolution", "title": "Discovery and remodeling of Vibrio natriegens as a microbial platform for efficient formic acid biorefinery.", "to": "improved formic acid-utilizing ability"}, {"from": "introduction of foreign indigoidine-forming pathway", "title": "Discovery and remodeling of Vibrio natriegens as a microbial platform for efficient formic acid biorefinery.", "to": "production of 29.0 grams per liter indigoidine"}, {"from": "introduction of foreign indigoidine-forming pathway", "title": "Discovery and remodeling of Vibrio natriegens as a microbial platform for efficient formic acid biorefinery.", "to": "consumption of 165.3 grams per liter formate"}, {"from": "consumption rate of 2.3 grams per liter per hour", "title": "Discovery and remodeling of Vibrio natriegens as a microbial platform for efficient formic acid biorefinery.", "to": "consumption of 165.3 grams per liter formate"}, {"from": "horizontal gene transfer", "title": "Agarose biodegradation by deep-sea bacterium Vibrio natriegens WPAGA4 with the agarases through horizontal gene transfer.", "to": "agarose-degrading ability of Vibrio natriegens WPAGA4"}, {"from": "horizontal gene transfer", "title": "Agarose biodegradation by deep-sea bacterium Vibrio natriegens WPAGA4 with the agarases through horizontal gene transfer.", "to": "agarose degradation pathway of Vibrio natriegens WPAGA4"}, {"from": "Aga3418", "title": "Agarose biodegradation by deep-sea bacterium Vibrio natriegens WPAGA4 with the agarases through horizontal gene transfer.", "to": "agarose degradation product neoagarobiose"}, {"from": "Aga3420", "title": "Agarose biodegradation by deep-sea bacterium Vibrio natriegens WPAGA4 with the agarases through horizontal gene transfer.", "to": "agarose degradation product neoagarobiose"}, {"from": "Aga3472", "title": "Agarose biodegradation by deep-sea bacterium Vibrio natriegens WPAGA4 with the agarases through horizontal gene transfer.", "to": "agarose degradation product neoagarobiose"}, {"from": "Aga3419", "title": "Agarose biodegradation by deep-sea bacterium Vibrio natriegens WPAGA4 with the agarases through horizontal gene transfer.", "to": "agarose degradation product neoagarobiose"}, {"from": "Aga3419", "title": "Agarose biodegradation by deep-sea bacterium Vibrio natriegens WPAGA4 with the agarases through horizontal gene transfer.", "to": "agarose degradation product neoagarotetraose"}, {"from": "expression level ratio of Aga3418", "title": "Agarose biodegradation by deep-sea bacterium Vibrio natriegens WPAGA4 with the agarases through horizontal gene transfer.", "to": "Aga3419, Aga3420, and Aga3472, optimal agarose degradation by Vibrio natriegens WPAGA4"}, {"from": "genomic information", "title": "Agarose biodegradation by deep-sea bacterium Vibrio natriegens WPAGA4 with the agarases through horizontal gene transfer.", "to": "occurrence of horizontal gene transfer"}, {"from": "expression level ratio of agarase genes", "title": "Agarose biodegradation by deep-sea bacterium Vibrio natriegens WPAGA4 with the agarases through horizontal gene transfer.", "to": "crucial factor for agarose degradation by Vibrio natriegens WPAGA4"}, {"from": "horizontal gene transfer", "title": "Agarose biodegradation by deep-sea bacterium Vibrio natriegens WPAGA4 with the agarases through horizontal gene transfer.", "to": "promotion of deep-sea carbon cycle"}, {"from": "Vibrio natriegens", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for high-yield production of challenging expression targets and for protein deuteration.", "to": "production of soluble proteins in high yields"}, {"from": "Vmax\u2122 X2", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for high-yield production of challenging expression targets and for protein deuteration.", "to": "production of soluble bacterial and fungal proteins in milligram scale"}, {"from": "Vibrio natriegens", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for high-yield production of challenging expression targets and for protein deuteration.", "to": "alternative host platform for production of proteins"}, {"from": "cholera toxin", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for high-yield production of challenging expression targets and for protein deuteration.", "to": "secreted by Type II secretion system"}, {"from": "N-acetyl glucosamine binding protein A", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for high-yield production of challenging expression targets and for protein deuteration.", "to": "secreted by Type II secretion system"}, {"from": "heat-labile enterotoxin", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for high-yield production of challenging expression targets and for protein deuteration.", "to": "secreted by Type II secretion system"}, {"from": "cholera toxin", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for high-yield production of challenging expression targets and for protein deuteration.", "to": "recovered from growth media"}, {"from": "N-acetyl glucosamine binding protein A", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for high-yield production of challenging expression targets and for protein deuteration.", "to": "recovered from growth media"}, {"from": "heat-labile enterotoxin", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for high-yield production of challenging expression targets and for protein deuteration.", "to": "recovered from growth media"}, {"from": "Vmax\u2122 X2", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for high-yield production of challenging expression targets and for protein deuteration.", "to": "6- to 26-fold increase in Protein yields compared to Escherichia coli"}, {"from": "Vmax\u2122 X2", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for high-yield production of challenging expression targets and for protein deuteration.", "to": "3-fold increase in yield using deuterated minimal media compared to Escherichia coli"}, {"from": "Vmax\u2122 X2", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for high-yield production of challenging expression targets and for protein deuteration.", "to": "simplified downstream purification procedure"}, {"from": "Escherichia coli", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for high-yield production of challenging expression targets and for protein deuteration.", "to": "struggled to produce soluble proteins in milligram scale"}, {"from": "Vmax\u2122 X2", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for high-yield production of challenging expression targets and for protein deuteration.", "to": "promising host for production of challenging expression targets"}, {"from": "Vmax\u2122 X2", "title": "Using \u003ci\u003eVibrio natriegens\u003c/i\u003e for high-yield production of challenging expression targets and for protein deuteration.", "to": "promising host for protein deuteration in amounts suitable for structural biology studies"}, {"from": "Vibrio natriegens", "title": "Nonsterilized Fermentation of Crude Glycerol for Polyhydroxybutyrate Production by Metabolically Engineered \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "Polyhydroxybutyrate production"}, {"from": "inhibitory substances presence", "title": "Nonsterilized Fermentation of Crude Glycerol for Polyhydroxybutyrate Production by Metabolically Engineered \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "organic wastes poor utilization"}, {"from": "slow microbial growth", "title": "Nonsterilized Fermentation of Crude Glycerol for Polyhydroxybutyrate Production by Metabolically Engineered \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "organic wastes poor utilization"}, {"from": "high energy input for feedstock sterilization", "title": "Nonsterilized Fermentation of Crude Glycerol for Polyhydroxybutyrate Production by Metabolically Engineered \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "organic wastes poor utilization"}, {"from": "base editing", "title": "Nonsterilized Fermentation of Crude Glycerol for Polyhydroxybutyrate Production by Metabolically Engineered \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "key genes identification"}, {"from": "key genes overexpression", "title": "Nonsterilized Fermentation of Crude Glycerol for Polyhydroxybutyrate Production by Metabolically Engineered \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "Polyhydroxybutyrate production"}, {"from": "glycerol", "title": "Nonsterilized Fermentation of Crude Glycerol for Polyhydroxybutyrate Production by Metabolically Engineered \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "Polyhydroxybutyrate production"}, {"from": "sugars", "title": "Nonsterilized Fermentation of Crude Glycerol for Polyhydroxybutyrate Production by Metabolically Engineered \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "Polyhydroxybutyrate production"}, {"from": "Vibrio natriegens metabolic engineering", "title": "Nonsterilized Fermentation of Crude Glycerol for Polyhydroxybutyrate Production by Metabolically Engineered \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "Polyhydroxybutyrate productivity increase"}, {"from": "Vibrio natriegens metabolic engineering", "title": "Nonsterilized Fermentation of Crude Glycerol for Polyhydroxybutyrate Production by Metabolically Engineered \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "tolerance to toxic substances"}, {"from": "Vibrio natriegens metabolic engineering", "title": "Nonsterilized Fermentation of Crude Glycerol for Polyhydroxybutyrate Production by Metabolically Engineered \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "activity maintenance under nonsterilized fermentation conditions"}, {"from": "Engineered Escherichia coli MG1655", "title": "Nonsterilized Fermentation of Crude Glycerol for Polyhydroxybutyrate Production by Metabolically Engineered \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "lower Polyhydroxybutyrate productivity than engineered Vibrio natriegens"}, {"from": "Engineered Escherichia coli MG1655", "title": "Nonsterilized Fermentation of Crude Glycerol for Polyhydroxybutyrate Production by Metabolically Engineered \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "lower tolerance to toxic substances than engineered Vibrio natriegens"}, {"from": "tyrosinase enzyme Tyr1", "title": "Rapid, high-titer biosynthesis of melanin using the marine bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "melanin production biosynthesis"}, {"from": "tyrosine", "title": "Rapid, high-titer biosynthesis of melanin using the marine bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "melanin production biosynthesis"}, {"from": "Vibrio natriegens", "title": "Rapid, high-titer biosynthesis of melanin using the marine bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "melanin production biosynthesis"}, {"from": "copper responsive promoter", "title": "Rapid, high-titer biosynthesis of melanin using the marine bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "heterologous protein expression regulation"}, {"from": "recombinant microbial system", "title": "Rapid, high-titer biosynthesis of melanin using the marine bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "melanin production biosynthesis"}, {"from": "optimized minimally defined medium", "title": "Rapid, high-titer biosynthesis of melanin using the marine bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "melanin production yield"}, {"from": "optimized minimally defined medium", "title": "Rapid, high-titer biosynthesis of melanin using the marine bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "volumetric productivity of melanin production"}, {"from": "optimized minimally defined medium", "title": "Rapid, high-titer biosynthesis of melanin using the marine bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "conversion rates of tyrosine to melanin production"}, {"from": "high sodium sodium chloride concentrations", "title": "High-cell-density cultivation of Vibrio natriegens in a low-chloride chemically defined medium.", "to": "corrosion processes on metal surfaces"}, {"from": "low-sodium chloride chemically defined medium", "title": "High-cell-density cultivation of Vibrio natriegens in a low-chloride chemically defined medium.", "to": "high-cell-density fed-batch cultivations"}, {"from": "sodium sodium chloride", "title": "High-cell-density cultivation of Vibrio natriegens in a low-chloride chemically defined medium.", "to": "precipitation of ammonium magnesium phosphate"}, {"from": "sodium citrate", "title": "High-cell-density cultivation of Vibrio natriegens in a low-chloride chemically defined medium.", "to": "prevention of precipitates"}, {"from": "exponential feeding", "title": "High-cell-density cultivation of Vibrio natriegens in a low-chloride chemically defined medium.", "to": "biomass concentrations of more than 60 grams per liter"}, {"from": "disodium hydrogen phosphate", "title": "High-cell-density cultivation of Vibrio natriegens in a low-chloride chemically defined medium.", "to": "low-sodium chloride chemically defined medium"}, {"from": "disodium sulfate", "title": "High-cell-density cultivation of Vibrio natriegens in a low-chloride chemically defined medium.", "to": "low-sodium chloride chemically defined medium"}, {"from": "sodium citrate", "title": "High-cell-density cultivation of Vibrio natriegens in a low-chloride chemically defined medium.", "to": "low-sodium chloride chemically defined medium"}, {"from": "Vibrio natriegens ", "title": "Metabolic engineering of fast-growing Vibrio natriegens for efficient pyruvate production.", "to": "pyruvate production"}, {"from": "deletion of two prophage gene clusters ", "title": "Metabolic engineering of fast-growing Vibrio natriegens for efficient pyruvate production.", "to": "pyruvate production"}, {"from": "deletion of several essential genes for the biosynthesis of byproducts ", "title": "Metabolic engineering of fast-growing Vibrio natriegens for efficient pyruvate production.", "to": "pyruvate production"}, {"from": "down-regulation of aceE ", "title": "Metabolic engineering of fast-growing Vibrio natriegens for efficient pyruvate production.", "to": "reduced carbon flux into the tricarboxylic acid cycle"}, {"from": "fine-tuning of ppc gene expression ", "title": "Metabolic engineering of fast-growing Vibrio natriegens for efficient pyruvate production.", "to": "balance between cell growth and pyruvate synthesis"}, {"from": "PYR32 strain ", "title": "Metabolic engineering of fast-growing Vibrio natriegens for efficient pyruvate production.", "to": "54.22 grams per liter pyruvate production"}, {"from": "PYR32 strain ", "title": "Metabolic engineering of fast-growing Vibrio natriegens for efficient pyruvate production.", "to": "1.17 mol/mol yield of pyruvate"}, {"from": "PYR32 strain ", "title": "Metabolic engineering of fast-growing Vibrio natriegens for efficient pyruvate production.", "to": "3.39 grams per liter per hour average productivity of pyruvate"}, {"from": "PYR32 strain ", "title": "Metabolic engineering of fast-growing Vibrio natriegens for efficient pyruvate production.", "to": "efficient conversion of sucrose into pyruvate"}, {"from": "PYR32 strain ", "title": "Metabolic engineering of fast-growing Vibrio natriegens for efficient pyruvate production.", "to": "efficient conversion of gluconate into pyruvate"}, {"from": "Vibrio natriegens", "title": "[Secretory expression and fermentation optimization for extracellular production of pullulanase in \u003ci\u003eVibrio natriegens\u003c/i\u003e].", "to": "secretory expression of large molecular weight proteins"}, {"from": "Vibrio natriegens", "title": "[Secretory expression and fermentation optimization for extracellular production of pullulanase in \u003ci\u003eVibrio natriegens\u003c/i\u003e].", "to": "protein synthesis efficiency"}, {"from": "Pullulanase PulA", "title": "[Secretory expression and fermentation optimization for extracellular production of pullulanase in \u003ci\u003eVibrio natriegens\u003c/i\u003e].", "to": "secretory expression in Vibrio natriegens VnDX"}, {"from": "Pullulanase PulN2", "title": "[Secretory expression and fermentation optimization for extracellular production of pullulanase in \u003ci\u003eVibrio natriegens\u003c/i\u003e].", "to": "secretory expression in Vibrio natriegens VnDX"}, {"from": "signal peptide", "title": "[Secretory expression and fermentation optimization for extracellular production of pullulanase in \u003ci\u003eVibrio natriegens\u003c/i\u003e].", "to": "secretory expression"}, {"from": "fermentation temperature range", "title": "[Secretory expression and fermentation optimization for extracellular production of pullulanase in \u003ci\u003eVibrio natriegens\u003c/i\u003e].", "to": "secretory expression"}, {"from": "inducer concentration", "title": "[Secretory expression and fermentation optimization for extracellular production of pullulanase in \u003ci\u003eVibrio natriegens\u003c/i\u003e].", "to": "secretory expression"}, {"from": "glycine concentration", "title": "[Secretory expression and fermentation optimization for extracellular production of pullulanase in \u003ci\u003eVibrio natriegens\u003c/i\u003e].", "to": "secretory expression"}, {"from": "fermentation time", "title": "[Secretory expression and fermentation optimization for extracellular production of pullulanase in \u003ci\u003eVibrio natriegens\u003c/i\u003e].", "to": "secretory expression"}, {"from": "extracellular enzyme activities", "title": "[Secretory expression and fermentation optimization for extracellular production of pullulanase in \u003ci\u003eVibrio natriegens\u003c/i\u003e].", "to": "comparison between Vibrio natriegens VnDX and Escherichia coli BL21(DE3"}, {"from": "Pullulanase PulA extracellular enzyme activity in Vibrio natriegens VnDX", "title": "[Secretory expression and fermentation optimization for extracellular production of pullulanase in \u003ci\u003eVibrio natriegens\u003c/i\u003e].", "to": "61.6 U/mL"}, {"from": "Pullulanase PulN2 extracellular enzyme activity in Vibrio natriegens VnDX", "title": "[Secretory expression and fermentation optimization for extracellular production of pullulanase in \u003ci\u003eVibrio natriegens\u003c/i\u003e].", "to": "64.3 U/mL"}, {"from": "Pullulanase PulA extracellular enzyme activity in Vibrio natriegens VnDX", "title": "[Secretory expression and fermentation optimization for extracellular production of pullulanase in \u003ci\u003eVibrio natriegens\u003c/i\u003e].", "to": "110% of Escherichia coli BL21(DE3"}, {"from": "Pullulanase PulN2 extracellular enzyme activity in Vibrio natriegens VnDX", "title": "[Secretory expression and fermentation optimization for extracellular production of pullulanase in \u003ci\u003eVibrio natriegens\u003c/i\u003e].", "to": "62% of Escherichia coli BL21(DE3"}, {"from": "Vibrio natriegens", "title": "Novel expression system based on enhanced permeability of Vibrio natriegens cells induced by D,D- carboxypeptidase overexpression.", "to": "fast-growing"}, {"from": "Vibrio natriegens", "title": "Novel expression system based on enhanced permeability of Vibrio natriegens cells induced by D,D- carboxypeptidase overexpression.", "to": "non-pathogenic"}, {"from": "Vibrio natriegens", "title": "Novel expression system based on enhanced permeability of Vibrio natriegens cells induced by D,D- carboxypeptidase overexpression.", "to": "high-level recombinant protein production"}, {"from": "Vibrio natriegens", "title": "Novel expression system based on enhanced permeability of Vibrio natriegens cells induced by D,D- carboxypeptidase overexpression.", "to": "fast DNA propagation"}, {"from": "Vibrio natriegens", "title": "Novel expression system based on enhanced permeability of Vibrio natriegens cells induced by D,D- carboxypeptidase overexpression.", "to": "short generation time"}, {"from": "Vibrio natriegens", "title": "Novel expression system based on enhanced permeability of Vibrio natriegens cells induced by D,D- carboxypeptidase overexpression.", "to": "proteosynthetic activity"}, {"from": "Vibrio natriegens", "title": "Novel expression system based on enhanced permeability of Vibrio natriegens cells induced by D,D- carboxypeptidase overexpression.", "to": "versatile metabolism"}, {"from": "Vibrio natriegens", "title": "Novel expression system based on enhanced permeability of Vibrio natriegens cells induced by D,D- carboxypeptidase overexpression.", "to": "future industrial platform"}, {"from": "Vibrio natriegens", "title": "Novel expression system based on enhanced permeability of Vibrio natriegens cells induced by D,D- carboxypeptidase overexpression.", "to": "fermentation processes operating with high productivity"}, {"from": "enhanced membrane permeability", "title": "Novel expression system based on enhanced permeability of Vibrio natriegens cells induced by D,D- carboxypeptidase overexpression.", "to": "facilitating transport of target proteins into extracellular environment"}, {"from": "IMAC chromatography", "title": "Novel expression system based on enhanced permeability of Vibrio natriegens cells induced by D,D- carboxypeptidase overexpression.", "to": "purification of AfKatG from growth medium"}, {"from": "IMAC chromatography", "title": "Novel expression system based on enhanced permeability of Vibrio natriegens cells induced by D,D- carboxypeptidase overexpression.", "to": "purification of MDBP from growth medium"}, {"from": "IMAC chromatography", "title": "Novel expression system based on enhanced permeability of Vibrio natriegens cells induced by D,D- carboxypeptidase overexpression.", "to": "purification of purification of Taq polymerase from growth medium from growth medium"}, {"from": "AfKatG", "title": "Novel expression system based on enhanced permeability of Vibrio natriegens cells induced by D,D- carboxypeptidase overexpression.", "to": "total yield from growth medium"}, {"from": "MDBP", "title": "Novel expression system based on enhanced permeability of Vibrio natriegens cells induced by D,D- carboxypeptidase overexpression.", "to": "total yield from growth medium"}, {"from": "purification of Taq polymerase from growth medium", "title": "Novel expression system based on enhanced permeability of Vibrio natriegens cells induced by D,D- carboxypeptidase overexpression.", "to": "total yield from growth medium"}, {"from": "Vibrio natriegens based system", "title": "Novel expression system based on enhanced permeability of Vibrio natriegens cells induced by D,D- carboxypeptidase overexpression.", "to": "extracellular tag-less recombinant protein producer"}, {"from": "amino acid balancing", "title": "Amino acid balancing for the prediction and evaluation of protein concentrations in cell-free protein synthesis systems.", "to": "performance of Cell-free protein synthesis systems"}, {"from": "Escherichia coli system", "title": "Amino acid balancing for the prediction and evaluation of protein concentrations in cell-free protein synthesis systems.", "to": "high efficiency in the utilization of amino acids"}, {"from": "Vibrio natriegens based systems", "title": "Amino acid balancing for the prediction and evaluation of protein concentrations in cell-free protein synthesis systems.", "to": "fractional yields of over 70%"}, {"from": "Leishmania tarentolae based systems", "title": "Amino acid balancing for the prediction and evaluation of protein concentrations in cell-free protein synthesis systems.", "to": "fractional yields of over 70%"}, {"from": "Cell-free protein synthesis systems", "title": "Amino acid balancing for the prediction and evaluation of protein concentrations in cell-free protein synthesis systems.", "to": "protein molar concentration"}, {"from": "Ostreid herpesvirus-1", "title": "Identification and Characterization of Infectious Pathogens Associated with Mass Mortalities of Pacific Oyster (\u003ci\u003eCrassostrea gigas\u003c/i\u003e) Cultured in Northern China.", "to": "mass mortalities of hatchery larvae of Pacific oysters"}, {"from": "Vibrio natriegens", "title": "Identification and Characterization of Infectious Pathogens Associated with Mass Mortalities of Pacific Oyster (\u003ci\u003eCrassostrea gigas\u003c/i\u003e) Cultured in Northern China.", "to": "mass mortalities of Pacific oysters"}, {"from": "Vibrio alginolyticus", "title": "Identification and Characterization of Infectious Pathogens Associated with Mass Mortalities of Pacific Oyster (\u003ci\u003eCrassostrea gigas\u003c/i\u003e) Cultured in Northern China.", "to": "mass mortalities of Pacific oysters"}, {"from": "Pseudoalteromonas spp.", "title": "Identification and Characterization of Infectious Pathogens Associated with Mass Mortalities of Pacific Oyster (\u003ci\u003eCrassostrea gigas\u003c/i\u003e) Cultured in Northern China.", "to": "mass mortalities of Pacific oysters during the cold season"}, {"from": "Vibrio natriegens growth temperature range", "title": "Identification and Characterization of Infectious Pathogens Associated with Mass Mortalities of Pacific Oyster (\u003ci\u003eCrassostrea gigas\u003c/i\u003e) Cultured in Northern China.", "to": "faster growth at 25 degrees Celsius"}, {"from": "Vibrio natriegens haemolytic activity temperature range", "title": "Identification and Characterization of Infectious Pathogens Associated with Mass Mortalities of Pacific Oyster (\u003ci\u003eCrassostrea gigas\u003c/i\u003e) Cultured in Northern China.", "to": "more remarkable haemolytic activity at 25 degrees Celsius"}, {"from": "Vibrio natriegens siderophore production capacity temperature range", "title": "Identification and Characterization of Infectious Pathogens Associated with Mass Mortalities of Pacific Oyster (\u003ci\u003eCrassostrea gigas\u003c/i\u003e) Cultured in Northern China.", "to": "more remarkable siderophore production capacity at 25 degrees Celsius"}, {"from": "Vibrio alginolyticus growth temperature range", "title": "Identification and Characterization of Infectious Pathogens Associated with Mass Mortalities of Pacific Oyster (\u003ci\u003eCrassostrea gigas\u003c/i\u003e) Cultured in Northern China.", "to": "faster growth at 25 degrees Celsius"}, {"from": "Vibrio alginolyticus haemolytic activity temperature range", "title": "Identification and Characterization of Infectious Pathogens Associated with Mass Mortalities of Pacific Oyster (\u003ci\u003eCrassostrea gigas\u003c/i\u003e) Cultured in Northern China.", "to": "more remarkable haemolytic activity at 25 degrees Celsius"}, {"from": "Vibrio alginolyticus siderophore production capacity temperature range", "title": "Identification and Characterization of Infectious Pathogens Associated with Mass Mortalities of Pacific Oyster (\u003ci\u003eCrassostrea gigas\u003c/i\u003e) Cultured in Northern China.", "to": "more remarkable siderophore production capacity at 25 degrees Celsius"}, {"from": "Vibrio natriegens mortality rate temperature range", "title": "Identification and Characterization of Infectious Pathogens Associated with Mass Mortalities of Pacific Oyster (\u003ci\u003eCrassostrea gigas\u003c/i\u003e) Cultured in Northern China.", "to": "higher accumulative mortalities at 25 degrees Celsius"}, {"from": "Vibrio alginolyticus mortality rate temperature range", "title": "Identification and Characterization of Infectious Pathogens Associated with Mass Mortalities of Pacific Oyster (\u003ci\u003eCrassostrea gigas\u003c/i\u003e) Cultured in Northern China.", "to": "higher accumulative mortalities at 25 degrees Celsius"}, {"from": "Gluconobacter oxydans levansucrases", "title": "Investigation of biocatalytic production of lactosucrose and fructooligosaccharides using levansucrases and dairy by-products as starting materials.", "to": "lactosucrose production"}, {"from": "Gluconobacter oxydans levansucrases", "title": "Investigation of biocatalytic production of lactosucrose and fructooligosaccharides using levansucrases and dairy by-products as starting materials.", "to": "fructooligosaccharides production"}, {"from": "naturally competent Vibrio natriegens levansucrases", "title": "Investigation of biocatalytic production of lactosucrose and fructooligosaccharides using levansucrases and dairy by-products as starting materials.", "to": "lactosucrose production"}, {"from": "naturally competent Vibrio natriegens levansucrases", "title": "Investigation of biocatalytic production of lactosucrose and fructooligosaccharides using levansucrases and dairy by-products as starting materials.", "to": "fructooligosaccharides production"}, {"from": "Novosphingobium aromaticivorans levansucrases", "title": "Investigation of biocatalytic production of lactosucrose and fructooligosaccharides using levansucrases and dairy by-products as starting materials.", "to": "lactosucrose production"}, {"from": "Novosphingobium aromaticivorans levansucrases", "title": "Investigation of biocatalytic production of lactosucrose and fructooligosaccharides using levansucrases and dairy by-products as starting materials.", "to": "fructooligosaccharides production"}, {"from": "Burkholderia graminis levansucrases", "title": "Investigation of biocatalytic production of lactosucrose and fructooligosaccharides using levansucrases and dairy by-products as starting materials.", "to": "lactosucrose production"}, {"from": "Burkholderia graminis levansucrases", "title": "Investigation of biocatalytic production of lactosucrose and fructooligosaccharides using levansucrases and dairy by-products as starting materials.", "to": "fructooligosaccharides production"}, {"from": "whey permeate", "title": "Investigation of biocatalytic production of lactosucrose and fructooligosaccharides using levansucrases and dairy by-products as starting materials.", "to": "lactosucrose production"}, {"from": "whey permeate", "title": "Investigation of biocatalytic production of lactosucrose and fructooligosaccharides using levansucrases and dairy by-products as starting materials.", "to": "fructooligosaccharides production"}, {"from": "milk permeate", "title": "Investigation of biocatalytic production of lactosucrose and fructooligosaccharides using levansucrases and dairy by-products as starting materials.", "to": "lactosucrose production"}, {"from": "milk permeate", "title": "Investigation of biocatalytic production of lactosucrose and fructooligosaccharides using levansucrases and dairy by-products as starting materials.", "to": "fructooligosaccharides production"}, {"from": "naturally competent Vibrio natriegens levansucrases", "title": "Investigation of biocatalytic production of lactosucrose and fructooligosaccharides using levansucrases and dairy by-products as starting materials.", "to": "transfructosylation activity"}, {"from": "acceptor specificity of levansucrases", "title": "Investigation of biocatalytic production of lactosucrose and fructooligosaccharides using levansucrases and dairy by-products as starting materials.", "to": "end-product profile"}, {"from": "thermodynamic equilibrium", "title": "Investigation of biocatalytic production of lactosucrose and fructooligosaccharides using levansucrases and dairy by-products as starting materials.", "to": "end-product profile"}, {"from": "Gluconobacter oxydans levansucrases", "title": "Investigation of biocatalytic production of lactosucrose and fructooligosaccharides using levansucrases and dairy by-products as starting materials.", "to": "transfructosylation activity"}, {"from": "Novosphingobium aromaticivorans levansucrases", "title": "Investigation of biocatalytic production of lactosucrose and fructooligosaccharides using levansucrases and dairy by-products as starting materials.", "to": "transfructosylation activity"}, {"from": "Burkholderia graminis levansucrases", "title": "Investigation of biocatalytic production of lactosucrose and fructooligosaccharides using levansucrases and dairy by-products as starting materials.", "to": "transfructosylation activity"}, {"from": "coral bleaching event of 2020", "title": "Sedimentary \u003ci\u003eVibrio\u003c/i\u003e Blooms in the Xisha Islands May Associate with the 2020 Coral Bleaching Event.", "to": "bloom of vibrios"}, {"from": "spatial distance", "title": "Sedimentary \u003ci\u003eVibrio\u003c/i\u003e Blooms in the Xisha Islands May Associate with the 2020 Coral Bleaching Event.", "to": "Vibrio community"}, {"from": "coral species", "title": "Sedimentary \u003ci\u003eVibrio\u003c/i\u003e Blooms in the Xisha Islands May Associate with the 2020 Coral Bleaching Event.", "to": "Vibrio community"}, {"from": "Vibrio harveyi", "title": "Sedimentary \u003ci\u003eVibrio\u003c/i\u003e Blooms in the Xisha Islands May Associate with the 2020 Coral Bleaching Event.", "to": "relative abundance"}, {"from": "Vibrio harveyi", "title": "Sedimentary \u003ci\u003eVibrio\u003c/i\u003e Blooms in the Xisha Islands May Associate with the 2020 Coral Bleaching Event.", "to": "niche breadth"}, {"from": "Vibrio harveyi", "title": "Sedimentary \u003ci\u003eVibrio\u003c/i\u003e Blooms in the Xisha Islands May Associate with the 2020 Coral Bleaching Event.", "to": "Acroporidae"}, {"from": "TfoX", "title": "Systems biology of competency in Vibrio natriegens is revealed by applying novel data analytics to the transcriptome.", "to": "competency"}, {"from": "QstR", "title": "Systems biology of competency in Vibrio natriegens is revealed by applying novel data analytics to the transcriptome.", "to": "competency"}, {"from": "iron metabolism iModulon", "title": "Systems biology of competency in Vibrio natriegens is revealed by applying novel data analytics to the transcriptome.", "to": "competency"}, {"from": "translation iModulon", "title": "Systems biology of competency in Vibrio natriegens is revealed by applying novel data analytics to the transcriptome.", "to": "competency"}, {"from": "novel iModulon of unknown function", "title": "Systems biology of competency in Vibrio natriegens is revealed by applying novel data analytics to the transcriptome.", "to": "competency"}, {"from": "motility behavior iModulon", "title": "Systems biology of competency in Vibrio natriegens is revealed by applying novel data analytics to the transcriptome.", "to": "competency"}, {"from": "polycations iModulon", "title": "Systems biology of competency in Vibrio natriegens is revealed by applying novel data analytics to the transcriptome.", "to": "competency"}, {"from": "reactive oxygen species responses iModulon", "title": "Systems biology of competency in Vibrio natriegens is revealed by applying novel data analytics to the transcriptome.", "to": "competency"}, {"from": "gene deletion", "title": "Systems biology of competency in Vibrio natriegens is revealed by applying novel data analytics to the transcriptome.", "to": "competency"}, {"from": "aerobic glucose-limited fed-batch fermentation", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "biomass concentration"}, {"from": "aerobic glucose-limited fed-batch fermentation", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "viscosity of culture broth"}, {"from": "formation of exopolysaccharides", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "viscosity of culture broth"}, {"from": "deletion of transcriptional regulator cpsR", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "viscosity of culture broth"}, {"from": "deletion of transcriptional regulator cpsR", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "carbohydrate content of exopolysaccharides"}, {"from": "deletion of cps cluster", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "amount of exopolysaccharides secreted"}, {"from": "deletion of cps cluster", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "viscosity of culture broth"}, {"from": "deletion of cps cluster", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "exopolysaccharides composition"}, {"from": "anaerobic conditions on glucose", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "viscosity of culture broth"}, {"from": "deletion of cpsR in Vibrio natriegens Succ1", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "viscosity of culture broth"}, {"from": "deletion of cps cluster in Vibrio natriegens Succ1", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "viscosity of culture broth"}, {"from": "Vibrio natriegens Succ1 deletion of cpsR", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "final succinate concentration"}, {"from": "Vibrio natriegens Succ1 deletion of cpsR", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "volumetric productivity of succinate"}, {"from": "Vibrio natriegens Succ1 deletion of cpsR", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "product yield of succinate"}, {"from": "Vibrio natriegens Succ1 deletion of cps cluster", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "final succinate concentration"}, {"from": "Vibrio natriegens Succ1 deletion of cps cluster", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "volumetric productivity of succinate"}, {"from": "Vibrio natriegens Succ1 deletion of cps cluster", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "product yield of succinate"}, {"from": "high growth and substrate consumption rates", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "Vibrio natriegens as an emerging host for biotechnology"}, {"from": "feed-batch processes", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "high space-time yields in industrial processes"}, {"from": "anaerobic cultivation of Vibrio natriegens Succ1", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "increased viscosity at the end of cultivation"}, {"from": "deletion of transcriptional regulator cpsR and cps cluster in Vibrio natriegens Succ1", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "viscosity of culture broth reduction"}, {"from": "product yield of succinate in Vibrio natriegens Succ1 deletion of cpsR and cps cluster", "title": "Investigation of exopolysaccharide formation and its impact on anaerobic succinate production with Vibrio natriegens.", "to": "theoretical maximum succinate yield"}, {"from": "malonate semialdehyde ", "title": "Engineering Escherichia coli to assimilate \u03b2-alanine as a major carbon source.", "to": "beta-alanine"}, {"from": "beta-alanine ", "title": "Engineering Escherichia coli to assimilate \u03b2-alanine as a major carbon source.", "to": "malonate semialdehyde"}, {"from": "beta-alanine ", "title": "Engineering Escherichia coli to assimilate \u03b2-alanine as a major carbon source.", "to": "Vibrio natriegens Coenzyme A-acylating methyl malonate semialdehyde dehydrogenases"}, {"from": "Pseudomonas aeruginosa beta-alanine/pyruvate transaminase", "title": "Engineering Escherichia coli to assimilate \u03b2-alanine as a major carbon source.", "to": "malonate semialdehyde "}, {"from": "Vibrio natriegens Coenzyme A-acylating methyl malonate semialdehyde dehydrogenases", "title": "Engineering Escherichia coli to assimilate \u03b2-alanine as a major carbon source.", "to": "acetyl-Coenzyme A"}, {"from": "malonate semialdehyde ", "title": "Engineering Escherichia coli to assimilate \u03b2-alanine as a major carbon source.", "to": "acetyl-Coenzyme A"}, {"from": "Escherichia coli MG1655", "title": "Engineering Escherichia coli to assimilate \u03b2-alanine as a major carbon source.", "to": "beta-alanine"}, {"from": "Escherichia coli BA02", "title": "Engineering Escherichia coli to assimilate \u03b2-alanine as a major carbon source.", "to": "optical density at 600 nanometer"}, {"from": "Pseudomonas aeruginosa beta-alanine/pyruvate transaminase", "title": "Engineering Escherichia coli to assimilate \u03b2-alanine as a major carbon source.", "to": "Escherichia coli MG1655"}, {"from": "Vibrio natriegens Coenzyme A-acylating methyl malonate semialdehyde dehydrogenases", "title": "Engineering Escherichia coli to assimilate \u03b2-alanine as a major carbon source.", "to": "Escherichia coli MG1655"}, {"from": "Escherichia coli BA07\u2206", "title": "Engineering Escherichia coli to assimilate \u03b2-alanine as a major carbon source.", "to": "optical density at 600 nanometer"}, {"from": "Escherichia coli BA07\u2206", "title": "Engineering Escherichia coli to assimilate \u03b2-alanine as a major carbon source.", "to": "beta-alanine"}, {"from": "native 3-hydroxy acid dehydrogenase", "title": "Engineering Escherichia coli to assimilate \u03b2-alanine as a major carbon source.", "to": "3-hydroxypropionate production"}, {"from": "replication-induced gene copy numbers", "title": "The role of replication-induced chromosomal copy numbers in spatio-temporal gene regulation and evolutionary chromosome plasticity.", "to": "cellular functions"}, {"from": "replication-induced gene copy numbers", "title": "The role of replication-induced chromosomal copy numbers in spatio-temporal gene regulation and evolutionary chromosome plasticity.", "to": "central pathways"}, {"from": "replication-induced gene copy numbers", "title": "The role of replication-induced chromosomal copy numbers in spatio-temporal gene regulation and evolutionary chromosome plasticity.", "to": "spatio-temporal expression pattern"}, {"from": "growth phase dependent gene expression", "title": "The role of replication-induced chromosomal copy numbers in spatio-temporal gene regulation and evolutionary chromosome plasticity.", "to": "evolutionary gene migration"}, {"from": "chromosome evolution", "title": "The role of replication-induced chromosomal copy numbers in spatio-temporal gene regulation and evolutionary chromosome plasticity.", "to": "high growth rates related transient copy numbers"}, {"from": "replication-induced gene copy numbers", "title": "The role of replication-induced chromosomal copy numbers in spatio-temporal gene regulation and evolutionary chromosome plasticity.", "to": "transient chromosomal copy numbers"}, {"from": "replication-induced gene copy numbers", "title": "The role of replication-induced chromosomal copy numbers in spatio-temporal gene regulation and evolutionary chromosome plasticity.", "to": "growth phase dependent gene expression"}, {"from": "Vibrio natriegens", "title": "Anodic Respiration of Vibrio natriegens in a Bioelectrochemical System.", "to": "new standard biotechnological organism"}, {"from": "Vibrio natriegens", "title": "Anodic Respiration of Vibrio natriegens in a Bioelectrochemical System.", "to": "fast growth"}, {"from": "Vibrio natriegens productivity", "title": "Anodic Respiration of Vibrio natriegens in a Bioelectrochemical System.", "to": "surpasses Escherichia coli"}, {"from": "Vibrio natriegens", "title": "Anodic Respiration of Vibrio natriegens in a Bioelectrochemical System.", "to": "genomic tools availability"}, {"from": "Vibrio natriegens", "title": "Anodic Respiration of Vibrio natriegens in a Bioelectrochemical System.", "to": "similar extracellular electron transfer pathway to electroactive organisms"}, {"from": "Vibrio natriegens", "title": "Anodic Respiration of Vibrio natriegens in a Bioelectrochemical System.", "to": "donates electrons from metabolism to electrode"}, {"from": "Vibrio natriegens", "title": "Anodic Respiration of Vibrio natriegens in a Bioelectrochemical System.", "to": "electrons donation via direct contact"}, {"from": "Vibrio natriegens", "title": "Anodic Respiration of Vibrio natriegens in a Bioelectrochemical System.", "to": "electrons donation via artificial redox mediator"}, {"from": "artificial mediator use", "title": "Anodic Respiration of Vibrio natriegens in a Bioelectrochemical System.", "to": "current densities up to 196 microamperes per square centimeter"}, {"from": "direct electron transfer", "title": "Anodic Respiration of Vibrio natriegens in a Bioelectrochemical System.", "to": "6.6 microamperes per square centimeter achieved within first 24 hours of cultivation"}, {"from": "mediated system", "title": "Anodic Respiration of Vibrio natriegens in a Bioelectrochemical System.", "to": "production of formate from glucose"}, {"from": "mediated system", "title": "Anodic Respiration of Vibrio natriegens in a Bioelectrochemical System.", "to": "production of acetate from glucose"}, {"from": "mediated system", "title": "Anodic Respiration of Vibrio natriegens in a Bioelectrochemical System.", "to": "production of succinate from glucose"}, {"from": "Vibrio natriegens", "title": "Anodic Respiration of Vibrio natriegens in a Bioelectrochemical System.", "to": "superior electron-transfer capabilities"}, {"from": "Vibrio natriegens", "title": "Anodic Respiration of Vibrio natriegens in a Bioelectrochemical System.", "to": "outstanding metabolism"}, {"from": "Mycobacterial aminoacylase gene", "title": "Chaperone assisted recombinant expression of a mycobacterial aminoacylase in Vibrio natriegens and Escherichia coli capable of N-lauroyl-L-amino acid synthesis.", "to": "Escherichia coli"}, {"from": "Mycobacterial aminoacylase gene", "title": "Chaperone assisted recombinant expression of a mycobacterial aminoacylase in Vibrio natriegens and Escherichia coli capable of N-lauroyl-L-amino acid synthesis.", "to": "Vibrio natriegens"}, {"from": "isopropyl \u03b2-d-1-thiogalactopyranoside", "title": "Chaperone assisted recombinant expression of a mycobacterial aminoacylase in Vibrio natriegens and Escherichia coli capable of N-lauroyl-L-amino acid synthesis.", "to": "soluble aminoacylase in Vibrio natriegens"}, {"from": "lactose", "title": "Chaperone assisted recombinant expression of a mycobacterial aminoacylase in Vibrio natriegens and Escherichia coli capable of N-lauroyl-L-amino acid synthesis.", "to": "soluble recombinant protein in Escherichia coli"}, {"from": "GroEL/S chaperones overexpression", "title": "Chaperone assisted recombinant expression of a mycobacterial aminoacylase in Vibrio natriegens and Escherichia coli capable of N-lauroyl-L-amino acid synthesis.", "to": "increased aminoacylase activity in Vibrio natriegens"}, {"from": "GroEL/S chaperones overexpression", "title": "Chaperone assisted recombinant expression of a mycobacterial aminoacylase in Vibrio natriegens and Escherichia coli capable of N-lauroyl-L-amino acid synthesis.", "to": "increased aminoacylase activity in Escherichia coli"}, {"from": "Cpn60/10 chaperonins co-expression at 12 Celsius", "title": "Chaperone assisted recombinant expression of a mycobacterial aminoacylase in Vibrio natriegens and Escherichia coli capable of N-lauroyl-L-amino acid synthesis.", "to": "increased aminoacylase activity in Escherichia coli ArcticExpress"}, {"from": "pH 7.0", "title": "Chaperone assisted recombinant expression of a mycobacterial aminoacylase in Vibrio natriegens and Escherichia coli capable of N-lauroyl-L-amino acid synthesis.", "to": "stable aminoacylase activity"}, {"from": "70 Celsius", "title": "Chaperone assisted recombinant expression of a mycobacterial aminoacylase in Vibrio natriegens and Escherichia coli capable of N-lauroyl-L-amino acid synthesis.", "to": "maximum aminoacylase activity"}, {"from": "40 Celsius and pH 7.0 for 5 days", "title": "Chaperone assisted recombinant expression of a mycobacterial aminoacylase in Vibrio natriegens and Escherichia coli capable of N-lauroyl-L-amino acid synthesis.", "to": "aminoacylase stability"}, {"from": "short-chain acyl-amino acids preference", "title": "Chaperone assisted recombinant expression of a mycobacterial aminoacylase in Vibrio natriegens and Escherichia coli capable of N-lauroyl-L-amino acid synthesis.", "to": "aminoacylase activity"}, {"from": "N-lauroyl-L-methionine", "title": "Chaperone assisted recombinant expression of a mycobacterial aminoacylase in Vibrio natriegens and Escherichia coli capable of N-lauroyl-L-amino acid synthesis.", "to": "aminoacylase hydrolysis acceptance"}, {"from": "lauric acid and methionine in aqueous system", "title": "Chaperone assisted recombinant expression of a mycobacterial aminoacylase in Vibrio natriegens and Escherichia coli capable of N-lauroyl-L-amino acid synthesis.", "to": "lauroyl-methionine synthesis"}, {"from": "low-temperature range preservation", "title": "Transcriptomic responses of the fast-growing bacterium Vibrio natriegens during cold-induced loss of culturability.", "to": "DNA damage"}, {"from": "low-temperature range preservation", "title": "Transcriptomic responses of the fast-growing bacterium Vibrio natriegens during cold-induced loss of culturability.", "to": "suppression of respiratory electron transport chain"}, {"from": "increase in reactive oxygen species", "title": "Transcriptomic responses of the fast-growing bacterium Vibrio natriegens during cold-induced loss of culturability.", "to": "cold-induced loss of culturability"}, {"from": "volume of liquid culture medium", "title": "Transcriptomic responses of the fast-growing bacterium Vibrio natriegens during cold-induced loss of culturability.", "to": "relationship between cold-induced loss of culturability and oxygen availability"}, {"from": "deletion of Sodium-translocating NADH-ubiquinone oxidoreductase", "title": "Transcriptomic responses of the fast-growing bacterium Vibrio natriegens during cold-induced loss of culturability.", "to": "accelerated cold-induced loss of culturability"}, {"from": "deletion of Sodium-translocating NADH-ubiquinone oxidoreductase", "title": "Transcriptomic responses of the fast-growing bacterium Vibrio natriegens during cold-induced loss of culturability.", "to": "increase in intracellular reactive oxygen species levels"}, {"from": "marine bacteria", "title": "Taking Synthetic Biology to the Seas: From Blue Chassis Organisms to Marine Aquaforming.", "to": "synthetic biology"}, {"from": "marine bacteria", "title": "Taking Synthetic Biology to the Seas: From Blue Chassis Organisms to Marine Aquaforming.", "to": "metabolic engineering"}, {"from": "Vibrio natriegens", "title": "Taking Synthetic Biology to the Seas: From Blue Chassis Organisms to Marine Aquaforming.", "to": "emerging marine chassis organisms"}, {"from": "Halomonas bluephagenesis", "title": "Taking Synthetic Biology to the Seas: From Blue Chassis Organisms to Marine Aquaforming.", "to": "emerging marine chassis organisms"}, {"from": "genetically enhanced microbes", "title": "Taking Synthetic Biology to the Seas: From Blue Chassis Organisms to Marine Aquaforming.", "to": "valorization of algal polysaccharides"}, {"from": "synthetic biology", "title": "Taking Synthetic Biology to the Seas: From Blue Chassis Organisms to Marine Aquaforming.", "to": "tackling climate change"}, {"from": "synthetic biology", "title": "Taking Synthetic Biology to the Seas: From Blue Chassis Organisms to Marine Aquaforming.", "to": "tackling marine pollution"}, {"from": "Gram-negative bacterium", "title": "Vibrio natriegens genome-scale modeling reveals insights into halophilic adaptations and resource allocation.", "to": "Vibrio natriegens"}, {"from": "gram-negative bacterium", "title": "Vibrio natriegens genome-scale modeling reveals insights into halophilic adaptations and resource allocation.", "to": "exceptional high growth rates"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens genome-scale modeling reveals insights into halophilic adaptations and resource allocation.", "to": "standard biotechnological host"}, {"from": "genome-scale metabolic model", "title": "Vibrio natriegens genome-scale modeling reveals insights into halophilic adaptations and resource allocation.", "to": "Vibrio natriegens"}, {"from": "iLC858", "title": "Vibrio natriegens genome-scale modeling reveals insights into halophilic adaptations and resource allocation.", "to": "automated draft assembly"}, {"from": "iLC858", "title": "Vibrio natriegens genome-scale modeling reveals insights into halophilic adaptations and resource allocation.", "to": "extensive manual curation"}, {"from": "iLC858", "title": "Vibrio natriegens genome-scale modeling reveals insights into halophilic adaptations and resource allocation.", "to": "predicted yields"}, {"from": "iLC858", "title": "Vibrio natriegens genome-scale modeling reveals insights into halophilic adaptations and resource allocation.", "to": "central metabolic fluxes"}, {"from": "iLC858", "title": "Vibrio natriegens genome-scale modeling reveals insights into halophilic adaptations and resource allocation.", "to": "viable carbon substrates"}, {"from": "iLC858", "title": "Vibrio natriegens genome-scale modeling reveals insights into halophilic adaptations and resource allocation.", "to": "essential genes"}, {"from": "mass spectrometry-based proteomics data", "title": "Vibrio natriegens genome-scale modeling reveals insights into halophilic adaptations and resource allocation.", "to": "enzyme-encoding genes expression"}, {"from": "iLC858", "title": "Vibrio natriegens genome-scale modeling reveals insights into halophilic adaptations and resource allocation.", "to": "metabolic comparison between Escherichia coli and Vibrio natriegens"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens genome-scale modeling reveals insights into halophilic adaptations and resource allocation.", "to": "analysis of model architecture of respiratory and ATP-generating system"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens genome-scale modeling reveals insights into halophilic adaptations and resource allocation.", "to": "sodium-dependent oxaloacetate decarboxylase pump discovery"}, {"from": "mass spectrometry-based proteomics data", "title": "Vibrio natriegens genome-scale modeling reveals insights into halophilic adaptations and resource allocation.", "to": "investigation of halophilic adaptations"}, {"from": "iLC858", "title": "Vibrio natriegens genome-scale modeling reveals insights into halophilic adaptations and resource allocation.", "to": "Resource Balance Analysis model creation"}, {"from": "iLC858", "title": "Vibrio natriegens genome-scale modeling reveals insights into halophilic adaptations and resource allocation.", "to": "study of allocation of carbon resources"}, {"from": "Vibrio natriegens", "title": "Identification of Emerging Industrial Biotechnology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Novel High Salt-Tolerant and Feedstock Flexibility Electroactive Microorganism for Microbial Fuel Cell.", "to": "microbial fuel cells"}, {"from": "electroactive industrial microorganisms", "title": "Identification of Emerging Industrial Biotechnology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Novel High Salt-Tolerant and Feedstock Flexibility Electroactive Microorganism for Microbial Fuel Cell.", "to": "bioproduct and electricity production"}, {"from": "Vibrio natriegens", "title": "Identification of Emerging Industrial Biotechnology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Novel High Salt-Tolerant and Feedstock Flexibility Electroactive Microorganism for Microbial Fuel Cell.", "to": "next-generation industrial microorganism chassis"}, {"from": "Vibrio natriegens", "title": "Identification of Emerging Industrial Biotechnology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Novel High Salt-Tolerant and Feedstock Flexibility Electroactive Microorganism for Microbial Fuel Cell.", "to": "insufficient secretion of electron mediators"}, {"from": "electron mediators", "title": "Identification of Emerging Industrial Biotechnology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Novel High Salt-Tolerant and Feedstock Flexibility Electroactive Microorganism for Microbial Fuel Cell.", "to": "electronic transfer efficiency of Vibrio natriegens microbial fuel cells"}, {"from": "media and carbon sources", "title": "Identification of Emerging Industrial Biotechnology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Novel High Salt-Tolerant and Feedstock Flexibility Electroactive Microorganism for Microbial Fuel Cell.", "to": "bioelectricity generation performance of Vibrio natriegens microbial fuel cells"}, {"from": "1% glycerol", "title": "Identification of Emerging Industrial Biotechnology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Novel High Salt-Tolerant and Feedstock Flexibility Electroactive Microorganism for Microbial Fuel Cell.", "to": "highest power density in Vibrio natriegens microbial fuel cells"}, {"from": "Vibrio natriegens", "title": "Identification of Emerging Industrial Biotechnology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Novel High Salt-Tolerant and Feedstock Flexibility Electroactive Microorganism for Microbial Fuel Cell.", "to": "phenazine-1-carboxamide synthesis"}, {"from": "Vibrio natriegens", "title": "Identification of Emerging Industrial Biotechnology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Novel High Salt-Tolerant and Feedstock Flexibility Electroactive Microorganism for Microbial Fuel Cell.", "to": "phenazine-1-carboxylic acid synthesis"}, {"from": "Vibrio natriegens", "title": "Identification of Emerging Industrial Biotechnology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Novel High Salt-Tolerant and Feedstock Flexibility Electroactive Microorganism for Microbial Fuel Cell.", "to": "1-hydroxyphenazine synthesis"}, {"from": "shikimate pathway", "title": "Identification of Emerging Industrial Biotechnology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Novel High Salt-Tolerant and Feedstock Flexibility Electroactive Microorganism for Microbial Fuel Cell.", "to": "phenazine-1-carboxamide synthesis"}, {"from": "phenazine synthesis and modification pathways", "title": "Identification of Emerging Industrial Biotechnology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Novel High Salt-Tolerant and Feedstock Flexibility Electroactive Microorganism for Microbial Fuel Cell.", "to": "phenazine-1-carboxylic acid synthesis"}, {"from": "phenazine synthesis and modification pathways", "title": "Identification of Emerging Industrial Biotechnology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Novel High Salt-Tolerant and Feedstock Flexibility Electroactive Microorganism for Microbial Fuel Cell.", "to": "1-hydroxyphenazine synthesis"}, {"from": "Vibrio natriegens", "title": "Identification of Emerging Industrial Biotechnology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Novel High Salt-Tolerant and Feedstock Flexibility Electroactive Microorganism for Microbial Fuel Cell.", "to": "high salt-tolerant electroactive microorganism"}, {"from": "Vibrio natriegens", "title": "Identification of Emerging Industrial Biotechnology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Novel High Salt-Tolerant and Feedstock Flexibility Electroactive Microorganism for Microbial Fuel Cell.", "to": "feedstock flexibility electroactive microorganism"}, {"from": "Vibrio natriegens", "title": "Identification of Emerging Industrial Biotechnology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Novel High Salt-Tolerant and Feedstock Flexibility Electroactive Microorganism for Microbial Fuel Cell.", "to": "microbial electrofermentation application potential"}, {"from": "in silico exploration", "title": "Discovery and Enzymatic Screening of Genome-Mined Microbial Levanases to Produce Second-Generation \u03b2-(2,6)-Fructooligosaccharides: Catalytic Properties.", "to": "levanase family biodiversity"}, {"from": "sequence-driven approach", "title": "Discovery and Enzymatic Screening of Genome-Mined Microbial Levanases to Produce Second-Generation \u03b2-(2,6)-Fructooligosaccharides: Catalytic Properties.", "to": "levanase family biodiversity"}, {"from": "levanases", "title": "Discovery and Enzymatic Screening of Genome-Mined Microbial Levanases to Produce Second-Generation \u03b2-(2,6)-Fructooligosaccharides: Catalytic Properties.", "to": "hydrolyze low-molecular-weight levan"}, {"from": "levanases", "title": "Discovery and Enzymatic Screening of Genome-Mined Microbial Levanases to Produce Second-Generation \u03b2-(2,6)-Fructooligosaccharides: Catalytic Properties.", "to": "hydrolyze high-molecular-weight levan"}, {"from": "levanases", "title": "Discovery and Enzymatic Screening of Genome-Mined Microbial Levanases to Produce Second-Generation \u03b2-(2,6)-Fructooligosaccharides: Catalytic Properties.", "to": "thermal stability"}, {"from": "active levanases", "title": "Discovery and Enzymatic Screening of Genome-Mined Microbial Levanases to Produce Second-Generation \u03b2-(2,6)-Fructooligosaccharides: Catalytic Properties.", "to": "preferential hydrolysis of levan over inulin"}, {"from": "levanase hydrolysis", "title": "Discovery and Enzymatic Screening of Genome-Mined Microbial Levanases to Produce Second-Generation \u03b2-(2,6)-Fructooligosaccharides: Catalytic Properties.", "to": "levanbiose production"}, {"from": "levanase hydrolysis", "title": "Discovery and Enzymatic Screening of Genome-Mined Microbial Levanases to Produce Second-Generation \u03b2-(2,6)-Fructooligosaccharides: Catalytic Properties.", "to": "levantriose production"}, {"from": "levanase hydrolysis", "title": "Discovery and Enzymatic Screening of Genome-Mined Microbial Levanases to Produce Second-Generation \u03b2-(2,6)-Fructooligosaccharides: Catalytic Properties.", "to": "levantetraose production"}, {"from": "LEV9 from Belliella Baltica", "title": "Discovery and Enzymatic Screening of Genome-Mined Microbial Levanases to Produce Second-Generation \u03b2-(2,6)-Fructooligosaccharides: Catalytic Properties.", "to": "catalytic properties characterized"}, {"from": "LEV36 from Dyadobacter fermentans", "title": "Discovery and Enzymatic Screening of Genome-Mined Microbial Levanases to Produce Second-Generation \u03b2-(2,6)-Fructooligosaccharides: Catalytic Properties.", "to": "catalytic properties characterized"}, {"from": "LEV37 from Capnocytophaga ochracea", "title": "Discovery and Enzymatic Screening of Genome-Mined Microbial Levanases to Produce Second-Generation \u03b2-(2,6)-Fructooligosaccharides: Catalytic Properties.", "to": "catalytic properties characterized"}, {"from": "LEV79 from Vibrio natriegens", "title": "Discovery and Enzymatic Screening of Genome-Mined Microbial Levanases to Produce Second-Generation \u03b2-(2,6)-Fructooligosaccharides: Catalytic Properties.", "to": "catalytic properties characterized"}, {"from": "LEV91 from Paenarthrobacter aurescens", "title": "Discovery and Enzymatic Screening of Genome-Mined Microbial Levanases to Produce Second-Generation \u03b2-(2,6)-Fructooligosaccharides: Catalytic Properties.", "to": "catalytic properties characterized"}, {"from": "novel levanases identification", "title": "Discovery and Enzymatic Screening of Genome-Mined Microbial Levanases to Produce Second-Generation \u03b2-(2,6)-Fructooligosaccharides: Catalytic Properties.", "to": "production of levan-type FOSs"}, {"from": "microbial colony growth", "title": "Use of the speckle imaging sub-pixel correlation analysis in revealing a mechanism of microbial colony growth.", "to": "activity of cells on edges"}, {"from": "speckle imaging technology", "title": "Use of the speckle imaging sub-pixel correlation analysis in revealing a mechanism of microbial colony growth.", "to": "visualization of zones of increased microbial activity"}, {"from": "laser speckle imaging technique", "title": "Use of the speckle imaging sub-pixel correlation analysis in revealing a mechanism of microbial colony growth.", "to": "record of microbial colony growth"}, {"from": "laser speckle imaging technique", "title": "Use of the speckle imaging sub-pixel correlation analysis in revealing a mechanism of microbial colony growth.", "to": "visualization of microbial growth activity"}, {"from": "speckle imaging technique", "title": "Use of the speckle imaging sub-pixel correlation analysis in revealing a mechanism of microbial colony growth.", "to": "visualization of zones of increased microbial activity"}, {"from": "Vibrio natriegens", "title": "Use of the speckle imaging sub-pixel correlation analysis in revealing a mechanism of microbial colony growth.", "to": "laser speckle imaging technique effectiveness"}, {"from": "Escherichia coli", "title": "Use of the speckle imaging sub-pixel correlation analysis in revealing a mechanism of microbial colony growth.", "to": "laser speckle imaging technique effectiveness"}, {"from": "Staphylococcus aureus", "title": "Use of the speckle imaging sub-pixel correlation analysis in revealing a mechanism of microbial colony growth.", "to": "laser speckle imaging technique effectiveness"}, {"from": "speckle analysis system", "title": "Use of the speckle imaging sub-pixel correlation analysis in revealing a mechanism of microbial colony growth.", "to": "record of microbial colony growth"}, {"from": "speckle analysis system", "title": "Use of the speckle imaging sub-pixel correlation analysis in revealing a mechanism of microbial colony growth.", "to": "visualization of microbial growth activity"}, {"from": "Synechococcus elongatus", "title": "Engineering Photosynthetic Microbial Consortia for Carbon-Negative Biosynthesis.", "to": "carbon dioxide sequestration module"}, {"from": "naturally competent Vibrio natriegens", "title": "Engineering Photosynthetic Microbial Consortia for Carbon-Negative Biosynthesis.", "to": "sucrose utilization module"}, {"from": "Synechococcus elongatus", "title": "Engineering Photosynthetic Microbial Consortia for Carbon-Negative Biosynthesis.", "to": "naturally competent Vibrio natriegens"}, {"from": "carbon dioxide", "title": "Engineering Photosynthetic Microbial Consortia for Carbon-Negative Biosynthesis.", "to": "various valuable chemicals"}, {"from": "adaptation of the consensus -10 and -35 sequence boxes", "title": "Unlocking the strength of inducible promoters in Gram-negative bacteria.", "to": "improvement in strength of lac and tet promoters"}, {"from": "adaptation of the consensus -10 and -35 sequence boxes", "title": "Unlocking the strength of inducible promoters in Gram-negative bacteria.", "to": "improvement in control of lac and tet promoters"}, {"from": "adaptation of the consensus -10 and -35 sequence boxes", "title": "Unlocking the strength of inducible promoters in Gram-negative bacteria.", "to": "improvement in portability of lac and tet promoters"}, {"from": "independent control of the transcriptional regulators", "title": "Unlocking the strength of inducible promoters in Gram-negative bacteria.", "to": "improvement in strength of lac and tet promoters"}, {"from": "independent control of the transcriptional regulators", "title": "Unlocking the strength of inducible promoters in Gram-negative bacteria.", "to": "improvement in control of lac and tet promoters"}, {"from": "independent control of the transcriptional regulators", "title": "Unlocking the strength of inducible promoters in Gram-negative bacteria.", "to": "improvement in portability of lac and tet promoters"}, {"from": "incorporation of a strong ribosome binding site", "title": "Unlocking the strength of inducible promoters in Gram-negative bacteria.", "to": "improvement in strength of lac and tet promoters"}, {"from": "incorporation of a strong ribosome binding site", "title": "Unlocking the strength of inducible promoters in Gram-negative bacteria.", "to": "improvement in control of lac and tet promoters"}, {"from": "incorporation of a strong ribosome binding site", "title": "Unlocking the strength of inducible promoters in Gram-negative bacteria.", "to": "improvement in portability of lac and tet promoters"}, {"from": "use of the far-red fluorescent protein mCardinal", "title": "Unlocking the strength of inducible promoters in Gram-negative bacteria.", "to": "improvement in the signal-to-background ratio of promoter measurements"}, {"from": "production of cocaine esterase enzyme", "title": "Unlocking the strength of inducible promoters in Gram-negative bacteria.", "to": "validation of improvement in OFF state control and inducibility"}, {"from": "promoters portability", "title": "Unlocking the strength of inducible promoters in Gram-negative bacteria.", "to": "demonstration in Gram-negative species Pseudomonas putida and Vibrio natriegens"}, {"from": "Vibrio natriegens", "title": "Recombinant Protein Expression Chassis Library of \u003ci\u003eVibrio natriegens\u003c/i\u003e by Fine-Tuning the Expression of T7 RNA Polymerase.", "to": "fastest-growing bacteria"}, {"from": "Vibrio natriegens", "title": "Recombinant Protein Expression Chassis Library of \u003ci\u003eVibrio natriegens\u003c/i\u003e by Fine-Tuning the Expression of T7 RNA Polymerase.", "to": "doubling time less than 10 min"}, {"from": "T7 expression system", "title": "Recombinant Protein Expression Chassis Library of \u003ci\u003eVibrio natriegens\u003c/i\u003e by Fine-Tuning the Expression of T7 RNA Polymerase.", "to": "introduced into Vibrio natriegens"}, {"from": "T7 expression system", "title": "Recombinant Protein Expression Chassis Library of \u003ci\u003eVibrio natriegens\u003c/i\u003e by Fine-Tuning the Expression of T7 RNA Polymerase.", "to": "heterologous protein expression system"}, {"from": "commercial strain Vmax1", "title": "Recombinant Protein Expression Chassis Library of \u003ci\u003eVibrio natriegens\u003c/i\u003e by Fine-Tuning the Expression of T7 RNA Polymerase.", "to": "heterologous protein expression system"}, {"from": "variant VnDX", "title": "Recombinant Protein Expression Chassis Library of \u003ci\u003eVibrio natriegens\u003c/i\u003e by Fine-Tuning the Expression of T7 RNA Polymerase.", "to": "heterologous protein expression system"}, {"from": "Escherichia coli BL21(DE3)", "title": "Recombinant Protein Expression Chassis Library of \u003ci\u003eVibrio natriegens\u003c/i\u003e by Fine-Tuning the Expression of T7 RNA Polymerase.", "to": "backup expression chassis"}, {"from": "T7 RNA polymerase strength", "title": "Recombinant Protein Expression Chassis Library of \u003ci\u003eVibrio natriegens\u003c/i\u003e by Fine-Tuning the Expression of T7 RNA Polymerase.", "to": "not optimal for every recombinant protein"}, {"from": "promoter sequence change of T7 RNA polymerase", "title": "Recombinant Protein Expression Chassis Library of \u003ci\u003eVibrio natriegens\u003c/i\u003e by Fine-Tuning the Expression of T7 RNA Polymerase.", "to": "different expression strengths"}, {"from": "ribosome binding site sequence change of T7 RNA polymerase", "title": "Recombinant Protein Expression Chassis Library of \u003ci\u003eVibrio natriegens\u003c/i\u003e by Fine-Tuning the Expression of T7 RNA Polymerase.", "to": "different expression strengths"}, {"from": "VnDX variant library", "title": "Recombinant Protein Expression Chassis Library of \u003ci\u003eVibrio natriegens\u003c/i\u003e by Fine-Tuning the Expression of T7 RNA Polymerase.", "to": "fine-tuning T7 RNA polymerase"}, {"from": "industrially used enzyme glucose dehydrogenase", "title": "Recombinant Protein Expression Chassis Library of \u003ci\u003eVibrio natriegens\u003c/i\u003e by Fine-Tuning the Expression of T7 RNA Polymerase.", "to": "reporter protein"}, {"from": "variant VnDX-tet", "title": "Recombinant Protein Expression Chassis Library of \u003ci\u003eVibrio natriegens\u003c/i\u003e by Fine-Tuning the Expression of T7 RNA Polymerase.", "to": "increased glucose dehydrogenase activity by 109%"}, {"from": "promoter of T7 RNA polymerase changed from PlacUV5 to Ptet", "title": "Recombinant Protein Expression Chassis Library of \u003ci\u003eVibrio natriegens\u003c/i\u003e by Fine-Tuning the Expression of T7 RNA Polymerase.", "to": "increased glucose dehydrogenase activity"}, {"from": "VnDX-RBS12/pGDH", "title": "Recombinant Protein Expression Chassis Library of \u003ci\u003eVibrio natriegens\u003c/i\u003e by Fine-Tuning the Expression of T7 RNA Polymerase.", "to": "highest glucose dehydrogenase activity increased by 12.6%"}, {"from": "ribosome binding site sequence change", "title": "Recombinant Protein Expression Chassis Library of \u003ci\u003eVibrio natriegens\u003c/i\u003e by Fine-Tuning the Expression of T7 RNA Polymerase.", "to": "different T7 RNA polymerase translation levels"}, {"from": "VnDX variant library", "title": "Recombinant Protein Expression Chassis Library of \u003ci\u003eVibrio natriegens\u003c/i\u003e by Fine-Tuning the Expression of T7 RNA Polymerase.", "to": "choice for expressing various recombinant proteins"}, {"from": "Vibrio natriegens", "title": "Recombinant Protein Expression Chassis Library of \u003ci\u003eVibrio natriegens\u003c/i\u003e by Fine-Tuning the Expression of T7 RNA Polymerase.", "to": "expanded application for expressing various recombinant proteins"}, {"from": "Vibrio natriegens strain PWH3a", "title": "Complete Genome Sequence of Vibrio natriegens Strain PWH3a.", "to": "complete genome sequence"}, {"from": "Vibrio natriegens strain PWH3a", "title": "Complete Genome Sequence of Vibrio natriegens Strain PWH3a.", "to": "two circular chromosomes"}, {"from": "Vibrio natriegens strain PWH3a", "title": "Complete Genome Sequence of Vibrio natriegens Strain PWH3a.", "to": "4,650 coding sequences"}, {"from": "Vibrio natriegens strain PWH3a", "title": "Complete Genome Sequence of Vibrio natriegens Strain PWH3a.", "to": "34 rRNA"}, {"from": "Vibrio natriegens strain PWH3a", "title": "Complete Genome Sequence of Vibrio natriegens Strain PWH3a.", "to": "4 noncoding RNA"}, {"from": "Vibrio natriegens strain PWH3a", "title": "Complete Genome Sequence of Vibrio natriegens Strain PWH3a.", "to": "131 tRNA genes"}, {"from": "Vibrio natriegens strain PWH3a", "title": "Complete Genome Sequence of Vibrio natriegens Strain PWH3a.", "to": "one Mu-like prophage sequence"}, {"from": "functional promoters P1", "title": "Establishment of a salt-induced bioremediation platform from marine Vibrio natriegens.", "to": "Vibrio natriegens strain Vmax"}, {"from": "functional promoters P2-1", "title": "Establishment of a salt-induced bioremediation platform from marine Vibrio natriegens.", "to": "Vibrio natriegens strain Vmax"}, {"from": "functional promoters P2-2", "title": "Establishment of a salt-induced bioremediation platform from marine Vibrio natriegens.", "to": "Vibrio natriegens strain Vmax"}, {"from": "high salt concentrations", "title": "Establishment of a salt-induced bioremediation platform from marine Vibrio natriegens.", "to": "loss of microbial activities"}, {"from": "polyethylene terephthalate degradation", "title": "Establishment of a salt-induced bioremediation platform from marine Vibrio natriegens.", "to": "engineered strains"}, {"from": "chlorpyrifos degradation", "title": "Establishment of a salt-induced bioremediation platform from marine Vibrio natriegens.", "to": "engineered strains"}, {"from": "hexabromocyclododecanes degradation", "title": "Establishment of a salt-induced bioremediation platform from marine Vibrio natriegens.", "to": "engineered strains"}, {"from": "degradation rate", "title": "Establishment of a salt-induced bioremediation platform from marine Vibrio natriegens.", "to": "polyethylene terephthalate"}, {"from": "degradation rate", "title": "Establishment of a salt-induced bioremediation platform from marine Vibrio natriegens.", "to": "chlorpyrifos"}, {"from": "degradation rate", "title": "Establishment of a salt-induced bioremediation platform from marine Vibrio natriegens.", "to": "hexabromocyclododecanes"}, {"from": "expressing the chitin-binding protein GbpA", "title": "Establishment of a salt-induced bioremediation platform from marine Vibrio natriegens.", "to": "immobilization strategy"}, {"from": "marine bacteria V. natriegens Vmax", "title": "Establishment of a salt-induced bioremediation platform from marine Vibrio natriegens.", "to": "degradation of marine pollution"}, {"from": "multidrug-resistant Vibrio natriegens strain AbY-1805", "title": "Isolation and Characterization of a Novel Vibrio natriegens\u2014Infecting Phage and Its Potential Therapeutic Application in Abalone Aquaculture.", "to": "death of juvenile Pacific abalones"}, {"from": "lytic phage vB_VnaS-L3", "title": "Isolation and Characterization of a Novel Vibrio natriegens\u2014Infecting Phage and Its Potential Therapeutic Application in Abalone Aquaculture.", "to": "infect Vibrio natriegens AbY-1805"}, {"from": "lytic phage vB_VnaS-L3", "title": "Isolation and Characterization of a Novel Vibrio natriegens\u2014Infecting Phage and Its Potential Therapeutic Application in Abalone Aquaculture.", "to": "short latent period"}, {"from": "lytic phage vB_VnaS-L3", "title": "Isolation and Characterization of a Novel Vibrio natriegens\u2014Infecting Phage and Its Potential Therapeutic Application in Abalone Aquaculture.", "to": "large burst size"}, {"from": "treatment with vB_VnaS-L3", "title": "Isolation and Characterization of a Novel Vibrio natriegens\u2014Infecting Phage and Its Potential Therapeutic Application in Abalone Aquaculture.", "to": "reduced mortality of juvenile abalones"}, {"from": "treatment with vB_VnaS-L3", "title": "Isolation and Characterization of a Novel Vibrio natriegens\u2014Infecting Phage and Its Potential Therapeutic Application in Abalone Aquaculture.", "to": "maintained abalone feeding capacity"}, {"from": "lytic phage vB_VnaS-L3", "title": "Isolation and Characterization of a Novel Vibrio natriegens\u2014Infecting Phage and Its Potential Therapeutic Application in Abalone Aquaculture.", "to": "novel marine Siphoviridae-family phage"}, {"from": "lytic phage vB_VnaS-L3", "title": "Isolation and Characterization of a Novel Vibrio natriegens\u2014Infecting Phage and Its Potential Therapeutic Application in Abalone Aquaculture.", "to": "narrow host range"}, {"from": "lytic phage vB_VnaS-L3", "title": "Isolation and Characterization of a Novel Vibrio natriegens\u2014Infecting Phage and Its Potential Therapeutic Application in Abalone Aquaculture.", "to": "viability at wide range of pH"}, {"from": "lytic phage vB_VnaS-L3", "title": "Isolation and Characterization of a Novel Vibrio natriegens\u2014Infecting Phage and Its Potential Therapeutic Application in Abalone Aquaculture.", "to": "viability at wide range of temperature"}, {"from": "lytic phage vB_VnaS-L3", "title": "Isolation and Characterization of a Novel Vibrio natriegens\u2014Infecting Phage and Its Potential Therapeutic Application in Abalone Aquaculture.", "to": "viability at wide range of salinity"}, {"from": "stepwise metabolic engineering", "title": "A Highly Compatible Phototrophic Community for Carbon-Negative Biosynthesis.", "to": "cyanobacteria CO2 sequestration module"}, {"from": "cyanobacteria CO2 sequestration module", "title": "A Highly Compatible Phototrophic Community for Carbon-Negative Biosynthesis.", "to": "super-coupled module"}, {"from": "super-coupled module", "title": "A Highly Compatible Phototrophic Community for Carbon-Negative Biosynthesis.", "to": "Vibrio natriegens"}, {"from": "enhanced photosynthetic electron transport", "title": "A Highly Compatible Phototrophic Community for Carbon-Negative Biosynthesis.", "to": "intercellular communication"}, {"from": "extracellular vesicles", "title": "A Highly Compatible Phototrophic Community for Carbon-Negative Biosynthesis.", "to": "intercellular communication"}, {"from": "HCPC", "title": "A Highly Compatible Phototrophic Community for Carbon-Negative Biosynthesis.", "to": "channeling CO2 into valuable chemicals"}, {"from": "channeling CO2 into valuable chemicals", "title": "A Highly Compatible Phototrophic Community for Carbon-Negative Biosynthesis.", "to": "overall release of -22.27 to -606.59 kgCO2 e kg-1 in the end products"}, {"from": "Vibrio natriegens WPAGA4", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "16S rRNA gene sequence"}, {"from": "Vibrio natriegens WPAGA4", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "average nucleotide identity"}, {"from": "Vibrio natriegens WPAGA4", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "digital DNA-DNA hybridization"}, {"from": "Vibrio natriegens WPAGA4", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "4561 coding genes"}, {"from": "Vibrio natriegens WPAGA4", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "37 rRNA"}, {"from": "Vibrio natriegens WPAGA4", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "131 tRNA"}, {"from": "Vibrio natriegens WPAGA4", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "59 other non-coding RNA genes"}, {"from": "GH50 agarase gene", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "expression in Escherichia coli cells"}, {"from": "recombinant Aga3420", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "optimum temperature range of 40 Celsius"}, {"from": "recombinant Aga3420", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "optimum pH of 7.0"}, {"from": "recombinant Aga3420", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "product Neoagarobiose during agarose degradation"}, {"from": "recombinant Aga3420", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "stability at 10-30 Celsius for 50 min"}, {"from": "recombinant Aga3420", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "Km value of 2.8 mg/mL"}, {"from": "recombinant Aga3420", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "Vmax value of 78.1 U/mg"}, {"from": "recombinant Aga3420", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "kcat value of 376.9 s^-1"}, {"from": "recombinant Aga3420", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "relative activity of 59.7% at 10 Celsius"}, {"from": "recombinant Aga3420", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "relative activity of 41.2% at 0 Celsius"}, {"from": "Vibrio natriegens WPAGA4", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "capability to degrade and metabolize agarose in cold deep-sea environments"}, {"from": "recombinant Aga3420", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "suitability for industrial enzyme in NA2 production"}, {"from": "recombinant Aga3420", "title": "Agarose-Degrading Characteristics of a Deep-Sea Bacterium Vibrio Natriegens WPAGA4 and Its Cold-Adapted GH50 Agarase Aga3420.", "to": "industrial potential in medical and cosmetic fields"}, {"from": "Vibrio natriegens", "title": "Genome-Scale Modeling and Systems Metabolic Engineering of Vibrio natriegens for the Production of 1,3-Propanediol.", "to": "genome-scale metabolic models"}, {"from": "Vibrio natriegens", "title": "Genome-Scale Modeling and Systems Metabolic Engineering of Vibrio natriegens for the Production of 1,3-Propanediol.", "to": "genome-editing protocols"}, {"from": "metabolic engineering of Vibrio natriegens", "title": "Genome-Scale Modeling and Systems Metabolic Engineering of Vibrio natriegens for the Production of 1,3-Propanediol.", "to": "production of 1,3-propanediol"}, {"from": "nick/gap repair and negative supercoiling reaction", "title": "Enzymatic Supercoiling of Bacterial Chromosomes Facilitates Genome Manipulation.", "to": "negative supercoiling of the whole genomes extracted from Escherichia coli"}, {"from": "nick/gap repair and negative supercoiling reaction", "title": "Enzymatic Supercoiling of Bacterial Chromosomes Facilitates Genome Manipulation.", "to": "negative supercoiling of the whole genomes extracted from Vibrio natriegens"}, {"from": "Escherichia coli replication-cycle reaction", "title": "Enzymatic Supercoiling of Bacterial Chromosomes Facilitates Genome Manipulation.", "to": "enzymatic replication of 1-2 megabase chromosomes"}, {"from": "modified nick/gap repair and negative supercoiling reaction performed at a low salt concentration", "title": "Enzymatic Supercoiling of Bacterial Chromosomes Facilitates Genome Manipulation.", "to": "electroporation-ready 1 megabase chromosomes"}, {"from": "portable chromosome format", "title": "Enzymatic Supercoiling of Bacterial Chromosomes Facilitates Genome Manipulation.", "to": "genetically stable 1 megabase chromosomes"}, {"from": "Vibrio natriegens", "title": "Non-Sterilized Fermentation of 2,3-Butanediol with Seawater by Metabolic Engineered Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "low nutritional requirements"}, {"from": "Vibrio natriegens", "title": "Non-Sterilized Fermentation of 2,3-Butanediol with Seawater by Metabolic Engineered Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "high salt tolerance"}, {"from": "Vibrio natriegens", "title": "Non-Sterilized Fermentation of 2,3-Butanediol with Seawater by Metabolic Engineered Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "rapid growth rate"}, {"from": "Vibrio natriegens", "title": "Non-Sterilized Fermentation of 2,3-Butanediol with Seawater by Metabolic Engineered Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "metabolic engineering"}, {"from": "deleting two byproduct encoding genes", "title": "Non-Sterilized Fermentation of 2,3-Butanediol with Seawater by Metabolic Engineered Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "2,3-butanediol production"}, {"from": "non-sterilized fermentation", "title": "Non-Sterilized Fermentation of 2,3-Butanediol with Seawater by Metabolic Engineered Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "2,3-butanediol production"}, {"from": "seawater-based minimal medium", "title": "Non-Sterilized Fermentation of 2,3-Butanediol with Seawater by Metabolic Engineered Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "2,3-butanediol production"}, {"from": "optimized fermentative conditions", "title": "Non-Sterilized Fermentation of 2,3-Butanediol with Seawater by Metabolic Engineered Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "2,3-butanediol production of 41.27 grams per liter"}, {"from": "optimized fermentative conditions", "title": "Non-Sterilized Fermentation of 2,3-Butanediol with Seawater by Metabolic Engineered Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "productivity of 3.44 grams per liter per hour"}, {"from": "optimized fermentative conditions", "title": "Non-Sterilized Fermentation of 2,3-Butanediol with Seawater by Metabolic Engineered Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "yield of 0.39 grams per gram glucose"}, {"from": "recombinant strain Vibrio natriegens Delta frdA Delta ldhA-pETRABC", "title": "Non-Sterilized Fermentation of 2,3-Butanediol with Seawater by Metabolic Engineered Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "2,3-butanediol production of 41.27 grams per liter"}, {"from": "recombinant strain Vibrio natriegens Delta frdA Delta ldhA-pETRABC", "title": "Non-Sterilized Fermentation of 2,3-Butanediol with Seawater by Metabolic Engineered Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "productivity of 3.44 grams per liter per hour"}, {"from": "recombinant strain Vibrio natriegens Delta frdA Delta ldhA-pETRABC", "title": "Non-Sterilized Fermentation of 2,3-Butanediol with Seawater by Metabolic Engineered Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "yield of 0.39 grams per gram glucose"}, {"from": "Clustered regularly interspaced short palindromic repeat/CRISPR-associated gene systems", "title": "CRISPR-Cas9 Based Bacteriophage Genome Editing.", "to": "bacteriophages"}, {"from": "Clustered regularly interspaced short palindromic repeat/CRISPR-associated gene systems", "title": "CRISPR-Cas9 Based Bacteriophage Genome Editing.", "to": "study bacteriophage functional genomics"}, {"from": "bacteriophages", "title": "CRISPR-Cas9 Based Bacteriophage Genome Editing.", "to": "evolution of prokaryote CRISPR arrays"}, {"from": "phage Genome edit by natural transformation platform", "title": "CRISPR-Cas9 Based Bacteriophage Genome Editing.", "to": "phage gene diversity"}, {"from": "phage Genome edit by natural transformation platform", "title": "CRISPR-Cas9 Based Bacteriophage Genome Editing.", "to": "development of phage synthetic biology"}, {"from": "phage Genome edit by natural transformation platform", "title": "CRISPR-Cas9 Based Bacteriophage Genome Editing.", "to": "development of nanotechnology"}, {"from": "phage Genome edit by natural transformation platform", "title": "CRISPR-Cas9 Based Bacteriophage Genome Editing.", "to": "discovery of new molecular biology tools"}, {"from": "phage Genome edit by natural transformation platform", "title": "CRISPR-Cas9 Based Bacteriophage Genome Editing.", "to": "efficiency and accuracy of phage TT4P2 gene editing"}, {"from": "Clustered regularly interspaced short palindromic repeat/CRISPR-associated gene systems", "title": "CRISPR-Cas9 Based Bacteriophage Genome Editing.", "to": "target and cleave invaders"}, {"from": "CRISPR SWAPnDROP", "title": "A multifunctional system for genome editing and large-scale interspecies gene transfer.", "to": "large-scale in-vivo DNA transfer between bacterial species"}, {"from": "CRISPR SWAPnDROP", "title": "A multifunctional system for genome editing and large-scale interspecies gene transfer.", "to": "Genome edit by natural transformation in various species"}, {"from": "CRISPR SWAPnDROP", "title": "A multifunctional system for genome editing and large-scale interspecies gene transfer.", "to": "excision of large chromosomal regions"}, {"from": "CRISPR SWAPnDROP", "title": "A multifunctional system for genome editing and large-scale interspecies gene transfer.", "to": "transfer of large chromosomal regions"}, {"from": "CRISPR SWAPnDROP", "title": "A multifunctional system for genome editing and large-scale interspecies gene transfer.", "to": "integration of large chromosomal regions"}, {"from": "CRISPR SWAPnDROP", "title": "A multifunctional system for genome editing and large-scale interspecies gene transfer.", "to": "scarless Genome edit by natural transformation"}, {"from": "CRISPR SWAPnDROP", "title": "A multifunctional system for genome editing and large-scale interspecies gene transfer.", "to": "marker-free Genome edit by natural transformation"}, {"from": "CRISPR SWAPnDROP", "title": "A multifunctional system for genome editing and large-scale interspecies gene transfer.", "to": "iterative Genome edit by natural transformation"}, {"from": "CRISPR SWAPnDROP", "title": "A multifunctional system for genome editing and large-scale interspecies gene transfer.", "to": "parallel insertions and deletions in Genome edit by natural transformation"}, {"from": "CRISPR SWAPnDROP", "title": "A multifunctional system for genome editing and large-scale interspecies gene transfer.", "to": "DNA library applications"}, {"from": "CRISPR SWAPnDROP", "title": "A multifunctional system for genome editing and large-scale interspecies gene transfer.", "to": "recycling of standardized parts"}, {"from": "CRISPR SWAPnDROP multi-color scarless co-selection system", "title": "A multifunctional system for genome editing and large-scale interspecies gene transfer.", "to": "editing efficiency improvement"}, {"from": "CRISPR SWAPnDROP multi-color scarless co-selection system", "title": "A multifunctional system for genome editing and large-scale interspecies gene transfer.", "to": "visual quality controls"}, {"from": "Vibrio natriegens", "title": "Overexpression of recombinant proteins containing non-canonical amino acids in Vibrio natriegens: p-azido-L-phenylalanine as coupling site for \u003csup\u003e19\u003c/sup\u003eF-tags.", "to": "fastest growing organism"}, {"from": "Vibrio natriegens", "title": "Overexpression of recombinant proteins containing non-canonical amino acids in Vibrio natriegens: p-azido-L-phenylalanine as coupling site for \u003csup\u003e19\u003c/sup\u003eF-tags.", "to": "minimum doubling time of 9.4 minutes"}, {"from": "Vibrio natriegens", "title": "Overexpression of recombinant proteins containing non-canonical amino acids in Vibrio natriegens: p-azido-L-phenylalanine as coupling site for \u003csup\u003e19\u003c/sup\u003eF-tags.", "to": "utilize over 60 different carbon sources"}, {"from": "Vibrio natriegens", "title": "Overexpression of recombinant proteins containing non-canonical amino acids in Vibrio natriegens: p-azido-L-phenylalanine as coupling site for \u003csup\u003e19\u003c/sup\u003eF-tags.", "to": "non-pathogenic properties"}, {"from": "Vibrio natriegens", "title": "Overexpression of recombinant proteins containing non-canonical amino acids in Vibrio natriegens: p-azido-L-phenylalanine as coupling site for \u003csup\u003e19\u003c/sup\u003eF-tags.", "to": "alternative to E. coli"}, {"from": "engineered V. natriegens strain Vmax Express", "title": "Overexpression of recombinant proteins containing non-canonical amino acids in Vibrio natriegens: p-azido-L-phenylalanine as coupling site for \u003csup\u003e19\u003c/sup\u003eF-tags.", "to": "ability to incorporate non-canonical amino acid"}, {"from": "non-canonical amino acid p-azido-L-phenylalanine", "title": "Overexpression of recombinant proteins containing non-canonical amino acids in Vibrio natriegens: p-azido-L-phenylalanine as coupling site for \u003csup\u003e19\u003c/sup\u003eF-tags.", "to": "incorporation into recombinant proteins"}, {"from": "enhanced yellow fluorescent protein", "title": "Overexpression of recombinant proteins containing non-canonical amino acids in Vibrio natriegens: p-azido-L-phenylalanine as coupling site for \u003csup\u003e19\u003c/sup\u003eF-tags.", "to": "incorporation of p-azido-L-phenylalanine"}, {"from": "MlaC", "title": "Overexpression of recombinant proteins containing non-canonical amino acids in Vibrio natriegens: p-azido-L-phenylalanine as coupling site for \u003csup\u003e19\u003c/sup\u003eF-tags.", "to": "incorporation of p-azido-L-phenylalanine"}, {"from": "p-azido-L-phenylalanine incorporation into enhanced yellow fluorescent protein", "title": "Overexpression of recombinant proteins containing non-canonical amino acids in Vibrio natriegens: p-azido-L-phenylalanine as coupling site for \u003csup\u003e19\u003c/sup\u003eF-tags.", "to": "improved suppression efficiency"}, {"from": "p-azido-L-phenylalanine incorporation into enhanced yellow fluorescent protein", "title": "Overexpression of recombinant proteins containing non-canonical amino acids in Vibrio natriegens: p-azido-L-phenylalanine as coupling site for \u003csup\u003e19\u003c/sup\u003eF-tags.", "to": "protein titer"}, {"from": "expression levels of MlaC-p-azido-L-phenylalanine", "title": "Overexpression of recombinant proteins containing non-canonical amino acids in Vibrio natriegens: p-azido-L-phenylalanine as coupling site for \u003csup\u003e19\u003c/sup\u003eF-tags.", "to": "exceeded those of E. coli BL21 cells"}, {"from": "enhanced yellow fluorescent protein-p-azido-L-phenylalanine", "title": "Overexpression of recombinant proteins containing non-canonical amino acids in Vibrio natriegens: p-azido-L-phenylalanine as coupling site for \u003csup\u003e19\u003c/sup\u003eF-tags.", "to": "expression and isotopic labeling in minimal medium"}, {"from": "azido-group", "title": "Overexpression of recombinant proteins containing non-canonical amino acids in Vibrio natriegens: p-azido-L-phenylalanine as coupling site for \u003csup\u003e19\u003c/sup\u003eF-tags.", "to": "coupling site for NMR sensitive 19F-tags"}, {"from": "Vmax", "title": "Overexpression of recombinant proteins containing non-canonical amino acids in Vibrio natriegens: p-azido-L-phenylalanine as coupling site for \u003csup\u003e19\u003c/sup\u003eF-tags.", "to": "flexible expression host"}, {"from": "Vmax", "title": "Overexpression of recombinant proteins containing non-canonical amino acids in Vibrio natriegens: p-azido-L-phenylalanine as coupling site for \u003csup\u003e19\u003c/sup\u003eF-tags.", "to": "suitable for incorporation of non-canonical amino acids in recombinant proteins"}, {"from": "Vmax", "title": "Overexpression of recombinant proteins containing non-canonical amino acids in Vibrio natriegens: p-azido-L-phenylalanine as coupling site for \u003csup\u003e19\u003c/sup\u003eF-tags.", "to": "potential to surpass Protein yields of E. coli"}, {"from": "Vibrio natriegens", "title": "Overexpression of recombinant proteins containing non-canonical amino acids in Vibrio natriegens: p-azido-L-phenylalanine as coupling site for \u003csup\u003e19\u003c/sup\u003eF-tags.", "to": "expression of isotopically labeled proteins containing non-canonical amino acids"}, {"from": "isotopically labeled proteins containing non-canonical amino acids", "title": "Overexpression of recombinant proteins containing non-canonical amino acids in Vibrio natriegens: p-azido-L-phenylalanine as coupling site for \u003csup\u003e19\u003c/sup\u003eF-tags.", "to": "chemically modified for application in protein-observed 19F-NMR"}, {"from": "Vibrio natriegens", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "Genome editing"}, {"from": "Natural transformation", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "Incorporation of DNA into genome"}, {"from": "Natural transformation", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "Uptake of multiple DNA fragments"}, {"from": "Multiple DNA fragments", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "Multiple simultaneous edits"}, {"from": "NT-CRISPR", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "Combination of natural transformation with CRISPR-Cas9 counterselection"}, {"from": "Genome edit by natural transformation", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "Induction of Induction of CRISPR-Cas9 targeting wild type sequence"}, {"from": "Induction of CRISPR-Cas9 targeting wild type sequence", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "Death of non-edited cells"}, {"from": "Death of non-edited cells", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "High cell killing efficiencies"}, {"from": "High cell killing efficiencies", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "Dispensability of antibiotic resistance markers"}, {"from": "Dispensability of antibiotic resistance markers", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "Scarless edits"}, {"from": "Dispensability of antibiotic resistance markers", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "Markerless edits"}, {"from": "Scarless edits", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "Single-base precision"}, {"from": "Markerless edits", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "Single-base precision"}, {"from": "NT-CRISPR", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "Deletions"}, {"from": "NT-CRISPR", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "Integrations"}, {"from": "NT-CRISPR", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "Single-base modifications"}, {"from": "Editing efficiencies", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "Up to 100%"}, {"from": "NT-CRISPR", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "Simultaneous deletion of multiple chromosomal regions"}, {"from": "SpG Cas9", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "Compatibility with NT-CRISPR"}, {"from": "SpG Cas9", "title": "NT-CRISPR, combining natural transformation and CRISPR-Cas9 counterselection for markerless and scarless genome editing in Vibrio natriegens.", "to": "Broadens target spectrum"}, {"from": "Vibrio natriegens", "title": "Identification and Cross-Characterisation of Artificial Promoters and 5\u0027 Untranslated Regions in \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "synthetic biology applications"}, {"from": "Escherichia coli", "title": "Identification and Cross-Characterisation of Artificial Promoters and 5\u0027 Untranslated Regions in \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "Vibrio natriegens"}, {"from": "identification of artificial promoters portability and 5\u0027 untranslated regions", "title": "Identification and Cross-Characterisation of Artificial Promoters and 5\u0027 Untranslated Regions in \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "production of fluorescent proteins"}, {"from": "Vibrio natriegens", "title": "Identification and Cross-Characterisation of Artificial Promoters and 5\u0027 Untranslated Regions in \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "identification of artificial promoters portability and 5\u0027 untranslated regions"}, {"from": "Escherichia coli", "title": "Identification and Cross-Characterisation of Artificial Promoters and 5\u0027 Untranslated Regions in \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "identification of artificial promoters portability and 5\u0027 untranslated regions"}, {"from": "DNA sequence and motif analysis", "title": "Identification and Cross-Characterisation of Artificial Promoters and 5\u0027 Untranslated Regions in \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "artificial regulatory sequence"}, {"from": "DNA sequence and motif analysis", "title": "Identification and Cross-Characterisation of Artificial Promoters and 5\u0027 Untranslated Regions in \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "algorithms"}, {"from": "Vibrio natriegens", "title": "Rapid production of l-DOPA by Vibrio natriegens, an emerging next-generation whole-cell catalysis chassis.", "to": "rapid growth"}, {"from": "Vibrio natriegens", "title": "Rapid production of l-DOPA by Vibrio natriegens, an emerging next-generation whole-cell catalysis chassis.", "to": "high soluble protein expression system capacity"}, {"from": "Vibrio natriegens", "title": "Rapid production of l-DOPA by Vibrio natriegens, an emerging next-generation whole-cell catalysis chassis.", "to": "maximum soluble protein expression system at 37 degrees Celsius in 4 hours"}, {"from": "Escherichia coli", "title": "Rapid production of l-DOPA by Vibrio natriegens, an emerging next-generation whole-cell catalysis chassis.", "to": "maximum soluble protein expression system at 16 degrees Celsius in 16 hours"}, {"from": "Vibrio natriegens", "title": "Rapid production of l-DOPA by Vibrio natriegens, an emerging next-generation whole-cell catalysis chassis.", "to": "l-DOPA production"}, {"from": "Vibrio natriegens", "title": "Rapid production of l-DOPA by Vibrio natriegens, an emerging next-generation whole-cell catalysis chassis.", "to": "maximum productivity over 10.0 grams per liter per hour"}, {"from": "maximum productivity over 10.0 grams per liter per hour", "title": "Rapid production of l-DOPA by Vibrio natriegens, an emerging next-generation whole-cell catalysis chassis.", "to": "nearly two-fold higher than previously reported"}, {"from": "54.0 grams per liter l-DOPA", "title": "Rapid production of l-DOPA by Vibrio natriegens, an emerging next-generation whole-cell catalysis chassis.", "to": "catechol conversion rate greater than 95%"}, {"from": "Vibrio natriegens", "title": "Rapid production of l-DOPA by Vibrio natriegens, an emerging next-generation whole-cell catalysis chassis.", "to": "remarkable potential as a whole-cell catalysis chassis for the production of valuable chemicals"}, {"from": "metabolic engineering", "title": "Metabolic engineering of Vibrio natriegens for anaerobic succinate production.", "to": "high succinate titre"}, {"from": "metabolic engineering", "title": "Metabolic engineering of Vibrio natriegens for anaerobic succinate production.", "to": "high succinate yield"}, {"from": "metabolic engineering", "title": "Metabolic engineering of Vibrio natriegens for anaerobic succinate production.", "to": "high succinate productivity"}, {"from": "optimization of process conditions", "title": "Metabolic engineering of Vibrio natriegens for anaerobic succinate production.", "to": "high succinate titre"}, {"from": "optimization of process conditions", "title": "Metabolic engineering of Vibrio natriegens for anaerobic succinate production.", "to": "high succinate yield"}, {"from": "optimization of process conditions", "title": "Metabolic engineering of Vibrio natriegens for anaerobic succinate production.", "to": "high succinate productivity"}, {"from": "Vibrio natriegens", "title": "Metabolic engineering of Vibrio natriegens for anaerobic succinate production.", "to": "high biomass-specific succinate production rate"}, {"from": "Vibrio natriegens", "title": "Metabolic engineering of Vibrio natriegens for anaerobic succinate production.", "to": "succinate production"}, {"from": "reductive tricarboxylic acid cycle", "title": "Metabolic engineering of Vibrio natriegens for anaerobic succinate production.", "to": "succinate production"}, {"from": "oxidative tricarboxylic acid / glyoxylate pathway", "title": "Metabolic engineering of Vibrio natriegens for anaerobic succinate production.", "to": "succinate production"}, {"from": "minimal medium devoid of complex additives and expensive supplements", "title": "Metabolic engineering of Vibrio natriegens for anaerobic succinate production.", "to": "final succinate titre of 60.4 grams per liter"}, {"from": "minimal medium devoid of complex additives and expensive supplements", "title": "Metabolic engineering of Vibrio natriegens for anaerobic succinate production.", "to": "maximum succinate productivity of 20.8 grams per liter per hour"}, {"from": "minimal medium devoid of complex additives and expensive supplements", "title": "Metabolic engineering of Vibrio natriegens for anaerobic succinate production.", "to": "overall volumetric succinate productivity of 8.6 grams per liter per hour"}, {"from": "Vibrio natriegens", "title": "Biosensor-based isolation of amino acid-producing \u003ci\u003eVibrio natriegens\u003c/i\u003e strains.", "to": "promising new host for molecular biology and next generation bioprocesses"}, {"from": "Vibrio natriegens", "title": "Biosensor-based isolation of amino acid-producing \u003ci\u003eVibrio natriegens\u003c/i\u003e strains.", "to": "Gram-negative"}, {"from": "Vibrio natriegens", "title": "Biosensor-based isolation of amino acid-producing \u003ci\u003eVibrio natriegens\u003c/i\u003e strains.", "to": "slight-halophilic bacterium"}, {"from": "Vibrio natriegens", "title": "Biosensor-based isolation of amino acid-producing \u003ci\u003eVibrio natriegens\u003c/i\u003e strains.", "to": "high nutrient versatility"}, {"from": "Vibrio natriegens", "title": "Biosensor-based isolation of amino acid-producing \u003ci\u003eVibrio natriegens\u003c/i\u003e strains.", "to": "reported doubling time of under 10 minutes"}, {"from": "LysG", "title": "Biosensor-based isolation of amino acid-producing \u003ci\u003eVibrio natriegens\u003c/i\u003e strains.", "to": "expression in Vibrio natriegens"}, {"from": "LysG-based sensors", "title": "Biosensor-based isolation of amino acid-producing \u003ci\u003eVibrio natriegens\u003c/i\u003e strains.", "to": "detection of positively charged amino acids"}, {"from": "random mutagenesis", "title": "Biosensor-based isolation of amino acid-producing \u003ci\u003eVibrio natriegens\u003c/i\u003e strains.", "to": "LysG-based sensor used to screen for amino acid producer strains"}, {"from": "fluorescence-activated cell sorting", "title": "Biosensor-based isolation of amino acid-producing \u003ci\u003eVibrio natriegens\u003c/i\u003e strains.", "to": "sorting of sorting of highly fluorescent cells"}, {"from": "sorting of highly fluorescent cells", "title": "Biosensor-based isolation of amino acid-producing \u003ci\u003eVibrio natriegens\u003c/i\u003e strains.", "to": "potential amino acid producer cells"}, {"from": "amino acid producer strains", "title": "Biosensor-based isolation of amino acid-producing \u003ci\u003eVibrio natriegens\u003c/i\u003e strains.", "to": "producing L-lysine"}, {"from": "amino acid producer strains", "title": "Biosensor-based isolation of amino acid-producing \u003ci\u003eVibrio natriegens\u003c/i\u003e strains.", "to": "producing L-arginine"}, {"from": "amino acid producer strains", "title": "Biosensor-based isolation of amino acid-producing \u003ci\u003eVibrio natriegens\u003c/i\u003e strains.", "to": "producing L-histidine"}, {"from": "producer strains", "title": "Biosensor-based isolation of amino acid-producing \u003ci\u003eVibrio natriegens\u003c/i\u003e strains.", "to": "extracellularly producing up to 1 millimolar of effector amino acid"}, {"from": "genome sequencing of producer strains", "title": "Biosensor-based isolation of amino acid-producing \u003ci\u003eVibrio natriegens\u003c/i\u003e strains.", "to": "insight into the amino acid production metabolism of Vibrio natriegens"}, {"from": "expression and application of transcription factor-based biosensors in Vibrio natriegens", "title": "Biosensor-based isolation of amino acid-producing \u003ci\u003eVibrio natriegens\u003c/i\u003e strains.", "to": "insight into the underlying physiology"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens as a host for rapid biotechnology.", "to": "fast growth rate"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens as a host for rapid biotechnology.", "to": "doubling time of less than 10 min"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens as a host for rapid biotechnology.", "to": "high substrate uptake rates"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens as a host for rapid biotechnology.", "to": "metabolic prowess"}, {"from": "high substrate uptake rates", "title": "Vibrio natriegens as a host for rapid biotechnology.", "to": "next-generation workhorse for rapid molecular biology"}, {"from": "metabolic prowess", "title": "Vibrio natriegens as a host for rapid biotechnology.", "to": "next-generation workhorse for rapid molecular biology"}, {"from": "high substrate uptake rates", "title": "Vibrio natriegens as a host for rapid biotechnology.", "to": "next-generation workhorse for protein expression system"}, {"from": "metabolic prowess", "title": "Vibrio natriegens as a host for rapid biotechnology.", "to": "next-generation workhorse for protein expression system"}, {"from": "high substrate uptake rates", "title": "Vibrio natriegens as a host for rapid biotechnology.", "to": "next-generation workhorse for metabolic engineering"}, {"from": "metabolic prowess", "title": "Vibrio natriegens as a host for rapid biotechnology.", "to": "next-generation workhorse for metabolic engineering"}, {"from": "Vibrio natriegens Vmax X2", "title": "Genetic Code Expansion in the Engineered Organism Vmax X2: High Yield and Exceptional Fidelity.", "to": "unnatural amino acid mutagenesis"}, {"from": "Vibrio natriegens Vmax X2", "title": "Genetic Code Expansion in the Engineered Organism Vmax X2: High Yield and Exceptional Fidelity.", "to": "soluble Protein yields"}, {"from": "Vmax X2", "title": "Genetic Code Expansion in the Engineered Organism Vmax X2: High Yield and Exceptional Fidelity.", "to": "superfolder green fluorescent Protein yields"}, {"from": "superfolder green fluorescent Protein yields", "title": "Genetic Code Expansion in the Engineered Organism Vmax X2: High Yield and Exceptional Fidelity.", "to": "noncanonical \u03b1-amino acids"}, {"from": "superfolder green fluorescent Protein yields", "title": "Genetic Code Expansion in the Engineered Organism Vmax X2: High Yield and Exceptional Fidelity.", "to": "Top10"}, {"from": "superfolder green fluorescent Protein yields", "title": "Genetic Code Expansion in the Engineered Organism Vmax X2: High Yield and Exceptional Fidelity.", "to": "BL21"}, {"from": "superfolder green fluorescent Protein yields", "title": "Genetic Code Expansion in the Engineered Organism Vmax X2: High Yield and Exceptional Fidelity.", "to": "Escherichia coli C321.\u0394A.opt"}, {"from": "superfolder green fluorescent Protein yields", "title": "Genetic Code Expansion in the Engineered Organism Vmax X2: High Yield and Exceptional Fidelity.", "to": "Escherichia coli C321.\u0394A.exp"}, {"from": "Vmax X2", "title": "Genetic Code Expansion in the Engineered Organism Vmax X2: High Yield and Exceptional Fidelity.", "to": "misincorporated natural \u03b1-amino acids levels"}, {"from": "orthogonal aminoacyl tRNA synthetase", "title": "Genetic Code Expansion in the Engineered Organism Vmax X2: High Yield and Exceptional Fidelity.", "to": "misincorporated natural \u03b1-amino acids levels"}, {"from": "type VI secretion system", "title": "Engineering a customizable antibacterial T6SS-based platform in Vibrio natriegens.", "to": "Vibrio natriegens"}, {"from": "externally induced on/off switch", "title": "Engineering a customizable antibacterial T6SS-based platform in Vibrio natriegens.", "to": "type VI secretion system"}, {"from": "Vibrio natriegens", "title": "Engineering a customizable antibacterial T6SS-based platform in Vibrio natriegens.", "to": "effective antibacterial weapon"}, {"from": "delivered effector repertoire", "title": "Engineering a customizable antibacterial T6SS-based platform in Vibrio natriegens.", "to": "toxicity range"}, {"from": "Genetic engineering", "title": "Exploiting unconventional prokaryotic hosts for industrial biotechnology.", "to": "microbial metabolism"}, {"from": "Genetic engineering", "title": "Exploiting unconventional prokaryotic hosts for industrial biotechnology.", "to": "bioconversions"}, {"from": "Synthetic biology tools", "title": "Exploiting unconventional prokaryotic hosts for industrial biotechnology.", "to": "nonmodel organisms application"}, {"from": "nonmodel organisms application", "title": "Exploiting unconventional prokaryotic hosts for industrial biotechnology.", "to": "bio-based industry hosts"}, {"from": "Prokaryotic newcomers", "title": "Exploiting unconventional prokaryotic hosts for industrial biotechnology.", "to": "metabolic capabilities"}, {"from": "Prokaryotic newcomers", "title": "Exploiting unconventional prokaryotic hosts for industrial biotechnology.", "to": "cellular capabilities"}, {"from": "Prokaryotic newcomers", "title": "Exploiting unconventional prokaryotic hosts for industrial biotechnology.", "to": "industrial application potential"}, {"from": "Prokaryotic newcomers", "title": "Exploiting unconventional prokaryotic hosts for industrial biotechnology.", "to": "industrial application drawbacks"}, {"from": "Vibrio natriegens", "title": "The Marburg Collection: A Golden Gate DNA Assembly Framework for Synthetic Biology Applications in \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "chassis for synthetic and molecular biology"}, {"from": "Vibrio natriegens", "title": "The Marburg Collection: A Golden Gate DNA Assembly Framework for Synthetic Biology Applications in \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "Marburg Collection"}, {"from": "Marburg Collection", "title": "The Marburg Collection: A Golden Gate DNA Assembly Framework for Synthetic Biology Applications in \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "Golden Gate cloning toolbox"}, {"from": "Marburg Collection", "title": "The Marburg Collection: A Golden Gate DNA Assembly Framework for Synthetic Biology Applications in \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "de novo assembly of plasmids"}, {"from": "plasmid replication origin", "title": "The Marburg Collection: A Golden Gate DNA Assembly Framework for Synthetic Biology Applications in \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "de novo assembly of plasmids"}, {"from": "resistance part", "title": "The Marburg Collection: A Golden Gate DNA Assembly Framework for Synthetic Biology Applications in \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "de novo assembly of plasmids"}, {"from": "Marburg Collection", "title": "The Marburg Collection: A Golden Gate DNA Assembly Framework for Synthetic Biology Applications in \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "modular circuit assembly"}, {"from": "Marburg Collection", "title": "The Marburg Collection: A Golden Gate DNA Assembly Framework for Synthetic Biology Applications in \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "inversion of individual transcription units"}, {"from": "inversion of individual transcription units", "title": "The Marburg Collection: A Golden Gate DNA Assembly Framework for Synthetic Biology Applications in \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "reduce transcriptional crosstalk"}, {"from": "Marburg Collection", "title": "The Marburg Collection: A Golden Gate DNA Assembly Framework for Synthetic Biology Applications in \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "microplate reader measurement workflow"}, {"from": "microplate reader measurement workflow", "title": "The Marburg Collection: A Golden Gate DNA Assembly Framework for Synthetic Biology Applications in \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "quantitative characterization of genetic parts"}, {"from": "Escherichia coli", "title": "Development and comparison of cell-free protein synthesis systems derived from typical bacterial chassis.", "to": "Cell-free protein synthesis system"}, {"from": "Bacillus subtilis", "title": "Development and comparison of cell-free protein synthesis systems derived from typical bacterial chassis.", "to": "Cell-free protein synthesis system"}, {"from": "Corynebacterium glutamate", "title": "Development and comparison of cell-free protein synthesis systems derived from typical bacterial chassis.", "to": "Cell-free protein synthesis system"}, {"from": "Vibrio natriegens", "title": "Development and comparison of cell-free protein synthesis systems derived from typical bacterial chassis.", "to": "Cell-free protein synthesis system"}, {"from": "Codon usage", "title": "Development and comparison of cell-free protein synthesis systems derived from typical bacterial chassis.", "to": "Protein yield"}, {"from": "Plasmid synthesis competent cell selection", "title": "Development and comparison of cell-free protein synthesis systems derived from typical bacterial chassis.", "to": "Protein yield"}, {"from": "Plasmid concentration", "title": "Development and comparison of cell-free protein synthesis systems derived from typical bacterial chassis.", "to": "Protein yield"}, {"from": "Ribosomal binding site", "title": "Development and comparison of cell-free protein synthesis systems derived from typical bacterial chassis.", "to": "Protein yield"}, {"from": "Cell-free protein synthesis system reagent components", "title": "Development and comparison of cell-free protein synthesis systems derived from typical bacterial chassis.", "to": "Protein yield"}, {"from": "Cell-free protein synthesis systems", "title": "Development and comparison of cell-free protein synthesis systems derived from typical bacterial chassis.", "to": "Therapeutic protein production"}, {"from": "Cell-free protein synthesis systems", "title": "Development and comparison of cell-free protein synthesis systems derived from typical bacterial chassis.", "to": "Receptor-binding domain protein of SARS-CoV-2 production"}, {"from": "Vibrio natriegens", "title": "A novel global transcriptional perturbation target identified by forward genetics reprograms Vibrio natriegens for improving recombinant protein production.", "to": "protein expression system"}, {"from": "Ochrobactrum anthropi-derived \u03c9-transaminase", "title": "A novel global transcriptional perturbation target identified by forward genetics reprograms Vibrio natriegens for improving recombinant protein production.", "to": "strongly expressed in Escherichia coli"}, {"from": "Ochrobactrum anthropi-derived \u03c9-transaminase", "title": "A novel global transcriptional perturbation target identified by forward genetics reprograms Vibrio natriegens for improving recombinant protein production.", "to": "weakly in Vibrio natriegens"}, {"from": "Ochrobactrum anthropi-derived \u03c9-transaminase C-terminal", "title": "A novel global transcriptional perturbation target identified by forward genetics reprograms Vibrio natriegens for improving recombinant protein production.", "to": "green fluorescent protein fusion"}, {"from": "Vibrio natriegens mutants", "title": "A novel global transcriptional perturbation target identified by forward genetics reprograms Vibrio natriegens for improving recombinant protein production.", "to": "increase in protein yield of Ochrobactrum anthropi-derived \u03c9-transaminase"}, {"from": "Vibrio natriegens mutants", "title": "A novel global transcriptional perturbation target identified by forward genetics reprograms Vibrio natriegens for improving recombinant protein production.", "to": "increase in enzyme activity of Ochrobactrum anthropi-derived \u03c9-transaminase"}, {"from": "ultraviolet mutagenesis", "title": "A novel global transcriptional perturbation target identified by forward genetics reprograms Vibrio natriegens for improving recombinant protein production.", "to": "increase in protein yield of genes of interest"}, {"from": "fluorescence-activated cell sorting", "title": "A novel global transcriptional perturbation target identified by forward genetics reprograms Vibrio natriegens for improving recombinant protein production.", "to": "increase in protein yield of genes of interest"}, {"from": "OATA colorimetric assay", "title": "A novel global transcriptional perturbation target identified by forward genetics reprograms Vibrio natriegens for improving recombinant protein production.", "to": "increase in protein yield of genes of interest"}, {"from": "Y457 variants in rpoC", "title": "A novel global transcriptional perturbation target identified by forward genetics reprograms Vibrio natriegens for improving recombinant protein production.", "to": "increase in recombinant protein yield"}, {"from": "Y457 variants in rpoC", "title": "A novel global transcriptional perturbation target identified by forward genetics reprograms Vibrio natriegens for improving recombinant protein production.", "to": "increase in pET plasmids copy number"}, {"from": "Y457 variants in rpoC", "title": "A novel global transcriptional perturbation target identified by forward genetics reprograms Vibrio natriegens for improving recombinant protein production.", "to": "increase in soluble protein yield of genes of interest"}, {"from": "GOI-GFP fusion", "title": "A novel global transcriptional perturbation target identified by forward genetics reprograms Vibrio natriegens for improving recombinant protein production.", "to": "powerful tool of forward genetics"}, {"from": "Ter binding sites", "title": "High-Efficiency Protection of Linear DNA in Cell-Free Extracts from \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "protection of protection of linear expression templates"}, {"from": "DNA-binding protein Tus", "title": "High-Efficiency Protection of Linear DNA in Cell-Free Extracts from \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "protection of protection of linear expression templates"}, {"from": "protection of linear expression templates", "title": "High-Efficiency Protection of Linear DNA in Cell-Free Extracts from \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "mCherry encoding"}, {"from": "protection of linear expression templates", "title": "High-Efficiency Protection of Linear DNA in Cell-Free Extracts from \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "deGFP encoding"}, {"from": "Escherichia coli extracts", "title": "High-Efficiency Protection of Linear DNA in Cell-Free Extracts from \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "our method comparison"}, {"from": "GamS-mediated protection scheme", "title": "High-Efficiency Protection of Linear DNA in Cell-Free Extracts from \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "our method comparison"}, {"from": "Tus-Ter system", "title": "High-Efficiency Protection of Linear DNA in Cell-Free Extracts from \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "Vibrio natriegens extracts"}, {"from": "linear templates", "title": "High-Efficiency Protection of Linear DNA in Cell-Free Extracts from \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "plasmid-level expression"}, {"from": "Optimal conditions", "title": "High-cell-density fed-batch cultivations of Vibrio natriegens.", "to": "Generation times of less than 10 minutes"}, {"from": "Vibrio natriegens", "title": "High-cell-density fed-batch cultivations of Vibrio natriegens.", "to": "Fastest growing non-pathogenic bacterium"}, {"from": "Full genome availability", "title": "High-cell-density fed-batch cultivations of Vibrio natriegens.", "to": "Vibrio natriegens"}, {"from": "Genetic engineering tools availability", "title": "High-cell-density fed-batch cultivations of Vibrio natriegens.", "to": "Vibrio natriegens"}, {"from": "Wide range of carbon sources utilization", "title": "High-cell-density fed-batch cultivations of Vibrio natriegens.", "to": "Vibrio natriegens"}, {"from": "Vibrio natriegens", "title": "High-cell-density fed-batch cultivations of Vibrio natriegens.", "to": "Attractive host for biotechnological production"}, {"from": "Fed-batch cultivations", "title": "High-cell-density fed-batch cultivations of Vibrio natriegens.", "to": "High-cell-density cultivations"}, {"from": "Deep-well plates", "title": "High-cell-density fed-batch cultivations of Vibrio natriegens.", "to": "Fed-batch cultivations"}, {"from": "Lab-scale bioreactor cultivations", "title": "High-cell-density fed-batch cultivations of Vibrio natriegens.", "to": "Different temperature ranges in mineral salt medium"}, {"from": "Mineral salt medium", "title": "High-cell-density fed-batch cultivations of Vibrio natriegens.", "to": "Lab-scale bioreactor cultivations"}, {"from": "Exponential glucose feeding", "title": "High-cell-density fed-batch cultivations of Vibrio natriegens.", "to": "Cell death"}, {"from": "Constant glucose-feeding", "title": "High-cell-density fed-batch cultivations of Vibrio natriegens.", "to": "Cell death"}, {"from": "Initial NaCl concentrations of 15-18 grams per Liter", "title": "High-cell-density fed-batch cultivations of Vibrio natriegens.", "to": "Positive effect on cell growth"}, {"from": "Temperature reduction from 37 to 30 degrees Celsius", "title": "High-cell-density fed-batch cultivations of Vibrio natriegens.", "to": "Positive effect on cell growth"}, {"from": "Maximal growth rate", "title": "High-cell-density fed-batch cultivations of Vibrio natriegens.", "to": "1.36 per hour"}, {"from": "Specific oxygen uptake rate", "title": "High-cell-density fed-batch cultivations of Vibrio natriegens.", "to": "22 millimoles per gram CDW per hour"}, {"from": "High biomass yields", "title": "High-cell-density fed-batch cultivations of Vibrio natriegens.", "to": "Up to 55 grams per Liter"}, {"from": "Fed-batch strategies", "title": "High-cell-density fed-batch cultivations of Vibrio natriegens.", "to": "Potential of Vibrio natriegens as a strong producer in industrial biotechnology"}, {"from": "motility behavior", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "bacteria"}, {"from": "bacteria", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "environmental applications"}, {"from": "bacteria", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "food industry applications"}, {"from": "bacteria", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "biomedical applications"}, {"from": "species", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "Vibrio natriegens"}, {"from": "species", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "Magnetococcus marinus"}, {"from": "species", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "Pseudomonas putida"}, {"from": "species", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "Vibrio fischeri"}, {"from": "species", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "Escherichia coli"}, {"from": "microfluidic environments", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "various levels of confinement"}, {"from": "microfluidic environments", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "geometrical complexity"}, {"from": "moderate confinement", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "motility behavior"}, {"from": "Vibrio natriegens", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "moved parallel to the wall"}, {"from": "Vibrio fischeri", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "moved parallel to the wall"}, {"from": "Pseudomonas putida", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "stable movement parallel to the wall"}, {"from": "Escherichia coli", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "stable movement parallel to the wall"}, {"from": "Escherichia coli", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "incidental wall escape events"}, {"from": "Pseudomonas putida", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "incidental wall escape events"}, {"from": "Magnetococcus marinus", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "flipping between wall accumulator and wall escaper regimes"}, {"from": "tighter confining environments", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "motility governed by steric interactions"}, {"from": "steric interactions", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "motility"}, {"from": "mesoscale regions", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "impacts of hydrodynamics and steric interactions overlap"}, {"from": "hydrodynamics and steric interactions overlap", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "push bacteria in the same directions in linear channels"}, {"from": "hydrodynamics and steric interactions overlap", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "smooth bacterial movement"}, {"from": "hydrodynamics and steric interactions overlap", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "oppositional in meandered channels"}, {"from": "oppositional in meandered channels", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "chaotic movement"}, {"from": "oppositional in meandered channels", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "bacterial trapping"}, {"from": "microfluidic devices", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "single-cell genomic screening"}, {"from": "microfluidic devices", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "bacterial entrapment for diagnostics"}, {"from": "microfluidic devices", "title": "Patterns of bacterial motility in microfluidics-confining environments.", "to": "biocomputation"}, {"from": "Darobactin A", "title": "Optimization of heterologous Darobactin A expression and identification of the minimal biosynthetic gene cluster.", "to": "outer membrane protein BamA"}, {"from": "Darobactin A", "title": "Optimization of heterologous Darobactin A expression and identification of the minimal biosynthetic gene cluster.", "to": "Gram-negative bacteria"}, {"from": "Escherichia coli", "title": "Optimization of heterologous Darobactin A expression and identification of the minimal biosynthetic gene cluster.", "to": "Darobactin A production"}, {"from": "Vibrio natriegens", "title": "Optimization of heterologous Darobactin A expression and identification of the minimal biosynthetic gene cluster.", "to": "Darobactin A production"}, {"from": "expression constructs optimization", "title": "Optimization of heterologous Darobactin A expression and identification of the minimal biosynthetic gene cluster.", "to": "Darobactin A production titers increase"}, {"from": "expression constructs optimization", "title": "Optimization of heterologous Darobactin A expression and identification of the minimal biosynthetic gene cluster.", "to": "fermentation time decrease"}, {"from": "minimal Darobactin A biosynthetic gene cluster", "title": "Optimization of heterologous Darobactin A expression and identification of the minimal biosynthetic gene cluster.", "to": "Darobactin A production"}, {"from": "Euglena gracilis co-cultivated with naturally competent Vibrio natriegens", "title": "Global Metabolomics Reveals That \u003ci\u003eVibrio natriegens\u003c/i\u003e Enhances the Growth and Paramylon Synthesis of \u003ci\u003eEuglena gracilis\u003c/i\u003e.", "to": "increased total chlorophyll"}, {"from": "Euglena gracilis co-cultivated with naturally competent Vibrio natriegens", "title": "Global Metabolomics Reveals That \u003ci\u003eVibrio natriegens\u003c/i\u003e Enhances the Growth and Paramylon Synthesis of \u003ci\u003eEuglena gracilis\u003c/i\u003e.", "to": "increased microalgal growth"}, {"from": "Euglena gracilis co-cultivated with naturally competent Vibrio natriegens", "title": "Global Metabolomics Reveals That \u003ci\u003eVibrio natriegens\u003c/i\u003e Enhances the Growth and Paramylon Synthesis of \u003ci\u003eEuglena gracilis\u003c/i\u003e.", "to": "increased dry weight"}, {"from": "Euglena gracilis co-cultivated with naturally competent Vibrio natriegens", "title": "Global Metabolomics Reveals That \u003ci\u003eVibrio natriegens\u003c/i\u003e Enhances the Growth and Paramylon Synthesis of \u003ci\u003eEuglena gracilis\u003c/i\u003e.", "to": "increased storage sugar paramylon content"}, {"from": "Euglena gracilis co-cultivated with naturally competent Vibrio natriegens", "title": "Global Metabolomics Reveals That \u003ci\u003eVibrio natriegens\u003c/i\u003e Enhances the Growth and Paramylon Synthesis of \u003ci\u003eEuglena gracilis\u003c/i\u003e.", "to": "increase in methyl carbamate"}, {"from": "Euglena gracilis co-cultivated with naturally competent Vibrio natriegens", "title": "Global Metabolomics Reveals That \u003ci\u003eVibrio natriegens\u003c/i\u003e Enhances the Growth and Paramylon Synthesis of \u003ci\u003eEuglena gracilis\u003c/i\u003e.", "to": "increase in ectoine"}, {"from": "Euglena gracilis co-cultivated with naturally competent Vibrio natriegens", "title": "Global Metabolomics Reveals That \u003ci\u003eVibrio natriegens\u003c/i\u003e Enhances the Growth and Paramylon Synthesis of \u003ci\u003eEuglena gracilis\u003c/i\u003e.", "to": "increase in choline"}, {"from": "Euglena gracilis co-cultivated with naturally competent Vibrio natriegens", "title": "Global Metabolomics Reveals That \u003ci\u003eVibrio natriegens\u003c/i\u003e Enhances the Growth and Paramylon Synthesis of \u003ci\u003eEuglena gracilis\u003c/i\u003e.", "to": "increase in methyl N-methylanthranilate"}, {"from": "Euglena gracilis co-cultivated with naturally competent Vibrio natriegens", "title": "Global Metabolomics Reveals That \u003ci\u003eVibrio natriegens\u003c/i\u003e Enhances the Growth and Paramylon Synthesis of \u003ci\u003eEuglena gracilis\u003c/i\u003e.", "to": "increase in gentiatibetine"}, {"from": "Euglena gracilis co-cultivated with naturally competent Vibrio natriegens", "title": "Global Metabolomics Reveals That \u003ci\u003eVibrio natriegens\u003c/i\u003e Enhances the Growth and Paramylon Synthesis of \u003ci\u003eEuglena gracilis\u003c/i\u003e.", "to": "increase in 4R-aminopentanoic acid"}, {"from": "Euglena gracilis co-cultivated with naturally competent Vibrio natriegens", "title": "Global Metabolomics Reveals That \u003ci\u003eVibrio natriegens\u003c/i\u003e Enhances the Growth and Paramylon Synthesis of \u003ci\u003eEuglena gracilis\u003c/i\u003e.", "to": "increase in glu-val"}, {"from": "Vibrio natriegens", "title": "Metabolic engineering of Vibrio natriegens.", "to": "high growth rates"}, {"from": "Vibrio natriegens", "title": "Metabolic engineering of Vibrio natriegens.", "to": "high substrate consumption rates"}, {"from": "Vibrio natriegens", "title": "Metabolic engineering of Vibrio natriegens.", "to": "ability to utilize a variety of substrates"}, {"from": "Vibrio natriegens", "title": "Metabolic engineering of Vibrio natriegens.", "to": "easy to handle in lab"}, {"from": "development of wide range of genetic tools", "title": "Metabolic engineering of Vibrio natriegens.", "to": "genetic engineering of production strains"}, {"from": "comprehensive understanding of metabolism and its regulation", "title": "Metabolic engineering of Vibrio natriegens.", "to": "exploit full potential of Vibrio natriegens"}, {"from": "available genetic engineering tools", "title": "Metabolic engineering of Vibrio natriegens.", "to": "metabolic engineering of Vibrio natriegens"}, {"from": "obstacles", "title": "Metabolic engineering of Vibrio natriegens.", "to": "establish Vibrio natriegens as industrial production host"}, {"from": "laser speckle imaging technique", "title": "Laser speckle imaging for early detection of microbial colony forming units.", "to": "bacterial colony-forming units identification"}, {"from": "laser speckle imaging method", "title": "Laser speckle imaging for early detection of microbial colony forming units.", "to": "microbial activity assessment"}, {"from": "visual inspection under white light illumination", "title": "Laser speckle imaging for early detection of microbial colony forming units.", "to": "microbial activity assessment"}, {"from": "growth of Vibrio natriegens", "title": "Laser speckle imaging for early detection of microbial colony forming units.", "to": "bacterial colonies on the solid medium observation"}, {"from": "visual examination under white light illumination", "title": "Laser speckle imaging for early detection of microbial colony forming units.", "to": "bacterial colonies on the solid medium observation"}, {"from": "laser speckle correlation analysis", "title": "Laser speckle imaging for early detection of microbial colony forming units.", "to": "bacterial colonies on the solid medium observation"}, {"from": "Gompertz model", "title": "Laser speckle imaging for early detection of microbial colony forming units.", "to": "colony radius growth curves description"}, {"from": "laser speckle contrast imaging", "title": "Laser speckle imaging for early detection of microbial colony forming units.", "to": "colony growth detection"}, {"from": "signal processing", "title": "Laser speckle imaging for early detection of microbial colony forming units.", "to": "colony growth detection"}, {"from": "standard CFU counting method under white light illumination", "title": "Laser speckle imaging for early detection of microbial colony forming units.", "to": "colony growth detection"}, {"from": "Vibrio natriegens", "title": "Systems metabolic engineering of Vibrio natriegens for the production of 1,3-propanediol.", "to": "1,3-propanediol production"}, {"from": "heterologous pathway", "title": "Systems metabolic engineering of Vibrio natriegens for the production of 1,3-propanediol.", "to": "1,3-propanediol production"}, {"from": "engineering cellular transcriptional regulators", "title": "Systems metabolic engineering of Vibrio natriegens for the production of 1,3-propanediol.", "to": "1,3-propanediol production"}, {"from": "global transcriptomic analysis", "title": "Systems metabolic engineering of Vibrio natriegens for the production of 1,3-propanediol.", "to": "1,3-propanediol production"}, {"from": "cofactor engineering", "title": "Systems metabolic engineering of Vibrio natriegens for the production of 1,3-propanediol.", "to": "increased intracellular reducing power"}, {"from": "pathway engineering", "title": "Systems metabolic engineering of Vibrio natriegens for the production of 1,3-propanediol.", "to": "reduced toxic intermediate accumulation"}, {"from": "glycerol oxidation pathway engineering", "title": "Systems metabolic engineering of Vibrio natriegens for the production of 1,3-propanediol.", "to": "elimination of byproduct formation"}, {"from": "final engineered strain", "title": "Systems metabolic engineering of Vibrio natriegens for the production of 1,3-propanediol.", "to": "1,3-propanediol titer of 56.2 g/L"}, {"from": "final engineered strain", "title": "Systems metabolic engineering of Vibrio natriegens for the production of 1,3-propanediol.", "to": "1,3-propanediol yield of 0.61 mol/mol"}, {"from": "final engineered strain", "title": "Systems metabolic engineering of Vibrio natriegens for the production of 1,3-propanediol.", "to": "1,3-propanediol productivity of 2.36 g/L/h"}, {"from": "Escherichia coli", "title": "Genetic Code Expansion of \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "model bacteria in studies"}, {"from": "Vibrio natriegens", "title": "Genetic Code Expansion of \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "fast-growing gram-negative bacterium"}, {"from": "Escherichia coli", "title": "Genetic Code Expansion of \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "genetic code expansion methodologies"}, {"from": "Vibrio natriegens", "title": "Genetic Code Expansion of \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "adaptee of genetic code expansion methodology"}, {"from": "genetic code expansion", "title": "Genetic Code Expansion of \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "incorporation of unnatural amino acids into proteins"}, {"from": "incorporation of unnatural amino acids into proteins", "title": "Genetic Code Expansion of \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "demonstrated in Vibrio natriegens"}, {"from": "recombinant protein expression system systems", "title": "\u003ci\u003eVibrio natriegens\u003c/i\u003e as a pET-Compatible Expression Host Complementary to \u003ci\u003eEscherichia coli\u003c/i\u003e.", "to": "reduction in production cost of enzymes"}, {"from": "Vibrio natriegens", "title": "\u003ci\u003eVibrio natriegens\u003c/i\u003e as a pET-Compatible Expression Host Complementary to \u003ci\u003eEscherichia coli\u003c/i\u003e.", "to": "protein expression system host"}, {"from": "doubling time", "title": "\u003ci\u003eVibrio natriegens\u003c/i\u003e as a pET-Compatible Expression Host Complementary to \u003ci\u003eEscherichia coli\u003c/i\u003e.", "to": "Vibrio natriegens"}, {"from": "pET plasmids", "title": "\u003ci\u003eVibrio natriegens\u003c/i\u003e as a pET-Compatible Expression Host Complementary to \u003ci\u003eEscherichia coli\u003c/i\u003e.", "to": "Vibrio natriegens strain VnDX"}, {"from": "genes of interest", "title": "\u003ci\u003eVibrio natriegens\u003c/i\u003e as a pET-Compatible Expression Host Complementary to \u003ci\u003eEscherichia coli\u003c/i\u003e.", "to": "soluble expression in Vibrio natriegens"}, {"from": "genes of interest", "title": "\u003ci\u003eVibrio natriegens\u003c/i\u003e as a pET-Compatible Expression Host Complementary to \u003ci\u003eEscherichia coli\u003c/i\u003e.", "to": "soluble expression in Escherichia coli"}, {"from": "genes of interest", "title": "\u003ci\u003eVibrio natriegens\u003c/i\u003e as a pET-Compatible Expression Host Complementary to \u003ci\u003eEscherichia coli\u003c/i\u003e.", "to": "better expression in Vibrio natriegens"}, {"from": "Terrific Broth medium", "title": "\u003ci\u003eVibrio natriegens\u003c/i\u003e as a pET-Compatible Expression Host Complementary to \u003ci\u003eEscherichia coli\u003c/i\u003e.", "to": "consensus \"what to try first\" protocol for Vibrio natriegens"}, {"from": "genes of interest", "title": "\u003ci\u003eVibrio natriegens\u003c/i\u003e as a pET-Compatible Expression Host Complementary to \u003ci\u003eEscherichia coli\u003c/i\u003e.", "to": "increased catalytic efficiency under optimized expression conditions"}, {"from": "Vibrio natriegens", "title": "\u003ci\u003eVibrio natriegens\u003c/i\u003e as a pET-Compatible Expression Host Complementary to \u003ci\u003eEscherichia coli\u003c/i\u003e.", "to": "pET-compatible expression host"}, {"from": "genes of interest", "title": "\u003ci\u003eVibrio natriegens\u003c/i\u003e as a pET-Compatible Expression Host Complementary to \u003ci\u003eEscherichia coli\u003c/i\u003e.", "to": "highly expressed in Vibrio natriegens"}, {"from": "genes of interest", "title": "\u003ci\u003eVibrio natriegens\u003c/i\u003e as a pET-Compatible Expression Host Complementary to \u003ci\u003eEscherichia coli\u003c/i\u003e.", "to": "not expressed well in Escherichia coli"}, {"from": "Ciona intestinalis molecule against microbes-A24", "title": "Using \u003ci\u003eBacillus subtilis\u003c/i\u003e as a Host Cell to Express an Antimicrobial Peptide from the Marine Chordate \u003ci\u003eCiona intestinalis\u003c/i\u003e.", "to": "antimicrobial peptide"}, {"from": "plasmid expressing recombinant Ciona intestinalis molecule against microbes-A24", "title": "Using \u003ci\u003eBacillus subtilis\u003c/i\u003e as a Host Cell to Express an Antimicrobial Peptide from the Marine Chordate \u003ci\u003eCiona intestinalis\u003c/i\u003e.", "to": "Bacillus subtilis"}, {"from": "recombinant Ciona intestinalis molecule against microbes-A24-expressing Bacillus subtilis", "title": "Using \u003ci\u003eBacillus subtilis\u003c/i\u003e as a Host Cell to Express an Antimicrobial Peptide from the Marine Chordate \u003ci\u003eCiona intestinalis\u003c/i\u003e.", "to": "bactericidal activity"}, {"from": "bactericidal activity", "title": "Using \u003ci\u003eBacillus subtilis\u003c/i\u003e as a Host Cell to Express an Antimicrobial Peptide from the Marine Chordate \u003ci\u003eCiona intestinalis\u003c/i\u003e.", "to": "lactoferricin-expressing strains"}, {"from": "pepsin-digested extracts from recombinant Ciona intestinalis molecule against microbes-A24-expressing strains", "title": "Using \u003ci\u003eBacillus subtilis\u003c/i\u003e as a Host Cell to Express an Antimicrobial Peptide from the Marine Chordate \u003ci\u003eCiona intestinalis\u003c/i\u003e.", "to": "bactericidal activity against halophilic bacteria"}, {"from": "bactericidal activity of recombinant Ciona intestinalis molecule against microbes-A24-expressing Bacillus subtilis strains", "title": "Using \u003ci\u003eBacillus subtilis\u003c/i\u003e as a Host Cell to Express an Antimicrobial Peptide from the Marine Chordate \u003ci\u003eCiona intestinalis\u003c/i\u003e.", "to": "efficacy of Tetracycline against Vibrio natriegens"}, {"from": "bactericidal activity of recombinant Ciona intestinalis molecule against microbes-A24-expressing Bacillus subtilis strains", "title": "Using \u003ci\u003eBacillus subtilis\u003c/i\u003e as a Host Cell to Express an Antimicrobial Peptide from the Marine Chordate \u003ci\u003eCiona intestinalis\u003c/i\u003e.", "to": "efficacy of Tetracycline against clearance of Vibrio parahaemolyticus in vivo"}, {"from": "bactericidal activity of recombinant Ciona intestinalis molecule against microbes-A24-expressing Bacillus subtilis strains", "title": "Using \u003ci\u003eBacillus subtilis\u003c/i\u003e as a Host Cell to Express an Antimicrobial Peptide from the Marine Chordate \u003ci\u003eCiona intestinalis\u003c/i\u003e.", "to": "efficacy of Tetracycline against Vibrio alginolyticus"}, {"from": "recombinant Ciona intestinalis molecule against microbes-A24-expressing Bacillus subtilis strains", "title": "Using \u003ci\u003eBacillus subtilis\u003c/i\u003e as a Host Cell to Express an Antimicrobial Peptide from the Marine Chordate \u003ci\u003eCiona intestinalis\u003c/i\u003e.", "to": "protection against halophilic bacterial infection in marine fish and shellfish"}, {"from": "Alginate lyases", "title": "Characterization and enhanced extracellular overexpression of a new salt-activated alginate lyase.", "to": "Alginate oligosaccharides"}, {"from": "Alginate oligosaccharides", "title": "Characterization and enhanced extracellular overexpression of a new salt-activated alginate lyase.", "to": "Prebiotic effects"}, {"from": "Alginate oligosaccharides", "title": "Characterization and enhanced extracellular overexpression of a new salt-activated alginate lyase.", "to": "Antidiabetic effects"}, {"from": "Extracellular production of recombinant proteins in Escherichia coli", "title": "Characterization and enhanced extracellular overexpression of a new salt-activated alginate lyase.", "to": "Simplified downstream processing"}, {"from": "Extracellular production of recombinant proteins in Escherichia coli", "title": "Characterization and enhanced extracellular overexpression of a new salt-activated alginate lyase.", "to": "High productivity"}, {"from": "Aly01", "title": "Characterization and enhanced extracellular overexpression of a new salt-activated alginate lyase.", "to": "Release of trisaccharide from alginate"}, {"from": "Aly01", "title": "Characterization and enhanced extracellular overexpression of a new salt-activated alginate lyase.", "to": "Strictly Sodium Chloride activated"}, {"from": "Green fluorescent protein", "title": "Characterization and enhanced extracellular overexpression of a new salt-activated alginate lyase.", "to": "Successfully secreted in Escherichia coli with Aly01 signal peptide"}, {"from": "Terrific Broth medium supplemented with glycine and calcium", "title": "Characterization and enhanced extracellular overexpression of a new salt-activated alginate lyase.", "to": "Reduction of lag phase of protein secretion to 3 hours from 12 hours"}, {"from": "Calcium", "title": "Characterization and enhanced extracellular overexpression of a new salt-activated alginate lyase.", "to": "Remedied glycine-related cell growth impairment"}, {"from": "Enzyme productivity", "title": "Characterization and enhanced extracellular overexpression of a new salt-activated alginate lyase.", "to": "Reached maximum of 4.55 Units per milliliter"}, {"from": "Aly01 signal peptide", "title": "Characterization and enhanced extracellular overexpression of a new salt-activated alginate lyase.", "to": "Extracellular production of recombinant proteins in Escherichia coli"}, {"from": "Vibrio natriegens", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "synthetic biology"}, {"from": "genome editing", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "genetic modification"}, {"from": "genome editing tools", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "high efficiency"}, {"from": "genome editing tools", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "fidelity"}, {"from": "Vibrio natriegens strains", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "physiological characteristics"}, {"from": "CICC 10908 strain", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "fast growth"}, {"from": "CICC 10908 strain", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "stable growth"}, {"from": "CICC 10908 strain", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "genome editing study"}, {"from": "natural transformation system", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "Vibrio natriegens"}, {"from": "optimized natural transformation", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "antibiotic resistance marker cat-sacB integration"}, {"from": "optimized natural transformation", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "Kan(R"}, {"from": "double-selection cassette", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "seamless genome editing"}, {"from": "double-selection cassette", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "high efficiency"}, {"from": "double-selection cassette", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "fidelity"}, {"from": "gene deletion", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "positive rates"}, {"from": "gene complementation", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "positive rates"}, {"from": "gene insertion", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "positive rates"}, {"from": "gene substitution", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "positive rates"}, {"from": "transformation of recombinant plasmid", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "Vibrio natriegens"}, {"from": "seamless genome editing system", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "high efficiency"}, {"from": "seamless genome editing system", "title": "[Construction of seamless genome editing system for fast-growing Vibrio natriegens].", "to": "fidelity"}, {"from": "naturally competent Vibrio natriegens", "title": "Complete Genome Sequences of Two Vibrio natriegens Bacteriophages.", "to": "biotechnology applications"}, {"from": "naturally competent Vibrio natriegens", "title": "Complete Genome Sequences of Two Vibrio natriegens Bacteriophages.", "to": "metabolic engineering"}, {"from": "naturally competent Vibrio natriegens", "title": "Complete Genome Sequences of Two Vibrio natriegens Bacteriophages.", "to": "bacteriophages"}, {"from": "Vibrionaceae bacteria genome replication", "title": "Vibrionaceae core, shell and cloud genes are non-randomly distributed on Chr 1: An hypothesis that links the genomic location of genes with their intracellular placement.", "to": "increased expression of genes close to origin of replication of Chromosome 1 during fast growth"}, {"from": "origin of replication of Chromosome 1-proximate growth-related genes", "title": "Vibrionaceae core, shell and cloud genes are non-randomly distributed on Chr 1: An hypothesis that links the genomic location of genes with their intracellular placement.", "to": "increased gene expression during fast growth"}, {"from": "core genes", "title": "Vibrionaceae core, shell and cloud genes are non-randomly distributed on Chr 1: An hypothesis that links the genomic location of genes with their intracellular placement.", "to": "bias towards origin of replication of Chromosome 1"}, {"from": "softcore genes", "title": "Vibrionaceae core, shell and cloud genes are non-randomly distributed on Chr 1: An hypothesis that links the genomic location of genes with their intracellular placement.", "to": "bias towards origin of replication of Chromosome 1"}, {"from": "shell genes", "title": "Vibrionaceae core, shell and cloud genes are non-randomly distributed on Chr 1: An hypothesis that links the genomic location of genes with their intracellular placement.", "to": "overrepresentation close to terminus of Chromosome 1"}, {"from": "core and softcore genes", "title": "Vibrionaceae core, shell and cloud genes are non-randomly distributed on Chr 1: An hypothesis that links the genomic location of genes with their intracellular placement.", "to": "overrepresentation around origin of replication of Chromosome 1"}, {"from": "shell and cloud genes", "title": "Vibrionaceae core, shell and cloud genes are non-randomly distributed on Chr 1: An hypothesis that links the genomic location of genes with their intracellular placement.", "to": "overrepresentation close to terminus of Chromosome 1"}, {"from": "global gene expression patterns", "title": "Vibrionaceae core, shell and cloud genes are non-randomly distributed on Chr 1: An hypothesis that links the genomic location of genes with their intracellular placement.", "to": "correlation with chromosomal distance to origin of replication of Chromosome 1"}, {"from": "pangene categories", "title": "Vibrionaceae core, shell and cloud genes are non-randomly distributed on Chr 1: An hypothesis that links the genomic location of genes with their intracellular placement.", "to": "contribution to skewed expression pattern at fast-growing conditions"}, {"from": "softcore", "title": "Vibrionaceae core, shell and cloud genes are non-randomly distributed on Chr 1: An hypothesis that links the genomic location of genes with their intracellular placement.", "to": "shell, and cloud genes, responsible for elevated expression at slow-growing conditions"}, {"from": "core/softcore genes", "title": "Vibrionaceae core, shell and cloud genes are non-randomly distributed on Chr 1: An hypothesis that links the genomic location of genes with their intracellular placement.", "to": "enriched in spatially separated intracellular regions"}, {"from": "shell/cloud genes", "title": "Vibrionaceae core, shell and cloud genes are non-randomly distributed on Chr 1: An hypothesis that links the genomic location of genes with their intracellular placement.", "to": "enriched in spatially separated intracellular regions"}, {"from": "genomic location of genes", "title": "Vibrionaceae core, shell and cloud genes are non-randomly distributed on Chr 1: An hypothesis that links the genomic location of genes with their intracellular placement.", "to": "link to cellular placement"}, {"from": "naturally competent Vibrio natriegens", "title": "Genome Sequence of Vibrio natriegens Phage vB_VnaS-AQKL99.", "to": "Aquatic Killer 99"}, {"from": "simulated microgravity", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "growth rate of bacteria"}, {"from": "simulated microgravity", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "shear force"}, {"from": "Vibrio natriegens", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "generation time"}, {"from": "simulated microgravity", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "growth rate of Vibrio natriegens"}, {"from": "long-term culturing under simulated microgravity", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "mutant with slower growth rate"}, {"from": "transcriptome data", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "mechanical signal transduction genes up-regulated"}, {"from": "transcriptome data", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "flagellin biogenesis genes up-regulated"}, {"from": "transcriptome data", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "adaptive response genes down-regulated"}, {"from": "transcriptome data", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "anaerobic metabolism genes down-regulated"}, {"from": "transcriptome data", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "nitrogen metabolism genes down-regulated"}, {"from": "transcriptome data", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "chromosome segregation genes down-regulated"}, {"from": "transcriptome data", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "cell vitality genes down-regulated"}, {"from": "genome-wide chromosome conformation capture", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "changes of genome 3D organization"}, {"from": "changes of genome 3D organization", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "differentially expressed genes"}, {"from": "whole genome re-sequencing", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "mutant with slower growth rate"}, {"from": "structure variations", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "down-regulated genes"}, {"from": "decreased interaction frequency", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "well-orchestrated chromosomes replication"}, {"from": "simulated microgravity", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "local gene expression regulation"}, {"from": "local gene expression regulation", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "physiological characteristics alteration"}, {"from": "physiological characteristics alteration", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "growth rate"}, {"from": "physiological characteristics alteration", "title": "Changes in \u003ci\u003eVibrio natriegens\u003c/i\u003e Growth Under Simulated Microgravity.", "to": "metabolic capacity"}, {"from": "cobalamin class-III dependent enzymes", "title": "Heterologous expression of cobalamin dependent class-III enzymes.", "to": "reductive dehalogenases"}, {"from": "reductive dehalogenases", "title": "Heterologous expression of cobalamin dependent class-III enzymes.", "to": "organohalide respiration"}, {"from": "reductive dehalogenases", "title": "Heterologous expression of cobalamin dependent class-III enzymes.", "to": "carbon-halogen bonds cleaving"}, {"from": "carbon-halogen bonds cleaving", "title": "Heterologous expression of cobalamin dependent class-III enzymes.", "to": "bioremediation applications"}, {"from": "heterologous expression systems", "title": "Heterologous expression of cobalamin dependent class-III enzymes.", "to": "active holo-enzyme production"}, {"from": "active holo-enzyme", "title": "Heterologous expression of cobalamin dependent class-III enzymes.", "to": "iron-sulphur cluster incorporation"}, {"from": "active holo-enzyme", "title": "Heterologous expression of cobalamin dependent class-III enzymes.", "to": "cobalamin incorporation"}, {"from": "Bacillus megaterium", "title": "Heterologous expression of cobalamin dependent class-III enzymes.", "to": "cobalamin class-III dependent enzymes expression"}, {"from": "Escherichia coli HMS174(DE3)", "title": "Heterologous expression of cobalamin dependent class-III enzymes.", "to": "cobalamin class-III dependent enzymes expression"}, {"from": "Shimwellia blattae", "title": "Heterologous expression of cobalamin dependent class-III enzymes.", "to": "cobalamin class-III dependent enzymes expression"}, {"from": "Vibrio natrigenes", "title": "Heterologous expression of cobalamin dependent class-III enzymes.", "to": "cobalamin class-III dependent enzymes expression"}, {"from": "Nitratireductor pacificus pht-3B reductive dehalogenase", "title": "Heterologous expression of cobalamin dependent class-III enzymes.", "to": "model enzyme"}, {"from": "epoxyqueuosine reductase from Streptococcus thermophilus", "title": "Heterologous expression of cobalamin dependent class-III enzymes.", "to": "model enzyme"}, {"from": "co-expression of BtuB", "title": "Heterologous expression of cobalamin dependent class-III enzymes.", "to": "cobalamin incorporation"}, {"from": "Bacillus megaterium", "title": "Heterologous expression of cobalamin dependent class-III enzymes.", "to": "highest levels of cofactor incorporation"}, {"from": "co-expression of BtuB in E. coli", "title": "Heterologous expression of cobalamin dependent class-III enzymes.", "to": "appropriate balance between cofactor incorporation and protein yield"}, {"from": "rRNA transcription", "title": "Control of ribosome synthesis in bacteria: the important role of rRNA chain elongation rate.", "to": "ribosome synthesis"}, {"from": "transcriptional initiation rate of rRNA operon", "title": "Control of ribosome synthesis in bacteria: the important role of rRNA chain elongation rate.", "to": "rRNA synthesis"}, {"from": "rRNA chain elongation rate", "title": "Control of ribosome synthesis in bacteria: the important role of rRNA chain elongation rate.", "to": "bacterial ribosome synthesis capacity"}, {"from": "rRNA chain elongation rate of Escherichia coli", "title": "Control of ribosome synthesis in bacteria: the important role of rRNA chain elongation rate.", "to": "constant under nutrient limitation"}, {"from": "rRNA chain elongation rate of Escherichia coli", "title": "Control of ribosome synthesis in bacteria: the important role of rRNA chain elongation rate.", "to": "constant under chloramphenicol inhibition"}, {"from": "rRNA chain elongation rate", "title": "Control of ribosome synthesis in bacteria: the important role of rRNA chain elongation rate.", "to": "decreases under low temperature ranges"}, {"from": "rRNA chain elongation rate of Vibrio natriegens", "title": "Control of ribosome synthesis in bacteria: the important role of rRNA chain elongation rate.", "to": "bacterial ribosome synthesis capacity"}, {"from": "double stranded DNA-binding protein", "title": "Increasing cell-free gene expression yields from linear templates in Escherichia coli and Vibrio natriegens extracts by using DNA-binding proteins.", "to": "protection of linear expression templates"}, {"from": "single-chain Cro protein", "title": "Increasing cell-free gene expression yields from linear templates in Escherichia coli and Vibrio natriegens extracts by using DNA-binding proteins.", "to": "protection of linear expression templates"}, {"from": "CroP-LET method", "title": "Increasing cell-free gene expression yields from linear templates in Escherichia coli and Vibrio natriegens extracts by using DNA-binding proteins.", "to": "increase in superfolder green fluorescent protein expression levels"}, {"from": "Escherichia coli cell-free protein synthesis reactions", "title": "Increasing cell-free gene expression yields from linear templates in Escherichia coli and Vibrio natriegens extracts by using DNA-binding proteins.", "to": "increase in superfolder green fluorescent protein expression levels"}, {"from": "CroP-LET method", "title": "Increasing cell-free gene expression yields from linear templates in Escherichia coli and Vibrio natriegens extracts by using DNA-binding proteins.", "to": "enhancement in yields in cell-free protein synthesis platforms from nonmodel organisms"}, {"from": "Vibrio natriegens cell-free protein synthesis platform", "title": "Increasing cell-free gene expression yields from linear templates in Escherichia coli and Vibrio natriegens extracts by using DNA-binding proteins.", "to": "increase in superfolder green fluorescent protein yields"}, {"from": "Vibrio natriegens", "title": "Design and Reconstruction of Regulatory Parts for Fast-frowing \u003ci\u003eVibrio natriegens\u003c/i\u003e Synthetic Biology.", "to": "gene expression"}, {"from": "promoter region", "title": "Design and Reconstruction of Regulatory Parts for Fast-frowing \u003ci\u003eVibrio natriegens\u003c/i\u003e Synthetic Biology.", "to": "gene expression"}, {"from": "ribosome binding site element", "title": "Design and Reconstruction of Regulatory Parts for Fast-frowing \u003ci\u003eVibrio natriegens\u003c/i\u003e Synthetic Biology.", "to": "gene expression"}, {"from": "synthetic regulatory parts", "title": "Design and Reconstruction of Regulatory Parts for Fast-frowing \u003ci\u003eVibrio natriegens\u003c/i\u003e Synthetic Biology.", "to": "fluorescence intensities"}, {"from": "synthetic regulatory parts", "title": "Design and Reconstruction of Regulatory Parts for Fast-frowing \u003ci\u003eVibrio natriegens\u003c/i\u003e Synthetic Biology.", "to": "stability"}, {"from": "nutrient-rich media", "title": "Design and Reconstruction of Regulatory Parts for Fast-frowing \u003ci\u003eVibrio natriegens\u003c/i\u003e Synthetic Biology.", "to": "synthetic regulatory parts stability"}, {"from": "minimal media", "title": "Design and Reconstruction of Regulatory Parts for Fast-frowing \u003ci\u003eVibrio natriegens\u003c/i\u003e Synthetic Biology.", "to": "synthetic regulatory parts stability"}, {"from": "galactose permease", "title": "Design and Reconstruction of Regulatory Parts for Fast-frowing \u003ci\u003eVibrio natriegens\u003c/i\u003e Synthetic Biology.", "to": "glucose transport"}, {"from": "glucokinase", "title": "Design and Reconstruction of Regulatory Parts for Fast-frowing \u003ci\u003eVibrio natriegens\u003c/i\u003e Synthetic Biology.", "to": "glucose transport"}, {"from": "synthetic regulatory parts", "title": "Design and Reconstruction of Regulatory Parts for Fast-frowing \u003ci\u003eVibrio natriegens\u003c/i\u003e Synthetic Biology.", "to": "galactose permease expression modulation"}, {"from": "synthetic regulatory parts", "title": "Design and Reconstruction of Regulatory Parts for Fast-frowing \u003ci\u003eVibrio natriegens\u003c/i\u003e Synthetic Biology.", "to": "glucokinase"}, {"from": "Genome mining", "title": "Investigating the Product Profiles and Structural Relationships of New Levansucrases with Conventional and Non-Conventional Substrates.", "to": "Levansucrase enzymes with increased transfructosylating activity and wider acceptor promiscuity"}, {"from": "Levansucrase", "title": "Investigating the Product Profiles and Structural Relationships of New Levansucrases with Conventional and Non-Conventional Substrates.", "to": "Fructooligosaccharides synthesis"}, {"from": "Levansucrase", "title": "Investigating the Product Profiles and Structural Relationships of New Levansucrases with Conventional and Non-Conventional Substrates.", "to": "\u03b2-(2\u21926"}, {"from": "Levansucrase from Gluconobacter oxydans", "title": "Investigating the Product Profiles and Structural Relationships of New Levansucrases with Conventional and Non-Conventional Substrates.", "to": "Fructooligosaccharides with up to 13 units in length"}, {"from": "Levansucrase from Novosphingobium aromaticivorans", "title": "Investigating the Product Profiles and Structural Relationships of New Levansucrases with Conventional and Non-Conventional Substrates.", "to": "Fructooligosaccharides with up to 13 units in length"}, {"from": "Alignment of amino acid sequences and substrate docking with homology models", "title": "Investigating the Product Profiles and Structural Relationships of New Levansucrases with Conventional and Non-Conventional Substrates.", "to": "Structural elements causing differences in product spectra"}, {"from": "Raffinose", "title": "Investigating the Product Profiles and Structural Relationships of New Levansucrases with Conventional and Non-Conventional Substrates.", "to": "Preferred donor molecule for Levansucrase from Vibrio natriegens"}, {"from": "Raffinose", "title": "Investigating the Product Profiles and Structural Relationships of New Levansucrases with Conventional and Non-Conventional Substrates.", "to": "Preferred donor molecule for Levansucrase from Novosphingobium aromaticivorans"}, {"from": "Raffinose", "title": "Investigating the Product Profiles and Structural Relationships of New Levansucrases with Conventional and Non-Conventional Substrates.", "to": "Preferred donor molecule for Levansucrase from Paraburkolderia graminis"}, {"from": "Levansucrases", "title": "Investigating the Product Profiles and Structural Relationships of New Levansucrases with Conventional and Non-Conventional Substrates.", "to": "Wide acceptor promiscuity"}, {"from": "Levansucrases", "title": "Investigating the Product Profiles and Structural Relationships of New Levansucrases with Conventional and Non-Conventional Substrates.", "to": "Utilization of monosaccharides"}, {"from": "Levansucrases", "title": "Investigating the Product Profiles and Structural Relationships of New Levansucrases with Conventional and Non-Conventional Substrates.", "to": "Utilization of two alcohols"}, {"from": "Vibrio natriegens", "title": "A Hybrid Extracellular Electron Transfer Pathway Enhances the Survival of Vibrio natriegens.", "to": "extracellular electron transfer"}, {"from": "extracellular electron transfer", "title": "A Hybrid Extracellular Electron Transfer Pathway Enhances the Survival of Vibrio natriegens.", "to": "fermentative conditions with gluconate"}, {"from": "extracellular electron transfer", "title": "A Hybrid Extracellular Electron Transfer Pathway Enhances the Survival of Vibrio natriegens.", "to": "fermentative conditions with glucosamine"}, {"from": "extracellular electron transfer", "title": "A Hybrid Extracellular Electron Transfer Pathway Enhances the Survival of Vibrio natriegens.", "to": "fermentative conditions with pyruvate"}, {"from": "extracellular electron transfer", "title": "A Hybrid Extracellular Electron Transfer Pathway Enhances the Survival of Vibrio natriegens.", "to": "requires CymA"}, {"from": "extracellular electron transfer", "title": "A Hybrid Extracellular Electron Transfer Pathway Enhances the Survival of Vibrio natriegens.", "to": "requires PdsA"}, {"from": "extracellular electron transfer", "title": "A Hybrid Extracellular Electron Transfer Pathway Enhances the Survival of Vibrio natriegens.", "to": "requires MtrCAB"}, {"from": "extracellular electron transfer", "title": "A Hybrid Extracellular Electron Transfer Pathway Enhances the Survival of Vibrio natriegens.", "to": "Fe(III"}, {"from": "CymA", "title": "A Hybrid Extracellular Electron Transfer Pathway Enhances the Survival of Vibrio natriegens.", "to": "diverged from Shewanella"}, {"from": "CymA", "title": "A Hybrid Extracellular Electron Transfer Pathway Enhances the Survival of Vibrio natriegens.", "to": "from Vibrionaceae"}, {"from": "NapC", "title": "A Hybrid Extracellular Electron Transfer Pathway Enhances the Survival of Vibrio natriegens.", "to": "gammaproteobacteria"}, {"from": "extracellular electron transfer", "title": "A Hybrid Extracellular Electron Transfer Pathway Enhances the Survival of Vibrio natriegens.", "to": "anaerobic survival of Vibrio natriegens"}, {"from": "extracellular electron transfer", "title": "A Hybrid Extracellular Electron Transfer Pathway Enhances the Survival of Vibrio natriegens.", "to": "primary physiological function in Vibrionaceae"}, {"from": "Vibrio natriegens genes", "title": "A Hybrid Extracellular Electron Transfer Pathway Enhances the Survival of Vibrio natriegens.", "to": "functionally complemented Shewanella oneidensis mutants"}, {"from": "extracellular electron transfer", "title": "A Hybrid Extracellular Electron Transfer Pathway Enhances the Survival of Vibrio natriegens.", "to": "insoluble oxide minerals utilization"}, {"from": "extracellular electron transfer", "title": "A Hybrid Extracellular Electron Transfer Pathway Enhances the Survival of Vibrio natriegens.", "to": "enhanced survival of Vibrio natriegens under fermentative conditions"}, {"from": "Vibrio natriegens", "title": "Comparison of simple expression procedures in novel expression host Vibrio natriegens and established Escherichia coli system.", "to": "fast growth rates"}, {"from": "Vibrio natriegens", "title": "Comparison of simple expression procedures in novel expression host Vibrio natriegens and established Escherichia coli system.", "to": "short doubling time"}, {"from": "Vibrio natriegens", "title": "Comparison of simple expression procedures in novel expression host Vibrio natriegens and established Escherichia coli system.", "to": "high level of proteosynthetic activity"}, {"from": "Vibrio natriegens", "title": "Comparison of simple expression procedures in novel expression host Vibrio natriegens and established Escherichia coli system.", "to": "non-pathogenicity"}, {"from": "Vibrio natriegens", "title": "Comparison of simple expression procedures in novel expression host Vibrio natriegens and established Escherichia coli system.", "to": "easy scale-up process"}, {"from": "Vibrio natriegens", "title": "Comparison of simple expression procedures in novel expression host Vibrio natriegens and established Escherichia coli system.", "to": "potential for using alternative carbon sources"}, {"from": "Vibrio natriegens", "title": "Comparison of simple expression procedures in novel expression host Vibrio natriegens and established Escherichia coli system.", "to": "potential for further genetic and metabolic engineering"}, {"from": "Vibrio natriegens", "title": "Comparison of simple expression procedures in novel expression host Vibrio natriegens and established Escherichia coli system.", "to": "growth media"}, {"from": "Vibrio natriegens", "title": "Comparison of simple expression procedures in novel expression host Vibrio natriegens and established Escherichia coli system.", "to": "wide range of genetic tools"}, {"from": "Vibrio natriegens", "title": "Comparison of simple expression procedures in novel expression host Vibrio natriegens and established Escherichia coli system.", "to": "production of recombinant human growth hormone"}, {"from": "Vibrio natriegens", "title": "Comparison of simple expression procedures in novel expression host Vibrio natriegens and established Escherichia coli system.", "to": "production of yeast alcohol dehydrogenase"}, {"from": "Vibrio natriegens", "title": "Comparison of simple expression procedures in novel expression host Vibrio natriegens and established Escherichia coli system.", "to": "production of production of archaeal catalase-peroxidase"}, {"from": "sucrose", "title": "Comparison of simple expression procedures in novel expression host Vibrio natriegens and established Escherichia coli system.", "to": "higher biomass productions in Vibrio natriegens compared to Escherichia coli"}, {"from": "human growth hormone", "title": "Comparison of simple expression procedures in novel expression host Vibrio natriegens and established Escherichia coli system.", "to": "lower production in Vibrio natriegens compared to Escherichia coli"}, {"from": "yeast alcohol dehydrogenase", "title": "Comparison of simple expression procedures in novel expression host Vibrio natriegens and established Escherichia coli system.", "to": "lower production and decreased solubility in Vibrio natriegens compared to Escherichia coli"}, {"from": "production of archaeal catalase-peroxidase", "title": "Comparison of simple expression procedures in novel expression host Vibrio natriegens and established Escherichia coli system.", "to": "higher production in Vibrio natriegens compared to Escherichia coli"}, {"from": "Vibrio natriegens", "title": "Comparison of simple expression procedures in novel expression host Vibrio natriegens and established Escherichia coli system.", "to": "susceptibility to condition changes"}, {"from": "Vibrio natriegens", "title": "Comparison of simple expression procedures in novel expression host Vibrio natriegens and established Escherichia coli system.", "to": "inconsistent results"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens: an ultrafast-growing marine bacterium as emerging synthetic biology chassis.", "to": "non-pathogenic bacterium"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens: an ultrafast-growing marine bacterium as emerging synthetic biology chassis.", "to": "alternative chassis organism"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens: an ultrafast-growing marine bacterium as emerging synthetic biology chassis.", "to": "synthetic biology efforts"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens: an ultrafast-growing marine bacterium as emerging synthetic biology chassis.", "to": "establishment of molecular biology tools"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens: an ultrafast-growing marine bacterium as emerging synthetic biology chassis.", "to": "biotechnological production cycles"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens: an ultrafast-growing marine bacterium as emerging synthetic biology chassis.", "to": "natural ecological niche"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens: an ultrafast-growing marine bacterium as emerging synthetic biology chassis.", "to": "interactions with other organisms"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens: an ultrafast-growing marine bacterium as emerging synthetic biology chassis.", "to": "genomic organization"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens: an ultrafast-growing marine bacterium as emerging synthetic biology chassis.", "to": "diverse metabolism"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens: an ultrafast-growing marine bacterium as emerging synthetic biology chassis.", "to": "synthetic biology chassis"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens: an ultrafast-growing marine bacterium as emerging synthetic biology chassis.", "to": "methods"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens: an ultrafast-growing marine bacterium as emerging synthetic biology chassis.", "to": "research community growth"}, {"from": "natural clay nanotubes loaded with biocide", "title": "Development of Marine Antifouling Epoxy Coating Enhanced with Clay Nanotubes.", "to": "extended protection against marine microorganisms"}, {"from": "natural clay nanotubes loaded with silver", "title": "Development of Marine Antifouling Epoxy Coating Enhanced with Clay Nanotubes.", "to": "extended protection against marine microorganisms"}, {"from": "halloysite formulations with biocide-loaded nanoclay", "title": "Development of Marine Antifouling Epoxy Coating Enhanced with Clay Nanotubes.", "to": "extended DCOIT release time to 12 months"}, {"from": "epoxy-halloysite nanocomposites", "title": "Development of Marine Antifouling Epoxy Coating Enhanced with Clay Nanotubes.", "to": "less adhesion and proliferation of marine bacteria Vibrio natriegens"}, {"from": "DCOIT-loaded natural environmentally friendly halloysite clay", "title": "Development of Marine Antifouling Epoxy Coating Enhanced with Clay Nanotubes.", "to": "improved antifouling properties of epoxy coating"}, {"from": "silver particles synthesized on halloysite nanotubes", "title": "Development of Marine Antifouling Epoxy Coating Enhanced with Clay Nanotubes.", "to": "enhanced antifouling property of epoxy resin"}, {"from": "MnO particles and halloysite natural mixture", "title": "Development of Marine Antifouling Epoxy Coating Enhanced with Clay Nanotubes.", "to": "used as nonbiocide additive to marine coating"}, {"from": "Ag-halloysite composite", "title": "Development of Marine Antifouling Epoxy Coating Enhanced with Clay Nanotubes.", "to": "no visible fouling in short-term White Sea water test"}, {"from": "MnO particles and halloysite natural mixture", "title": "Development of Marine Antifouling Epoxy Coating Enhanced with Clay Nanotubes.", "to": "no visible fouling in short-term White Sea water test"}, {"from": "Vibrio natriegens", "title": "Melanin Produced by the Fast-Growing Marine Bacterium Vibrio natriegens through Heterologous Biosynthesis: Characterization and Application.", "to": "melanin production"}, {"from": "heterologous tyrosinase gene", "title": "Melanin Produced by the Fast-Growing Marine Bacterium Vibrio natriegens through Heterologous Biosynthesis: Characterization and Application.", "to": "melanin production"}, {"from": "melanin", "title": "Melanin Produced by the Fast-Growing Marine Bacterium Vibrio natriegens through Heterologous Biosynthesis: Characterization and Application.", "to": "derived from dihydroxyindole-2-carboxylic acid"}, {"from": "melanin", "title": "Melanin Produced by the Fast-Growing Marine Bacterium Vibrio natriegens through Heterologous Biosynthesis: Characterization and Application.", "to": "polymer"}, {"from": "melanin", "title": "Melanin Produced by the Fast-Growing Marine Bacterium Vibrio natriegens through Heterologous Biosynthesis: Characterization and Application.", "to": "nanoparticle structure"}, {"from": "melanin", "title": "Melanin Produced by the Fast-Growing Marine Bacterium Vibrio natriegens through Heterologous Biosynthesis: Characterization and Application.", "to": "protection of cells from UV irradiation"}, {"from": "melanin", "title": "Melanin Produced by the Fast-Growing Marine Bacterium Vibrio natriegens through Heterologous Biosynthesis: Characterization and Application.", "to": "redox activity"}, {"from": "melanin", "title": "Melanin Produced by the Fast-Growing Marine Bacterium Vibrio natriegens through Heterologous Biosynthesis: Characterization and Application.", "to": "adsorption of trinitrotoluene"}, {"from": "melanin nanoparticles", "title": "Melanin Produced by the Fast-Growing Marine Bacterium Vibrio natriegens through Heterologous Biosynthesis: Characterization and Application.", "to": "photoprotective properties"}, {"from": "expressing heterologous tyrosinase gene", "title": "Melanin Produced by the Fast-Growing Marine Bacterium Vibrio natriegens through Heterologous Biosynthesis: Characterization and Application.", "to": "melanin nanoparticles"}, {"from": "expressing heterologous tyrosinase gene with inducible promoters", "title": "Melanin Produced by the Fast-Growing Marine Bacterium Vibrio natriegens through Heterologous Biosynthesis: Characterization and Application.", "to": "physical and chemical properties of melanin"}, {"from": "melanin", "title": "Melanin Produced by the Fast-Growing Marine Bacterium Vibrio natriegens through Heterologous Biosynthesis: Characterization and Application.", "to": "physical and chemical properties of melanin"}, {"from": "Vibrio natriegens", "title": "Exploiting the Feedstock Flexibility of the Emergent Synthetic Biology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e for Engineered Natural Product Production.", "to": "genetic tractability"}, {"from": "Vibrio natriegens", "title": "Exploiting the Feedstock Flexibility of the Emergent Synthetic Biology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e for Engineered Natural Product Production.", "to": "feedstock flexibility"}, {"from": "Vibrio natriegens", "title": "Exploiting the Feedstock Flexibility of the Emergent Synthetic Biology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e for Engineered Natural Product Production.", "to": "beta-carotene production"}, {"from": "Vibrio natriegens", "title": "Exploiting the Feedstock Flexibility of the Emergent Synthetic Biology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e for Engineered Natural Product Production.", "to": "violacein production"}, {"from": "beta-carotene production", "title": "Exploiting the Feedstock Flexibility of the Emergent Synthetic Biology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e for Engineered Natural Product Production.", "to": "beta-carotene production"}, {"from": "Vibrio campbellii", "title": "Exploiting the Feedstock Flexibility of the Emergent Synthetic Biology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e for Engineered Natural Product Production.", "to": "beta-carotene production"}, {"from": "mevalonate biosynthetic pathway", "title": "Exploiting the Feedstock Flexibility of the Emergent Synthetic Biology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e for Engineered Natural Product Production.", "to": "precursor abundance"}, {"from": "Lactobacillus acidophilus", "title": "Exploiting the Feedstock Flexibility of the Emergent Synthetic Biology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e for Engineered Natural Product Production.", "to": "mevalonate biosynthetic pathway"}, {"from": "biosynthetic gene cluster", "title": "Exploiting the Feedstock Flexibility of the Emergent Synthetic Biology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e for Engineered Natural Product Production.", "to": "violacein production"}, {"from": "Chromobacterium violaceum", "title": "Exploiting the Feedstock Flexibility of the Emergent Synthetic Biology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e for Engineered Natural Product Production.", "to": "biosynthetic gene cluster"}, {"from": "Vibrio natriegens", "title": "Exploiting the Feedstock Flexibility of the Emergent Synthetic Biology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e for Engineered Natural Product Production.", "to": "small molecule drug production"}, {"from": "Vibrio natriegens", "title": "Exploiting the Feedstock Flexibility of the Emergent Synthetic Biology Chassis \u003ci\u003eVibrio natriegens\u003c/i\u003e for Engineered Natural Product Production.", "to": "natural product production with industrially-relevant feedstocks"}, {"from": "aerobic metabolism", "title": "OxyR Is a Convergent Target for Mutations Acquired during Adaptation to Oxidative Stress-Prone Metabolic States.", "to": "oxidative stress"}, {"from": "peroxide-sensing transcription factor", "title": "OxyR Is a Convergent Target for Mutations Acquired during Adaptation to Oxidative Stress-Prone Metabolic States.", "to": "mutations"}, {"from": "mutations", "title": "OxyR Is a Convergent Target for Mutations Acquired during Adaptation to Oxidative Stress-Prone Metabolic States.", "to": "oxidized conformation of OxyR"}, {"from": "oxidized conformation of OxyR", "title": "OxyR Is a Convergent Target for Mutations Acquired during Adaptation to Oxidative Stress-Prone Metabolic States.", "to": "constitutive expression of genes"}, {"from": "constitutive activity of OxyR", "title": "OxyR Is a Convergent Target for Mutations Acquired during Adaptation to Oxidative Stress-Prone Metabolic States.", "to": "reduces DNA damage from reactive oxygen species"}, {"from": "reactive oxygen species", "title": "OxyR Is a Convergent Target for Mutations Acquired during Adaptation to Oxidative Stress-Prone Metabolic States.", "to": "activity of the SOS response regulator LexA"}, {"from": "constitutive activity of OxyR", "title": "OxyR Is a Convergent Target for Mutations Acquired during Adaptation to Oxidative Stress-Prone Metabolic States.", "to": "lower growth"}, {"from": "lower growth", "title": "OxyR Is a Convergent Target for Mutations Acquired during Adaptation to Oxidative Stress-Prone Metabolic States.", "to": "calculations of proteome allocation"}, {"from": "calculations of proteome allocation", "title": "OxyR Is a Convergent Target for Mutations Acquired during Adaptation to Oxidative Stress-Prone Metabolic States.", "to": "genome-scale models of metabolism and macromolecular expression"}, {"from": "oxidative stress", "title": "OxyR Is a Convergent Target for Mutations Acquired during Adaptation to Oxidative Stress-Prone Metabolic States.", "to": "genetic changes in oxyR"}, {"from": "ribosome binding sites", "title": "Selection of High Producers From Combinatorial Libraries for the Production of Recombinant Proteins in \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "protein synthesis"}, {"from": "N-terminal affinity tags", "title": "Selection of High Producers From Combinatorial Libraries for the Production of Recombinant Proteins in \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "protein synthesis"}, {"from": "mRNA secondary structures", "title": "Selection of High Producers From Combinatorial Libraries for the Production of Recombinant Proteins in \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "protein synthesis"}, {"from": "ribosome binding sites", "title": "Selection of High Producers From Combinatorial Libraries for the Production of Recombinant Proteins in \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "expression optimization"}, {"from": "N-terminal affinity tags", "title": "Selection of High Producers From Combinatorial Libraries for the Production of Recombinant Proteins in \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "expression optimization"}, {"from": "periplasmic translocation sequences", "title": "Selection of High Producers From Combinatorial Libraries for the Production of Recombinant Proteins in \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "expression optimization"}, {"from": "Green fluorescent protein", "title": "Selection of High Producers From Combinatorial Libraries for the Production of Recombinant Proteins in \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "identification of high producers"}, {"from": "thrombin cleavage site", "title": "Selection of High Producers From Combinatorial Libraries for the Production of Recombinant Proteins in \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "removal of fusion tags"}, {"from": "antimicrobial peptide", "title": "Selection of High Producers From Combinatorial Libraries for the Production of Recombinant Proteins in \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "expression in Escherichia coli"}, {"from": "antifungal peptide", "title": "Selection of High Producers From Combinatorial Libraries for the Production of Recombinant Proteins in \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "expression in Escherichia coli"}, {"from": "urate oxidase", "title": "Selection of High Producers From Combinatorial Libraries for the Production of Recombinant Proteins in \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "expression in Escherichia coli"}, {"from": "antimicrobial peptide", "title": "Selection of High Producers From Combinatorial Libraries for the Production of Recombinant Proteins in \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "expression in Vibrio natriegens"}, {"from": "antifungal peptide", "title": "Selection of High Producers From Combinatorial Libraries for the Production of Recombinant Proteins in \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "expression in Vibrio natriegens"}, {"from": "urate oxidase", "title": "Selection of High Producers From Combinatorial Libraries for the Production of Recombinant Proteins in \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "expression in Vibrio natriegens"}, {"from": "Escherichia coli ssYahJ secretion signal", "title": "Selection of High Producers From Combinatorial Libraries for the Production of Recombinant Proteins in \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "translocation to periplasm in Vibrio natriegens"}, {"from": "combinatorial library approach", "title": "Selection of High Producers From Combinatorial Libraries for the Production of Recombinant Proteins in \u003ci\u003eEscherichia coli\u003c/i\u003e and \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "selection of high-producer clones"}, {"from": "naturally competent Vibrio natriegens", "title": "Complete Genome Sequence of Vibrio natriegens Phage Phriendly.", "to": "podophage Phriendly"}, {"from": "Silver nanoparticles particle size", "title": "Antibacterial activity of silver nanoparticles of different particle size against Vibrio Natriegens.", "to": "Minimum inhibitory concentration value"}, {"from": "Silver nanoparticles particle size", "title": "Antibacterial activity of silver nanoparticles of different particle size against Vibrio Natriegens.", "to": "Minimum bactericidal concentrations value"}, {"from": "Silver nanoparticles particle size", "title": "Antibacterial activity of silver nanoparticles of different particle size against Vibrio Natriegens.", "to": "Antibacterial activity"}, {"from": "Antibacterial activity", "title": "Antibacterial activity of silver nanoparticles of different particle size against Vibrio Natriegens.", "to": "Reactive oxygen species generation by bacteria"}, {"from": "Antibacterial activity", "title": "Antibacterial activity of silver nanoparticles of different particle size against Vibrio Natriegens.", "to": "Bacterial membrane damage"}, {"from": "Reactive oxygen species", "title": "Antibacterial activity of silver nanoparticles of different particle size against Vibrio Natriegens.", "to": "Bacterial membrane damage"}, {"from": "spaceflight conditions", "title": "Introducing \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Microbial Model Organism for Microgravity Research.", "to": "physiological microbial alterations"}, {"from": "microgravity", "title": "Introducing \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Microbial Model Organism for Microgravity Research.", "to": "increased growth yields"}, {"from": "microgravity", "title": "Introducing \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Microbial Model Organism for Microgravity Research.", "to": "decreased antibiotic susceptibility"}, {"from": "Vibrio natriegens", "title": "Introducing \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Microbial Model Organism for Microgravity Research.", "to": "fast generation time"}, {"from": "simulated microgravity", "title": "Introducing \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Microbial Model Organism for Microgravity Research.", "to": "Vibrio natriegens biomass increase"}, {"from": "normal terrestrial gravity", "title": "Introducing \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Microbial Model Organism for Microgravity Research.", "to": "final cell population"}, {"from": "simulated microgravity", "title": "Introducing \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Microbial Model Organism for Microgravity Research.", "to": "final cell population"}, {"from": "simulated microgravity", "title": "Introducing \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Microbial Model Organism for Microgravity Research.", "to": "no change in susceptibility to rifampicin"}, {"from": "normal terrestrial gravity", "title": "Introducing \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Microbial Model Organism for Microgravity Research.", "to": "no change in susceptibility to rifampicin"}, {"from": "Vibrio natriegens", "title": "Introducing \u003ci\u003eVibrio natriegens\u003c/i\u003e as a Microbial Model Organism for Microgravity Research.", "to": "model organism for microbial microgravity research"}, {"from": "Vibrio natriegens", "title": "Synthetic Biology Tools for the Fast-Growing Marine Bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "resistance markers"}, {"from": "Vibrio natriegens", "title": "Synthetic Biology Tools for the Fast-Growing Marine Bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "promoters portability"}, {"from": "Vibrio natriegens", "title": "Synthetic Biology Tools for the Fast-Growing Marine Bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "ribosomal binding sites"}, {"from": "Vibrio natriegens", "title": "Synthetic Biology Tools for the Fast-Growing Marine Bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "reporters"}, {"from": "Vibrio natriegens", "title": "Synthetic Biology Tools for the Fast-Growing Marine Bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "terminators"}, {"from": "Vibrio natriegens", "title": "Synthetic Biology Tools for the Fast-Growing Marine Bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "degradation tags"}, {"from": "Vibrio natriegens", "title": "Synthetic Biology Tools for the Fast-Growing Marine Bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "origin of replication sequences"}, {"from": "Vibrio natriegens", "title": "Synthetic Biology Tools for the Fast-Growing Marine Bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "plasmid backbones"}, {"from": "genetic parts", "title": "Synthetic Biology Tools for the Fast-Growing Marine Bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "plasmid stability"}, {"from": "genetic parts", "title": "Synthetic Biology Tools for the Fast-Growing Marine Bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "plasmid copy numbers"}, {"from": "genetic parts", "title": "Synthetic Biology Tools for the Fast-Growing Marine Bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "production load on cells"}, {"from": "constructs", "title": "Synthetic Biology Tools for the Fast-Growing Marine Bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "chemical induction"}, {"from": "constructs", "title": "Synthetic Biology Tools for the Fast-Growing Marine Bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "optogenetic induction"}, {"from": "engineered circuit behavior", "title": "Synthetic Biology Tools for the Fast-Growing Marine Bacterium \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "Vibrio natriegens"}, {"from": "naturally competent Vibrio natriegens MK3", "title": "Effect of biosurfactant derived from Vibrio natriegens MK3 against Vibrio harveyi biofilm and virulence.", "to": "glycolipid biosurfactant"}, {"from": "glycolipid biosurfactant", "title": "Effect of biosurfactant derived from Vibrio natriegens MK3 against Vibrio harveyi biofilm and virulence.", "to": "decrease in surface tension"}, {"from": "glycolipid biosurfactant", "title": "Effect of biosurfactant derived from Vibrio natriegens MK3 against Vibrio harveyi biofilm and virulence.", "to": "reduction in biofilm formation"}, {"from": "glycolipid biosurfactant", "title": "Effect of biosurfactant derived from Vibrio natriegens MK3 against Vibrio harveyi biofilm and virulence.", "to": "reduction in bioluminescence"}, {"from": "glycolipid biosurfactant", "title": "Effect of biosurfactant derived from Vibrio natriegens MK3 against Vibrio harveyi biofilm and virulence.", "to": "reduction in extracellular polysaccharide synthesis"}, {"from": "glycolipid biosurfactant", "title": "Effect of biosurfactant derived from Vibrio natriegens MK3 against Vibrio harveyi biofilm and virulence.", "to": "reduction in quorum sensing"}, {"from": "glycolipid biosurfactant", "title": "Effect of biosurfactant derived from Vibrio natriegens MK3 against Vibrio harveyi biofilm and virulence.", "to": "antipathogenic effect against Vibrio harveyi"}, {"from": "cocultivation with Vibrio natriegens", "title": "Improvement of Euglena gracilis Paramylon Production through a Cocultivation Strategy with the Indole-3-Acetic Acid-Producing Bacterium Vibrio natriegens.", "to": "increase in biomass productivity of Euglena gracilis"}, {"from": "cocultivation with Vibrio natriegens", "title": "Improvement of Euglena gracilis Paramylon Production through a Cocultivation Strategy with the Indole-3-Acetic Acid-Producing Bacterium Vibrio natriegens.", "to": "increase in paramylon content of Euglena gracilis"}, {"from": "bacterial inoculum density", "title": "Improvement of Euglena gracilis Paramylon Production through a Cocultivation Strategy with the Indole-3-Acetic Acid-Producing Bacterium Vibrio natriegens.", "to": "growth of Euglena gracilis"}, {"from": "timing of inoculation", "title": "Improvement of Euglena gracilis Paramylon Production through a Cocultivation Strategy with the Indole-3-Acetic Acid-Producing Bacterium Vibrio natriegens.", "to": "growth of Euglena gracilis"}, {"from": "optimal deployment of Vibrio natriegens", "title": "Improvement of Euglena gracilis Paramylon Production through a Cocultivation Strategy with the Indole-3-Acetic Acid-Producing Bacterium Vibrio natriegens.", "to": "increase in biomass productivity of Euglena gracilis by more than 20%"}, {"from": "optimal deployment of Vibrio natriegens", "title": "Improvement of Euglena gracilis Paramylon Production through a Cocultivation Strategy with the Indole-3-Acetic Acid-Producing Bacterium Vibrio natriegens.", "to": "increase in paramylon content of Euglena gracilis by more than 35%"}, {"from": "indole-3-acetic acid biosynthesized by Vibrio natriegens", "title": "Improvement of Euglena gracilis Paramylon Production through a Cocultivation Strategy with the Indole-3-Acetic Acid-Producing Bacterium Vibrio natriegens.", "to": "growth of Euglena gracilis"}, {"from": "cocultivation", "title": "Improvement of Euglena gracilis Paramylon Production through a Cocultivation Strategy with the Indole-3-Acetic Acid-Producing Bacterium Vibrio natriegens.", "to": "significantly larger paramylon granules in Euglena gracilis"}, {"from": "cocultivation with indole-3-acetic acid-producing Vibrio natriegens", "title": "Improvement of Euglena gracilis Paramylon Production through a Cocultivation Strategy with the Indole-3-Acetic Acid-Producing Bacterium Vibrio natriegens.", "to": "enhanced production of paramylon in Euglena gracilis"}, {"from": "DNA-damaging agent mitomycin C", "title": "Generation of a Prophage-Free Variant of the Fast-Growing Bacterium Vibrio natriegens.", "to": "prophage regions induction"}, {"from": "standard cultivation conditions", "title": "Generation of a Prophage-Free Variant of the Fast-Growing Bacterium Vibrio natriegens.", "to": "spontaneous activation of prophage regions"}, {"from": "standard cultivation conditions", "title": "Generation of a Prophage-Free Variant of the Fast-Growing Bacterium Vibrio natriegens.", "to": "spontaneous activation of VNP1"}, {"from": "standard cultivation conditions", "title": "Generation of a Prophage-Free Variant of the Fast-Growing Bacterium Vibrio natriegens.", "to": "spontaneous activation of VNP2"}, {"from": "mCherry fusion to the major capsid proteins", "title": "Generation of a Prophage-Free Variant of the Fast-Growing Bacterium Vibrio natriegens.", "to": "confirmation of activation of VNP1"}, {"from": "Transmission electron microscopy", "title": "Generation of a Prophage-Free Variant of the Fast-Growing Bacterium Vibrio natriegens.", "to": "release of phage particles belonging to the Siphoviridae family"}, {"from": "removal of proviral load", "title": "Generation of a Prophage-Free Variant of the Fast-Growing Bacterium Vibrio natriegens.", "to": "improved robustness toward DNA-damaging conditions"}, {"from": "removal of proviral load", "title": "Generation of a Prophage-Free Variant of the Fast-Growing Bacterium Vibrio natriegens.", "to": "reduced cell lysis under hypo-osmotic conditions"}, {"from": "removal of proviral load", "title": "Generation of a Prophage-Free Variant of the Fast-Growing Bacterium Vibrio natriegens.", "to": "increased pyruvate production"}, {"from": "prophage-free variant", "title": "Generation of a Prophage-Free Variant of the Fast-Growing Bacterium Vibrio natriegens.", "to": "outcompeted the wild type in a competitive growth experiment"}, {"from": "prophage-free strain", "title": "Generation of a Prophage-Free Variant of the Fast-Growing Bacterium Vibrio natriegens.", "to": "higher tolerance toward DNA damage"}, {"from": "prophage-free strain", "title": "Generation of a Prophage-Free Variant of the Fast-Growing Bacterium Vibrio natriegens.", "to": "higher tolerance toward hypo-osmotic stress"}, {"from": "prophage elements", "title": "Generation of a Prophage-Free Variant of the Fast-Growing Bacterium Vibrio natriegens.", "to": "detrimental effects on growth and fitness"}, {"from": "prophage-free variant", "title": "Generation of a Prophage-Free Variant of the Fast-Growing Bacterium Vibrio natriegens.", "to": "promising platform for biotechnological applications"}, {"from": "aggregates", "title": "Direct Detection of Heterotrophic Diazotrophs Associated with Planktonic Aggregates.", "to": "Nitrogen fixation"}, {"from": "heterotrophic diazotrophs", "title": "Direct Detection of Heterotrophic Diazotrophs Associated with Planktonic Aggregates.", "to": "Nitrogen fixation"}, {"from": "polysaccharides", "title": "Direct Detection of Heterotrophic Diazotrophs Associated with Planktonic Aggregates.", "to": "aggregates"}, {"from": "nitrogenase-immunolabeling", "title": "Direct Detection of Heterotrophic Diazotrophs Associated with Planktonic Aggregates.", "to": "active heterotrophic diazotrophs"}, {"from": "Alcian blue", "title": "Direct Detection of Heterotrophic Diazotrophs Associated with Planktonic Aggregates.", "to": "polysaccharides"}, {"from": "concanavalin-A", "title": "Direct Detection of Heterotrophic Diazotrophs Associated with Planktonic Aggregates.", "to": "polysaccharides"}, {"from": "nucleic-acid staining", "title": "Direct Detection of Heterotrophic Diazotrophs Associated with Planktonic Aggregates.", "to": "total bacteria"}, {"from": "artificial aggregates", "title": "Direct Detection of Heterotrophic Diazotrophs Associated with Planktonic Aggregates.", "to": "active heterotrophic diazotrophs"}, {"from": "natural aggregates", "title": "Direct Detection of Heterotrophic Diazotrophs Associated with Planktonic Aggregates.", "to": "active heterotrophic diazotrophs"}, {"from": "Escherichia coli-based expression systems", "title": "Vibrio natriegens: An Alternative Expression System for the High-Yield Production of Isotopically Labeled Proteins.", "to": "recombinant proteins"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens: An Alternative Expression System for the High-Yield Production of Isotopically Labeled Proteins.", "to": "recombinant proteins"}, {"from": "M9 minimal medium", "title": "Vibrio natriegens: An Alternative Expression System for the High-Yield Production of Isotopically Labeled Proteins.", "to": "Protein yield"}, {"from": "early time point of induction", "title": "Vibrio natriegens: An Alternative Expression System for the High-Yield Production of Isotopically Labeled Proteins.", "to": "Protein yield"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens: An Alternative Expression System for the High-Yield Production of Isotopically Labeled Proteins.", "to": "cost-effectiveness"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens: An Alternative Expression System for the High-Yield Production of Isotopically Labeled Proteins.", "to": "convenience"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens: An Alternative Expression System for the High-Yield Production of Isotopically Labeled Proteins.", "to": "time efficiency"}, {"from": "FKBP", "title": "Vibrio natriegens: An Alternative Expression System for the High-Yield Production of Isotopically Labeled Proteins.", "to": "Protein yield"}, {"from": "EYFP", "title": "Vibrio natriegens: An Alternative Expression System for the High-Yield Production of Isotopically Labeled Proteins.", "to": "Protein yield"}, {"from": "FKBP", "title": "Vibrio natriegens: An Alternative Expression System for the High-Yield Production of Isotopically Labeled Proteins.", "to": "correct folding"}, {"from": "EYFP", "title": "Vibrio natriegens: An Alternative Expression System for the High-Yield Production of Isotopically Labeled Proteins.", "to": "correct folding"}, {"from": "Vibrio natriegens", "title": "\u003ci\u003eBacillus subtilis\u003c/i\u003e Inhibits \u003ci\u003eVibrio natriegens\u003c/i\u003e-Induced Corrosion via Biomineralization in Seawater.", "to": "corrosion of steel materials"}, {"from": "Bacillus subtilis", "title": "\u003ci\u003eBacillus subtilis\u003c/i\u003e Inhibits \u003ci\u003eVibrio natriegens\u003c/i\u003e-Induced Corrosion via Biomineralization in Seawater.", "to": "formation of a biomineralized film"}, {"from": "biomineralized film", "title": "\u003ci\u003eBacillus subtilis\u003c/i\u003e Inhibits \u003ci\u003eVibrio natriegens\u003c/i\u003e-Induced Corrosion via Biomineralization in Seawater.", "to": "inhibition of Vibrio natriegens growth"}, {"from": "biomineralized film", "title": "\u003ci\u003eBacillus subtilis\u003c/i\u003e Inhibits \u003ci\u003eVibrio natriegens\u003c/i\u003e-Induced Corrosion via Biomineralization in Seawater.", "to": "mitigation of corrosion of steel in seawater"}, {"from": "biomineralized film", "title": "\u003ci\u003eBacillus subtilis\u003c/i\u003e Inhibits \u003ci\u003eVibrio natriegens\u003c/i\u003e-Induced Corrosion via Biomineralization in Seawater.", "to": "stable anti-corrosion activity"}, {"from": "formation of a biomineralized film", "title": "\u003ci\u003eBacillus subtilis\u003c/i\u003e Inhibits \u003ci\u003eVibrio natriegens\u003c/i\u003e-Induced Corrosion via Biomineralization in Seawater.", "to": "extracellular polymeric substances and calcite composition"}, {"from": "inhibition of Vibrio natriegens growth", "title": "\u003ci\u003eBacillus subtilis\u003c/i\u003e Inhibits \u003ci\u003eVibrio natriegens\u003c/i\u003e-Induced Corrosion via Biomineralization in Seawater.", "to": "formation of the biomineralized film"}, {"from": "clustered regularly interspaced short palindromic repeats interference screen", "title": "Functional genomics of the rapidly replicating bacterium Vibrio natriegens by CRISPRi.", "to": "minimal set of genes required for rapid wild-type growth"}, {"from": "predicted protein-coding genes", "title": "Functional genomics of the rapidly replicating bacterium Vibrio natriegens by CRISPRi.", "to": "core genes comprising putative essential and growth-supporting genes"}, {"from": "core genes", "title": "Functional genomics of the rapidly replicating bacterium Vibrio natriegens by CRISPRi.", "to": "enriched for respiratory pathways"}, {"from": "core genes", "title": "Functional genomics of the rapidly replicating bacterium Vibrio natriegens by CRISPRi.", "to": "located on chromosome 1"}, {"from": "growth-neutral duplicates of core genes", "title": "Functional genomics of the rapidly replicating bacterium Vibrio natriegens by CRISPRi.", "to": "located on chromosome 1"}, {"from": "clustered regularly interspaced short palindromic repeats interference screen", "title": "Functional genomics of the rapidly replicating bacterium Vibrio natriegens by CRISPRi.", "to": "metabolic pathway annotations"}, {"from": "functional genomics", "title": "Functional genomics of the rapidly replicating bacterium Vibrio natriegens by CRISPRi.", "to": "acceleration of biological studies and engineering of Vibrio natriegens"}, {"from": "Vibrio natriegens", "title": "Cell-free Protein Expression Using the Rapidly Growing Bacterium Vibrio natriegens.", "to": "emerging emerging microbial host for biotechnology"}, {"from": "preparation protocol", "title": "Cell-free Protein Expression Using the Rapidly Growing Bacterium Vibrio natriegens.", "to": "Vibrio natriegens crude cell extracts"}, {"from": "preparation protocol", "title": "Cell-free Protein Expression Using the Rapidly Growing Bacterium Vibrio natriegens.", "to": "reduced cost"}, {"from": "preparation protocol", "title": "Cell-free Protein Expression Using the Rapidly Growing Bacterium Vibrio natriegens.", "to": "user accessibility"}, {"from": "Cell-free protein synthesis", "title": "Cell-free Protein Expression Using the Rapidly Growing Bacterium Vibrio natriegens.", "to": "super folder Green Fluorescent Protein"}, {"from": "Cell-free protein synthesis", "title": "Cell-free Protein Expression Using the Rapidly Growing Bacterium Vibrio natriegens.", "to": "\u003e 260 micrograms per milliliter"}, {"from": "Cell-free protein synthesis", "title": "Cell-free Protein Expression Using the Rapidly Growing Bacterium Vibrio natriegens.", "to": "3 hours"}, {"from": "protocol", "title": "Cell-free Protein Expression Using the Rapidly Growing Bacterium Vibrio natriegens.", "to": "integration into protein synthesis pipelines"}, {"from": "protocol", "title": "Cell-free Protein Expression Using the Rapidly Growing Bacterium Vibrio natriegens.", "to": "advances in bio-production"}, {"from": "protocol", "title": "Cell-free Protein Expression Using the Rapidly Growing Bacterium Vibrio natriegens.", "to": "advances in synthetic biology applications"}, {"from": "Escherichia coli", "title": "\u003ci\u003eVibrio natriegens\u003c/i\u003e as Host for Expression of Multisubunit Membrane Protein Complexes.", "to": "lack of specific accessory genes for membrane insertion or cofactor assembly"}, {"from": "Vibrio natriegens", "title": "\u003ci\u003eVibrio natriegens\u003c/i\u003e as Host for Expression of Multisubunit Membrane Protein Complexes.", "to": "suitable expression host for membrane-bound proteins from inhibition of growth of Vibrio cholerae"}, {"from": "NADH:quinone oxidoreductase", "title": "\u003ci\u003eVibrio natriegens\u003c/i\u003e as Host for Expression of Multisubunit Membrane Protein Complexes.", "to": "increased overall NADH:quinone oxidoreduction activity of membranes from transformed Vibrio natriegens"}, {"from": "NADH:quinone oxidoreductase", "title": "\u003ci\u003eVibrio natriegens\u003c/i\u003e as Host for Expression of Multisubunit Membrane Protein Complexes.", "to": "sensitivity toward Silver ion"}, {"from": "inhibition of growth of Vibrio cholerae NADH:quinone oxidoreductase", "title": "\u003ci\u003eVibrio natriegens\u003c/i\u003e as Host for Expression of Multisubunit Membrane Protein Complexes.", "to": "complete assembly in Vibrio natriegens"}, {"from": "secondary transport system Mrp from inhibition of growth of Vibrio cholerae", "title": "\u003ci\u003eVibrio natriegens\u003c/i\u003e as Host for Expression of Multisubunit Membrane Protein Complexes.", "to": "functional state in Vibrio natriegens demonstrated by in vivo Lithium ion transport"}, {"from": "Vibrio natriegens", "title": "\u003ci\u003eVibrio natriegens\u003c/i\u003e as Host for Expression of Multisubunit Membrane Protein Complexes.", "to": "promising expression host for the production of membrane protein complexes from Gram-negative pathogens"}, {"from": "Vibrio natriegens", "title": "Establishing a Cell-Free Vibrio natriegens Expression System.", "to": "fast growing bacterium"}, {"from": "Vibrio natriegens", "title": "Establishing a Cell-Free Vibrio natriegens Expression System.", "to": "microbial host for biotechnology"}, {"from": "Vibrio natriegens cell-free expression system", "title": "Establishing a Cell-Free Vibrio natriegens Expression System.", "to": "superfolder GFP production"}, {"from": "simplified crude extract preparation protocol", "title": "Establishing a Cell-Free Vibrio natriegens Expression System.", "to": "superfolder GFP production"}, {"from": "small-scale batch reaction", "title": "Establishing a Cell-Free Vibrio natriegens Expression System.", "to": "superfolder GFP production"}, {"from": "culturing conditions", "title": "Establishing a Cell-Free Vibrio natriegens Expression System.", "to": "translation kinetics"}, {"from": "culturing conditions", "title": "Establishing a Cell-Free Vibrio natriegens Expression System.", "to": "protein yield"}, {"from": "growth media", "title": "Establishing a Cell-Free Vibrio natriegens Expression System.", "to": "translation kinetics"}, {"from": "growth media", "title": "Establishing a Cell-Free Vibrio natriegens Expression System.", "to": "protein yield"}, {"from": "cell density", "title": "Establishing a Cell-Free Vibrio natriegens Expression System.", "to": "translation kinetics"}, {"from": "cell density", "title": "Establishing a Cell-Free Vibrio natriegens Expression System.", "to": "protein yield"}, {"from": "incubation temperature ranges", "title": "Establishing a Cell-Free Vibrio natriegens Expression System.", "to": "maximal protein yield"}, {"from": "ions", "title": "Establishing a Cell-Free Vibrio natriegens Expression System.", "to": "ribosomal stability"}, {"from": "ribosomal stability", "title": "Establishing a Cell-Free Vibrio natriegens Expression System.", "to": "improved yield"}, {"from": "Vibrio natriegens cell-free expression system", "title": "Establishing a Cell-Free Vibrio natriegens Expression System.", "to": "metabolic engineering applications"}, {"from": "Vibrio natriegens cell-free expression system", "title": "Establishing a Cell-Free Vibrio natriegens Expression System.", "to": "synthetic biology applications"}, {"from": "Cell-free protein synthesis systems", "title": "Establishing a High-Yielding Cell-Free Protein Synthesis Platform Derived from Vibrio natriegens.", "to": "proteins"}, {"from": "Cell-free protein synthesis systems", "title": "Establishing a High-Yielding Cell-Free Protein Synthesis Platform Derived from Vibrio natriegens.", "to": "genetic regulatory element libraries"}, {"from": "Cell-free protein synthesis systems", "title": "Establishing a High-Yielding Cell-Free Protein Synthesis Platform Derived from Vibrio natriegens.", "to": "optimization of biosynthetic pathways"}, {"from": "Cell-free protein synthesis systems", "title": "Establishing a High-Yielding Cell-Free Protein Synthesis Platform Derived from Vibrio natriegens.", "to": "sensing biomarkers"}, {"from": "Vibrio natriegens", "title": "Establishing a High-Yielding Cell-Free Protein Synthesis Platform Derived from Vibrio natriegens.", "to": "cell-free protein synthesis systems"}, {"from": "sonication", "title": "Establishing a High-Yielding Cell-Free Protein Synthesis Platform Derived from Vibrio natriegens.", "to": "highly active extracts"}, {"from": "simplified crude extract preparation protocol", "title": "Establishing a High-Yielding Cell-Free Protein Synthesis Platform Derived from Vibrio natriegens.", "to": "synthesis of superfolder green fluorescent protein"}, {"from": "cell-free reaction conditions", "title": "Establishing a High-Yielding Cell-Free Protein Synthesis Platform Derived from Vibrio natriegens.", "to": "synthesis of superfolder green fluorescent protein"}, {"from": "Vibrio natriegens-derived cell-free protein synthesis platform", "title": "Establishing a High-Yielding Cell-Free Protein Synthesis Platform Derived from Vibrio natriegens.", "to": "production of antimicrobial peptides"}, {"from": "simplified crude extract preparation protocol", "title": "Establishing a High-Yielding Cell-Free Protein Synthesis Platform Derived from Vibrio natriegens.", "to": "lyophilization capability"}, {"from": "simplified crude extract preparation protocol", "title": "Establishing a High-Yielding Cell-Free Protein Synthesis Platform Derived from Vibrio natriegens.", "to": "coupling with Vmax Express strain"}, {"from": "knockout library", "title": "Establishing a High-Yielding Cell-Free Protein Synthesis Platform Derived from Vibrio natriegens.", "to": "increase in system productivity"}, {"from": "sponge-bacteria interactions", "title": "A controlled aquarium system and approach to study the role of sponge-bacteria interactions using Aplysilla rosea and Vibrio natriegens.", "to": "microbial diversity"}, {"from": "sponge-bacteria interactions", "title": "A controlled aquarium system and approach to study the role of sponge-bacteria interactions using Aplysilla rosea and Vibrio natriegens.", "to": "sponge metabolite profiles"}, {"from": "sponge-bacteria interactions", "title": "A controlled aquarium system and approach to study the role of sponge-bacteria interactions using Aplysilla rosea and Vibrio natriegens.", "to": "bioactivity"}, {"from": "controlled aquarium system", "title": "A controlled aquarium system and approach to study the role of sponge-bacteria interactions using Aplysilla rosea and Vibrio natriegens.", "to": "sponge-bacteria interactions"}, {"from": "Aplysilla rosea", "title": "A controlled aquarium system and approach to study the role of sponge-bacteria interactions using Aplysilla rosea and Vibrio natriegens.", "to": "Vibrio natriegens interaction"}, {"from": "Vibrio natriegens exposure", "title": "A controlled aquarium system and approach to study the role of sponge-bacteria interactions using Aplysilla rosea and Vibrio natriegens.", "to": "shifts in microbial communities"}, {"from": "Vibrio natriegens exposure", "title": "A controlled aquarium system and approach to study the role of sponge-bacteria interactions using Aplysilla rosea and Vibrio natriegens.", "to": "increased bacterial diversity"}, {"from": "Vibrio natriegens", "title": "Cell-Free Protein Synthesis From Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "cell-free protein synthesis system"}, {"from": "cell-free extract", "title": "Cell-Free Protein Synthesis From Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "eGFP synthesis"}, {"from": "fast-growing Vibrio natriegens cultures", "title": "Cell-Free Protein Synthesis From Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "cell-free extract"}, {"from": "cell-free protein synthesis system", "title": "Cell-Free Protein Synthesis From Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "rRNA stability"}, {"from": "cell-free protein synthesis system", "title": "Cell-Free Protein Synthesis From Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "substrate consumption"}, {"from": "magnesium homeostasis induction", "title": "Cell-Free Protein Synthesis From Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "rRNA stability"}, {"from": "cell-free protein synthesis system", "title": "Cell-Free Protein Synthesis From Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "translation capacity"}, {"from": "Vibrio natriegens genes", "title": "Cell-Free Protein Synthesis From Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "protein expression"}, {"from": "in vitro screening", "title": "Cell-Free Protein Synthesis From Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "promoter strength testing"}, {"from": "promoter strength testing", "title": "Cell-Free Protein Synthesis From Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "gene expression efficiency"}, {"from": "cell-free protein synthesis toolbox", "title": "Cell-Free Protein Synthesis From Fast-Growing \u003ci\u003eVibrio natriegens\u003c/i\u003e.", "to": "Vibrio natriegens platform establishment"}, {"from": "Vibrio natriegens", "title": "Speeding up bioproduction of selenium nanoparticles by using Vibrio natriegens as microbial factory.", "to": "aerobic growth"}, {"from": "Vibrio natriegens", "title": "Speeding up bioproduction of selenium nanoparticles by using Vibrio natriegens as microbial factory.", "to": "survival in 100 mM selenite"}, {"from": "selenite", "title": "Speeding up bioproduction of selenium nanoparticles by using Vibrio natriegens as microbial factory.", "to": "Vibrio natriegens growth"}, {"from": "Vibrio natriegens", "title": "Speeding up bioproduction of selenium nanoparticles by using Vibrio natriegens as microbial factory.", "to": "selenium nanoparticles production"}, {"from": "Vibrio natriegens", "title": "Speeding up bioproduction of selenium nanoparticles by using Vibrio natriegens as microbial factory.", "to": "bioremediation of selenite"}, {"from": "selenite", "title": "Speeding up bioproduction of selenium nanoparticles by using Vibrio natriegens as microbial factory.", "to": "selenium nanoparticles production by Vibrio natriegens"}, {"from": "Vibrio natriegens", "title": "Speeding up bioproduction of selenium nanoparticles by using Vibrio natriegens as microbial factory.", "to": "faster selenium nanoparticles production compared to other microorganisms"}, {"from": "selenite-containing LB medium at 30 degrees Celsius", "title": "Speeding up bioproduction of selenium nanoparticles by using Vibrio natriegens as microbial factory.", "to": "selenium nanoparticles production"}, {"from": "exponential growth phase of Vibrio natriegens", "title": "Speeding up bioproduction of selenium nanoparticles by using Vibrio natriegens as microbial factory.", "to": "selenium nanoparticles release after cell lysis"}, {"from": "CcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "inhibitory activities against elastase"}, {"from": "CcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "inhibitory activities against subtilisin A"}, {"from": "CcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "inhibitory activities against proteinase K"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "competitive inhibitory effects"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "antimicrobial activity"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "binding to Staphylococcus aureus"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "binding to Bacillus subtilis"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "binding to Escherichia coli"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "binding to Vibrio vulnificus"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "binding to Vibrio cholerae"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "binding to Vibrio natriegens"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "binding to Vibrio mimicus"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "binding to Vibrio alginolyticus"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "binding to Vibrio parahaemolyticus"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "binding to Candida albicans"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "binding to Candida parapsilokis"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "binding to Candida glabrata"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "inhibition of growth of Staphylococcus aureus"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "inhibition of growth of Bacillus subtilis"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "inhibition of growth of Escherichia coli"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "inhibition of growth of Vibrio vulnificus"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "inhibition of growth of Vibrio cholerae"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "inhibition of growth of Vibrio natriegens"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "inhibition of growth of Vibrio mimicus"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "inhibition of growth of Vibrio alginolyticus"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "inhibition of growth of Vibrio parahaemolyticus"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "inhibition of growth of Candida albicans"}, {"from": "rCcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "inhibition of growth of Candida parapsilokis"}, {"from": "CcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "antibacterial activities"}, {"from": "CcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "antifungal activities"}, {"from": "CcKPI1", "title": "Antimicrobial activity of a newly identified Kazal-type serine proteinase inhibitor, CcKPI1, from the jellyfish Cyanea capillata.", "to": "role in immune defence of Cyanea capillata"}, {"from": "Vibrio natriegens", "title": "Metabolism of the fast-growing bacterium Vibrio natriegens elucidated by \u003csup\u003e13\u003c/sup\u003eC metabolic flux analysis.", "to": "core metabolism model"}, {"from": "Vibrio natriegens", "title": "Metabolism of the fast-growing bacterium Vibrio natriegens elucidated by \u003csup\u003e13\u003c/sup\u003eC metabolic flux analysis.", "to": "biomass composition"}, {"from": "Vibrio natriegens", "title": "Metabolism of the fast-growing bacterium Vibrio natriegens elucidated by \u003csup\u003e13\u003c/sup\u003eC metabolic flux analysis.", "to": "high-resolution carbon-13 metabolic flux analysis"}, {"from": "high-resolution carbon-13 metabolic flux analysis", "title": "Metabolism of the fast-growing bacterium Vibrio natriegens elucidated by \u003csup\u003e13\u003c/sup\u003eC metabolic flux analysis.", "to": "intracellular fluxes estimation"}, {"from": "intracellular fluxes estimation", "title": "Metabolism of the fast-growing bacterium Vibrio natriegens elucidated by \u003csup\u003e13\u003c/sup\u003eC metabolic flux analysis.", "to": "parallel labeling experiments optimal tracers [1,2-carbon-13]glucose"}, {"from": "Vibrio natriegens", "title": "Metabolism of the fast-growing bacterium Vibrio natriegens elucidated by \u003csup\u003e13\u003c/sup\u003eC metabolic flux analysis.", "to": "growth rate of 1.70 1/h"}, {"from": "Vibrio natriegens", "title": "Metabolism of the fast-growing bacterium Vibrio natriegens elucidated by \u003csup\u003e13\u003c/sup\u003eC metabolic flux analysis.", "to": "doubling time of 24 minutes"}, {"from": "Vibrio natriegens", "title": "Metabolism of the fast-growing bacterium Vibrio natriegens elucidated by \u003csup\u003e13\u003c/sup\u003eC metabolic flux analysis.", "to": "glucose uptake rate of 3.90g/g/h"}, {"from": "Vibrio natriegens core metabolism", "title": "Metabolism of the fast-growing bacterium Vibrio natriegens elucidated by \u003csup\u003e13\u003c/sup\u003eC metabolic flux analysis.", "to": "similarity to E. coli core metabolism"}, {"from": "Vibrio natriegens core metabolism", "title": "Metabolism of the fast-growing bacterium Vibrio natriegens elucidated by \u003csup\u003e13\u003c/sup\u003eC metabolic flux analysis.", "to": "lower normalized flux through oxidative pentose phosphate pathway by 33%"}, {"from": "Vibrio natriegens", "title": "Metabolism of the fast-growing bacterium Vibrio natriegens elucidated by \u003csup\u003e13\u003c/sup\u003eC metabolic flux analysis.", "to": "energy metabolism insights"}, {"from": "Vibrio natriegens", "title": "Metabolism of the fast-growing bacterium Vibrio natriegens elucidated by \u003csup\u003e13\u003c/sup\u003eC metabolic flux analysis.", "to": "redox metabolism insights"}, {"from": "biomass-specific substrate consumption rate", "title": "High Substrate Uptake Rates Empower Vibrio natriegens as Production Host for Industrial Biotechnology.", "to": "growth rate"}, {"from": "Vibrio natriegens", "title": "High Substrate Uptake Rates Empower Vibrio natriegens as Production Host for Industrial Biotechnology.", "to": "biomass-specific substrate consumption rate"}, {"from": "Vibrio natriegens", "title": "High Substrate Uptake Rates Empower Vibrio natriegens as Production Host for Industrial Biotechnology.", "to": "growth rate"}, {"from": "Vibrio natriegens", "title": "High Substrate Uptake Rates Empower Vibrio natriegens as Production Host for Industrial Biotechnology.", "to": "rapid growth in rapid growth in BHIN complex medium"}, {"from": "rapid growth in BHIN complex medium", "title": "High Substrate Uptake Rates Empower Vibrio natriegens as Production Host for Industrial Biotechnology.", "to": "growth rate of Vibrio natriegens"}, {"from": "Vibrio natriegens", "title": "High Substrate Uptake Rates Empower Vibrio natriegens as Production Host for Industrial Biotechnology.", "to": "high biomass-specific substrate consumption rate under aerobic conditions"}, {"from": "Vibrio natriegens", "title": "High Substrate Uptake Rates Empower Vibrio natriegens as Production Host for Industrial Biotechnology.", "to": "high biomass-specific substrate consumption rate under anaerobic conditions"}, {"from": "genetically engineered Vibrio natriegens", "title": "High Substrate Uptake Rates Empower Vibrio natriegens as Production Host for Industrial Biotechnology.", "to": "overall volumetric productivity of alanine"}, {"from": "Vibrio natriegens", "title": "High Substrate Uptake Rates Empower Vibrio natriegens as Production Host for Industrial Biotechnology.", "to": "promising microbial platform for industrial fermentation processes"}, {"from": "Vibrio natriegens", "title": "High Substrate Uptake Rates Empower Vibrio natriegens as Production Host for Industrial Biotechnology.", "to": "versatility in metabolism"}, {"from": "Vibrio natriegens", "title": "High Substrate Uptake Rates Empower Vibrio natriegens as Production Host for Industrial Biotechnology.", "to": "rapid conversion rates"}, {"from": "Photobacterium sp. TKY1 ", "title": "Environmental Impact of Tributyltin-Resistant Marine Bacteria in the Indigenous Microbial Population of Tributyltin-Polluted Surface Sediments.", "to": "tributyltin (TBT"}, {"from": "Photobacterium sp. TKY1 ", "title": "Environmental Impact of Tributyltin-Resistant Marine Bacteria in the Indigenous Microbial Population of Tributyltin-Polluted Surface Sediments.", "to": "number of surviving cells"}, {"from": "tributyltin (TBT) chloride ", "title": "Environmental Impact of Tributyltin-Resistant Marine Bacteria in the Indigenous Microbial Population of Tributyltin-Polluted Surface Sediments.", "to": "Photobacterium ganghwense JCM 12487 sensitivity"}, {"from": "tributyltin (TBT) chloride ", "title": "Environmental Impact of Tributyltin-Resistant Marine Bacteria in the Indigenous Microbial Population of Tributyltin-Polluted Surface Sediments.", "to": "Photobacterium halotolerans LMG 22194 sensitivity"}, {"from": "Vibrio natriegens ATCC 14048 ", "title": "Environmental Impact of Tributyltin-Resistant Marine Bacteria in the Indigenous Microbial Population of Tributyltin-Polluted Surface Sediments.", "to": "reduction in surviving cells in presence of Photobacterium sp. TKY1 and tributyltin (TBT"}, {"from": "genetic tools", "title": "Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens.", "to": "development of Vibrio natriegens"}, {"from": "natural competence induction", "title": "Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens.", "to": "Vibrio natriegens"}, {"from": "MuGENT", "title": "Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens.", "to": "integration of multiple genome edits"}, {"from": "MuGENT", "title": "Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens.", "to": "high-efficiency genome editing"}, {"from": "MuGENT", "title": "Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens.", "to": "generation of highly complex mutant populations"}, {"from": "generation of highly complex mutant populations", "title": "Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens.", "to": "metabolic engineering efforts"}, {"from": "PHB biosynthesis genes expression targeting via MuGENT", "title": "Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens.", "to": "enhanced PHB production in Vibrio natriegens"}, {"from": "edited strains", "title": "Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens.", "to": "\u223c100 times more PHB production than parent isolate"}, {"from": "edited strains", "title": "Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens.", "to": "\u223c3.3 times more PHB production than rationally designed strain"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens as a fast-growing host for molecular biology.", "to": "bacterial host"}, {"from": "Vibrio natriegens", "title": "Vibrio natriegens as a fast-growing host for molecular biology.", "to": "fastest growth rate"}, {"from": "wide range of genetic tools", "title": "Vibrio natriegens as a fast-growing host for molecular biology.", "to": "engineer Vibrio natriegens"}, {"from": "methods", "title": "Vibrio natriegens as a fast-growing host for molecular biology.", "to": "engineer Vibrio natriegens"}, {"from": "engineered strains", "title": "Vibrio natriegens as a fast-growing host for molecular biology.", "to": "advantages in biotech processes"}, {"from": "Escherichia coli recA::luxCDABE bioreporter", "title": "Construction of a bioreporter by heterogeneously expressing a Vibrio natriegens recA::luxCDABE fusion in Escherichia coli, and genotoxicity assessments of petrochemical-contaminated groundwater in northern China.", "to": "genotoxicity assessments"}, {"from": "recA promoter", "title": "Construction of a bioreporter by heterogeneously expressing a Vibrio natriegens recA::luxCDABE fusion in Escherichia coli, and genotoxicity assessments of petrochemical-contaminated groundwater in northern China.", "to": "Escherichia coli recA::luxCDABE bioreporter construction"}, {"from": "Vibrio natriegens", "title": "Construction of a bioreporter by heterogeneously expressing a Vibrio natriegens recA::luxCDABE fusion in Escherichia coli, and genotoxicity assessments of petrochemical-contaminated groundwater in northern China.", "to": "recA promoter source"}, {"from": "mitomycin C induction", "title": "Construction of a bioreporter by heterogeneously expressing a Vibrio natriegens recA::luxCDABE fusion in Escherichia coli, and genotoxicity assessments of petrochemical-contaminated groundwater in northern China.", "to": "dose-response relationship in Escherichia coli recA::luxCDABE bioreporter"}, {"from": "pollutants induction", "title": "Construction of a bioreporter by heterogeneously expressing a Vibrio natriegens recA::luxCDABE fusion in Escherichia coli, and genotoxicity assessments of petrochemical-contaminated groundwater in northern China.", "to": "calculation as mitomycin C equivalents by Escherichia coli recA::luxCDABE bioreporter"}, {"from": "environmental samples induction", "title": "Construction of a bioreporter by heterogeneously expressing a Vibrio natriegens recA::luxCDABE fusion in Escherichia coli, and genotoxicity assessments of petrochemical-contaminated groundwater in northern China.", "to": "calculation as mitomycin C equivalents by Escherichia coli recA::luxCDABE bioreporter"}, {"from": "temperature range", "title": "Construction of a bioreporter by heterogeneously expressing a Vibrio natriegens recA::luxCDABE fusion in Escherichia coli, and genotoxicity assessments of petrochemical-contaminated groundwater in northern China.", "to": "genotoxicity evaluation by Escherichia coli recA::luxCDABE bioreporter"}, {"from": "pH range", "title": "Construction of a bioreporter by heterogeneously expressing a Vibrio natriegens recA::luxCDABE fusion in Escherichia coli, and genotoxicity assessments of petrochemical-contaminated groundwater in northern China.", "to": "genotoxicity evaluation by Escherichia coli recA::luxCDABE bioreporter"}, {"from": "salinity range", "title": "Construction of a bioreporter by heterogeneously expressing a Vibrio natriegens recA::luxCDABE fusion in Escherichia coli, and genotoxicity assessments of petrochemical-contaminated groundwater in northern China.", "to": "genotoxicity evaluation by Escherichia coli recA::luxCDABE bioreporter"}, {"from": "environmental media", "title": "Construction of a bioreporter by heterogeneously expressing a Vibrio natriegens recA::luxCDABE fusion in Escherichia coli, and genotoxicity assessments of petrochemical-contaminated groundwater in northern China.", "to": "genotoxicity assessment by Escherichia coli recA::luxCDABE bioreporter"}, {"from": "Vibrio natriegens recA expression in Escherichia coli", "title": "Construction of a bioreporter by heterogeneously expressing a Vibrio natriegens recA::luxCDABE fusion in Escherichia coli, and genotoxicity assessments of petrochemical-contaminated groundwater in northern China.", "to": "indication of a LexA-like regulator in Vibrio natriegens"}, {"from": "Vibrio natriegens recA putative SOS box", "title": "Construction of a bioreporter by heterogeneously expressing a Vibrio natriegens recA::luxCDABE fusion in Escherichia coli, and genotoxicity assessments of petrochemical-contaminated groundwater in northern China.", "to": "similarity to Escherichia coli SOS box"}, {"from": "groundwater samples from a petrochemical-contaminated site", "title": "Construction of a bioreporter by heterogeneously expressing a Vibrio natriegens recA::luxCDABE fusion in Escherichia coli, and genotoxicity assessments of petrochemical-contaminated groundwater in northern China.", "to": "genotoxicity evaluation by Escherichia coli recA::luxCDABE bioreporter"}, {"from": "chemical analyses of contamination levels", "title": "Construction of a bioreporter by heterogeneously expressing a Vibrio natriegens recA::luxCDABE fusion in Escherichia coli, and genotoxicity assessments of petrochemical-contaminated groundwater in northern China.", "to": "genotoxicity assessment by Escherichia coli recA::luxCDABE bioreporter"}, {"from": "Porphyra haitanensis", "title": "[Screening and identification of a bacterium capable of converting agar to neoagaro oligosaccharides].", "to": "Vibrio natriegens"}, {"from": "Seawater", "title": "[Screening and identification of a bacterium capable of converting agar to neoagaro oligosaccharides].", "to": "Vibrio natriegens"}, {"from": "Medium containing 1 per thousand agar", "title": "[Screening and identification of a bacterium capable of converting agar to neoagaro oligosaccharides].", "to": "Vibrio natriegens"}, {"from": "Dilution-plate method", "title": "[Screening and identification of a bacterium capable of converting agar to neoagaro oligosaccharides].", "to": "Vibrio natriegens"}, {"from": "Liquid culture", "title": "[Screening and identification of a bacterium capable of converting agar to neoagaro oligosaccharides].", "to": "Crude enzymes"}, {"from": "DNS method", "title": "[Screening and identification of a bacterium capable of converting agar to neoagaro oligosaccharides].", "to": "Neoagaro oligosaccharides"}, {"from": "16S rDNA sequence analysis", "title": "[Screening and identification of a bacterium capable of converting agar to neoagaro oligosaccharides].", "to": "Vibrio natriegens"}, {"from": "Morphological characteristics", "title": "[Screening and identification of a bacterium capable of converting agar to neoagaro oligosaccharides].", "to": "Vibrio natriegens"}, {"from": "Physiological biochemical characteristics", "title": "[Screening and identification of a bacterium capable of converting agar to neoagaro oligosaccharides].", "to": "Vibrio natriegens"}, {"from": "Basic Local Alignment Search Tool", "title": "[Screening and identification of a bacterium capable of converting agar to neoagaro oligosaccharides].", "to": "Vibrio natriegens"}, {"from": "HPLC", "title": "[Screening and identification of a bacterium capable of converting agar to neoagaro oligosaccharides].", "to": "Neoagaro oligosaccharides"}, {"from": "Vibrio natriegens", "title": "[Screening and identification of a bacterium capable of converting agar to neoagaro oligosaccharides].", "to": "Neoagaro oligosaccharides"}, {"from": "Dasycladus vermicularis", "title": "Sulfated phenolic acids from Dasycladales siphonous green algae.", "to": "sulfated hydroxylated coumarin derivative"}, {"from": "sulfated hydroxylated coumarin derivative", "title": "Sulfated phenolic acids from Dasycladales siphonous green algae.", "to": "protein cross linker"}, {"from": "comprehensive monitoring technique", "title": "Sulfated phenolic acids from Dasycladales siphonous green algae.", "to": "identification of 4-(sulfooxy"}, {"from": "4-(sulfooxy)phenylacetic acid", "title": "Sulfated phenolic acids from Dasycladales siphonous green algae.", "to": "proof of mass spectrometry-based structure"}, {"from": "4-(sulfooxy)benzoic acid", "title": "Sulfated phenolic acids from Dasycladales siphonous green algae.", "to": "proof of mass spectrometry-based structure"}, {"from": "4-(sulfooxy)phenylacetic acid", "title": "Sulfated phenolic acids from Dasycladales siphonous green algae.", "to": "desulfated phenols"}, {"from": "4-(sulfooxy)benzoic acid", "title": "Sulfated phenolic acids from Dasycladales siphonous green algae.", "to": "desulfated phenols"}, {"from": "sulfatases of bacteria", "title": "Sulfated phenolic acids from Dasycladales siphonous green algae.", "to": "desulfated phenols"}, {"from": "desulfated forms", "title": "Sulfated phenolic acids from Dasycladales siphonous green algae.", "to": "increased activity in biofouling experiments"}, {"from": "sulfatation", "title": "Sulfated phenolic acids from Dasycladales siphonous green algae.", "to": "detoxification measure"}, {"from": "Zosteric acid", "title": "Sulfation mediates activity of zosteric acid against biofilm formation.", "to": "para-coumaric acid"}, {"from": "para-coumaric acid", "title": "Sulfation mediates activity of zosteric acid against biofilm formation.", "to": "biofilm inhibiting activity"}, {"from": "Zosteric acid", "title": "Sulfation mediates activity of zosteric acid against biofilm formation.", "to": "inactive"}, {"from": "Zosteric acid", "title": "Sulfation mediates activity of zosteric acid against biofilm formation.", "to": "sulfatase activity"}, {"from": "sulfatase activity", "title": "Sulfation mediates activity of zosteric acid against biofilm formation.", "to": "para-coumaric acid"}, {"from": "para-coumaric acid", "title": "Sulfation mediates activity of zosteric acid against biofilm formation.", "to": "Escherichia coli inhibition"}, {"from": "para-coumaric acid", "title": "Sulfation mediates activity of zosteric acid against biofilm formation.", "to": "Vibrio natriegens inhibition"}, {"from": "mixed electron donors", "title": "Universally improving effect of mixed electron donors on the CO\u2082 fixing efficiency of non-photosynthetic microbial communities from marine environments.", "to": "carbon dioxide fixing efficiency"}, {"from": "non-photosynthetic microbial communities", "title": "Universally improving effect of mixed electron donors on the CO\u2082 fixing efficiency of non-photosynthetic microbial communities from marine environments.", "to": "carbon dioxide fixing efficiency"}, {"from": "mixed electron donors", "title": "Universally improving effect of mixed electron donors on the CO\u2082 fixing efficiency of non-photosynthetic microbial communities from marine environments.", "to": "energy utilization efficiency"}, {"from": "nitrite", "title": "Universally improving effect of mixed electron donors on the CO\u2082 fixing efficiency of non-photosynthetic microbial communities from marine environments.", "to": "energy utilization efficiency"}, {"from": "thiosulfate", "title": "Universally improving effect of mixed electron donors on the CO\u2082 fixing efficiency of non-photosynthetic microbial communities from marine environments.", "to": "energy utilization efficiency"}, {"from": "microbial inoculation concentration", "title": "Universally improving effect of mixed electron donors on the CO\u2082 fixing efficiency of non-photosynthetic microbial communities from marine environments.", "to": "net amount of carbon dioxide fixing"}, {"from": "autotrophic bacteria", "title": "Universally improving effect of mixed electron donors on the CO\u2082 fixing efficiency of non-photosynthetic microbial communities from marine environments.", "to": "energy/carbon source recycling"}, {"from": "heterotrophic bacteria", "title": "Universally improving effect of mixed electron donors on the CO\u2082 fixing efficiency of non-photosynthetic microbial communities from marine environments.", "to": "energy/carbon source recycling"}, {"from": "naturally competent Vibrio natriegens", "title": "Universally improving effect of mixed electron donors on the CO\u2082 fixing efficiency of non-photosynthetic microbial communities from marine environments.", "to": "non-photosynthetic microbial communities"}, {"from": "symbiotic bacteria", "title": "Universally improving effect of mixed electron donors on the CO\u2082 fixing efficiency of non-photosynthetic microbial communities from marine environments.", "to": "non-photosynthetic microbial communities"}, {"from": "C-type lectin", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "pattern-recognition proteins of the non-self-innate immune system in invertebrates"}, {"from": "EsCTL1", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "169 amino acids"}, {"from": "EsCTL2", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "164 amino acids"}, {"from": "EsCTL1", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "carbohydrate-recognition domain"}, {"from": "EsCTL2", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "carbohydrate-recognition domain"}, {"from": "EsCTL1", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "not clustered with other reported lectins from crabs"}, {"from": "EsCTL2", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "not clustered with other reported lectins from crabs"}, {"from": "EsCTL1", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "expressed only in the hepatopancreas"}, {"from": "EsCTL2", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "expressed only in the hepatopancreas"}, {"from": "challenge with lipopolysaccharide", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "regulation of expression levels of EsCTL1 and EsCTL2"}, {"from": "challenge with peptidoglycan", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "regulation of expression levels of EsCTL1 and EsCTL2"}, {"from": "challenge with Staphylococcus aureus", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "regulation of expression levels of EsCTL1 and EsCTL2"}, {"from": "challenge with Aeromonas hydrophila", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "regulation of expression levels of EsCTL1 and EsCTL2"}, {"from": "recombinant EsCTL1", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "agglutination of both Gram-positive bacteria"}, {"from": "recombinant EsCTL2", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "agglutination of both Gram-positive bacteria"}, {"from": "recombinant EsCTL1", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "agglutination of both Gram-negative bacteria"}, {"from": "recombinant EsCTL2", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "agglutination of both Gram-negative bacteria"}, {"from": "recombinant EsCTL1", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "direct binding to lipopolysaccharide"}, {"from": "recombinant EsCTL2", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "direct binding to lipopolysaccharide"}, {"from": "recombinant EsCTL1", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "direct binding to peptidoglycan"}, {"from": "recombinant EsCTL2", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "direct binding to peptidoglycan"}, {"from": "recombinant EsCTL1", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "direct binding to microorganisms"}, {"from": "recombinant EsCTL2", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "direct binding to microorganisms"}, {"from": "rEsCTL1", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "facilitation of the clearance of Vibrio parahaemolyticus in vivo"}, {"from": "EsCTL1", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "roles in the anti-bacterial immunity of Chinese mitten crab"}, {"from": "EsCTL2", "title": "Function of two novel single-CRD containing C-type lectins in innate immunity from Eriocheir sinensis.", "to": "roles in the anti-bacterial immunity of Chinese mitten crab"}, {"from": "molecular biological analysis", "title": "Genes similar to the Vibrio parahaemolyticus virulence-related genes tdh, tlh, and vscC2 occur in other vibrionaceae species isolated from a pristine estuary.", "to": "detection of Vibrio parahaemolyticus"}, {"from": "virulence factor genes", "title": "Genes similar to the Vibrio parahaemolyticus virulence-related genes tdh, tlh, and vscC2 occur in other vibrionaceae species isolated from a pristine estuary.", "to": "determinations of Vibrio parahaemolyticus population numbers"}, {"from": "virulence factor genes", "title": "Genes similar to the Vibrio parahaemolyticus virulence-related genes tdh, tlh, and vscC2 occur in other vibrionaceae species isolated from a pristine estuary.", "to": "prevalence of pathogenic Vibrio parahaemolyticus strains"}, {"from": "Photobacterium damselae", "title": "Genes similar to the Vibrio parahaemolyticus virulence-related genes tdh, tlh, and vscC2 occur in other vibrionaceae species isolated from a pristine estuary.", "to": "isolated from pristine salt marsh estuary"}, {"from": "Vibrio diabolicus", "title": "Genes similar to the Vibrio parahaemolyticus virulence-related genes tdh, tlh, and vscC2 occur in other vibrionaceae species isolated from a pristine estuary.", "to": "isolated from pristine salt marsh estuary"}, {"from": "Vibrio harveyi", "title": "Genes similar to the Vibrio parahaemolyticus virulence-related genes tdh, tlh, and vscC2 occur in other vibrionaceae species isolated from a pristine estuary.", "to": "isolated from pristine salt marsh estuary"}, {"from": "Vibrio natriegens", "title": "Genes similar to the Vibrio parahaemolyticus virulence-related genes tdh, tlh, and vscC2 occur in other vibrionaceae species isolated from a pristine estuary.", "to": "isolated from pristine salt marsh estuary"}, {"from": "strains similar to Vibrio tubiashii", "title": "Genes similar to the Vibrio parahaemolyticus virulence-related genes tdh, tlh, and vscC2 occur in other vibrionaceae species isolated from a pristine estuary.", "to": "isolated from pristine salt marsh estuary"}, {"from": "Vibrio parahaemolyticus hemolysin genes tdh", "title": "Genes similar to the Vibrio parahaemolyticus virulence-related genes tdh, tlh, and vscC2 occur in other vibrionaceae species isolated from a pristine estuary.", "to": "examined using established PCR primers and protocols"}, {"from": "Vibrio parahaemolyticus hemolysin genes trh", "title": "Genes similar to the Vibrio parahaemolyticus virulence-related genes tdh, tlh, and vscC2 occur in other vibrionaceae species isolated from a pristine estuary.", "to": "examined using established PCR primers and protocols"}, {"from": "Vibrio parahaemolyticus hemolysin gene tlh", "title": "Genes similar to the Vibrio parahaemolyticus virulence-related genes tdh, tlh, and vscC2 occur in other vibrionaceae species isolated from a pristine estuary.", "to": "examined using established PCR primers and protocols"}, {"from": "Vibrio parahaemolyticus type III secretion system 2 alpha gene vscC2", "title": "Genes similar to the Vibrio parahaemolyticus virulence-related genes tdh, tlh, and vscC2 occur in other vibrionaceae species isolated from a pristine estuary.", "to": "examined using established PCR primers and protocols"}, {"from": "virulence-related genes", "title": "Genes similar to the Vibrio parahaemolyticus virulence-related genes tdh, tlh, and vscC2 occur in other vibrionaceae species isolated from a pristine estuary.", "to": "occurrence at high frequencies in non-Vibrio parahaemolyticus Vibrionaceae species"}, {"from": "Vibrio diabolicus", "title": "Genes similar to the Vibrio parahaemolyticus virulence-related genes tdh, tlh, and vscC2 occur in other vibrionaceae species isolated from a pristine estuary.", "to": "contained virulence-related genes"}, {"from": "contained virulence-related genes", "title": "Genes similar to the Vibrio parahaemolyticus virulence-related genes tdh, tlh, and vscC2 occur in other vibrionaceae species isolated from a pristine estuary.", "to": "does not ensure correct identification of virulent Vibrio parahaemolyticus"}, {"from": "Vibrio parahaemolyticus-like virulence factors", "title": "Genes similar to the Vibrio parahaemolyticus virulence-related genes tdh, tlh, and vscC2 occur in other vibrionaceae species isolated from a pristine estuary.", "to": "potentially complicates tracking of Vibrio parahaemolyticus infections outbreaks"}, {"from": "EsCTLDcp", "title": "An LDLa domain-containing C-type lectin is involved in the innate immunity of Eriocheir sinensis.", "to": "Chinese mitten crab Eriocheir sinensis"}, {"from": "EsCTLDcp", "title": "An LDLa domain-containing C-type lectin is involved in the innate immunity of Eriocheir sinensis.", "to": "pathogens"}, {"from": "EsCTLDcp", "title": "An LDLa domain-containing C-type lectin is involved in the innate immunity of Eriocheir sinensis.", "to": "gram-positive bacteria Staphylococcus aureus"}, {"from": "EsCTLDcp", "title": "An LDLa domain-containing C-type lectin is involved in the innate immunity of Eriocheir sinensis.", "to": "Vibrio parahaemolyticus-like virulence factors"}, {"from": "EsCTLDcp", "title": "An LDLa domain-containing C-type lectin is involved in the innate immunity of Eriocheir sinensis.", "to": "gram-negative bacteria Aeromonas hydrophila"}, {"from": "EsCTLDcp", "title": "An LDLa domain-containing C-type lectin is involved in the innate immunity of Eriocheir sinensis.", "to": "bacteria Bacillus thuringiensis"}, {"from": "EsCTLDcp", "title": "An LDLa domain-containing C-type lectin is involved in the innate immunity of Eriocheir sinensis.", "to": "bacteria Bacillus subtilis"}, {"from": "EsCTLDcp", "title": "An LDLa domain-containing C-type lectin is involved in the innate immunity of Eriocheir sinensis.", "to": "bacteria Escherichia coli"}, {"from": "EsCTLDcp", "title": "An LDLa domain-containing C-type lectin is involved in the innate immunity of Eriocheir sinensis.", "to": "bacteria Vibrio natriegens"}, {"from": "EsCTLDcp", "title": "An LDLa domain-containing C-type lectin is involved in the innate immunity of Eriocheir sinensis.", "to": "envelope protein of white spot syndrome virus VP24"}, {"from": "EsCTLDcp", "title": "An LDLa domain-containing C-type lectin is involved in the innate immunity of Eriocheir sinensis.", "to": "innate immunity of Eriocheir sinensis"}, {"from": "EsCTLDcp", "title": "An LDLa domain-containing C-type lectin is involved in the innate immunity of Eriocheir sinensis.", "to": "hepatopancreas expression regulation"}, {"from": "EsCTLDcp", "title": "An LDLa domain-containing C-type lectin is involved in the innate immunity of Eriocheir sinensis.", "to": "agglutination of bacteria"}, {"from": "EsCTLDcp", "title": "An LDLa domain-containing C-type lectin is involved in the innate immunity of Eriocheir sinensis.", "to": "binding to various bacteria"}, {"from": "EsCTLDcp", "title": "An LDLa domain-containing C-type lectin is involved in the innate immunity of Eriocheir sinensis.", "to": "clearance of Vibrio parahaemolyticus in vivo"}, {"from": "Vibrio cholerae", "title": "The mosaic accessory gene structures of the SXT/R391-like integrative and conjugative elements derived from Vibrio spp. isolated from aquatic products and environment in the Yangtze River Estuary, China.", "to": "antibiotic resistance"}, {"from": "clearance of Vibrio parahaemolyticus in vivo", "title": "The mosaic accessory gene structures of the SXT/R391-like integrative and conjugative elements derived from Vibrio spp. isolated from aquatic products and environment in the Yangtze River Estuary, China.", "to": "antibiotic resistance"}, {"from": "antibiotic resistance", "title": "The mosaic accessory gene structures of the SXT/R391-like integrative and conjugative elements derived from Vibrio spp. isolated from aquatic products and environment in the Yangtze River Estuary, China.", "to": "Escherichia coli"}, {"from": "SXT/R391-like integrative and conjugative elements", "title": "The mosaic accessory gene structures of the SXT/R391-like integrative and conjugative elements derived from Vibrio spp. isolated from aquatic products and environment in the Yangtze River Estuary, China.", "to": "antibiotic resistance"}, {"from": "Vibrio strains", "title": "The mosaic accessory gene structures of the SXT/R391-like integrative and conjugative elements derived from Vibrio spp. isolated from aquatic products and environment in the Yangtze River Estuary, China.", "to": "SXT/R391-like integrative and conjugative elements"}, {"from": "Vibrio cholerae", "title": "The mosaic accessory gene structures of the SXT/R391-like integrative and conjugative elements derived from Vibrio spp. isolated from aquatic products and environment in the Yangtze River Estuary, China.", "to": "SXT/R391-like integrative and conjugative elements"}, {"from": "clearance of Vibrio parahaemolyticus in vivo", "title": "The mosaic accessory gene structures of the SXT/R391-like integrative and conjugative elements derived from Vibrio spp. isolated from aquatic products and environment in the Yangtze River Estuary, China.", "to": "SXT/R391-like integrative and conjugative elements"}, {"from": "Vibrio alginolyticus", "title": "The mosaic accessory gene structures of the SXT/R391-like integrative and conjugative elements derived from Vibrio spp. isolated from aquatic products and environment in the Yangtze River Estuary, China.", "to": "SXT/R391-like integrative and conjugative elements"}, {"from": "Vibrio natriegens", "title": "The mosaic accessory gene structures of the SXT/R391-like integrative and conjugative elements derived from Vibrio spp. isolated from aquatic products and environment in the Yangtze River Estuary, China.", "to": "SXT/R391-like integrative and conjugative elements"}, {"from": "SXT/R391-like integrative and conjugative elements", "title": "The mosaic accessory gene structures of the SXT/R391-like integrative and conjugative elements derived from Vibrio spp. isolated from aquatic products and environment in the Yangtze River Estuary, China.", "to": "novel genes"}, {"from": "SXT/R391-like integrative and conjugative elements", "title": "The mosaic accessory gene structures of the SXT/R391-like integrative and conjugative elements derived from Vibrio spp. isolated from aquatic products and environment in the Yangtze River Estuary, China.", "to": "ICEs-chromosome junction sequences"}, {"from": "SXT/R391-like integrative and conjugative elements", "title": "The mosaic accessory gene structures of the SXT/R391-like integrative and conjugative elements derived from Vibrio spp. isolated from aquatic products and environment in the Yangtze River Estuary, China.", "to": "core-genes for ICE transfer"}, {"from": "SXT/R391-like integrative and conjugative elements", "title": "The mosaic accessory gene structures of the SXT/R391-like integrative and conjugative elements derived from Vibrio spp. isolated from aquatic products and environment in the Yangtze River Estuary, China.", "to": "novel cluster"}, {"from": "Vibrio natriegens strains", "title": "Draft Genome Sequence of the Fast-Growing Bacterium Vibrio natriegens Strain DSMZ 759.", "to": "annotated draft genome sequence"}, {"from": "Vibrio natriegens", "title": "Draft Genome Sequence of the Fast-Growing Marine Bacterium Vibrio natriegens Strain ATCC 14048.", "to": "generation time of \u003c10 min"}, {"from": "Vibrio natriegens", "title": "Draft Genome Sequence of the Fast-Growing Marine Bacterium Vibrio natriegens Strain ATCC 14048.", "to": "sequenced genome"}, {"from": "Vibrio natriegens ATCC 14048", "title": "Draft Genome Sequence of the Fast-Growing Marine Bacterium Vibrio natriegens Strain ATCC 14048.", "to": "sequenced genome"}, {"from": "Monoclonal antibodies raised against Vibrio parahaemolyticus cell extracts", "title": "Production and characterization of a novel monoclonal antibody against Vibrio parahaemolyticus F0F1 ATP synthase\u0027s delta subunit and its application for rapid identification of the pathogen.", "to": "MAb-VP34"}, {"from": "MAb-VP34", "title": "Production and characterization of a novel monoclonal antibody against Vibrio parahaemolyticus F0F1 ATP synthase\u0027s delta subunit and its application for rapid identification of the pathogen.", "to": "Reacted in enzyme-linked immunosorbent assays"}, {"from": "MAb-VP34", "title": "Production and characterization of a novel monoclonal antibody against Vibrio parahaemolyticus F0F1 ATP synthase\u0027s delta subunit and its application for rapid identification of the pathogen.", "to": "Highly specific for Vibrio parahaemolyticus"}, {"from": "MAb-VP34", "title": "Production and characterization of a novel monoclonal antibody against Vibrio parahaemolyticus F0F1 ATP synthase\u0027s delta subunit and its application for rapid identification of the pathogen.", "to": "Recognized Vibrio parahaemolyticus F(0"}, {"from": "1) ATP synthase\u0027s delta subunit)\n(MAb-VP34", "title": "Production and characterization of a novel monoclonal antibody against Vibrio parahaemolyticus F0F1 ATP synthase\u0027s delta subunit and its application for rapid identification of the pathogen.", "to": "Used to develop immunodot blotting assay (VP-Dot"}, {"from": "Immunodot blotting assay (VP-Dot)", "title": "Production and characterization of a novel monoclonal antibody against Vibrio parahaemolyticus F0F1 ATP synthase\u0027s delta subunit and its application for rapid identification of the pathogen.", "to": "Rapid identification of Vibrio parahaemolyticus colonies"}, {"from": "Immunodot blotting assay (VP-Dot)", "title": "Production and characterization of a novel monoclonal antibody against Vibrio parahaemolyticus F0F1 ATP synthase\u0027s delta subunit and its application for rapid identification of the pathogen.", "to": "Reduced labor, time, Rapid identification of Vibrio parahaemolyticus colonies"}, {"from": "packhorse lobster", "title": "Vibrio jasicida sp. nov., a member of the Harveyi clade, isolated from marine animals (packhorse lobster, abalone and Atlantic salmon).", "to": "Vibrio jasicida sp. nov."}, {"from": "abalone", "title": "Vibrio jasicida sp. nov., a member of the Harveyi clade, isolated from marine animals (packhorse lobster, abalone and Atlantic salmon).", "to": "Vibrio jasicida sp. nov."}, {"from": "Atlantic salmon", "title": "Vibrio jasicida sp. nov., a member of the Harveyi clade, isolated from marine animals (packhorse lobster, abalone and Atlantic salmon).", "to": "Vibrio jasicida sp. nov."}, {"from": "0.5-8.0% NaCl", "title": "Vibrio jasicida sp. nov., a member of the Harveyi clade, isolated from marine animals (packhorse lobster, abalone and Atlantic salmon).", "to": "Vibrio jasicida sp. nov."}, {"from": "optimum 3.0-6.0% NaCl", "title": "Vibrio jasicida sp. nov., a member of the Harveyi clade, isolated from marine animals (packhorse lobster, abalone and Atlantic salmon).", "to": "Vibrio jasicida sp. nov."}, {"from": "10-37 \u00b0C", "title": "Vibrio jasicida sp. nov., a member of the Harveyi clade, isolated from marine animals (packhorse lobster, abalone and Atlantic salmon).", "to": "Vibrio jasicida sp. nov."}, {"from": "optimum 25-30 \u00b0C", "title": "Vibrio jasicida sp. nov., a member of the Harveyi clade, isolated from marine animals (packhorse lobster, abalone and Atlantic salmon).", "to": "Vibrio jasicida sp. nov."}, {"from": "Vibrio jasicida sp. nov.", "title": "Vibrio jasicida sp. nov., a member of the Harveyi clade, isolated from marine animals (packhorse lobster, abalone and Atlantic salmon).", "to": "45.9 mol% DNA G+C content"}, {"from": "Vibrio harveyi JMH612", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "N-acyl-homoserine lactone"}, {"from": "Agrobacterium tumefaciens KYC55", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "N-acyl-homoserine lactone"}, {"from": "Agrobacterium tumefaciens KYC55", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "thin-layer chromatography"}, {"from": "Vibrio mediterranei VIB296", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "N-acyl-homoserine lactone diversity"}, {"from": "Aliivibrio logei VIB414", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "N-acyl-homoserine lactone diversity"}, {"from": "luxS gene", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "autoinducer-2 activity"}, {"from": "Aliivibrio logei VIB414", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "cholerae autoinducer-1-like activity"}, {"from": "Vibrio campbellii VIB285", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "cholerae autoinducer-1-like activity"}, {"from": "Vibrio furnissii VIB293", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "cholerae autoinducer-1-like activity"}, {"from": "Vibrio pomeroyi LMG20537", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "cholerae autoinducer-1-like activity"}, {"from": "Vibrio harveyi VIB571", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "cholerae autoinducer-1-like activity"}, {"from": "Vibrio harveyi VIB645", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "cholerae autoinducer-1-like activity"}, {"from": "Aliivibrio logei VIB414", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "N-acyl-homoserine lactone activity"}, {"from": "Aliivibrio logei VIB414", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "autoinducer-2 activity"}, {"from": "Vibrio campbellii VIB285", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "N-acyl-homoserine lactone activity"}, {"from": "Vibrio campbellii VIB285", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "autoinducer-2 activity"}, {"from": "Vibrio furnissii VIB293", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "N-acyl-homoserine lactone activity"}, {"from": "Vibrio furnissii VIB293", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "autoinducer-2 activity"}, {"from": "Vibrio pomeroyi LMG20537", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "N-acyl-homoserine lactone activity"}, {"from": "Vibrio pomeroyi LMG20537", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "autoinducer-2 activity"}, {"from": "Vibrio harveyi VIB571", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "N-acyl-homoserine lactone activity"}, {"from": "Vibrio harveyi VIB571", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "autoinducer-2 activity"}, {"from": "Vibrio harveyi VIB645", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "N-acyl-homoserine lactone activity"}, {"from": "Vibrio harveyi VIB645", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "autoinducer-2 activity"}, {"from": "Vibrionaceae strains", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "quorum sensing signal molecule diversity"}, {"from": "Vibrionaceae strains", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "N-acyl-homoserine lactone production"}, {"from": "Vibrionaceae strains", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "autoinducer-2 production"}, {"from": "Vibrionaceae strains", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "cholerae autoinducer-1-like activity"}, {"from": "environmental conditions", "title": "Detection of quorum sensing signal molecules in the family Vibrionaceae.", "to": "quorum sensing signal molecule diversity"}, {"from": "plant growth-promoting bacteria strains", "title": "[Genotypic analysis and plant growth-promoting ability of four plant growth-promoting bacteria from mangrove].", "to": "phosphate-solubilization capacity"}, {"from": "plant growth-promoting bacteria strains", "title": "[Genotypic analysis and plant growth-promoting ability of four plant growth-promoting bacteria from mangrove].", "to": "nitrogen-fixation capacity"}, {"from": "phosphate-solubilization capacity", "title": "[Genotypic analysis and plant growth-promoting ability of four plant growth-promoting bacteria from mangrove].", "to": "plant growth"}, {"from": "nitrogen-fixation capacity", "title": "[Genotypic analysis and plant growth-promoting ability of four plant growth-promoting bacteria from mangrove].", "to": "plant growth"}, {"from": "strain HN011", "title": "[Genotypic analysis and plant growth-promoting ability of four plant growth-promoting bacteria from mangrove].", "to": "Vibrio natriegens"}, {"from": "SZ7-1", "title": "[Genotypic analysis and plant growth-promoting ability of four plant growth-promoting bacteria from mangrove].", "to": "Klebsiella oxytoca"}, {"from": "SZ002", "title": "[Genotypic analysis and plant growth-promoting ability of four plant growth-promoting bacteria from mangrove].", "to": "Paenibacillus sp."}, {"from": "SZ002 nifH gene", "title": "[Genotypic analysis and plant growth-promoting ability of four plant growth-promoting bacteria from mangrove].", "to": "Klebsiella genus"}, {"from": "plant growth-promoting bacteria strains", "title": "[Genotypic analysis and plant growth-promoting ability of four plant growth-promoting bacteria from mangrove].", "to": "increased plant dry weight"}, {"from": "plant growth-promoting bacteria strains", "title": "[Genotypic analysis and plant growth-promoting ability of four plant growth-promoting bacteria from mangrove].", "to": "increased total Nitrogen in plants"}, {"from": "plant growth-promoting bacteria strains", "title": "[Genotypic analysis and plant growth-promoting ability of four plant growth-promoting bacteria from mangrove].", "to": "increased total Phosphorus in plants"}, {"from": "Nucleotide excision repair", "title": "Identification of Vibrio natriegens uvrA and uvrB genes and analysis of gene regulation using transcriptional reporter plasmids.", "to": "DNA lesions"}, {"from": "Escherichia coli", "title": "Identification of Vibrio natriegens uvrA and uvrB genes and analysis of gene regulation using transcriptional reporter plasmids.", "to": "Nucleotide excision repair"}, {"from": "Vibrio natriegens", "title": "Identification of Vibrio natriegens uvrA and uvrB genes and analysis of gene regulation using transcriptional reporter plasmids.", "to": "DNA damage"}, {"from": "uvrA", "title": "Identification of Vibrio natriegens uvrA and uvrB genes and analysis of gene regulation using transcriptional reporter plasmids.", "to": "DNA repair capabilities"}, {"from": "uvrB", "title": "Identification of Vibrio natriegens uvrA and uvrB genes and analysis of gene regulation using transcriptional reporter plasmids.", "to": "DNA repair capabilities"}, {"from": "Escherichia coli uvrA mutants", "title": "Identification of Vibrio natriegens uvrA and uvrB genes and analysis of gene regulation using transcriptional reporter plasmids.", "to": "uvrA"}, {"from": "Escherichia coli uvrB mutants", "title": "Identification of Vibrio natriegens uvrA and uvrB genes and analysis of gene regulation using transcriptional reporter plasmids.", "to": "uvrB"}, {"from": "luciferase gene", "title": "Identification of Vibrio natriegens uvrA and uvrB genes and analysis of gene regulation using transcriptional reporter plasmids.", "to": "transcriptional fusions"}, {"from": "uvrA regulatory regions", "title": "Identification of Vibrio natriegens uvrA and uvrB genes and analysis of gene regulation using transcriptional reporter plasmids.", "to": "luminescence induction"}, {"from": "UV irradiation", "title": "Identification of Vibrio natriegens uvrA and uvrB genes and analysis of gene regulation using transcriptional reporter plasmids.", "to": "luminescence induction"}, {"from": "dichloro chalcone containing marine paint", "title": "2-Methoxy-2\u0027,4\u0027-dichloro chalcone as an antimicrofoulant against marine bacterial biofilm.", "to": "reduction in maximum biofilm thickness on polymethylmethacrylate"}, {"from": "dichloro chalcone containing marine paint", "title": "2-Methoxy-2\u0027,4\u0027-dichloro chalcone as an antimicrofoulant against marine bacterial biofilm.", "to": "increase in surface roughness"}, {"from": "dichloro chalcone containing marine paint", "title": "2-Methoxy-2\u0027,4\u0027-dichloro chalcone as an antimicrofoulant against marine bacterial biofilm.", "to": "decrease in surface hydrophobicity"}, {"from": "dichloro chalcone", "title": "2-Methoxy-2\u0027,4\u0027-dichloro chalcone as an antimicrofoulant against marine bacterial biofilm.", "to": "large number of dead cells on coated surfaces"}, {"from": "Bacillus flexus", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "marine biofouling"}, {"from": "Pseudomonas fluorescens", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "marine biofouling"}, {"from": "Vibrio natriegens", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "marine biofouling"}, {"from": "Chalcone derivatives", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "Bacillus flexus"}, {"from": "Chalcone derivatives", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "Pseudomonas fluorescens"}, {"from": "Chalcone derivatives", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "Vibrio natriegens"}, {"from": "Compounds C-5", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "Bacillus flexus"}, {"from": "Compounds C-5", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "Pseudomonas fluorescens"}, {"from": "Compounds C-5", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "Vibrio natriegens"}, {"from": "Compounds C-16", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "Bacillus flexus"}, {"from": "Compounds C-16", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "Pseudomonas fluorescens"}, {"from": "Compounds C-16", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "Vibrio natriegens"}, {"from": "Compounds C-24", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "Bacillus flexus"}, {"from": "Compounds C-24", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "Pseudomonas fluorescens"}, {"from": "Compounds C-24", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "Vibrio natriegens"}, {"from": "Compounds C-33", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "Bacillus flexus"}, {"from": "Compounds C-33", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "Pseudomonas fluorescens"}, {"from": "Compounds C-33", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "Vibrio natriegens"}, {"from": "Compounds C-37", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "Bacillus flexus"}, {"from": "Compounds C-37", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "Pseudomonas fluorescens"}, {"from": "Compounds C-37", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "Vibrio natriegens"}, {"from": "Hydroxyl substitutions", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "antibiofouling activity"}, {"from": "Jurs descriptors", "title": "Antibacterial activity and QSAR of chalcones against biofilm-producing bacteria isolated from marine waters.", "to": "antibiofouling activity"}, {"from": "naturally competent Vibrio natriegens", "title": "Marine bacterioplankton can increase evaporation and gas transfer bymetabolizing insoluble surfactants from the air-seawater interface.", "to": "hexadecanoic acid"}, {"from": "hexadecanoic acid", "title": "Marine bacterioplankton can increase evaporation and gas transfer bymetabolizing insoluble surfactants from the air-seawater interface.", "to": "evaporative fluxes"}, {"from": "concentration of naturally competent Vibrio natriegens cells", "title": "Marine bacterioplankton can increase evaporation and gas transfer bymetabolizing insoluble surfactants from the air-seawater interface.", "to": "removal of hexadecanoic acid"}, {"from": "quantity of hexadecanoic acid at the interface", "title": "Marine bacterioplankton can increase evaporation and gas transfer bymetabolizing insoluble surfactants from the air-seawater interface.", "to": "evaporative fluxes"}, {"from": "Ultraviolet radiation", "title": "Effects of Ultraviolet Radiation on the Gram-positive marine bacterium Microbacterium maritypicum.", "to": "Microbacterium maritypicum resistance"}, {"from": "Ultraviolet radiation", "title": "Effects of Ultraviolet Radiation on the Gram-positive marine bacterium Microbacterium maritypicum.", "to": "Vibrio natriegens resistance"}, {"from": "NaCl range 0-1000 mM", "title": "Effects of Ultraviolet Radiation on the Gram-positive marine bacterium Microbacterium maritypicum.", "to": "Microbacterium maritypicum growth"}, {"from": "NaCl range 250-1000 mM", "title": "Effects of Ultraviolet Radiation on the Gram-positive marine bacterium Microbacterium maritypicum.", "to": "Vibrio natriegens growth"}, {"from": "NaCl optimum growth 0-400 mM", "title": "Effects of Ultraviolet Radiation on the Gram-positive marine bacterium Microbacterium maritypicum.", "to": "Microbacterium maritypicum optimum growth"}, {"from": "NaCl optimum growth 250-600 mM", "title": "Effects of Ultraviolet Radiation on the Gram-positive marine bacterium Microbacterium maritypicum.", "to": "Vibrio natriegens optimum growth"}, {"from": "Ultraviolet A and B radiation", "title": "Effects of Ultraviolet Radiation on the Gram-positive marine bacterium Microbacterium maritypicum.", "to": "Microbacterium maritypicum resistance"}, {"from": "Ultraviolet A and B radiation", "title": "Effects of Ultraviolet Radiation on the Gram-positive marine bacterium Microbacterium maritypicum.", "to": "Vibrio natriegens resistance"}, {"from": "Ultraviolet C radiation", "title": "Effects of Ultraviolet Radiation on the Gram-positive marine bacterium Microbacterium maritypicum.", "to": "Microbacterium maritypicum resistance"}, {"from": "Ultraviolet C radiation", "title": "Effects of Ultraviolet Radiation on the Gram-positive marine bacterium Microbacterium maritypicum.", "to": "Vibrio natriegens resistance"}, {"from": "Ultraviolet C radiation", "title": "Effects of Ultraviolet Radiation on the Gram-positive marine bacterium Microbacterium maritypicum.", "to": "Photoreactivation in Microbacterium maritypicum"}, {"from": "Ultraviolet C radiation", "title": "Effects of Ultraviolet Radiation on the Gram-positive marine bacterium Microbacterium maritypicum.", "to": "Photoreactivation in Vibrio natriegens"}, {"from": "NaCl 0.0 M", "title": "Effects of Ultraviolet Radiation on the Gram-positive marine bacterium Microbacterium maritypicum.", "to": "Microbacterium maritypicum UV-C resistance"}, {"from": "NaCl 0.6 M", "title": "Effects of Ultraviolet Radiation on the Gram-positive marine bacterium Microbacterium maritypicum.", "to": "Microbacterium maritypicum UV-C resistance"}, {"from": "Vibrio spp. strain M11", "title": "Production of poly-beta-hydroxybutyrate (PHB) by Vibrio spp. isolated from marine environment.", "to": "poly-beta-hydroxybutyrate"}, {"from": "Vibrio spp. strain M14", "title": "Production of poly-beta-hydroxybutyrate (PHB) by Vibrio spp. isolated from marine environment.", "to": "poly-beta-hydroxybutyrate"}, {"from": "Vibrio spp. strain M20", "title": "Production of poly-beta-hydroxybutyrate (PHB) by Vibrio spp. isolated from marine environment.", "to": "poly-beta-hydroxybutyrate"}, {"from": "Vibrio spp. strain M31", "title": "Production of poly-beta-hydroxybutyrate (PHB) by Vibrio spp. isolated from marine environment.", "to": "poly-beta-hydroxybutyrate"}, {"from": "growth on acetate", "title": "Production of poly-beta-hydroxybutyrate (PHB) by Vibrio spp. isolated from marine environment.", "to": "poly-beta-hydroxybutyrate"}, {"from": "growth on glycerol", "title": "Production of poly-beta-hydroxybutyrate (PHB) by Vibrio spp. isolated from marine environment.", "to": "poly-beta-hydroxybutyrate"}, {"from": "growth on succinate", "title": "Production of poly-beta-hydroxybutyrate (PHB) by Vibrio spp. isolated from marine environment.", "to": "poly-beta-hydroxybutyrate"}, {"from": "growth on glucose", "title": "Production of poly-beta-hydroxybutyrate (PHB) by Vibrio spp. isolated from marine environment.", "to": "poly-beta-hydroxybutyrate"}, {"from": "growth on sucrose", "title": "Production of poly-beta-hydroxybutyrate (PHB) by Vibrio spp. isolated from marine environment.", "to": "poly-beta-hydroxybutyrate"}, {"from": "sodium chloride in medium", "title": "Production of poly-beta-hydroxybutyrate (PHB) by Vibrio spp. isolated from marine environment.", "to": "high poly-beta-hydroxybutyrate concentration in Vibrio spp. strain M11"}, {"from": "Vibrio natriegens characteristic", "title": "Production of poly-beta-hydroxybutyrate (PHB) by Vibrio spp. isolated from marine environment.", "to": "exploitable trait for poly-beta-hydroxybutyrate production"}, {"from": "DNA rearrangements", "title": "Plasticity of the gene functions for DNA replication in the T4-like phages.", "to": "mutation resilience"}, {"from": "DNA rearrangements", "title": "Plasticity of the gene functions for DNA replication in the T4-like phages.", "to": "T4-like phages adaptation"}, {"from": "mutation resilience", "title": "Plasticity of the gene functions for DNA replication in the T4-like phages.", "to": "T4-like phages adaptation"}, {"from": "genome size", "title": "Plasticity of the gene functions for DNA replication in the T4-like phages.", "to": "T4-like phages divergence"}, {"from": "host range", "title": "Plasticity of the gene functions for DNA replication in the T4-like phages.", "to": "T4-like phages divergence"}, {"from": "content and genetic organization of T4-like genes for DNA metabolism", "title": "Plasticity of the gene functions for DNA replication in the T4-like phages.", "to": "T4-like phages divergence"}, {"from": "mutational drift of the predicted T4-like gene products and their regulatory sites", "title": "Plasticity of the gene functions for DNA replication in the T4-like phages.", "to": "T4-like phages divergence"}, {"from": "content of open-reading frames that have no counterparts in T4 or other known organisms", "title": "Plasticity of the gene functions for DNA replication in the T4-like phages.", "to": "T4-like phages divergence"}, {"from": "diversity in sequence of DNA replication proteins", "title": "Plasticity of the gene functions for DNA replication in the T4-like phages.", "to": "T4-like phages adaptation"}, {"from": "diversity in molecular form of DNA replication proteins", "title": "Plasticity of the gene functions for DNA replication in the T4-like phages.", "to": "T4-like phages adaptation"}, {"from": "sequence divergence in protein-protein and protein-nucleic acid interaction sites", "title": "Plasticity of the gene functions for DNA replication in the T4-like phages.", "to": "T4-like phages adaptation"}, {"from": "phage-encoded DNA polymerase gp43 split forms", "title": "Plasticity of the gene functions for DNA replication in the T4-like phages.", "to": "T4-like phages genetic variability"}, {"from": "Vibrio natriegens", "title": "[Biosynthesis and accumulation of poly(3-hydroxybutyrate) in Vibrio natriegens].", "to": "poly(3-hydroxybutyrate"}, {"from": "glucose", "title": "[Biosynthesis and accumulation of poly(3-hydroxybutyrate) in Vibrio natriegens].", "to": "poly(3-hydroxybutyrate"}, {"from": "gluconate", "title": "[Biosynthesis and accumulation of poly(3-hydroxybutyrate) in Vibrio natriegens].", "to": "poly(3-hydroxybutyrate"}, {"from": "frutose", "title": "[Biosynthesis and accumulation of poly(3-hydroxybutyrate) in Vibrio natriegens].", "to": "poly(3-hydroxybutyrate"}, {"from": "molasses", "title": "[Biosynthesis and accumulation of poly(3-hydroxybutyrate) in Vibrio natriegens].", "to": "poly(3-hydroxybutyrate"}, {"from": "poly(3-hydroxybutyrate) accumulation", "title": "[Biosynthesis and accumulation of poly(3-hydroxybutyrate) in Vibrio natriegens].", "to": "cell growth"}, {"from": "PHA polymerase activity", "title": "[Biosynthesis and accumulation of poly(3-hydroxybutyrate) in Vibrio natriegens].", "to": "poly(3-hydroxybutyrate"}, {"from": "beta-ketothiolase activity", "title": "[Biosynthesis and accumulation of poly(3-hydroxybutyrate) in Vibrio natriegens].", "to": "poly(3-hydroxybutyrate"}, {"from": "acetoacetyl-Coenzyme A reductase activity", "title": "[Biosynthesis and accumulation of poly(3-hydroxybutyrate) in Vibrio natriegens].", "to": "poly(3-hydroxybutyrate"}, {"from": "de novo fatty acid synthesis", "title": "[Biosynthesis and accumulation of poly(3-hydroxybutyrate) in Vibrio natriegens].", "to": "poly(3-hydroxybutyrate"}, {"from": "cerulenin", "title": "[Biosynthesis and accumulation of poly(3-hydroxybutyrate) in Vibrio natriegens].", "to": "de novo fatty acid synthesis"}, {"from": "solar radiation", "title": "Estimation of biologically damaging UV levels in marine surface waters with DNA and viral dosimeters.", "to": "pyrimidine dimers formation in calf-thymus DNA"}, {"from": "pyrimidine dimers", "title": "Estimation of biologically damaging UV levels in marine surface waters with DNA and viral dosimeters.", "to": "decay rates of viral infectivity"}, {"from": "solar radiation", "title": "Estimation of biologically damaging UV levels in marine surface waters with DNA and viral dosimeters.", "to": "decay rates of viral infectivity"}, {"from": "depth of water column", "title": "Estimation of biologically damaging UV levels in marine surface waters with DNA and viral dosimeters.", "to": "decay of solar radiation"}, {"from": "depth of water column", "title": "Estimation of biologically damaging UV levels in marine surface waters with DNA and viral dosimeters.", "to": "decay rates of viral infectivity"}, {"from": "Solar ultraviolet radiation", "title": "RecA Expression in Response to Solar UVR in the Marine Bacterium Vibrio natriegens.", "to": "cell damage"}, {"from": "Solar ultraviolet radiation", "title": "RecA Expression in Response to Solar UVR in the Marine Bacterium Vibrio natriegens.", "to": "DNA repair"}, {"from": "Reduction in the earth\u0027s stratospheric ozone layer", "title": "RecA Expression in Response to Solar UVR in the Marine Bacterium Vibrio natriegens.", "to": "increased solar ultraviolet radiation effects"}, {"from": "RecA protein", "title": "RecA Expression in Response to Solar UVR in the Marine Bacterium Vibrio natriegens.", "to": "DNA damage repair indication"}, {"from": "RecA protein", "title": "RecA Expression in Response to Solar UVR in the Marine Bacterium Vibrio natriegens.", "to": "regulation of Dark Repair Systems"}, {"from": "Dark Repair Systems", "title": "RecA Expression in Response to Solar UVR in the Marine Bacterium Vibrio natriegens.", "to": "excision repair"}, {"from": "Dark Repair Systems", "title": "RecA Expression in Response to Solar UVR in the Marine Bacterium Vibrio natriegens.", "to": "postreplication recombinational repair"}, {"from": "Dark Repair Systems", "title": "RecA Expression in Response to Solar UVR in the Marine Bacterium Vibrio natriegens.", "to": "mutagenic or SOS repair"}, {"from": "Solar UVB radiation", "title": "RecA Expression in Response to Solar UVR in the Marine Bacterium Vibrio natriegens.", "to": "reduced Vibrio natriegens viability"}, {"from": "Solar UVA radiation", "title": "RecA Expression in Response to Solar UVR in the Marine Bacterium Vibrio natriegens.", "to": "reduced Vibrio natriegens viability"}, {"from": "Unfiltered solar radiation exposure", "title": "RecA Expression in Response to Solar UVR in the Marine Bacterium Vibrio natriegens.", "to": "reduced Vibrio natriegens survival"}, {"from": "UVB blocked radiation exposure", "title": "RecA Expression in Response to Solar UVR in the Marine Bacterium Vibrio natriegens.", "to": "reduced Vibrio natriegens survival"}, {"from": "Visible light exposure", "title": "RecA Expression in Response to Solar UVR in the Marine Bacterium Vibrio natriegens.", "to": "no effect on Vibrio natriegens survival"}, {"from": "Solar radiation", "title": "RecA Expression in Response to Solar UVR in the Marine Bacterium Vibrio natriegens.", "to": "induced RecA protein expression"}, {"from": "UVB-induced cyclobutyl pyrimidine dimers", "title": "RecA Expression in Response to Solar UVR in the Marine Bacterium Vibrio natriegens.", "to": "correlation with RecA expression"}, {"from": "UVA radiation", "title": "RecA Expression in Response to Solar UVR in the Marine Bacterium Vibrio natriegens.", "to": "induced RecA protein expression"}, {"from": "RecA-regulated repair", "title": "RecA Expression in Response to Solar UVR in the Marine Bacterium Vibrio natriegens.", "to": "survival after solar ultraviolet radiation exposure"}, {"from": "RecA expression", "title": "RecA Expression in Response to Solar UVR in the Marine Bacterium Vibrio natriegens.", "to": "monitor of bacterial repair after solar UVR exposure"}, {"from": "Vibrio natriegens", "title": "rRNA promoter activity in the fast-growing bacterium Vibrio natriegens.", "to": "high rate of protein synthesis"}, {"from": "increase in ribosome numbers", "title": "rRNA promoter activity in the fast-growing bacterium Vibrio natriegens.", "to": "increasing growth rate"}, {"from": "Vibrio natriegens", "title": "rRNA promoter activity in the fast-growing bacterium Vibrio natriegens.", "to": "large number of rRNA operons"}, {"from": "Vibrio natriegens rRNA promoters", "title": "rRNA promoter activity in the fast-growing bacterium Vibrio natriegens.", "to": "extremely strong"}, {"from": "Vibrio natriegens rRNA core promoters", "title": "rRNA promoter activity in the fast-growing bacterium Vibrio natriegens.", "to": "active in vitro"}, {"from": "Vibrio natriegens RNAP", "title": "rRNA promoter activity in the fast-growing bacterium Vibrio natriegens.", "to": "activation by UP elements"}, {"from": "Escherichia coli transcription factor Fis", "title": "rRNA promoter activity in the fast-growing bacterium Vibrio natriegens.", "to": "activated Vibrio natriegens rrn P1 promoters in vitro"}, {"from": "high capacity for rRNA transcription", "title": "rRNA promoter activity in the fast-growing bacterium Vibrio natriegens.", "to": "high capacity for ribosome synthesis in Vibrio natriegens"}, {"from": "high capacity for rRNA transcription in Vibrio natriegens", "title": "rRNA promoter activity in the fast-growing bacterium Vibrio natriegens.", "to": "high gene dose"}, {"from": "high capacity for rRNA transcription in Vibrio natriegens", "title": "rRNA promoter activity in the fast-growing bacterium Vibrio natriegens.", "to": "activation by UP elements"}, {"from": "Vibrio parahaemolyticus", "title": "Structures of ribonuclease P RNAs of Vibrio core species.", "to": "RNase P RNA structure"}, {"from": "Vibrio alginolyticus", "title": "Structures of ribonuclease P RNAs of Vibrio core species.", "to": "RNase P RNA structure"}, {"from": "Vibrio carchariae", "title": "Structures of ribonuclease P RNAs of Vibrio core species.", "to": "RNase P RNA structure"}, {"from": "Vibrio natriegens", "title": "Structures of ribonuclease P RNAs of Vibrio core species.", "to": "RNase P RNA structure"}, {"from": "Vibrio campbellii", "title": "Structures of ribonuclease P RNAs of Vibrio core species.", "to": "RNase P RNA structure"}, {"from": "Vibrio proteolyticus", "title": "Structures of ribonuclease P RNAs of Vibrio core species.", "to": "RNase P RNA structure"}, {"from": "Vibrio pelagius", "title": "Structures of ribonuclease P RNAs of Vibrio core species.", "to": "RNase P RNA structure"}, {"from": "Vibrio harveyi", "title": "Structures of ribonuclease P RNAs of Vibrio core species.", "to": "RNase P RNA structure"}, {"from": "RNase P RNA structure", "title": "Structures of ribonuclease P RNAs of Vibrio core species.", "to": "P3 helices"}, {"from": "RNase P RNA structure", "title": "Structures of ribonuclease P RNAs of Vibrio core species.", "to": "P12 helices"}, {"from": "P3 helices", "title": "Structures of ribonuclease P RNAs of Vibrio core species.", "to": "tandem repeats"}, {"from": "P3 helices", "title": "Structures of ribonuclease P RNAs of Vibrio core species.", "to": "palindromic sequence"}, {"from": "P3 helices", "title": "Structures of ribonuclease P RNAs of Vibrio core species.", "to": "stem structure"}, {"from": "P3 helices", "title": "Structures of ribonuclease P RNAs of Vibrio core species.", "to": "repetition number"}, {"from": "Vibrio harveyi", "title": "Structures of ribonuclease P RNAs of Vibrio core species.", "to": "four repetitions of P3 helices"}, {"from": "Vibrio alginolyticus", "title": "Structures of ribonuclease P RNAs of Vibrio core species.", "to": "four repetitions of P3 helices"}, {"from": "Vibrio proteolyticus", "title": "Structures of ribonuclease P RNAs of Vibrio core species.", "to": "four repetitions of P3 helices"}, {"from": "RNase P RNA genes", "title": "Structures of ribonuclease P RNAs of Vibrio core species.", "to": "location between two open reading frames"}, {"from": "RNase P RNA genes", "title": "Structures of ribonuclease P RNAs of Vibrio core species.", "to": "similarity to hypothetical proteins in Escherichia coli"}, {"from": "Vibrio anguillarum", "title": "Effects of bacteria on the growth of an amoeba infecting the gills of turbot.", "to": "Little or no growth of trophozoites"}, {"from": "Aeromonas salmonicida", "title": "Effects of bacteria on the growth of an amoeba infecting the gills of turbot.", "to": "Little or no growth of trophozoites"}, {"from": "Streptococcus sp.", "title": "Effects of bacteria on the growth of an amoeba infecting the gills of turbot.", "to": "Little or no growth of trophozoites"}, {"from": "Pasteurella piscicida", "title": "Effects of bacteria on the growth of an amoeba infecting the gills of turbot.", "to": "Little or no growth of trophozoites"}, {"from": "Vibrio vulnificus", "title": "Effects of bacteria on the growth of an amoeba infecting the gills of turbot.", "to": "Little or no growth of trophozoites"}, {"from": "Vibrio campbelli", "title": "Effects of bacteria on the growth of an amoeba infecting the gills of turbot.", "to": "Little or no growth of trophozoites"}, {"from": "Vibrio fluvialis", "title": "Effects of bacteria on the growth of an amoeba infecting the gills of turbot.", "to": "Little or no growth of trophozoites"}, {"from": "Pseudomonas dondorofii", "title": "Effects of bacteria on the growth of an amoeba infecting the gills of turbot.", "to": "Little or no growth of trophozoites"}, {"from": "Aeromonas hydrophila", "title": "Effects of bacteria on the growth of an amoeba infecting the gills of turbot.", "to": "Growth of trophozoites"}, {"from": "Vibrio natriegens", "title": "Effects of bacteria on the growth of an amoeba infecting the gills of turbot.", "to": "Growth of trophozoites"}, {"from": "Pseudomonas dondorofii", "title": "Effects of bacteria on the growth of an amoeba infecting the gills of turbot.", "to": "Growth of trophozoites"}, {"from": "Escherichia coli", "title": "Effects of bacteria on the growth of an amoeba infecting the gills of turbot.", "to": "Growth of trophozoites"}, {"from": "Paraphysomonas imperforata", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "Vibrio natriegens"}, {"from": "Pteridomonas danica", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "Vibrio natriegens"}, {"from": "Cafeteria roenbergensis", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "Vibrio natriegens"}, {"from": "incubation temperature", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "timing of the maximum of flagellate numbers"}, {"from": "incubation temperature", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "concentrations at the time of maximum flagellate numbers"}, {"from": "incubation temperature", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "gross growth efficiency"}, {"from": "Pteridomonas danica", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "effectiveness as grazer"}, {"from": "Paraphysomonas imperforata", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "effectiveness as grazer"}, {"from": "Cafeteria roenbergensis", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "effectiveness as grazer"}, {"from": "Pteridomonas danica", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "threshold bacterial concentrations"}, {"from": "Paraphysomonas imperforata", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "threshold bacterial concentrations"}, {"from": "Cafeteria roenbergensis", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "threshold bacterial concentrations"}, {"from": "Pteridomonas danica", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "percentage of Vibrio natriegens consumed"}, {"from": "Paraphysomonas imperforata", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "percentage of Vibrio natriegens consumed"}, {"from": "Cafeteria roenbergensis", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "percentage of Vibrio natriegens consumed"}, {"from": "incubation temperature", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "peak concentrations of flagellates"}, {"from": "incubation temperature", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "numbers of flagellates produced"}, {"from": "incubation temperature", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "numbers of bacteria eaten"}, {"from": "bacterial concentration", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "time to reach peak flagellate numbers"}, {"from": "bacterial concentration", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "gross growth efficiency"}, {"from": "bacterial concentration", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "numbers of flagellates produced"}, {"from": "flagellate species", "title": "Grazing Characteristics and Growth Efficiencies at Two Different Temperatures for Three Nanoflagellates Fed with Vibrio Bacteria at Three Different Concentrations.", "to": "niche specialization"}, {"from": "Vibrio pelagius", "title": "Vibrio pelagius: differences of the type strain deposited at various culture collections.", "to": "diversity"}, {"from": "16S rRNA sequence analyses", "title": "Vibrio pelagius: differences of the type strain deposited at various culture collections.", "to": "Vibrio pelagius strains CECT 4202T"}, {"from": "16S rRNA sequence analyses", "title": "Vibrio pelagius: differences of the type strain deposited at various culture collections.", "to": "Vibrio pelagius strains ATCC 25916T"}, {"from": "genomic DNA-DNA similarity studies", "title": "Vibrio pelagius: differences of the type strain deposited at various culture collections.", "to": "Vibrio pelagius strains CECT 4202T"}, {"from": "genomic DNA-DNA similarity studies", "title": "Vibrio pelagius: differences of the type strain deposited at various culture collections.", "to": "Vibrio pelagius strains ATCC 25916T"}, {"from": "physiological investigations", "title": "Vibrio pelagius: differences of the type strain deposited at various culture collections.", "to": "Vibrio pelagius strains CECT 4202T"}, {"from": "physiological investigations", "title": "Vibrio pelagius: differences of the type strain deposited at various culture collections.", "to": "Vibrio pelagius strains ATCC 25916T"}, {"from": "Vibrio pelagius strains CECT 4202T", "title": "Vibrio pelagius: differences of the type strain deposited at various culture collections.", "to": "originally described type species"}, {"from": "Vibrio pelagius strains ATCC 25916T", "title": "Vibrio pelagius: differences of the type strain deposited at various culture collections.", "to": "originally described type species"}, {"from": "Vibrio natriegens", "title": "Vibrio pelagius: differences of the type strain deposited at various culture collections.", "to": "Vibrio pelagius strains NCIMB 1900T"}, {"from": "liquid flow velocity", "title": "Biofilm development in a membrane-aerated biofilm reactor: effect of flow velocity on performance.", "to": "mass transfer in the diffusion boundary layer"}, {"from": "liquid flow velocity", "title": "Biofilm development in a membrane-aerated biofilm reactor: effect of flow velocity on performance.", "to": "biomass detachment rate from the biofilm"}, {"from": "liquid flow velocity", "title": "Biofilm development in a membrane-aerated biofilm reactor: effect of flow velocity on performance.", "to": "maximum maximum biofilm thickness"}, {"from": "overall mass transfer coefficient", "title": "Biofilm development in a membrane-aerated biofilm reactor: effect of flow velocity on performance.", "to": "maximum maximum biofilm thickness"}, {"from": "liquid flow velocity", "title": "Biofilm development in a membrane-aerated biofilm reactor: effect of flow velocity on performance.", "to": "biofilm structure"}, {"from": "maximum biofilm thickness", "title": "Biofilm development in a membrane-aerated biofilm reactor: effect of flow velocity on performance.", "to": "reaction and diffusion"}, {"from": "effective diffusivity", "title": "Biofilm development in a membrane-aerated biofilm reactor: effect of flow velocity on performance.", "to": "reaction and diffusion"}, {"from": "yield of biomass on acetate", "title": "Biofilm development in a membrane-aerated biofilm reactor: effect of flow velocity on performance.", "to": "reaction and diffusion"}, {"from": "endogenous respiration", "title": "Biofilm development in a membrane-aerated biofilm reactor: effect of flow velocity on performance.", "to": "yield coefficients"}, {"from": "acetate uptake rates", "title": "Biofilm development in a membrane-aerated biofilm reactor: effect of flow velocity on performance.", "to": "mathematical model"}, {"from": "oxygen uptake rates", "title": "Biofilm development in a membrane-aerated biofilm reactor: effect of flow velocity on performance.", "to": "mathematical model"}, {"from": "UV-B radiation", "title": "Marine bacterial isolates display diverse responses to UV-B radiation.", "to": "survival of marine bacteria"}, {"from": "UV-B radiation", "title": "Marine bacterial isolates display diverse responses to UV-B radiation.", "to": "cyclobutane pyrimidine dimers"}, {"from": "Sphingomonas sp. strain RB2256", "title": "Marine bacterial isolates display diverse responses to UV-B radiation.", "to": "little DNA damage"}, {"from": "Sphingomonas sp. strain RB2256", "title": "Marine bacterial isolates display diverse responses to UV-B radiation.", "to": "high level of UV-B resistance"}, {"from": "growth phase", "title": "Marine bacterial isolates display diverse responses to UV-B radiation.", "to": "UV-B sensitivity of marine bacteria"}, {"from": "starvation", "title": "Marine bacterial isolates display diverse responses to UV-B radiation.", "to": "UV-B sensitivity of marine bacteria"}, {"from": "UV-B exposure", "title": "Marine bacterial isolates display diverse responses to UV-B radiation.", "to": "rates of photoreactivation restoration of infectivity"}, {"from": "UV-A exposure", "title": "Marine bacterial isolates display diverse responses to UV-B radiation.", "to": "rates of photoreactivation restoration of infectivity"}, {"from": "cool white light exposure", "title": "Marine bacterial isolates display diverse responses to UV-B radiation.", "to": "rates of photoreactivation restoration of infectivity"}, {"from": "UV-B radiation", "title": "Marine bacterial isolates display diverse responses to UV-B radiation.", "to": "microbial community structure in surface water"}, {"from": "Vibrio natriegens", "title": "Marine bacterial isolates display diverse responses to UV-B radiation.", "to": "survival and CPD data"}, {"from": "photoreactivation restoration of infectivity plus dark repair", "title": "Marine bacterial isolates display diverse responses to UV-B radiation.", "to": "photoadaptation"}, {"from": "X-ray microanalysis", "title": "The inorganic ion content of native aquatic bacteria.", "to": "magnesium concentration"}, {"from": "X-ray microanalysis", "title": "The inorganic ion content of native aquatic bacteria.", "to": "sodium concentration"}, {"from": "X-ray microanalysis", "title": "The inorganic ion content of native aquatic bacteria.", "to": "potassium to sodium ratio"}, {"from": "magnesium", "title": "The inorganic ion content of native aquatic bacteria.", "to": "cell turgor"}, {"from": "chloride", "title": "The inorganic ion content of native aquatic bacteria.", "to": "cell turgor"}, {"from": "sodium concentration", "title": "The inorganic ion content of native aquatic bacteria.", "to": "potassium to sodium ratio"}, {"from": "Vibrio natriegens", "title": "The inorganic ion content of native aquatic bacteria.", "to": "high internal chloride concentration"}, {"from": "magnesium", "title": "The inorganic ion content of native aquatic bacteria.", "to": "dominating cation in marine bacterioplankton"}, {"from": "chloride", "title": "The inorganic ion content of native aquatic bacteria.", "to": "adaptation in actively growing marine bacteria"}, {"from": "magnesium", "title": "The inorganic ion content of native aquatic bacteria.", "to": "adaptation in actively growing marine bacteria"}, {"from": "sunlight-damaged natural viral communities", "title": "Photoreactivation compensates for UV damage and restores infectivity to natural marine virus communities.", "to": "photoreactivation restoration of infectivity"}, {"from": "Vibrio natriegens", "title": "Photoreactivation compensates for UV damage and restores infectivity to natural marine virus communities.", "to": "restoration of infectivity to natural viral communities"}, {"from": "\u003e 312-nm-wavelength sunlight", "title": "Photoreactivation compensates for UV damage and restores infectivity to natural marine virus communities.", "to": "restoration of infectivity to sunlight-damaged viruses in oceanic waters"}, {"from": "370 to 550 nm wavelengths", "title": "Photoreactivation compensates for UV damage and restores infectivity to natural marine virus communities.", "to": "restoration of infectivity to damaged viruses"}, {"from": "photoreactivation", "title": "Photoreactivation compensates for UV damage and restores infectivity to natural marine virus communities.", "to": "compensation for sunlight-induced DNA damage in natural viral communities"}, {"from": "aerobic N-replete conditions", "title": "N2 fixation in marine heterotrophic bacteria: dynamics of environmental and molecular regulation.", "to": "anaerobic N-deplete conditions"}, {"from": "anaerobic N-deplete conditions", "title": "N2 fixation in marine heterotrophic bacteria: dynamics of environmental and molecular regulation.", "to": "transcriptional regulation of N2 fixation"}, {"from": "anaerobic N-deplete conditions", "title": "N2 fixation in marine heterotrophic bacteria: dynamics of environmental and molecular regulation.", "to": "post-translational regulation of N2 fixation"}, {"from": "nifHDK mRNA", "title": "N2 fixation in marine heterotrophic bacteria: dynamics of environmental and molecular regulation.", "to": "nitrogenase enzyme"}, {"from": "nifHDK mRNA levels highest at 140 min postshift", "title": "N2 fixation in marine heterotrophic bacteria: dynamics of environmental and molecular regulation.", "to": "nitrogenase enzyme levels highest between 6 and 15 h postshift"}, {"from": "nifHDK mRNA undetectable between 6 and 9 h", "title": "N2 fixation in marine heterotrophic bacteria: dynamics of environmental and molecular regulation.", "to": "nitrogenase activity peak between 6 and 9 h postshift"}, {"from": "nifHDK mRNA absent", "title": "N2 fixation in marine heterotrophic bacteria: dynamics of environmental and molecular regulation.", "to": "nitrogenase enzyme presence in Vibrio natriegens"}, {"from": "anaerobic conditions", "title": "N2 fixation in marine heterotrophic bacteria: dynamics of environmental and molecular regulation.", "to": "nitrogenase stability in Vibrio natriegens"}, {"from": "N2-fixing conditions to non-N2-fixing conditions", "title": "N2 fixation in marine heterotrophic bacteria: dynamics of environmental and molecular regulation.", "to": "disappearance of nifHDK mRNA"}, {"from": "N2-fixing conditions to non-N2-fixing conditions", "title": "N2 fixation in marine heterotrophic bacteria: dynamics of environmental and molecular regulation.", "to": "disappearance of Fe protein"}, {"from": "non-N2-fixing conditions to N2-fixing conditions", "title": "N2 fixation in marine heterotrophic bacteria: dynamics of environmental and molecular regulation.", "to": "reappearance of nifHDK mRNA"}, {"from": "non-N2-fixing conditions to N2-fixing conditions", "title": "N2 fixation in marine heterotrophic bacteria: dynamics of environmental and molecular regulation.", "to": "disappearance of Fe protein"}, {"from": "immunological detection", "title": "N2 fixation in marine heterotrophic bacteria: dynamics of environmental and molecular regulation.", "to": "positive correlation with nitrogenase activity"}, {"from": "Vibrio natriegens anaerobic induction", "title": "N2 fixation in marine heterotrophic bacteria: dynamics of environmental and molecular regulation.", "to": "continued N2 fixation"}, {"from": "Fluorescently stained viruses", "title": "Fluorescently Labeled Virus Probes Show that Natural Virus Populations Can Control the Structure of Marine Microbial Communities.", "to": "Specific strains of bacteria identification"}, {"from": "Fluorescently stained viruses", "title": "Fluorescently Labeled Virus Probes Show that Natural Virus Populations Can Control the Structure of Marine Microbial Communities.", "to": "Cyanobacteria identification"}, {"from": "YOYO-1", "title": "Fluorescently Labeled Virus Probes Show that Natural Virus Populations Can Control the Structure of Marine Microbial Communities.", "to": "Fluorescent staining"}, {"from": "POPO-1", "title": "Fluorescently Labeled Virus Probes Show that Natural Virus Populations Can Control the Structure of Marine Microbial Communities.", "to": "Fluorescent staining"}, {"from": "Host cells", "title": "Fluorescently Labeled Virus Probes Show that Natural Virus Populations Can Control the Structure of Marine Microbial Communities.", "to": "Quantification efficiency with fluorescently labeled virus probes"}, {"from": "Strain PWH3a addition", "title": "Fluorescently Labeled Virus Probes Show that Natural Virus Populations Can Control the Structure of Marine Microbial Communities.", "to": "Increase in virus concentration"}, {"from": "Strain PWH3a addition", "title": "Fluorescently Labeled Virus Probes Show that Natural Virus Populations Can Control the Structure of Marine Microbial Communities.", "to": "Collapse in populations of strain PWH3a"}, {"from": "Nutrient-enrichment", "title": "Fluorescently Labeled Virus Probes Show that Natural Virus Populations Can Control the Structure of Marine Microbial Communities.", "to": "Increase in virus concentration infecting strain PWH3a"}, {"from": "Nutrient-enrichment", "title": "Fluorescently Labeled Virus Probes Show that Natural Virus Populations Can Control the Structure of Marine Microbial Communities.", "to": "Collapse in populations of strain PWH3a"}, {"from": "Quantification efficiency with fluorescently labeled virus probes", "title": "Fluorescently Labeled Virus Probes Show that Natural Virus Populations Can Control the Structure of Marine Microbial Communities.", "to": "Bacteria quantification in microbial communities"}, {"from": "Low abundances of viruses", "title": "Fluorescently Labeled Virus Probes Show that Natural Virus Populations Can Control the Structure of Marine Microbial Communities.", "to": "Control over microbial community structure"}, {"from": "Viral abundance", "title": "Distribution of viral abundance in the reef environment of Key Largo, Florida.", "to": "salinity range"}, {"from": "Viral direct counts", "title": "Distribution of viral abundance in the reef environment of Key Largo, Florida.", "to": "Vibrio natriegens plaque titers"}, {"from": "Viral abundance", "title": "Distribution of viral abundance in the reef environment of Key Largo, Florida.", "to": "Bacterial direct counts"}, {"from": "Viral abundance", "title": "Distribution of viral abundance in the reef environment of Key Largo, Florida.", "to": "Chlorophyll a measurements"}, {"from": "Sediment viral parameters", "title": "Distribution of viral abundance in the reef environment of Key Largo, Florida.", "to": "Water column microbial data"}, {"from": "Sediment viral parameters", "title": "Distribution of viral abundance in the reef environment of Key Largo, Florida.", "to": "Water column viral data"}, {"from": "Sediment viral parameters", "title": "Distribution of viral abundance in the reef environment of Key Largo, Florida.", "to": "Water column salinity data"}, {"from": "Coliphage presence", "title": "Distribution of viral abundance in the reef environment of Key Largo, Florida.", "to": "Fecal pollution indicators"}, {"from": "comparative analysis of the 16S rRNAs", "title": "Towards a phylogeny of the genus Vibrio based on 16S rRNA sequences.", "to": "inter- and intrageneric relationships of the genus Vibrio"}, {"from": "comparative analysis of the 16S rRNAs", "title": "Towards a phylogeny of the genus Vibrio based on 16S rRNA sequences.", "to": "neighboring taxon of the family Enterobacteriaceae"}, {"from": "Vibrio alginolyticus", "title": "Towards a phylogeny of the genus Vibrio based on 16S rRNA sequences.", "to": "core of the genus Vibrio"}, {"from": "Vibrio campbellii", "title": "Towards a phylogeny of the genus Vibrio based on 16S rRNA sequences.", "to": "core of the genus Vibrio"}, {"from": "Vibrio natriegens", "title": "Towards a phylogeny of the genus Vibrio based on 16S rRNA sequences.", "to": "core of the genus Vibrio"}, {"from": "Vibrio harveyi", "title": "Towards a phylogeny of the genus Vibrio based on 16S rRNA sequences.", "to": "core of the genus Vibrio"}, {"from": "Vibrio proteolyticus", "title": "Towards a phylogeny of the genus Vibrio based on 16S rRNA sequences.", "to": "core of the genus Vibrio"}, {"from": "Vibrio parahaemolyticus", "title": "Towards a phylogeny of the genus Vibrio based on 16S rRNA sequences.", "to": "core of the genus Vibrio"}, {"from": "Vibrio vulnificus", "title": "Towards a phylogeny of the genus Vibrio based on 16S rRNA sequences.", "to": "core of the genus Vibrio"}, {"from": "Vibrio (Listonella) anguillarum", "title": "Towards a phylogeny of the genus Vibrio based on 16S rRNA sequences.", "to": "outskirts of the genus Vibrio"}, {"from": "Vibrio diazotrophicus", "title": "Towards a phylogeny of the genus Vibrio based on 16S rRNA sequences.", "to": "outskirts of the genus Vibrio"}, {"from": "Vibrio hollisae", "title": "Towards a phylogeny of the genus Vibrio based on 16S rRNA sequences.", "to": "outskirts of the genus Vibrio"}, {"from": "variable regions around positions 80", "title": "Towards a phylogeny of the genus Vibrio based on 16S rRNA sequences.", "to": "180, and 450, target sites for genus- and species-specific oligonucleotide probes"}, {"from": "variable regions around positions 80", "title": "Towards a phylogeny of the genus Vibrio based on 16S rRNA sequences.", "to": "180, and 450, target sites for polymerase chain reaction primers"}, {"from": "Fourier transform-infrared spectroscopy", "title": "Fourier transform-infrared spectroscopic methods for microbial ecology: analysis of bacteria, bacteria-polymer mixtures and biofilms.", "to": "bacteria"}, {"from": "Fourier transform-infrared spectroscopy", "title": "Fourier transform-infrared spectroscopic methods for microbial ecology: analysis of bacteria, bacteria-polymer mixtures and biofilms.", "to": "bacteria-polymer mixtures"}, {"from": "Fourier transform-infrared spectroscopy", "title": "Fourier transform-infrared spectroscopic methods for microbial ecology: analysis of bacteria, bacteria-polymer mixtures and biofilms.", "to": "digester samples"}, {"from": "Fourier transform-infrared spectroscopy", "title": "Fourier transform-infrared spectroscopic methods for microbial ecology: analysis of bacteria, bacteria-polymer mixtures and biofilms.", "to": "microbial biofilms"}, {"from": "Diffuse reflectance Fourier transform-infrared analysis", "title": "Fourier transform-infrared spectroscopic methods for microbial ecology: analysis of bacteria, bacteria-polymer mixtures and biofilms.", "to": "structural information"}, {"from": "bacteria", "title": "Fourier transform-infrared spectroscopic methods for microbial ecology: analysis of bacteria, bacteria-polymer mixtures and biofilms.", "to": "dominant amide I band"}, {"from": "bacteria", "title": "Fourier transform-infrared spectroscopic methods for microbial ecology: analysis of bacteria, bacteria-polymer mixtures and biofilms.", "to": "strong carbonyl stretch at approximately 1740 centimeters-1"}, {"from": "Fourier transform-infrared spectroscopy", "title": "Fourier transform-infrared spectroscopic methods for microbial ecology: analysis of bacteria, bacteria-polymer mixtures and biofilms.", "to": "differences in microbial community structure"}, {"from": "specific band ratios", "title": "Fourier transform-infrared spectroscopic methods for microbial ecology: analysis of bacteria, bacteria-polymer mixtures and biofilms.", "to": "composition of bacteria"}, {"from": "specific band ratios", "title": "Fourier transform-infrared spectroscopic methods for microbial ecology: analysis of bacteria, bacteria-polymer mixtures and biofilms.", "to": "composition of extracellular or intracellular storage product polymer mixtures"}, {"from": "bacteria-gum arabic", "title": "Fourier transform-infrared spectroscopic methods for microbial ecology: analysis of bacteria, bacteria-polymer mixtures and biofilms.", "to": "amide I/carbohydrate C-O approximately 1150 centimeters-1"}, {"from": "bacteria-poly-beta-hydroxybutyrate", "title": "Fourier transform-infrared spectroscopic methods for microbial ecology: analysis of bacteria, bacteria-polymer mixtures and biofilms.", "to": "amide I/carbonyl approximately 1740 centimeters-1"}, {"from": "key band ratios", "title": "Fourier transform-infrared spectroscopic methods for microbial ecology: analysis of bacteria, bacteria-polymer mixtures and biofilms.", "to": "compositions of the material"}, {"from": "Fourier transform-infrared spectroscopy", "title": "Fourier transform-infrared spectroscopic methods for microbial ecology: analysis of bacteria, bacteria-polymer mixtures and biofilms.", "to": "bacteria grown under differing nutrient conditions"}, {"from": "Vibrio natriegens biofilm", "title": "Fourier transform-infrared spectroscopic methods for microbial ecology: analysis of bacteria, bacteria-polymer mixtures and biofilms.", "to": "increase in bands at approximately 1440 and 1090 centimeters-1"}, {"from": "flow-through attenuated total reflectance", "title": "Fourier transform-infrared spectroscopic methods for microbial ecology: analysis of bacteria, bacteria-polymer mixtures and biofilms.", "to": "real-time changes in biofilm formation"}, {"from": "flow-through attenuated total reflectance", "title": "Fourier transform-infrared spectroscopic methods for microbial ecology: analysis of bacteria, bacteria-polymer mixtures and biofilms.", "to": "interactions within adherent microbial consortia"}, {"from": "Alteromonas haloplanktis 214", "title": "Relationship between ion requirements for respiration and membrane transport in a marine bacterium.", "to": "Na+"}, {"from": "Alteromonas haloplanktis 214", "title": "Relationship between ion requirements for respiration and membrane transport in a marine bacterium.", "to": "Li+"}, {"from": "Toluene-treated cells", "title": "Relationship between ion requirements for respiration and membrane transport in a marine bacterium.", "to": "Na+"}, {"from": "Na+", "title": "Relationship between ion requirements for respiration and membrane transport in a marine bacterium.", "to": "K+ sparing requirement requirement"}, {"from": "Alteromonas haloplanktis", "title": "Relationship between ion requirements for respiration and membrane transport in a marine bacterium.", "to": "Na+"}, {"from": "Alteromonas haloplanktis", "title": "Relationship between ion requirements for respiration and membrane transport in a marine bacterium.", "to": "Li+"}, {"from": "Alteromonas haloplanktis", "title": "Relationship between ion requirements for respiration and membrane transport in a marine bacterium.", "to": "Na+ requirement for alpha-aminoisobutyric acid uptake"}, {"from": "Vibrio natriegens", "title": "Relationship between ion requirements for respiration and membrane transport in a marine bacterium.", "to": "Na+"}, {"from": "Photobacterium phosphoreum", "title": "Relationship between ion requirements for respiration and membrane transport in a marine bacterium.", "to": "Na+"}, {"from": "Vibrio fischeri", "title": "Relationship between ion requirements for respiration and membrane transport in a marine bacterium.", "to": "K+"}, {"from": "Toluene-treated cells", "title": "Relationship between ion requirements for respiration and membrane transport in a marine bacterium.", "to": "Na+ activation"}, {"from": "Na+", "title": "Relationship between ion requirements for respiration and membrane transport in a marine bacterium.", "to": "K+ sparing requirement"}, {"from": "temperature", "title": "An ecological study of bacteriophages of Vibrio natriegens.", "to": "longer latent periods"}, {"from": "temperature", "title": "An ecological study of bacteriophages of Vibrio natriegens.", "to": "reduced burst sizes"}, {"from": "anaerobic conditions", "title": "An ecological study of bacteriophages of Vibrio natriegens.", "to": "longer latent periods"}, {"from": "anaerobic conditions", "title": "An ecological study of bacteriophages of Vibrio natriegens.", "to": "reduced burst size for phage nt-6"}, {"from": "Na(+) presence ", "title": "Influence of Na+ on synthesis of macromolecules by a marine bacterium.", "to": "significant quantity of (14"}, {"from": "K(+) presence ", "title": "Influence of Na+ on synthesis of macromolecules by a marine bacterium.", "to": "significant quantity of (14"}, {"from": "Na(+) presence ", "title": "Influence of Na+ on synthesis of macromolecules by a marine bacterium.", "to": "mannitol minimal culture medium proliferation"}, {"from": "K(+) presence ", "title": "Influence of Na+ on synthesis of macromolecules by a marine bacterium.", "to": "mannitol minimal culture medium proliferation"}, {"from": "Na(+) presence ", "title": "Influence of Na+ on synthesis of macromolecules by a marine bacterium.", "to": "protein synthesis increase"}, {"from": "K(+) presence ", "title": "Influence of Na+ on synthesis of macromolecules by a marine bacterium.", "to": "protein synthesis increase"}, {"from": "Na(+) presence ", "title": "Influence of Na+ on synthesis of macromolecules by a marine bacterium.", "to": "deoxyribonucleic acid quantity increase"}, {"from": "K(+) presence ", "title": "Influence of Na+ on synthesis of macromolecules by a marine bacterium.", "to": "deoxyribonucleic acid quantity increase"}, {"from": "Na(+) presence ", "title": "Influence of Na+ on synthesis of macromolecules by a marine bacterium.", "to": "ribonucleic acid quantity increase"}, {"from": "K(+) presence ", "title": "Influence of Na+ on synthesis of macromolecules by a marine bacterium.", "to": "ribonucleic acid quantity increase"}, {"from": "Na(+) presence ", "title": "Influence of Na+ on synthesis of macromolecules by a marine bacterium.", "to": "significant amount of mannitol-binding protein synthesis"}, {"from": "K(+) presence ", "title": "Influence of Na+ on synthesis of macromolecules by a marine bacterium.", "to": "significant amount of mannitol-binding protein synthesis"}, {"from": "Na(+) absence ", "title": "Influence of Na+ on synthesis of macromolecules by a marine bacterium.", "to": "significant amount of mannitol-binding protein synthesis"}, {"from": "0.3 m K(+) presence ", "title": "Influence of Na+ on synthesis of macromolecules by a marine bacterium.", "to": "significant amount of mannitol-binding protein synthesis"}, {"from": "mannitol presence ", "title": "Influence of Na+ on synthesis of macromolecules by a marine bacterium.", "to": "nicotinamide adenine dinucleotide-linked mannitol-1-phosphate dehydrogenase presence"}, {"from": "broth-grown cells ", "title": "Influence of Na+ on synthesis of macromolecules by a marine bacterium.", "to": "mannitol dehydrogenase absence"}, {"from": "mannitol-grown cells ", "title": "Influence of Na+ on synthesis of macromolecules by a marine bacterium.", "to": "mannitol dehydrogenase absence"}, {"from": "Vibrio natriegens", "title": NaN, "to": "ability to take up 14C-labeled mannitol"}, {"from": "Na(+)", "title": NaN, "to": "ability to take up 14C-labeled mannitol in Vibrio natriegens"}, {"from": "K(+)", "title": NaN, "to": "ability to take up 14C-labeled mannitol in Vibrio natriegens"}, {"from": "0.3 m K(+)", "title": NaN, "to": "significant quantity of label uptake in Vibrio natriegens in absence of Na(+"}, {"from": "Na(+)", "title": NaN, "to": "proliferation of Vibrio natriegens in mannitol minimal culture medium"}, {"from": "K(+)", "title": NaN, "to": "proliferation of Vibrio natriegens in mannitol minimal culture medium"}, {"from": "Na(+)", "title": NaN, "to": "increase in protein synthesis in Vibrio natriegens"}, {"from": "K(+)", "title": NaN, "to": "increase in protein synthesis in Vibrio natriegens"}, {"from": "Na(+)", "title": NaN, "to": "increase in deoxyribonucleic acid synthesis in Vibrio natriegens"}, {"from": "K(+)", "title": NaN, "to": "increase in deoxyribonucleic acid synthesis in Vibrio natriegens"}, {"from": "Na(+)", "title": NaN, "to": "increase in ribonucleic acid synthesis in Vibrio natriegens"}, {"from": "K(+)", "title": NaN, "to": "increase in ribonucleic acid synthesis in Vibrio natriegens"}, {"from": "Na(+)", "title": NaN, "to": "synthesis of mannitolbinding protein in Vibrio natriegens"}, {"from": "K(+)", "title": NaN, "to": "synthesis of mannitolbinding protein in Vibrio natriegens"}, {"from": "0.3 m K(+)", "title": NaN, "to": "small quantity of mannitolbinding protein produced in Vibrio natriegens in absence of Na(+"}, {"from": "mannitol and Na(+) and K(+)", "title": NaN, "to": "synthesis of mannitolbinding protein in Vibrio natriegens"}, {"from": "mannitol and 0.3 m K(+) but no Na(+)", "title": NaN, "to": "small quantity of mannitolbinding protein produced in Vibrio natriegens"}, {"from": "mannitol", "title": NaN, "to": "presence of nicotinamide adenine dinucleotide-linked mannitol-1-phosphate dehydrogenase in mannitol-grown Vibrio natriegens"}, {"from": "mannitol", "title": NaN, "to": "absence of mannitol dehydrogenase in mannitol-grown Vibrio natriegens"}]);

                  nodeColors = {};
                  allNodes = nodes.get({ returnType: "Object" });
                  for (nodeId in allNodes) {
                    nodeColors[nodeId] = allNodes[nodeId].color;
                  }
                  allEdges = edges.get({ returnType: "Object" });
                  // adding nodes and edges to the graph
                  data = {nodes: nodes, edges: edges};

                  var options = {"physics": {"barnesHut": {"gravitationalConstant": -80000, "centralGravity": 0.3, "springLength": 95, "springConstant": 0.04, "damping": 0.09, "avoidOverlap": 0}, "maxVelocity": 50, "minVelocity": 0.1, "solver": "barnesHut", "timestep": 0.5, "stabilization": {"enabled": true, "iterations": 1000, "updateInterval": 25, "onlyDynamicEdges": false, "fit": true}}};

                  


                  

                  network = new vis.Network(container, data, options);

                  

                  

                  


                  
                      network.on("stabilizationProgress", function(params) {
                          document.getElementById('loadingBar').removeAttribute("style");
                          var maxWidth = 496;
                          var minWidth = 20;
                          var widthFactor = params.iterations/params.total;
                          var width = Math.max(minWidth,maxWidth * widthFactor);
                          document.getElementById('bar').style.width = width + 'px';
                          document.getElementById('text').innerHTML = Math.round(widthFactor*100) + '%';
                      });
                      network.once("stabilizationIterationsDone", function() {
                          document.getElementById('text').innerHTML = '100%';
                          document.getElementById('bar').style.width = '496px';
                          document.getElementById('loadingBar').style.opacity = 0;
                          // really clean the dom element
                          setTimeout(function () {document.getElementById('loadingBar').style.display = 'none';}, 500);
                      });
                  

                  return network;

              }
              drawGraph();
        </script>
    </body>
</html>